# Inflammatory Bowel Disease 2015 {#ibd2015}

**Last updated:** `r Sys.Date()`

**Code version:** `r system("git log -1 --format='%H'", intern = TRUE)`

The GWAS summary statistics corresponding to @ibd_ng_2015 are provided by the [International Inflammatory Bowel Disease Genetics Consortium (IIBDGC)](http://www.ibdgenetics.org/downloads.html).

Below is the basic information about these datasets (excerpt from the `README` of IIBDGC data).

```
These files are the summary association statistics corresponding to Liu, van Sommeren et al, Nature Genetics, 2015. There are two groups of files: (1) GWAS meta-analyses, and (2) GWAS plus Immunochip trans-ancestry MANTRA meta-analyses. Each group has outputs for Crohn's disease (CD), ulcerative colitis (UC), and both inflammatory bowel diseases (IBD) together.

Group 1 files:
These files have been filtered to only include variants with INFO>0.3 and with more than 2 studies/datasets inlcuded in the analysis. 
EUR.CD.gwas_info03_filtered.assoc.gz - European CD 1KGP imputed GWAS meta-analysis
EUR.IBD.gwas_info03_filtered.assoc.gz - European IBD 1KGP imputed GWAS meta-analysis
EUR.UC.gwas_info03_filtered.assoc.gz - European UC 1KGP imputed GWAS meta-analysis

Columns:
CHR - chromosome
SNP - rsid
BP - position in b37
A1 - effect allele 
A2 - other allele
FRQ_A_XXXX - effect allele frequency in XXXX cases
FRQ_U_XXXXX - effect allele frequency in XXXXX controls
INFO - IMPUTE2 info score
OR - odds ratio
SE - standard error
P - P value
Direction - direction of effect in each study included int he meta-analysis
HetISqt - Between study Heterogenity score - I squared 
HetPVa - P value for heterogeneity test
nstudies - the maximum number of datasets/studies that could be included 
studies_included - the number of datasets/studies actually included in the analysis for this variant
```

The following analyses have been performed on Inflammatory Bowel Disease (IBD) 2015 dataset.

- GSEA Round 1 (Section \@ref(ibd2015-1))
- GSEA Round 2 (Section \@ref(ibd2015-2))
