# Neuroticism 2016 {#neu2016}

**Last updated:** `r Sys.Date()`

**Code version:** `r system("git log -1 --format='%H'", intern = TRUE)`

The GWAS summary statistics corresponding to @dsneu_ng_2016 are provided by the [Social Science Genetic Association Consortium (SSGAC)](http://www.thessgac.org/data).

Below is the basic information about these datasets (excerpt from the [`README`](http://ssgac.org/documents/ReadMe_genetic_variants_associated_with_swb.txt) of SSGAC data).

```
To limit the possibilities of identifiability, significant digits for betas and standard errors are restrcted to three decimal places, and no sample allele frequencies are provided (the reported allele frequencies are from the CEU, GBR and TSI populations in the 1000G sample).

For SNPs passing quality control, we provide results from these four meta-analyses:

1. SWB_Full.txt --- subjective well-being meta-analysis of all cohorts except 23andMe. All SNPs.
2. SWB_10K.txt --- subjective well-being meta-analysis of all cohorts. Pruned set of 10,000 SNPs. 
3. Neuroticism_Full.txt --- neuroticism meta-analysis of all cohorts. All SNPs.
4. DS_Full.txt --- depressive symptoms meta-analysis of all cohorts. All SNPs.

Each file consists of the following columns:

MarkerName: SNP rs number.
CHR: chromosome number.
POS: base pair position.
A1: effect allele.
A2: other allele.
EAF: A1 frequency in 1000G sample (CEU, GBR and TSI individuals).
Beta: Standardized regression coefficient. 
SE: standard error of beta.
Pval: Nominal p-value of the null hypothesis that the coefficient is equal to zero.
```

The following analyses have been performed on Neuroticism (NEU) 2016 dataset.

- GSEA Round 1 (Section \@ref(neu2016-1))
