--- 
title: "Integrated Enrichment Analysis of Genetic Variants and Biological Pathways"
author: "Xiang Zhu"
date: "`r Sys.Date()`"
site: bookdown::bookdown_site
output: bookdown::gitbook
documentclass: book
bibliography: [~/Dropbox/rss/Writing/rss_shared/rss.bib]
biblio-style: apalike
link-citations: yes
github-repo: xiangzhu/rss-gsea
---

# Basic Information

**Who**

[Xiang Zhu](https://github.com/xiangzhu) is a graduate student in the Department of Statistics at the University of Chicago, and his dissertation advisor is Prof. [Matthew Stephens](http://stephenslab.uchicago.edu/index.html). 

**What**

This is my notebook for a research project. The aim of this project is to develop statistical methods for integrated enrichment analysis of genome-wide association studies (GWAS) summary data and biological pathways.
Specifically, I combine the likelihood function from @rss_2016 with the prior distribution from @integrated_plos_2013, and then perform Bayesian analysis. 

To learn more about other research projects, please visit the [Stephens Lab website](http://stephenslab.uchicago.edu).

**How**

I make this notebook using many wonderful publicly available tools,
including [`rarkdown`][rmarkdown], [`knitr`][knitr], [`bookdown`][bookdown] and [`pandoc`][pandoc].
This notebook is largely motivated by the idea from the [singleCellseq][scs] project at the [Gilad Lab](http://giladlab.uchicago.edu/).
To host the notebook, I use [GitHub Pages][gh-pages].

[rmarkdown]: http://rmarkdown.rstudio.com/
[knitr]: http://yihui.name/knitr/
[bookdown]: https://bookdown.org/
[pandoc]: http://pandoc.org/
[rmd-site]: https://github.com/rstudio/rmarkdown/tree/gh-pages
[gh-pages]: https://pages.github.com/
[scs]: https://github.com/jdblischak/singleCellSeq

## License 

Currently the notebook is licensed under the [Creative Commons Attribution-NoDerivatives 4.0 International License (CC BY-ND 4.0)][site-license].
This means you are free to include content from this site in your presentation, blog, or social media posts, as long you cite it.
The "no derivatives" clause means you cannot modify the results and distribute them.
Once the results are published, I will remove "no derivatives" clause.

The source code in the notebook is licensed under the [GNU General Public License, version 3.0 (GPLv3)][code-license].

## Cite

If you use any results from this notebook, please cite it by including the following (or something similar):

```{block, note-text, type='rmdimportant'}
Zhu, X. (2016). Gene set enrichment analysis of summary data from genome-wide association studies. *Stephens Laboratory Notebook at the University of Chicago*. Retrieved from https://github.com/xiangzhu/rss-gsea
```

[code-license]: https://www.gnu.org/licenses/gpl-3.0.txt
[site-license]: https://creativecommons.org/licenses/by-nd/4.0/

## Contact

If you have any question about this notebook, please feel free to contact me.

- Create a new [issue](https://github.com/xiangzhu/rss-gsea/issues) in GitHub.
- Send an email to `xiangzhu@uchicago.edu`.

## News

- **03/04/2016**: Xiang presented this work at the 2016 Novembre-He-Stephens meeting. [[Slides]](http://www.stat.uchicago.edu/~xiangzhu/NHS_20160304.pdf)
