# Heart Rate 2013

**Last updated:** `r Sys.Date()`

**Code version:** `r system("git log -1 --format='%H'", intern = TRUE)`

The GWAS summary statistics corresponding to @hr_ng_2013 are provided by the [HRgene Consortium](https://walker05.u.hpc.mssm.edu/).

Below is the basic information about these datasets (excerpt from the `readme` file).
```
Read-me file accompanying the summary statistics of the genome-wide association study (GWAS) for heart rate

This study was coordinated by Ruth JF Loos and Marcel den Hoed, and was made possible by the effort of the HRgene consortium

The summary statistics file contains the following information:

rsid - the SNP identifier as per the HapMap2 reference panel
CHR - chromosome
POS - position of the SNP on the chromosome, based on the HapMap2 reference panel (build 36)
EFFECT_ALLELE - the allele for which the beta and effect allele frequency are provided
OTHER_ALLELE - the alternative, non-effect allele
EAF - the effect allele frequency
beta - the effect size in beats/min/effect allele
SE - standard error of the beta in beats/min
N_TOTAL - the sample size available for the SNP; up to 85,787 individuals
P_VALUE - the p-value for association of the SNP with heart rate
P_VALUE_GCADJ - the p-value for association of the SNP with heart rate after adjusting for population stratification (genomic control)

Please see our publication for more information:
den Hoed et al. Identification of heart rate-associated loci and their effects on cardiac conduction and rhythm disorders. Nat Genet 45 (6); 621-631 [PMID 23583979]

---------------
Last updated on the 1st of April 2015
---------------
```

The following analyses have been performed on Heart Rate 2013 dataset.

- GSEA Round 1 (Section \@ref(hr2013-1))
- GSEA Round 2 (Section \@ref(hr2013-2))
- GTEx Analysis (Section \@ref(hr2013-3))
