--- 
title: "Integrated Enrichment Analysis of Genetic Variants, Biological Pathways and Tissues Across 31 Human Phenotypes"
author: "Xiang Zhu"
date: "`r Sys.Date()`"
knit: bookdown::render_book
output: bookdown::gitbook
documentclass: book
bibliography: [~/Dropbox/rss/Writing/rss_shared/rss.bib]
biblio-style: apalike
link-citations: yes
github-repo: xiangzhu/rss-gsea
---

```{r,include=FALSE}
# ulimit -s 32768 

# load packages
suppressPackageStartupMessages(library(R.matlab))
suppressPackageStartupMessages(library(DT))
suppressPackageStartupMessages(library(knitr))
suppressPackageStartupMessages(library(plyr))
suppressPackageStartupMessages(library(reshape2))

# load self-defined subroutines
source("src/util.R")

# load gtex tissue names
source("~/Dropbox/rss/Data/gtex_database/color/load_gtex_color.R")
tissue.df <- color.dat[, c("tissue.name","tissue.abbr")]
names(tissue.df) <- c("tissue","id")
```

# Basic Information

**Who**

[Xiang Zhu](https://github.com/xiangzhu) is a graduate student in the Department of Statistics at the University of Chicago, and his dissertation advisor is Prof. [Matthew Stephens](http://stephenslab.uchicago.edu/index.html). 

**What**

This is my online notebook for sharing large-scale analysis **results**^[Details of analysis methods can be found in the corresponding manuscript and supplementary materials.] from a research project, the aim of which is to develop statistical methods for integrated enrichment analysis of genome-wide association studies (GWAS) summary data and biological pathways or tissue-based gene sets.

**How**

I make this notebook using many wonderful publicly available tools,
including [`rmarkdown`][rmarkdown], [`knitr`][knitr], [`bookdown`][bookdown], [`pandoc`][pandoc] and [`DT`][dt].
This notebook is largely motivated by the idea from the [singleCellseq][scs] project at the [Gilad Lab](http://giladlab.uchicago.edu/).
To host the notebook, I use [GitHub Pages][gh-pages].

[rmarkdown]: http://rmarkdown.rstudio.com/
[knitr]: http://yihui.name/knitr/
[bookdown]: https://bookdown.org/
[pandoc]: http://pandoc.org/
[dt]: https://rstudio.github.io/DT/
[gh-pages]: https://pages.github.com/
[scs]: https://github.com/jdblischak/singleCellSeq

**Contact**

If you have any question about this notebook, please feel free to contact me.

- Create a new [issue](https://github.com/xiangzhu/rss-gsea/issues) in GitHub.
- Send an email to `xiangzhu@uchicago.edu`.

**License & Cite** 

Currently the notebook is licensed under the [Creative Commons Attribution-NoDerivatives 4.0 International License (CC BY-ND 4.0)][site-license].
This means you are free to include content from this site in your presentation, blog, or social media posts, as long you cite it.
The "no derivatives" clause means you cannot modify the results and distribute them.
Once the results are published (manuscript in preparation), I will remove "no derivatives" clause.

The source code in the notebook is licensed under the [GNU General Public License, version 3.0 (GPLv3)][code-license].

If you use any results from this notebook, please cite it by including the following text:

```{block, note-text, type='rmdcite'}
Zhu, X. (2016). Integrated Enrichment Analysis of Genetic Variants, Biological Pathways and Tissues Across 31 Human Phenotypes. *Stephens Laboratory Notebook at the University of Chicago*. Retrieved from https://github.com/xiangzhu/rss-gsea
```

[code-license]: https://www.gnu.org/licenses/gpl-3.0.txt
[site-license]: https://creativecommons.org/licenses/by-nd/4.0/

