<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of depressive symptoms</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.5/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css" data-origin="pandoc">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(title);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' || rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<div class="container-fluid main-container">

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->



<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of depressive symptoms</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#report"> Report <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-02-18
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rss-gsea/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.2.0). The <em>Report</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="report" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20180626code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20180626)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20180626code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree6e12a0cfbfbbfea4a3991e6c49bde22ce9ab0b2ftargetblank6e12a0ca"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/6e12a0cfbfbbfea4a3991e6c49bde22ce9ab0b2f" target="_blank">6e12a0c</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree6e12a0cfbfbbfea4a3991e6c49bde22ce9ab0b2ftargetblank6e12a0ca" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/73f3c80eafd4669d84f817685bc7f2ded7510544/docs/ds_2016.html" target="_blank">73f3c80</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/ds_2016.html" target="_blank">1c85967</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/7f4fff7927b1fbcc72c01c1cf452eb120f738f0a/docs/ds_2016.html" target="_blank">7f4fff7</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/a66e841a1ff46d7ae89513857003c2f9869fdc96/analysis/ds_2016.Rmd" target="_blank">a66e841</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-01
</td>
<td>
wflow_publish(“ds_2016.Rmd”)
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/27089181">“Genetic Variants Associated with Subjective Well-Being, Depressive Symptoms, and Neuroticism Identified Through Genome-Wide Analyses” (<em>Nature Genetics</em>, 2016)</a>. The summary data file is available at <a href="https://www.thessgac.org/data" class="uri">https://www.thessgac.org/data</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["Regulation by TREX1<br/>(126, reactome, BioSystems)","fructose 2,6-bisphosphate synthesis/dephosphorylation<br/>(415, humancyc, PC)","Conjugation of phenylacetate with glutamine<br/>(19, reactome, PC)","Nef mediated downregulation of CD28 cell surface expression<br/>(48, reactome, PC)","MCM complex<br/>(769, kegg, BioSystems)","Proepithelin Conversion to Epithelin and Wound Repair Control<br/>(115, biocarta, BioCarta)","Conjugation of benzoate with glycine<br/>(530, reactome, PC)","Amino Acid conjugation<br/>(897, reactome, PC)","negative regulation of TCF-dependent signaling by DVL-interacting proteins<br/>(591, reactome, PC)","DNA Damage Bypass<br/>(196, reactome, PC)","glucocorticoid biosynthesis<br/>(313, biocyc, BioSystems)","2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA<br/>(2, humancyc, PC)","vitamin D3 biosynthesis<br/>(144, biocyc, BioSystems)","Rhesus glycoproteins mediate ammonium transport.<br/>(232, reactome, PC)","mineralocorticoid biosynthesis<br/>(312, biocyc, BioSystems)","guanosine nucleotides degradation<br/>(310, biocyc, BioSystems)","GABA A (rho) receptor activation<br/>(28, reactome, PC)","GABA A (rho) receptor activation<br/>(307, reactome, BioSystems)","Proline catabolism<br/>(51, reactome, PC)","Formation of the Editosome<br/>(26, reactome, PC)","progesterone biosynthesis<br/>(105, humancyc, PC)","Breakdown of the nuclear lamina<br/>(189, reactome, PC)","malate-aspartate shuttle<br/>(425, humancyc, PC)","tryptophan degradation<br/>(1060, humancyc, PC)","betaKlotho-mediated ligand binding<br/>(256, reactome, PC)","serotonin degradation<br/>(756, biocyc, BioSystems)","Transfer of LPS from LBP carrier to CD14<br/>(74, reactome, PC)","Beta oxidation of palmitoyl-CoA to myristoyl-CoA<br/>(188, reactome, PC)","Synaptic Proteins at the Synaptic Junction<br/>(452, biocarta, BioCarta)","acetate conversion to acetyl-CoA<br/>(158, biocyc, BioSystems)","dopamine degradation<br/>(92, humancyc, PC)","Amino acid conjugation of benzoic acid<br/>(499, wiki, BioSystems)","1,25-dihydroxyvitamin D3 biosynthesis<br/>(174, humancyc, PC)","tetrahydrofolate salvage from 5,10-methenyltetrahydrofolate<br/>(108, humancyc, PC)","ethanol degradation IV<br/>(642, biocyc, BioSystems)","Steroid hormone biosynthesis, cholesterol => prognenolone => progesterone<br/>(326, kegg, BioSystems)","Synthesis of IPs in the nucleus<br/>(404, reactome, PC)","Mineralocorticoid biosynthesis<br/>(677, reactome, PC)","Synthesis of GDP-mannose<br/>(573, reactome, PC)","Inhibition of Huntington's disease neurodegeneration by histone deacetylase inhibitors<br/>(447, biocarta, BioCarta)","eicosapentaenoate biosynthesis<br/>(1336, humancyc, PC)","Regulation of PAK-2p34 activity by PS-GAP/RHG10<br/>(122, reactome, BioSystems)","Mitochondrial ABC transporters<br/>(215, reactome, PC)","Glycosphingolipid biosynthesis, lacto-series, LacCer => Lc4Cer<br/>(497, kegg, BioSystems)","creatine-phosphate biosynthesis<br/>(478, biocyc, BioSystems)","acyl-CoA hydrolysis<br/>(82, humancyc, PC)","serotonin degradation<br/>(837, humancyc, PC)","Transcription from mitochondrial promoters<br/>(409, reactome, PC)","glycine biosynthesis<br/>(97, humancyc, PC)","aspirin triggered resolvin D biosynthesis<br/>(88, humancyc, PC)","Defective CD320 causes methylmalonic aciduria<br/>(129, reactome, BioSystems)","Biotin metabolism<br/>(342, kegg, BioSystems)","5-aminoimidazole ribonucleotide biosynthesis<br/>(176, humancyc, PC)","glycine cleavage<br/>(266, humancyc, PC)","ARL13B-mediated ciliary trafficking of INPP5E<br/>(306, reactome, BioSystems)","sulfate activation for sulfonation<br/>(107, humancyc, PC)","Activation of C3 and C5<br/>(651, reactome, PC)","Alternative complement activation<br/>(356, reactome, PC)","Interactions of Tat with host cellular proteins<br/>(38, reactome, PC)","Post-transcriptional silencing by small RNAs<br/>(812, reactome, PC)","Cholecalciferol biosynthesis<br/>(151, kegg, BioSystems)","Lactose synthesis<br/>(296, reactome, BioSystems)","Insulin effects increased synthesis of Xylulose-5-Phosphate<br/>(37, reactome, PC)","Xanthine and guanine salvage pathway<br/>(81, panther, PC)","DNA replication initiation<br/>(532, reactome, PC)","Signalling to ERK5<br/>(62, reactome, PC)","E-cadherin signaling events<br/>(170, pid, BioSystems)","tRNA splicing<br/>(596, humancyc, PC)","Ascorbate degradation<br/>(120, panther, PANTHER)","FMO oxidises nucleophiles<br/>(202, reactome, PC)","DNA polymerase epsilon complex<br/>(492, kegg, BioSystems)","TWIK-releated acid-sensitive K+ channel (TASK)<br/>(70, reactome, PC)","Glypican 2 network<br/>(32, pid, PC)","Rattus norvegicus<br/>(57, mirtarbase, PC)","Mitotic Metaphase/Anaphase Transition<br/>(44, reactome, PC)","PLK2 and PLK4 events<br/>(50, pid, PC)","Glutamine glutamate conversion<br/>(206, panther, PC)","retinoate biosynthesis II<br/>(314, biocyc, BioSystems)","retinoate biosynthesis II<br/>(433, humancyc, PC)","Fructose catabolism<br/>(27, reactome, PC)","dolichyl-diphosphooligosaccharide biosynthesis<br/>(951, humancyc, PC)","PDGF Pathway<br/>(137, wiki, BioSystems)","Creatine pathway<br/>(884, kegg, BioSystems)","Inhibition of Host mRNA Processing and RNA Silencing<br/>(36, reactome, PC)","Agmatine biosynthesis<br/>(10, reactome, PC)","Polyamine biosynthesis, arginine => agmatine => putrescine => spermidine<br/>(495, kegg, BioSystems)","palmitate biosynthesis<br/>(758, biocyc, BioSystems)","Defective ACTH causes Obesity and Pro-opiomelanocortinin deficiency (POMCD)<br/>(128, reactome, BioSystems)","Sterols are 12-hydroxylated by CYP8B1<br/>(63, reactome, PC)","Overview of telomerase RNA component gene hTerc Transcriptional Regulation<br/>(739, biocarta, BioCarta)","Defective CHST14 causes EDS, musculocontractural type<br/>(751, reactome, BioSystems)","Activation of BMF and translocation to mitochondria<br/>(181, reactome, PC)","icosapentaenoate biosynthesis II (metazoa)<br/>(1189, biocyc, BioSystems)","Beta-oxidation of very long chain fatty acids<br/>(525, reactome, PC)","gamma-linolenate biosynthesis<br/>(1362, biocyc, BioSystems)","&gamma;-linolenate biosynthesis<br/>(1377, humancyc, PC)","UTP and CTP dephosphorylation I<br/>(79, humancyc, PC)","miRNA Regulation of DNA Damage Response<br/>(1088, wiki, BioSystems)","Erythrocytes take up carbon dioxide and release oxygen<br/>(1018, reactome, PC)","L-kynurenine degradation<br/>(1398, humancyc, PC)"],[2,4,2,2,6,2,5,8,5,3,3,2,2,3,3,3,2,3,2,2,2,3,4,9,3,6,2,3,4,2,2,4,3,2,5,3,4,6,5,4,12,2,3,4,4,2,7,4,2,2,2,3,3,3,3,2,6,4,2,7,2,3,2,2,5,2,2,5,2,3,4,2,2,2,2,2,3,3,4,2,8,2,7,2,2,4,6,2,2,6,6,3,10,5,12,13,2,9,9,13],[52,469,106,150,480,145,241,302,438,255,174,163,166,248,98,430,188,309,163,75,115,225,374,888,242,409,182,160,291,226,266,226,224,168,570,217,326,363,403,340,1006,290,271,387,273,211,464,331,194,176,171,336,269,342,214,236,230,307,143,534,134,350,198,338,354,214,197,494,152,246,376,273,96,188,199,122,264,217,378,171,725,113,441,140,128,240,306,198,192,503,509,261,961,406,1162,1250,214,785,640,1242],[395.103999382724,163.30936884441,154.424646159987,147.644092569321,147.156088494427,140.534453282216,139.760581360885,139.550038457729,105.145131027658,103.734253279802,95.6859687902645,93.9028025501519,71.516414224134,69.6814423124302,69.3073125861627,67.8513624008001,66.2358853718376,65.8678276615479,62.6721633688352,61.5871690666169,59.6651064951401,59.1906100080181,58.1535064633234,56.4952148088256,56.4727879959779,55.0313281996569,52.9026491259139,50.58883164753,50.0919366237025,45.8901489246498,45.8457839220466,45.7488113590902,45.1247838602137,42.1336108416194,41.0461597509733,40.7557679759433,40.6116597619772,40.3426317936384,39.1356948648778,38.3554526122837,36.9256292904083,35.9096507956165,35.8037622132835,35.4564365150653,35.1825714325699,33.1063685439038,33.0308149479208,32.9748980015554,32.6726633114905,32.3540388464409,31.5999141823304,31.4774287822814,31.3024320137952,31.0517154313684,30.8977038032635,30.8023151602049,30.5938616907885,30.3581088587687,28.7896513623157,28.5733380522149,26.9585087017814,26.8942773267121,26.2749963547148,26.2162903423041,25.5369080458589,25.0742276113849,24.8982131718046,23.7954008228758,23.7025451571363,23.1719655045194,22.6222536560767,22.3360566233347,20.863967561447,19.727484851428,18.6382243828811,18.2829324178349,18.1000369371622,17.3288092913893,16.9146392866367,16.8285145775658,16.7444577306692,16.2293752281701,15.7909157752421,15.7001251909109,15.0198128513365,14.240083579567,13.528145196274,13.0148487609472,12.7069105373806,12.5758902354684,12.3507259069854,11.7174254417433,11.0650540135016,11.0286300731378,10.8967972921161,10.8957435181334,10.6894629539541,6.69443628202439,-0.203267949735455,-0.252035389545167],["-5.6296<br/>[-5.9789, -5.4217]","-5.6295<br/>[-5.9787, -5.4216]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6295<br/>[-5.9787, -5.4216]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6295<br/>[-5.9787, -5.4216]","-5.6296<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9789, -5.4217]","-5.6295<br/>[-5.9787, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6294<br/>[-5.9786, -5.4215]","-5.6296<br/>[-5.9788, -5.4216]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9787, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6295<br/>[-5.9788, -5.4216]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6293<br/>[-5.9785, -5.4214]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4216]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9787, -5.4216]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9789, -5.4217]","-5.6295<br/>[-5.9787, -5.4215]","-5.6296<br/>[-5.9789, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9787, -5.4216]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9789, -5.4217]","-5.6298<br/>[-5.9789, -5.4221]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9789, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6294<br/>[-5.9786, -5.4215]","-5.6297<br/>[-5.9789, -5.4218]","-5.6295<br/>[-5.9787, -5.4216]","-5.6296<br/>[-5.9789, -5.4217]","-5.6298<br/>[-5.9789, -5.4220]","-5.6297<br/>[-5.9789, -5.4219]","-5.6295<br/>[-5.9788, -5.4216]","-5.6296<br/>[-5.9788, -5.4217]","-5.6296<br/>[-5.9788, -5.4217]","-5.6295<br/>[-5.9787, -5.4216]","-5.6295<br/>[-5.9787, -5.4215]","-5.6296<br/>[-5.9788, -5.4216]","-5.6293<br/>[-5.9785, -5.4214]","-5.6295<br/>[-5.9788, -5.4216]","-5.6292<br/>[-5.9785, -5.4213]","-5.6292<br/>[-5.9784, -5.4213]","-5.6296<br/>[-5.9788, -5.4217]","-5.6294<br/>[-5.9786, -5.4214]","-5.6294<br/>[-5.9787, -5.4215]","-5.6293<br/>[-5.9785, -5.4214]"],["-0.6357<br/>[-0.6363, -0.6348]","-2.3673<br/>[-2.3700, -2.3626]","-1.3236<br/>[-1.3260, -1.3207]","-1.5346<br/>[-1.5716, -1.3475]","-1.5493<br/>[-1.6729, -1.5420]","-1.5475<br/>[-1.5578, -1.5350]","-1.9014<br/>[-1.9046, -1.8967]","-1.9970<br/>[-2.0024, -1.9899]","-2.3355<br/>[-2.3403, -2.3275]","-2.0987<br/>[-2.1053, -2.0884]","-1.9367<br/>[-1.9394, -1.9318]","-0.3621<br/>[-0.3623, -0.3516]","-1.6129<br/>[-1.6173, -1.6067]","-1.7832<br/>[-1.7918, -1.7729]","-1.3856<br/>[-1.3887, -1.3813]","-2.3236<br/>[-2.3323, -2.3098]","-1.9670<br/>[-1.9730, -1.9571]","-2.1511<br/>[-2.1858, -2.1147]","-1.9015<br/>[-1.9109, -1.8866]","-0.9606<br/>[-0.9691, -0.9513]","-1.4533<br/>[-1.4580, -1.4468]","-2.0334<br/>[-2.0506, -2.0137]","-2.2671<br/>[-2.2718, -2.2593]","-2.0417<br/>[-2.0438, -2.0396]","-1.3685<br/>[-1.3686, -1.3661]","-2.3050<br/>[-2.3106, -2.2959]","-1.7756<br/>[-1.7824, -1.7657]","-1.8934<br/>[-1.9029, -1.8781]","-2.1439<br/>[-2.1627, -2.1160]","-1.7281<br/>[-1.7430, -1.7131]","-2.1080<br/>[-2.1231, -2.0870]","-1.7457<br/>[-1.7513, -1.7382]","-1.7400<br/>[-1.7474, -1.7306]","-1.6158<br/>[-1.6224, -1.6070]","-2.4194<br/>[-2.4532, -2.3839]","-2.0187<br/>[-2.0353, -1.9947]","-2.2071<br/>[-2.2121, -2.1990]","-2.2491<br/>[-2.2587, -2.2340]","-2.2965<br/>[-2.3041, -2.2846]","-2.2071<br/>[-2.2300, -2.1745]","-2.6771<br/>[-2.7007, -2.6411]","-1.9344<br/>[-1.9803, -1.8899]","-2.1213<br/>[-2.1317, -2.1059]","-2.2752<br/>[-2.2865, -2.2578]","-1.7257<br/>[-1.7353, -1.7146]","-2.0135<br/>[-2.0232, -1.9984]","-2.3578<br/>[-2.3654, -2.3455]","-2.2108<br/>[-2.2187, -2.1985]","-1.6369<br/>[-1.6708, -1.6058]","-1.6354<br/>[-1.6425, -1.6263]","-1.9000<br/>[-1.9309, -1.8594]","-2.2091<br/>[-2.2251, -2.1849]","-1.7814<br/>[-1.8187, -1.7445]","-2.2172<br/>[-2.2322, -2.1959]","-2.0174<br/>[-2.0292, -1.9989]","-2.0596<br/>[-2.0715, -2.0425]","-2.0514<br/>[-2.0605, -2.0362]","-2.1728<br/>[-2.1858, -2.1525]","-1.5400<br/>[-1.5511, -1.5261]","-2.4122<br/>[-2.4264, -2.3899]","-1.5150<br/>[-1.5236, -1.5040]","-2.2302<br/>[-2.2428, -2.2107]","-1.9826<br/>[-1.9954, -1.9633]","-2.2077<br/>[-2.2274, -2.1790]","-2.2336<br/>[-2.2479, -2.2128]","-2.0239<br/>[-2.0293, -2.0149]","-1.7586<br/>[-1.8162, -1.6474]","-2.3599<br/>[-2.3920, -2.3138]","-1.8737<br/>[-1.8807, -1.8622]","-2.0819<br/>[-2.0898, -2.0690]","-2.2593<br/>[-2.2740, -2.2369]","-2.1174<br/>[-2.1348, -2.0913]","-1.3597<br/>[-1.3744, -1.3434]","-1.8186<br/>[-1.8934, -1.7603]","-1.9858<br/>[-1.9977, -1.9673]","-1.4615<br/>[-1.4806, -1.4360]","-2.0887<br/>[-2.1194, -2.0501]","-2.0277<br/>[-2.0353, -2.0147]","-2.2659<br/>[-2.2763, -2.2498]","-1.9087<br/>[-1.9311, -1.8770]","-2.5419<br/>[-2.5590, -2.5145]","-1.6962<br/>[-1.7492, -1.6298]","-2.3244<br/>[-2.3428, -2.2949]","-1.8357<br/>[-1.8448, -1.8200]","-1.6369<br/>[-1.7140, -1.5811]","-1.9330<br/>[-2.0212, -1.8634]","-2.1670<br/>[-2.1840, -2.1392]","-1.9662<br/>[-1.9948, -1.9242]","-1.9482<br/>[-1.9801, -1.9143]","-2.3877<br/>[-2.4003, -2.3674]","-2.3918<br/>[-2.4054, -2.3701]","-2.1044<br/>[-2.1154, -2.0862]","-2.6543<br/>[-2.6806, -2.6139]","-2.2916<br/>[-2.3074, -2.2675]","-2.7388<br/>[-2.7636, -2.7012]","-2.7722<br/>[-2.7956, -2.7373]","-2.0110<br/>[-2.0290, -1.9834]","-2.5818<br/>[-2.5935, -2.5624]","-4.7030<br/>[-7.5431, -4.0131]","-4.7779<br/>[-7.3574, -4.1007]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(ID, database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Spleen","Small Intestine - Terminal Ileum","Cells - EBV-transformed lymphocytes","Lung","Pancreas","Colon - Transverse","Stomach","Adrenal Gland","Esophagus - Mucosa","Breast - Mammary Tissue","Brain - Cerebellar Hemisphere","Prostate","Adipose - Subcutaneous","Uterus","Vagina","Nerve - Tibial","Brain - Hypothalamus","Brain - Hippocampus","Muscle - Skeletal","Brain - Cerebellum","Pituitary","Esophagus - Gastroesophageal Junction","Whole Blood","Skin - Sun Exposed (Lower leg)","Ovary","Heart - Left Ventricle","Adipose - Visceral (Omentum)","Brain - Anterior cingulate cortex (BA24)","Colon - Sigmoid","Skin - Not Sun Exposed (Suprapubic)","Brain - Cortex","Artery - Tibial","Esophagus - Muscularis","Thyroid","Cells - Transformed fibroblasts","Brain - Nucleus accumbens (basal ganglia)","Testis","Heart - Atrial Appendage","Artery - Coronary","Brain - Frontal Cortex (BA9)","Artery - Aorta","Liver","Brain - Caudate (basal ganglia)","Brain - Putamen (basal ganglia)"],[4665,5317,5603,5614,5783,5908,5957,6416,6242,6342,6397,6359,6374,6421,6483,6436,6566,6531,6460,6553,6618,6486,6615,6522,6683,6441,6586,6703,6605,6609,6666,6721,6696,6916,6797,6862,6807,6699,6892,6930,7005,7022,7294,7369],[-0.261312461270591,-0.27617633982261,-0.277151818427069,-0.28152771886962,-0.284872380236319,-0.288199674199491,-0.288943697105158,-0.29354989055355,-0.293850848399484,-0.294916702388208,-0.29502346990928,-0.295615753198895,-0.29561852770687,-0.295816048537127,-0.29636586477407,-0.296583930457208,-0.296841319138261,-0.297901241620168,-0.297929807016762,-0.297943009092416,-0.298147385593555,-0.298292965216574,-0.298385068790765,-0.299018085656008,-0.299183260840683,-0.299540484164338,-0.299809618399626,-0.299851673606608,-0.300428604339826,-0.300559005610248,-0.301063981493031,-0.301588256082702,-0.30199176963913,-0.302566065460181,-0.302696469435642,-0.302810410057299,-0.303144502849238,-0.303822300228082,-0.304339527688359,-0.305055180907321,-0.306534748954517,-0.306585058917436,-0.309841509564191,-0.312215060128982],["-5.6280<br/>[-5.9771, -5.4202]","-5.6277<br/>[-5.9769, -5.4199]","-5.6278<br/>[-5.9768, -5.4201]","-5.6277<br/>[-5.9768, -5.4199]","-5.6276<br/>[-5.9767, -5.4198]","-5.6275<br/>[-5.9767, -5.4197]","-5.6275<br/>[-5.9766, -5.4197]","-5.6275<br/>[-5.9765, -5.4198]","-5.6274<br/>[-5.9765, -5.4196]","-5.6274<br/>[-5.9765, -5.4196]","-5.6274<br/>[-5.9765, -5.4197]","-5.6274<br/>[-5.9765, -5.4196]","-5.6274<br/>[-5.9765, -5.4196]","-5.6274<br/>[-5.9765, -5.4196]","-5.6274<br/>[-5.9765, -5.4197]","-5.6274<br/>[-5.9765, -5.4196]","-5.6274<br/>[-5.9764, -5.4197]","-5.6273<br/>[-5.9764, -5.4196]","-5.6273<br/>[-5.9764, -5.4195]","-5.6273<br/>[-5.9764, -5.4196]","-5.6274<br/>[-5.9764, -5.4196]","-5.6273<br/>[-5.9764, -5.4195]","-5.6274<br/>[-5.9764, -5.4196]","-5.6273<br/>[-5.9764, -5.4195]","-5.6273<br/>[-5.9764, -5.4196]","-5.6273<br/>[-5.9764, -5.4194]","-5.6273<br/>[-5.9764, -5.4195]","-5.6273<br/>[-5.9764, -5.4196]","-5.6273<br/>[-5.9764, -5.4195]","-5.6273<br/>[-5.9764, -5.4195]","-5.6273<br/>[-5.9764, -5.4195]","-5.6273<br/>[-5.9764, -5.4195]","-5.6272<br/>[-5.9764, -5.4195]","-5.6273<br/>[-5.9763, -5.4196]","-5.6272<br/>[-5.9763, -5.4195]","-5.6273<br/>[-5.9763, -5.4195]","-5.6272<br/>[-5.9763, -5.4195]","-5.6272<br/>[-5.9763, -5.4194]","-5.6272<br/>[-5.9763, -5.4195]","-5.6272<br/>[-5.9763, -5.4194]","-5.6272<br/>[-5.9762, -5.4194]","-5.6272<br/>[-5.9762, -5.4194]","-5.6271<br/>[-5.9761, -5.4194]","-5.6270<br/>[-5.9761, -5.4193]"],["-5.5140<br/>[-7.6787, -4.8804]","-5.6156<br/>[-7.7378, -4.9860]","-5.3690<br/>[-7.4178, -4.7553]","-5.5828<br/>[-7.6885, -4.9548]","-5.5954<br/>[-7.6956, -4.9683]","-5.6797<br/>[-7.7677, -5.0571]","-5.6662<br/>[-7.7133, -5.0421]","-5.4511<br/>[-7.5159, -4.8292]","-5.6518<br/>[-7.7253, -5.0296]","-5.6014<br/>[-7.6719, -4.9785]","-5.5518<br/>[-7.6091, -4.9304]","-5.6307<br/>[-7.7004, -5.0070]","-5.6124<br/>[-7.6809, -4.9875]","-5.5709<br/>[-7.6392, -4.9469]","-5.5399<br/>[-7.6069, -4.9154]","-5.6055<br/>[-7.6319, -4.9815]","-5.4942<br/>[-7.5080, -4.8725]","-5.5913<br/>[-7.6421, -4.9731]","-5.6761<br/>[-7.7404, -5.0554]","-5.5705<br/>[-7.5794, -4.9516]","-5.5186<br/>[-7.5683, -4.9000]","-5.6700<br/>[-7.7322, -5.0496]","-5.5309<br/>[-7.5899, -4.9108]","-5.6793<br/>[-7.7394, -5.0579]","-5.5146<br/>[-7.5679, -4.8936]","-5.8697<br/>[-7.9350, -5.2476]","-5.6571<br/>[-7.7138, -5.0344]","-5.5370<br/>[-7.5421, -4.9157]","-5.6804<br/>[-7.6973, -5.0579]","-5.6855<br/>[-7.7413, -5.0627]","-5.6552<br/>[-7.6975, -5.0339]","-5.6231<br/>[-7.6763, -5.0019]","-5.6841<br/>[-7.7360, -5.0624]","-5.4999<br/>[-7.5371, -4.8795]","-5.6139<br/>[-7.6644, -4.9939]","-5.5600<br/>[-7.5969, -4.9414]","-5.6331<br/>[-7.6819, -5.0131]","-5.8542<br/>[-7.9078, -5.2368]","-5.6210<br/>[-7.6673, -4.9998]","-5.6331<br/>[-7.6654, -5.0154]","-5.6484<br/>[-7.6886, -5.0291]","-5.6364<br/>[-7.6685, -5.0179]","-5.5764<br/>[-7.5958, -4.9597]","-5.6533<br/>[-7.6676, -5.0375]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Pituitary","Spleen","Liver","Skin_Sun_Exposed_(Lower_leg)","Skin_Not_Sun_Exposed_(Suprapubic)","Vagina","Esophagus_Mucosa","Cells_EBV-transformed_lymphocytes","Whole_Blood","Kidney_Cortex","Colon_Transverse","Pancreas","Testis","Lung","Muscle_Skeletal","Minor_Salivary_Gland","Small_Intestine_Terminal_Ileum","Prostate","Stomach","Breast_Mammary_Tissue","Adipose_Visceral_(Omentum)","Adipose_Subcutaneous","Cells_Transformed_fibroblasts","Brain_Cerebellum","Colon_Sigmoid","Heart_Left_Ventricle","Heart_Atrial_Appendage","Brain_Hippocampus","Brain_Amygdala","Thyroid","Brain_Nucleus_accumbens_(basal_ganglia)","Ovary","Brain_Caudate_(basal_ganglia)","Brain_Putamen_(basal_ganglia)","Brain_Cortex","Uterus","Brain_Hypothalamus","Artery_Coronary","Adrenal_Gland","Nerve_Tibial","Artery_Tibial","Brain_Spinal_cord_(cervical_c-1)","Brain_Substantia_nigra","Brain_Cerebellar_Hemisphere","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Frontal_Cortex_(BA9)","Esophagus_Muscularis","Esophagus_Gastroesophageal_Junction","Artery_Aorta"],[10361,7904,8932,7323,7387,7317,7436,8423,7915,9630,8352,8210,8422,9030,9430,9186,9522,9853,10052,10535,10632,10833,11217,11729,11586,11687,11778,11876,12065,12782,12200,12344,12444,12568,12759,13203,13013,13113,13121,13280,13231,13385,13454,14306,13904,14414,14840,14710,15021],[0.699026152786999,-0.29088704557465,-0.297004105871294,-0.312851602753609,-0.313827070599076,-0.313864158268405,-0.315483723777224,-0.316700075806313,-0.31794078966025,-0.318816735974098,-0.327608309854585,-0.327700882912484,-0.330130471489163,-0.337714461199159,-0.340473204335334,-0.340834875369971,-0.344392252851179,-0.347375418855097,-0.350648244787213,-0.356613896445558,-0.357954807310607,-0.36069568253424,-0.365062820791894,-0.367992222057443,-0.368403609246369,-0.370230652833556,-0.371216766758179,-0.373040842998823,-0.375102640256718,-0.375828793131468,-0.376125710606779,-0.377122450829452,-0.378323087232389,-0.37917119594477,-0.382116258647516,-0.383284630092241,-0.383370579544014,-0.385017462900749,-0.386265370194157,-0.387714387255502,-0.38813831933341,-0.388450085258062,-0.389498059338882,-0.392536685494804,-0.393452875456124,-0.398574410463878,-0.399152640840942,-0.399545451524121,-0.403804362486595],["-5.7896<br/>[-6.2001, -5.5301]","-5.6281<br/>[-5.9762, -5.4213]","-5.6285<br/>[-5.9761, -5.4218]","-5.6270<br/>[-5.9761, -5.4192]","-5.6270<br/>[-5.9761, -5.4192]","-5.6270<br/>[-5.9761, -5.4192]","-5.6269<br/>[-5.9761, -5.4191]","-5.6271<br/>[-5.9759, -5.4195]","-5.6270<br/>[-5.9759, -5.4193]","-5.6277<br/>[-5.9757, -5.4208]","-5.6267<br/>[-5.9757, -5.4189]","-5.6266<br/>[-5.9758, -5.4188]","-5.6266<br/>[-5.9757, -5.4188]","-5.6264<br/>[-5.9755, -5.4187]","-5.6264<br/>[-5.9753, -5.4187]","-5.6263<br/>[-5.9754, -5.4185]","-5.6262<br/>[-5.9753, -5.4185]","-5.6261<br/>[-5.9751, -5.4184]","-5.6260<br/>[-5.9751, -5.4183]","-5.6259<br/>[-5.9749, -5.4182]","-5.6258<br/>[-5.9748, -5.4181]","-5.6257<br/>[-5.9748, -5.4180]","-5.6256<br/>[-5.9746, -5.4179]","-5.6256<br/>[-5.9744, -5.4180]","-5.6255<br/>[-5.9745, -5.4178]","-5.6254<br/>[-5.9744, -5.4177]","-5.6254<br/>[-5.9744, -5.4177]","-5.6253<br/>[-5.9743, -5.4176]","-5.6252<br/>[-5.9743, -5.4175]","-5.6255<br/>[-5.9741, -5.4181]","-5.6252<br/>[-5.9742, -5.4175]","-5.6252<br/>[-5.9742, -5.4175]","-5.6251<br/>[-5.9741, -5.4175]","-5.6251<br/>[-5.9741, -5.4175]","-5.6250<br/>[-5.9740, -5.4173]","-5.6250<br/>[-5.9739, -5.4175]","-5.6250<br/>[-5.9739, -5.4174]","-5.6249<br/>[-5.9739, -5.4173]","-5.6248<br/>[-5.9739, -5.4172]","-5.6248<br/>[-5.9738, -5.4171]","-5.6248<br/>[-5.9738, -5.4170]","-5.6248<br/>[-5.9738, -5.4171]","-5.6247<br/>[-5.9737, -5.4170]","-5.6247<br/>[-5.9735, -5.4173]","-5.6246<br/>[-5.9736, -5.4169]","-5.6244<br/>[-5.9734, -5.4167]","-5.6244<br/>[-5.9733, -5.4170]","-5.6244<br/>[-5.9733, -5.4168]","-5.6242<br/>[-5.9731, -5.4166]"],["-3.9995<br/>[-4.0701, -3.9266]","-4.9316<br/>[-6.9091, -4.3195]","-4.9166<br/>[-6.6846, -4.3739]","-5.7513<br/>[-7.7389, -5.1321]","-5.7502<br/>[-7.7752, -5.1337]","-5.8582<br/>[-7.8841, -5.2429]","-5.8339<br/>[-7.8162, -5.2157]","-5.3042<br/>[-7.0914, -4.7390]","-5.5268<br/>[-7.5161, -4.9132]","-5.1114<br/>[-6.7823, -4.6050]","-5.7296<br/>[-7.7076, -5.1161]","-5.9110<br/>[-7.9021, -5.2983]","-5.8529<br/>[-7.8376, -5.2413]","-5.8001<br/>[-7.7566, -5.1920]","-5.6457<br/>[-7.5980, -5.0352]","-5.8919<br/>[-7.8468, -5.2847]","-5.8270<br/>[-7.7704, -5.2188]","-5.7474<br/>[-7.6816, -5.1419]","-5.8120<br/>[-7.6844, -5.2071]","-5.8233<br/>[-7.7406, -5.2191]","-5.8408<br/>[-7.7519, -5.2374]","-5.8748<br/>[-7.7785, -5.2724]","-5.8795<br/>[-7.7735, -5.2781]","-5.7134<br/>[-7.6026, -5.1138]","-5.8272<br/>[-7.6774, -5.2253]","-5.8822<br/>[-7.7632, -5.2821]","-5.8832<br/>[-7.7622, -5.2862]","-5.9433<br/>[-7.8237, -5.3457]","-5.9485<br/>[-7.8244, -5.3509]","-5.6364<br/>[-7.3779, -5.0787]","-5.9167<br/>[-7.7897, -5.3185]","-5.8807<br/>[-7.7500, -5.2870]","-5.8924<br/>[-7.7608, -5.2967]","-5.8637<br/>[-7.6898, -5.2671]","-5.9387<br/>[-7.7979, -5.3425]","-5.7425<br/>[-7.5923, -5.1497]","-5.8488<br/>[-7.7038, -5.2539]","-5.8942<br/>[-7.7463, -5.3021]","-5.9841<br/>[-7.8349, -5.3890]","-5.9745<br/>[-7.8212, -5.3802]","-6.0574<br/>[-7.9050, -5.4617]","-5.9519<br/>[-7.7962, -5.3575]","-5.9827<br/>[-7.8242, -5.3900]","-5.7022<br/>[-7.5292, -5.1111]","-5.9557<br/>[-7.7900, -5.3623]","-5.9867<br/>[-7.8101, -5.3956]","-5.7895<br/>[-7.5627, -5.2004]","-5.8696<br/>[-7.6911, -5.2790]","-5.9702<br/>[-7.7822, -5.3826]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["19","12","18","1","13","15","20","7","17","10","6","14","2","11","9","3","8","16","4","5"],[6541,7719,7966,8494,8480,8556,8977,9060,9336,9212,9648,10060,10154,11103,10233,10846,10754,10894,12973,12195],[-0.250872830040795,-0.261902264903014,-0.273450257045087,-0.282809213548035,-0.286245076506903,-0.286588757656899,-0.290938112233648,-0.292883109796289,-0.294077867219248,-0.295857223246821,-0.299163140817415,-0.305053674623454,-0.305247986593833,-0.307885538147853,-0.30876310789527,-0.311188178146649,-0.312607524868716,-0.316491153559136,-0.318813021387052,-0.331220732798152],["-5.6274<br/>[-5.9764, -5.4197]","-5.6273<br/>[-5.9761, -5.4199]","-5.6270<br/>[-5.9759, -5.4193]","-5.6267<br/>[-5.9757, -5.4190]","-5.6265<br/>[-5.9757, -5.4188]","-5.6265<br/>[-5.9756, -5.4188]","-5.6265<br/>[-5.9755, -5.4188]","-5.6264<br/>[-5.9754, -5.4187]","-5.6264<br/>[-5.9754, -5.4188]","-5.6263<br/>[-5.9754, -5.4185]","-5.6262<br/>[-5.9752, -5.4186]","-5.6261<br/>[-5.9751, -5.4184]","-5.6261<br/>[-5.9750, -5.4184]","-5.6264<br/>[-5.9748, -5.4192]","-5.6259<br/>[-5.9750, -5.4182]","-5.6259<br/>[-5.9748, -5.4184]","-5.6259<br/>[-5.9748, -5.4182]","-5.6257<br/>[-5.9747, -5.4180]","-5.6262<br/>[-5.9743, -5.4193]","-5.6252<br/>[-5.9742, -5.4175]"],["-5.4629<br/>[-7.5058, -4.8421]","-5.2423<br/>[-7.2356, -4.6318]","-5.5094<br/>[-7.5074, -4.8966]","-5.6162<br/>[-7.5492, -5.0036]","-5.8756<br/>[-7.8556, -5.2616]","-5.8145<br/>[-7.7846, -5.2027]","-5.7140<br/>[-7.6847, -5.1046]","-5.7777<br/>[-7.7362, -5.1658]","-5.6381<br/>[-7.5754, -5.0308]","-5.8571<br/>[-7.8114, -5.2499]","-5.7036<br/>[-7.6365, -5.0966]","-5.7684<br/>[-7.6647, -5.1696]","-5.6998<br/>[-7.6270, -5.0927]","-5.4468<br/>[-7.1977, -4.9074]","-5.8771<br/>[-7.7607, -5.2748]","-5.6097<br/>[-7.5102, -5.0102]","-5.7288<br/>[-7.6347, -5.1251]","-5.8825<br/>[-7.7817, -5.2804]","-5.2882<br/>[-7.0274, -4.7097]","-5.9311<br/>[-7.8018, -5.3326]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
<br> <br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" title="1">devtools<span class="op">::</span><span class="kw">session_info</span>()</a></code></pre></div>
<pre><code>─ Session info ──────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.5.2 (2018-12-20)
 os       macOS Mojave 10.14.3        
 system   x86_64, darwin15.6.0        
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2019-02-18                  

─ Packages ──────────────────────────────────────────────────────────────
 package     * version date       lib source        
 assertthat    0.2.0   2017-04-11 [1] CRAN (R 3.5.0)
 backports     1.1.3   2018-12-14 [1] CRAN (R 3.5.0)
 callr         3.1.1   2018-12-21 [1] CRAN (R 3.5.0)
 cli           1.0.1   2018-09-25 [1] CRAN (R 3.5.0)
 crayon        1.3.4   2017-09-16 [1] CRAN (R 3.5.0)
 crosstalk     1.0.0   2016-12-21 [1] CRAN (R 3.5.0)
 desc          1.2.0   2018-05-01 [1] CRAN (R 3.5.0)
 devtools      2.0.1   2018-10-26 [1] CRAN (R 3.5.1)
 digest        0.6.18  2018-10-10 [1] CRAN (R 3.5.0)
 dplyr       * 0.8.0.1 2019-02-15 [1] CRAN (R 3.5.2)
 DT          * 0.5     2018-11-05 [1] CRAN (R 3.5.0)
 evaluate      0.13    2019-02-12 [1] CRAN (R 3.5.2)
 fs            1.2.6   2018-08-23 [1] CRAN (R 3.5.0)
 git2r         0.24.0  2019-01-07 [1] CRAN (R 3.5.2)
 glue          1.3.0   2018-07-17 [1] CRAN (R 3.5.0)
 htmltools     0.3.6   2017-04-28 [1] CRAN (R 3.5.0)
 htmlwidgets   1.3     2018-09-30 [1] CRAN (R 3.5.0)
 httpuv        1.4.5.1 2018-12-18 [1] CRAN (R 3.5.0)
 jsonlite      1.6     2018-12-07 [1] CRAN (R 3.5.0)
 knitr         1.21    2018-12-10 [1] CRAN (R 3.5.2)
 later         0.8.0   2019-02-11 [1] CRAN (R 3.5.2)
 magrittr      1.5     2014-11-22 [1] CRAN (R 3.5.0)
 memoise       1.1.0   2017-04-21 [1] CRAN (R 3.5.0)
 mime          0.6     2018-10-05 [1] CRAN (R 3.5.1)
 pillar        1.3.1   2018-12-15 [1] CRAN (R 3.5.0)
 pkgbuild      1.0.2   2018-10-16 [1] CRAN (R 3.5.0)
 pkgconfig     2.0.2   2018-08-16 [1] CRAN (R 3.5.0)
 pkgload       1.0.2   2018-10-29 [1] CRAN (R 3.5.0)
 plyr        * 1.8.4   2016-06-08 [1] CRAN (R 3.5.0)
 prettyunits   1.0.2   2015-07-13 [1] CRAN (R 3.5.0)
 processx      3.2.1   2018-12-05 [1] CRAN (R 3.5.0)
 promises      1.0.1   2018-04-13 [1] CRAN (R 3.5.0)
 ps            1.3.0   2018-12-21 [1] CRAN (R 3.5.0)
 purrr         0.3.0   2019-01-27 [1] CRAN (R 3.5.2)
 R.matlab    * 3.6.2   2018-09-27 [1] CRAN (R 3.5.0)
 R.methodsS3   1.7.1   2016-02-16 [1] CRAN (R 3.5.0)
 R.oo          1.22.0  2018-04-22 [1] CRAN (R 3.5.0)
 R.utils       2.8.0   2019-02-14 [1] CRAN (R 3.5.2)
 R6            2.4.0   2019-02-14 [1] CRAN (R 3.5.2)
 Rcpp          1.0.0   2018-11-07 [1] CRAN (R 3.5.0)
 remotes       2.0.2   2018-10-30 [1] CRAN (R 3.5.0)
 rlang         0.3.1   2019-01-08 [1] CRAN (R 3.5.2)
 rmarkdown     1.11    2018-12-08 [1] CRAN (R 3.5.0)
 rprojroot     1.3-2   2018-01-03 [1] CRAN (R 3.5.0)
 sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.5.0)
 shiny         1.2.0   2018-11-02 [1] CRAN (R 3.5.0)
 stringi       1.3.1   2019-02-13 [1] CRAN (R 3.5.2)
 stringr       1.4.0   2019-02-10 [1] CRAN (R 3.5.2)
 testthat      2.0.1   2018-10-13 [1] CRAN (R 3.5.0)
 tibble        2.0.1   2019-01-12 [1] CRAN (R 3.5.2)
 tidyselect    0.2.5   2018-10-11 [1] CRAN (R 3.5.0)
 usethis       1.4.0   2018-08-14 [1] CRAN (R 3.5.0)
 whisker       0.3-2   2013-04-28 [1] CRAN (R 3.5.0)
 withr         2.1.2   2018-03-15 [1] CRAN (R 3.5.0)
 workflowr     1.2.0   2019-02-14 [1] CRAN (R 3.5.2)
 xfun          0.4     2018-10-23 [1] CRAN (R 3.5.0)
 xtable        1.8-3   2018-08-29 [1] CRAN (R 3.5.0)
 yaml          2.2.0   2018-07-25 [1] CRAN (R 3.5.0)

[1] /Library/Frameworks/R.framework/Versions/3.5/Resources/library</code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
