<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2018-09-16" />

<title>Overview of 4,026 pre-processed gene sets</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Overview of 4,026 pre-processed gene sets</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2018-09-16</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-10-19</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/0515710d2e5e392193030ff6a3bed77596befbb2" target="_blank">0515710</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/0515710d2e5e392193030ff6a3bed77596befbb2/analysis/gene_set.Rmd" target="_blank">0515710</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-19
</td>
<td style="text-align:left;">
wflow_publish(“analysis/gene_set.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/gene_set.html" target="_blank">1c85967</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-05
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/0324740b5df4b7ed5ad5ae90510a14a1b85c4ec7/docs/gene_set.html" target="_blank">0324740</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-18
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/bcd04240a676c6f2acedc470d885f6ea9dd5ca62/analysis/gene_set.Rmd" target="_blank">bcd0424</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-18
</td>
<td style="text-align:left;">
wflow_publish(“analysis/gene_set.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/ddd848072123585e01f30360d5a55eeaece1e27f/docs/gene_set.html" target="_blank">ddd8480</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-17
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/decce81d337a7df15aa72274bb8871937eb64ee5/analysis/gene_set.Rmd" target="_blank">decce81</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-17
</td>
<td style="text-align:left;">
wflow_publish(“analysis/gene_set.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/32d3a605d73dd7ef14bf6dc0453b43343381cde2/docs/gene_set.html" target="_blank">32d3a60</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-16
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/8572f1a0008455bea10ad24e3a9c3fb70b4a60b4/analysis/gene_set.Rmd" target="_blank">8572f1a</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-16
</td>
<td style="text-align:left;">
wflow_publish(“analysis/gene_set.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<p>All 4,026 gene sets used in <a href="https://www.nature.com/articles/s41467-018-06805-x">Zhu and Stephens (2018)</a> are freely available at <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data"><code>xiangzhu/rss-gsea</code></a>, where the folder <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data/biological_pathway"><code>biological_pathway</code></a> contains 3,913 biological pathways, and the folder <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data/tissue_set"><code>tissue_set</code></a> contains 113 GTEx tissue-based gene sets. These gene sets can be referenced in a journal’s “Data availability” section as <a href="https://zenodo.org/badge/latestdoi/55633948"><img src="https://zenodo.org/badge/55633948.svg" alt="DOI" /></a>.</p>
<pre><code>data/
├── README.md
├── biological_pathway
│   ├── gene_37.3.mat
│   └── pathway.mat
└── tissue_set
    ├── de_genes
    ├── he_genes
    └── se_genes

5 directories, 3 files</code></pre>
<div id="biological-pathways" class="section level2">
<h2>Biological pathways</h2>
<p>The 3,913 GTEx biological pathway used in <a href="https://www.nature.com/articles/s41467-018-06805-x">Zhu and Stephens (2018)</a> are available in the folder <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data/biological_pathway"><code>biological_pathway</code></a>, which are represented by two files <code>gene_37.3.mat</code> and <code>pathway.mat</code>.</p>
<p>The file <code>gene_37.3.mat</code> contains basic information of genes.</p>
<pre><code>&gt;&gt; load gene_37.3.mat
&gt;&gt; gene
gene =
  struct with fields:
        id: [18732x1 double]
    symbol: {18732x1 cell}
       chr: [18732x1 double]
      desc: {18732x1 cell}
     start: [18732x1 double]
      stop: [18732x1 double]

&gt;&gt; [gene.id(10) gene.chr(10) gene.start(10) gene.stop(10)]
ans =
          18          16     8768444     8878432

&gt;&gt; gene.symbol(10)
ans =
  1x1 cell array
    {&#39;ABAT&#39;}

&gt;&gt; gene.desc(10)
ans =
  1x1 cell array
    {&#39;4-aminobutyrate aminotransferase&#39;}</code></pre>
<p>Note that only 18,313 genes mapped to reference sequence were used in our analyses.</p>
<pre><code>&gt;&gt; [min(gene.start) min(gene.stop)]
ans =
    -1    -1

&gt;&gt; inref_genes = ~(gene.start == -1 | gene.stop == -1);
&gt;&gt; sum(inref_genes)
ans =
       18313</code></pre>
<p>The file <code>pathway.mat</code> contains basic information of pathways.</p>
<pre><code>&gt;&gt; load pathway.mat
&gt;&gt; pathway
pathway =
  struct with fields:
       label: {4076x1 cell}
    database: {4076x1 cell}
      source: {4076x1 cell}
       genes: [18732x4076 double]
    synonyms: {4076x1 cell}

&gt;&gt; pathway.label(100)
ans =
  1x1 cell array
    {&#39;Activation of NOXA and translocation to mitochondria&#39;}

&gt;&gt; pathway.database(100)
ans =
  1x1 cell array
    {&#39;PC&#39;}

&gt;&gt; pathway.source(100)
ans =
  1x1 cell array
    {&#39;reactome&#39;}</code></pre>
<p>The gene-pathway information is represented as a sparse zero-one matrix <code>pathway.genes</code>, where <code>genes(i,j)==1</code> if gene <code>i</code> is a member of pathway <code>j</code> and <code>genes(i,j)==0</code> otherwise.</p>
<pre><code>&gt;&gt; genes = pathway.genes;
&gt;&gt; whos genes
  Name           Size                Bytes  Class     Attributes
  genes      18732x4076            3257512  double    sparse

&gt;&gt; genes(:,100)

ans =
      (1243,1)              1
      (3410,1)              1
      (4567,1)              1
      (4668,1)              1  </code></pre>
<p>Finally, our analyses only used 3,913 of 4,076 pathways that</p>
<ul>
<li>include 2-499 RefSeq-mapped genes;</li>
<li>have clear <code>database</code> and <code>source</code> definitions;</li>
<li>exclude one pathway <code>Viral RNP Complexes in the Host Cell Nucleus (PC, reactome)</code> (because no HapMap3 SNP was mapped to this pathway).</li>
</ul>
<pre><code>&gt;&gt; numgenes = pathway.genes&#39; * inref_genes;
&gt;&gt; size(numgenes)
ans =
        4076           1

&gt;&gt; paths = find(numgenes &gt; 1 &amp; numgenes &lt; 500);
&gt;&gt; size(paths)
ans =
        3916           1

&gt;&gt; database = pathway.database;
&gt;&gt; source = pathway.source;
&gt;&gt; database_na = find(not(cellfun(&#39;isempty&#39;, strfind(database, &#39;NA&#39;))));
&gt;&gt; source_na = find(not(cellfun(&#39;isempty&#39;, strfind(source, &#39;NA&#39;))));
&gt;&gt; length(union(database_na, source_na))
ans =
     2

&gt;&gt; label = pathway.label;
&gt;&gt; pathway_exclude = &#39;Viral RNP Complexes in the Host Cell Nucleus&#39;;
&gt;&gt; label_include = find(cellfun(&#39;isempty&#39;, strfind(label, pathway_exclude)));
&gt;&gt; label_exclude = setdiff(1:4076, label_include);
&gt;&gt; label(label_exclude)
ans =
  1x1 cell array
    {&#39;Viral RNP Complexes in the Host Cell Nucleus&#39;}

&gt;&gt; database(label_exclude)
ans =
  1x1 cell array
    {&#39;PC&#39;}

&gt;&gt; source(label_exclude)
ans =
  1x1 cell array
    {&#39;reactome&#39;}</code></pre>
</div>
<div id="tissue-based-gene-sets" class="section level2">
<h2>Tissue-based gene sets</h2>
<p>The 113 GTEx tissue-based gene sets used in <a href="https://www.nature.com/articles/s41467-018-06805-x">Zhu and Stephens (2018)</a> are available in the folder <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data/tissue_set"><code>tissue_set</code></a>. There are 44 “highly expressed” (HE) gene sets, 49 “selectively expressed” (SE) gene sets and 20 “distinctively expressed” (DE) gene sets. The creation of SE sets uses a method described in <a href="https://doi.org/10.1101/311563">Yang et al (2018)</a>. The creation of DE sets uses a method described in <a href="https://www.ncbi.nlm.nih.gov/pubmed/28333934">Dey et al (2017)</a>.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb7-1" data-line-number="1"><span class="fu">ls</span> data/tissue_set/he_genes <span class="kw">|</span> <span class="fu">wc</span> -l</a></code></pre></div>
<pre><code>      44</code></pre>
<div class="sourceCode" id="cb9"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb9-1" data-line-number="1"><span class="fu">ls</span> data/tissue_set/se_genes <span class="kw">|</span> <span class="fu">wc</span> -l</a></code></pre></div>
<pre><code>      49</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb11-1" data-line-number="1"><span class="fu">ls</span> data/tissue_set/de_genes <span class="kw">|</span> <span class="fu">wc</span> -l</a></code></pre></div>
<pre><code>      20</code></pre>
<p>Each of the tissue-based gene sets has the following format.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb13-1" data-line-number="1"><span class="fu">head</span> -n 5 data/tissue_set/he_genes/gene_names_expr_LIVER.txt</a></code></pre></div>
<pre><code>ensembl_gene_id chromosome_name start_position  end_position
ENSG00000002933 7   150497491   150502208
ENSG00000072778 17  7120444 7128592
ENSG00000075624 7   5566782 5603415
ENSG00000087086 19  49468558    49470135</code></pre>
<p>Note that the gene information of tissue-based sets was provided by GTEx, which may not be the same as <code>gene_37.3.mat</code> above.</p>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb15-1" data-line-number="1">devtools<span class="op">::</span><span class="kw">session_info</span>()</a></code></pre></div>
<pre><code>Session info -------------------------------------------------------------</code></pre>
<pre><code> setting  value                       
 version  R version 3.5.1 (2018-07-02)
 system   x86_64, darwin15.6.0        
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2018-10-19                  </code></pre>
<pre><code>Packages -----------------------------------------------------------------</code></pre>
<pre><code> package     * version date       source        
 backports     1.1.2   2017-12-13 CRAN (R 3.5.0)
 base        * 3.5.1   2018-07-05 local         
 compiler      3.5.1   2018-07-05 local         
 datasets    * 3.5.1   2018-07-05 local         
 devtools      1.13.6  2018-06-27 CRAN (R 3.5.0)
 digest        0.6.17  2018-09-12 CRAN (R 3.5.0)
 evaluate      0.11    2018-07-17 CRAN (R 3.5.0)
 git2r         0.23.0  2018-07-17 CRAN (R 3.5.0)
 graphics    * 3.5.1   2018-07-05 local         
 grDevices   * 3.5.1   2018-07-05 local         
 htmltools     0.3.6   2017-04-28 CRAN (R 3.5.0)
 knitr         1.20    2018-02-20 CRAN (R 3.5.0)
 magrittr      1.5     2014-11-22 CRAN (R 3.5.0)
 memoise       1.1.0   2017-04-21 CRAN (R 3.5.0)
 methods     * 3.5.1   2018-07-05 local         
 R.methodsS3   1.7.1   2016-02-16 CRAN (R 3.5.0)
 R.oo          1.22.0  2018-04-22 CRAN (R 3.5.0)
 R.utils       2.7.0   2018-08-27 CRAN (R 3.5.0)
 Rcpp          0.12.19 2018-10-01 CRAN (R 3.5.0)
 rmarkdown     1.10    2018-06-11 CRAN (R 3.5.0)
 rprojroot     1.3-2   2018-01-03 CRAN (R 3.5.0)
 stats       * 3.5.1   2018-07-05 local         
 stringi       1.2.4   2018-07-20 CRAN (R 3.5.0)
 stringr       1.3.1   2018-05-10 CRAN (R 3.5.0)
 tools         3.5.1   2018-07-05 local         
 utils       * 3.5.1   2018-07-05 local         
 whisker       0.3-2   2013-04-28 CRAN (R 3.5.0)
 withr         2.1.2   2018-03-15 CRAN (R 3.5.0)
 workflowr     1.1.1   2018-07-06 CRAN (R 3.5.0)
 yaml          2.2.0   2018-07-25 CRAN (R 3.5.0)</code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
