<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of gout</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
<li>
  <a href="https://stephenslab.github.io/rss/news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Enrichment results of gout</h1>
<h4 class="author">Xiang Zhu</h4>
<h4 class="date">2017-04-12</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-09-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rss-gsea/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20180626code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20180626)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20180626code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20180626)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree0d0131b608937a899299b4dd8cf4ff994ecea3cbtargetblank0d0131ba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/rss-gsea/tree/0d0131b608937a899299b4dd8cf4ff994ecea3cb" target="_blank">0d0131b</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree0d0131b608937a899299b4dd8cf4ff994ecea3cbtargetblank0d0131ba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/rss-gsea/tree/0d0131b608937a899299b4dd8cf4ff994ecea3cb" target="_blank">0d0131b</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/gout_2013.Rmd</code>) and HTML
(<code>docs/gout_2013.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/dedfe81645d7f81d60203d25ab700459d91649ab/docs/gout_2013.html" target="_blank">dedfe81</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2024-09-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/72237818c1b8a1963ebe64d9a5776caf4d1ca79e/docs/gout_2013.html" target="_blank">7223781</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2019-02-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/73f3c80eafd4669d84f817685bc7f2ded7510544/docs/gout_2013.html" target="_blank">73f3c80</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/gout_2013.html" target="_blank">1c85967</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/47e1648e2c99d6590a27190f0e885ed8bcfee4ae/docs/gout_2013.html" target="_blank">47e1648</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/8c82bddcc5674e1ce1a825f0572445a66a7504b4/analysis/gout_2013.Rmd" target="_blank">8c82bdd</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-02
</td>
<td>
wflow_publish("gout_2013.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics
published in the paper <a
href="https://www.ncbi.nlm.nih.gov/pubmed/23263486">“Genome-Wide
Association Analyses Identify 18 New Loci Associated with Serum Urate
Concentrations” (<em>Nature Genetics</em>, 2013)</a>. The summary data
file is available at <a
href="http://metabolomics.helmholtz-muenchen.de/gugc"
class="uri">http://metabolomics.helmholtz-muenchen.de/gugc</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three
quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model
against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong>
proportion of trait-associated SNPs that are “outside” the gene
set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong>
proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas
the last two capture the magnitude of enrichment. For each gene set, we
report these three quantities in the last three columns of tables below,
on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-31fbc4f90f9b8b0a7aa7" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-31fbc4f90f9b8b0a7aa7">{"x":{"filter":"none","vertical":false,"data":[["Toxicity of botulinum toxin type B (BoNT/B)<br/>(246, reactome, PC)","Toxicity of botulinum toxin type G (BoNT/G)<br/>(249, reactome, PC)","Cholecalciferol biosynthesis<br/>(151, kegg, BioSystems)","vitamin D3 biosynthesis<br/>(144, biocyc, BioSystems)","Msp/Ron Receptor Signaling Pathway<br/>(737, biocarta, BioCarta)","amb2 Integrin signaling<br/>(2482, pid, PC)","Acetate utilization<br/>(178, panther, PC)","Salvage pyrimidine deoxyribonucleotides<br/>(61, panther, PC)","pyrimidine ribonucleosides salvage I<br/>(309, biocyc, BioSystems)","1,25-dihydroxyvitamin D3 biosynthesis<br/>(174, humancyc, PC)","Synthesis of IPs in the nucleus<br/>(404, reactome, PC)","inositol pyrophosphates biosynthesis<br/>(834, humancyc, PC)","Synthesis of pyrophosphates in the cytosol<br/>(939, reactome, PC)","Termination of translesion DNA synthesis<br/>(2401, reactome, BioSystems)","STAT6-mediated induction of chemokines<br/>(235, reactome, PC)","IL 4 signaling pathway<br/>(1165, biocarta, BioCarta)","JAK/STAT signaling pathway<br/>(1431, panther, PANTHER)","JAK/STAT signaling pathway<br/>(1478, panther, PC)","Scavenging of heme from plasma<br/>(1355, reactome, BioSystems)","IL-10 Anti-inflammatory Signaling Pathway<br/>(1724, biocarta, BioCarta)","Scavenging of heme from plasma<br/>(1410, reactome, PC)","STING mediated induction of host immune responses<br/>(1647, reactome, PC)","STING mediated induction of host immune responses<br/>(1741, reactome, BioSystems)","IL22 Soluble Receptor Signaling Pathway <br/>(1657, biocarta, BioCarta)","JAK-STAT signaling<br/>(1859, kegg, BioSystems)","Regulation by TREX1<br/>(126, reactome, BioSystems)","V-type ATPase, eukaryotes<br/>(1980, kegg, BioSystems)","Signaling by FGFR1 fusion mutants<br/>(1802, reactome, BioSystems)","Signaling by FGFR1 fusion mutants<br/>(1836, reactome, PC)","Collecting duct acid secretion<br/>(2184, kegg, BioSystems)","Insulin receptor recycling<br/>(2060, reactome, PC)","Transferrin endocytosis and recycling<br/>(2211, reactome, PC)","UTP and CTP dephosphorylation I<br/>(79, humancyc, PC)","Thiamine metabolism<br/>(407, kegg, PC)","Osteoblast Signaling<br/>(1453, wiki, BioSystems)","pyrimidine deoxyribonucleotides de novo biosynthesis<br/>(1531, biocyc, BioSystems)","pyrimidine deoxyribonucleotides de novo biosynthesis<br/>(1501, humancyc, PC)","Interleukin-11 Signaling Pathway<br/>(2737, wiki, BioSystems)","Formation of the Editosome<br/>(26, reactome, PC)","Overview of telomerase protein component gene hTert Transcriptional Regulation <br/>(963, biocarta, BioCarta)","Overview of telomerase RNA component gene hTerc Transcriptional Regulation<br/>(739, biocarta, BioCarta)","pyrimidine deoxyribonucleotides biosynthesis from CTP<br/>(1530, biocyc, BioSystems)","NADH repair<br/>(220, humancyc, PC)","Glycolysis, core module involving three-carbon compounds<br/>(1198, kegg, BioSystems)","Glycolysis Pathway<br/>(1067, biocarta, BioCarta)","AKAP95 role in mitosis and chromosome dynamics<br/>(1257, biocarta, BioCarta)","Lysosphingolipid and LPA receptors<br/>(1029, reactome, PC)","ECM proteoglycans<br/>(3036, reactome, PC)","Androgen/estrogene/progesterone biosynthesis<br/>(859, panther, PANTHER)","Class II GLUTs<br/>(365, reactome, PC)","Facilitative Na+-independent glucose transporters<br/>(1225, reactome, PC)","Facilitative Na+-independent glucose transporters<br/>(1434, reactome, BioSystems)","Hypertrophy Model<br/>(1921, wiki, BioSystems)","Osteopontin Signaling<br/>(1294, wiki, BioSystems)","Transport of glucose and other sugars, bile salts and organic acids, metal ions and amine compounds<br/>(3496, reactome, PC)","Osteoclast Signaling<br/>(1687, wiki, BioSystems)","Transport of glucose and other sugars, bile salts and organic acids, metal ions and amine compounds<br/>(3518, reactome, BioSystems)","pyrimidine deoxyribonucleotides biosynthesis from CTP<br/>(1500, humancyc, PC)","SLC-mediated transmembrane transport<br/>(3847, reactome, PC)","SLC-mediated transmembrane transport<br/>(3852, reactome, BioSystems)","Abacavir transmembrane transport<br/>(514, reactome, PC)","Platelet degranulation<br/>(3307, reactome, PC)","Abacavir transport and metabolism<br/>(1104, reactome, PC)","Irinotecan Pathway<br/>(1451, wiki, BioSystems)","Response to elevated platelet cytosolic Ca2+<br/>(3355, reactome, PC)","Alpha9 beta1 integrin signaling events<br/>(1988, pid, PC)","Alpha4 beta1 integrin signaling events<br/>(2513, pid, PC)","Osteopontin-mediated events<br/>(2431, pid, PC)","Osteopontin-mediated events<br/>(2508, pid, BioSystems)","Integrins in angiogenesis<br/>(2878, pid, BioSystems)","TGF Beta Signaling Pathway<br/>(3031, wiki, BioSystems)","VxPx cargo-targeting to cilium<br/>(1917, reactome, BioSystems)","FGF signaling pathway<br/>(2903, pid, BioSystems)","ABC transporters<br/>(2792, kegg, BioSystems)","HIF-2-alpha transcription factor network<br/>(2466, pid, PC)","FGF signaling pathway<br/>(3037, pid, PC)","Degradation of the extracellular matrix<br/>(3442, reactome, PC)","Integrins in angiogenesis<br/>(3113, pid, PC)","Iron uptake and transport<br/>(2693, reactome, PC)","Regulation of toll-like receptor signaling pathway<br/>(3521, wiki, BioSystems)","Toll-like receptor signaling pathway<br/>(3557, kegg, BioSystems)","Toll-like receptor signaling pathway<br/>(3525, wiki, BioSystems)","Fluoropyrimidine Activity<br/>(2543, wiki, BioSystems)","Endochondral Ossification<br/>(3208, wiki, BioSystems)","Beta1 integrin cell surface interactions<br/>(3177, pid, PC)","Beta3 integrin cell surface interactions<br/>(2717, pid, PC)","Direct p53 effectors<br/>(3683, pid, BioSystems)","Direct p53 effectors<br/>(3703, pid, PC)","Cargo trafficking to the periciliary membrane<br/>(2947, reactome, BioSystems)","HIF-1-alpha transcription factor network<br/>(3190, pid, BioSystems)","HIF-1-alpha transcription factor network<br/>(3201, pid, PC)","Dopamine receptors<br/>(535, reactome, PC)","Bile secretion<br/>(3277, kegg, BioSystems)","Integrin cell surface interactions<br/>(3212, reactome, PC)","BDNF signaling pathway<br/>(3695, wiki, BioSystems)","Interactions of Tat with host cellular proteins<br/>(38, reactome, PC)","ECM-receptor interaction<br/>(3433, kegg, BioSystems)","Assembly of the primary cilium<br/>(3764, reactome, BioSystems)","Organic anion transport<br/>(560, reactome, PC)"],[3,3,2,2,6,32,3,2,3,3,4,7,8,30,3,10,13,14,12,17,13,16,17,16,19,2,21,18,19,25,23,26,2,4,13,14,14,40,2,8,6,14,3,10,9,11,9,54,7,4,11,13,20,11,95,16,98,14,251,254,5,74,10,13,79,22,33,31,32,46,53,20,47,42,32,54,87,60,39,98,102,98,33,65,64,40,133,139,49,64,65,5,70,66,136,2,85,173,5],[448,440,107,128,402,2569,322,244,357,177,284,591,732,2465,151,755,960,1040,912,1178,980,1222,1232,1305,1411,61,1698,1822,1867,1910,1877,2178,193,449,1348,1006,1080,3489,68,633,509,1044,317,565,761,804,899,6368,890,506,963,1126,1872,850,8901,1531,9319,1118,24511,25015,628,7107,1059,1210,7995,2694,3391,3304,3382,4731,5469,2419,4514,4488,2815,5134,9202,6412,3287,6897,6977,6993,3927,7640,7651,4417,11732,12072,4961,6189,6792,528,7652,6992,15052,126,10856,15460,203],[266.2800032649899,259.2035674024524,169.0275247789214,168.936823283061,148.1692616778697,147.0503767084754,124.2606046970852,118.3041114902009,117.8498056223834,106.5041799041078,90.44532629136287,90.21963614794385,90.12934294526832,89.11010919394239,79.95284521641798,78.78606504645786,78.44680908968208,78.35704772796421,78.26454208052942,78.25481707001276,78.22313586420329,78.20751828068757,78.2008848907324,78.03873386612543,77.96531709256276,61.04560552956429,36.30199127033057,36.26059418851388,36.23841405841082,36.21658409985716,36.11304194731462,36.03331988879832,33.64038536388139,32.55976397343827,30.57770332829038,22.93288079030026,22.87846715582339,20.48221376128888,17.18604202299396,15.83649871986767,15.80184105950375,13.85120151287929,13.41273145251315,13.29777585455615,12.93407357461975,12.90388254615198,12.72056930081838,10.1320613140682,7.991812028309453,4.469835361571771,3.880251860766158,3.736504485657202,3.295815015476728,2.941073568740818,2.152478707407405,2.128371358085386,2.102546729270432,1.889192998561422,1.744643767803447,1.722213896457295,1.535486334116902,1.42477957995859,1.272805204700424,1.241831734775401,1.220874655605819,1.192019429570572,1.032199595022226,1.02310770106651,1.012715281807932,0.901145499869511,0.883026267593747,0.8740782336291665,0.8694870735813588,0.8423462188906259,0.8349798966528098,0.8195205372611181,0.7950535979554824,0.7690258245140601,0.7549083005866064,0.7107422907723385,0.7083837436489515,0.705087402283046,0.6772657957384975,0.6741644706923982,0.6704252300511711,0.6555721267553669,0.6088846908877329,0.5939091184138412,0.5867156242004344,0.5206429969238145,0.473097752622221,0.4564127399728627,0.4458568599571649,0.4175902628929491,0.4100662729696376,0.2825765843980976,0.2728546849953255,0.1163490318422436,0.09030863610992192],["-5.3409<br/>[-5.6089, -5.1455]","-5.3409<br/>[-5.6089, -5.1455]","-5.3409<br/>[-5.6089, -5.1454]","-5.3409<br/>[-5.6089, -5.1454]","-5.3408<br/>[-5.6088, -5.1453]","-5.3409<br/>[-5.6084, -5.1459]","-5.3408<br/>[-5.6088, -5.1453]","-5.3408<br/>[-5.6089, -5.1453]","-5.3408<br/>[-5.6088, -5.1453]","-5.3408<br/>[-5.6089, -5.1454]","-5.3408<br/>[-5.6088, -5.1453]","-5.3407<br/>[-5.6087, -5.1452]","-5.3406<br/>[-5.6087, -5.1452]","-5.3402<br/>[-5.6081, -5.1449]","-5.3409<br/>[-5.6089, -5.1453]","-5.3407<br/>[-5.6087, -5.1452]","-5.3406<br/>[-5.6086, -5.1451]","-5.3406<br/>[-5.6086, -5.1451]","-5.3408<br/>[-5.6087, -5.1455]","-5.3405<br/>[-5.6085, -5.1451]","-5.3408<br/>[-5.6087, -5.1454]","-5.3405<br/>[-5.6085, -5.1451]","-5.3405<br/>[-5.6085, -5.1451]","-5.3405<br/>[-5.6085, -5.1450]","-5.3404<br/>[-5.6084, -5.1450]","-5.3409<br/>[-5.6089, -5.1454]","-5.3404<br/>[-5.6083, -5.1450]","-5.3404<br/>[-5.6083, -5.1450]","-5.3403<br/>[-5.6083, -5.1450]","-5.3404<br/>[-5.6083, -5.1450]","-5.3404<br/>[-5.6083, -5.1450]","-5.3402<br/>[-5.6081, -5.1449]","-5.3409<br/>[-5.6089, -5.1454]","-5.3408<br/>[-5.6088, -5.1454]","-5.3432<br/>[-5.6102, -5.1483]","-5.3406<br/>[-5.6086, -5.1452]","-5.3406<br/>[-5.6086, -5.1452]","-5.3399<br/>[-5.6077, -5.1446]","-5.3409<br/>[-5.6089, -5.1454]","-5.3408<br/>[-5.6088, -5.1454]","-5.3407<br/>[-5.6087, -5.1452]","-5.3407<br/>[-5.6086, -5.1453]","-5.3408<br/>[-5.6088, -5.1454]","-5.3408<br/>[-5.6088, -5.1454]","-5.3407<br/>[-5.6087, -5.1453]","-5.3407<br/>[-5.6087, -5.1453]","-5.3406<br/>[-5.6086, -5.1451]","-5.3390<br/>[-5.6066, -5.1439]","-5.3406<br/>[-5.6086, -5.1451]","-5.4415<br/>[-5.8327, -5.2069]","-5.4414<br/>[-5.8326, -5.2068]","-5.4413<br/>[-5.8325, -5.2067]","-5.3406<br/>[-5.6122, -5.1453]","-5.3407<br/>[-5.6087, -5.1453]","-5.4422<br/>[-5.8321, -5.2087]","-5.3405<br/>[-5.6084, -5.1451]","-5.4421<br/>[-5.8319, -5.2086]","-5.3407<br/>[-5.6086, -5.1453]","-5.4440<br/>[-5.8315, -5.2125]","-5.4438<br/>[-5.8313, -5.2123]","-5.4414<br/>[-5.8185, -5.2068]","-5.3391<br/>[-5.6103, -5.1441]","-5.4412<br/>[-5.8184, -5.2067]","-5.4411<br/>[-5.8183, -5.2065]","-5.3387<br/>[-5.6100, -5.1438]","-5.3404<br/>[-5.6081, -5.1452]","-5.3400<br/>[-5.6078, -5.1448]","-5.3400<br/>[-5.6078, -5.1447]","-5.3399<br/>[-5.6077, -5.1447]","-5.3398<br/>[-5.6074, -5.1447]","-5.3400<br/>[-5.6073, -5.1453]","-5.4408<br/>[-5.8179, -5.2063]","-5.3395<br/>[-5.6073, -5.1443]","-5.4406<br/>[-5.8174, -5.2064]","-5.4409<br/>[-5.8179, -5.2065]","-5.3394<br/>[-5.6071, -5.1442]","-5.3401<br/>[-5.6064, -5.1462]","-5.3393<br/>[-5.6068, -5.1444]","-5.4407<br/>[-5.8177, -5.2062]","-5.3390<br/>[-5.6065, -5.1440]","-5.3390<br/>[-5.6065, -5.1440]","-5.3389<br/>[-5.6065, -5.1439]","-5.4405<br/>[-5.8174, -5.2061]","-5.3390<br/>[-5.6063, -5.1442]","-5.3389<br/>[-5.6063, -5.1440]","-5.3400<br/>[-5.6075, -5.1450]","-5.3383<br/>[-5.6052, -5.1438]","-5.3382<br/>[-5.6050, -5.1437]","-5.4403<br/>[-5.8171, -5.2060]","-5.4405<br/>[-5.8170, -5.2065]","-5.4403<br/>[-5.8168, -5.2063]","-5.3409<br/>[-5.6089, -5.1455]","-5.4407<br/>[-5.8171, -5.2069]","-5.3389<br/>[-5.6065, -5.1440]","-5.3364<br/>[-5.6035, -5.1419]","-5.3420<br/>[-5.6095, -5.1469]","-5.3381<br/>[-5.6052, -5.1435]","-5.4388<br/>[-5.8144, -5.2056]","-5.3428<br/>[-5.6100, -5.1480]"],["-2.1479<br/>[-2.1673, -2.1199]","-2.1369<br/>[-2.1663, -2.0989]","-1.4157<br/>[-1.4245, -1.4029]","-1.4904<br/>[-1.5028, -1.4757]","-2.1219<br/>[-2.1263, -2.1163]","-2.8826<br/>[-2.9268, -2.8446]","-2.0146<br/>[-2.2004, -1.9866]","-2.0756<br/>[-2.0856, -2.0635]","-2.2430<br/>[-2.2508, -2.2318]","-1.9196<br/>[-1.9454, -1.8854]","-2.1472<br/>[-2.1518, -2.1390]","-2.4633<br/>[-2.4704, -2.4527]","-2.5563<br/>[-2.5631, -2.5468]","-3.0758<br/>[-3.0897, -3.0555]","-1.8757<br/>[-1.8777, -1.8723]","-2.5727<br/>[-2.5764, -2.5666]","-2.6778<br/>[-2.6809, -2.6733]","-2.7127<br/>[-2.7156, -2.7076]","-2.6533<br/>[-2.6584, -2.6464]","-2.7674<br/>[-2.7699, -2.7638]","-2.6846<br/>[-2.6895, -2.6772]","-2.7830<br/>[-2.7857, -2.7786]","-2.7865<br/>[-2.7892, -2.7821]","-2.8119<br/>[-2.8143, -2.8081]","-2.8459<br/>[-2.8482, -2.8418]","-1.4763<br/>[-1.4836, -1.4641]","-2.8903<br/>[-2.9276, -2.8409]","-2.9196<br/>[-2.9571, -2.8671]","-2.9308<br/>[-2.9693, -2.8812]","-2.9379<br/>[-2.9779, -2.8846]","-2.9359<br/>[-2.9721, -2.8890]","-3.0034<br/>[-3.0360, -2.9604]","-1.7803<br/>[-1.8055, -1.7525]","-2.1519<br/>[-2.1688, -2.1273]","-2.3607<br/>[-2.3710, -2.3345]","-2.6719<br/>[-2.7006, -2.6342]","-2.6992<br/>[-2.7304, -2.6543]","-2.6830<br/>[-2.6914, -2.6629]","-1.1564<br/>[-1.1680, -1.1373]","-2.4314<br/>[-2.4970, -2.3484]","-2.3863<br/>[-2.4052, -2.3560]","-2.5297<br/>[-2.5516, -2.4923]","-2.0061<br/>[-2.0227, -1.9831]","-2.2274<br/>[-2.2695, -2.1812]","-2.3673<br/>[-2.4012, -2.3250]","-2.3926<br/>[-2.4243, -2.3535]","-2.4471<br/>[-2.4740, -2.4116]","-2.8020<br/>[-2.8109, -2.7789]","-2.6424<br/>[-2.6483, -2.6319]","-2.3340<br/>[-2.3990, -2.2599]","-2.6226<br/>[-2.6784, -2.5487]","-2.6937<br/>[-2.7473, -2.6259]","-2.8018<br/>[-2.9134, -2.5990]","-2.8612<br/>[-2.9292, -2.7482]","-3.5204<br/>[-3.6358, -3.3875]","-3.1097<br/>[-3.1847, -2.9700]","-3.5444<br/>[-3.6586, -3.4139]","-2.1455<br/>[-2.1622, -2.1194]","-3.9320<br/>[-4.0709, -3.8035]","-3.9427<br/>[-4.0795, -3.8144]","-2.6518<br/>[-2.7977, -2.4718]","-3.3908<br/>[-3.5400, -3.2078]","-2.9059<br/>[-3.0244, -2.7390]","-2.9659<br/>[-3.0823, -2.7834]","-3.4491<br/>[-3.8882, -3.3418]","-3.2508<br/>[-3.4295, -3.0346]","-3.3991<br/>[-3.5269, -3.2049]","-3.4039<br/>[-3.5184, -3.2281]","-3.4142<br/>[-3.5277, -3.2337]","-3.5335<br/>[-3.6739, -3.3469]","-3.5441<br/>[-3.7357, -3.3067]","-3.3025<br/>[-3.3832, -3.1705]","-3.5568<br/>[-3.6538, -3.4076]","-3.4062<br/>[-3.6508, -3.1030]","-3.3435<br/>[-3.4483, -3.1903]","-3.6058<br/>[-3.7094, -3.4409]","-3.6433<br/>[-3.9546, -3.3626]","-3.6604<br/>[-3.8052, -3.4685]","-3.4181<br/>[-3.5162, -3.2651]","-3.7183<br/>[-3.8364, -3.5404]","-3.7203<br/>[-3.8413, -3.5384]","-3.7238<br/>[-3.8420, -3.5427]","-3.4968<br/>[-3.5932, -3.3436]","-3.7422<br/>[-3.8803, -3.5524]","-3.7421<br/>[-3.8816, -3.5471]","-3.5254<br/>[-3.6438, -3.3578]","-3.8317<br/>[-4.0633, -3.5615]","-3.8454<br/>[-4.0735, -3.5657]","-3.5951<br/>[-3.6940, -3.4364]","-3.6573<br/>[-3.7892, -3.4793]","-3.7036<br/>[-3.8299, -3.5310]","-2.6982<br/>[-3.2273, -2.5159]","-3.7365<br/>[-3.8804, -3.5479]","-3.7562<br/>[-3.8429, -3.6170]","-4.0238<br/>[-4.1732, -3.8096]","-2.0610<br/>[-4.4168, -1.7673]","-3.9210<br/>[-4.0331, -3.7514]","-4.0634<br/>[-4.1852, -3.8929]","-2.4073<br/>[-4.4785, -2.0234]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(ID, database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2,3]},{"name":"Pathway name<br/>(ID, database, repository)","targets":0},{"name":"# of genes","targets":1},{"name":"# of SNPs","targets":2},{"name":"Log10 BF","targets":3},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":4},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-c98d9295234ac2628b54" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c98d9295234ac2628b54">{"x":{"filter":"none","vertical":false,"data":[["Spleen","Muscle - Skeletal","Small Intestine - Terminal Ileum","Cells - EBV-transformed lymphocytes","Lung","Testis","Heart - Left Ventricle","Colon - Transverse","Stomach","Pancreas","Adrenal Gland","Nerve - Tibial","Heart - Atrial Appendage","Whole Blood","Brain - Cerebellar Hemisphere","Breast - Mammary Tissue","Brain - Frontal Cortex (BA9)","Ovary","Skin - Not Sun Exposed (Suprapubic)","Esophagus - Mucosa","Prostate","Brain - Cerebellum","Skin - Sun Exposed (Lower leg)","Esophagus - Gastroesophageal Junction","Adipose - Subcutaneous","Uterus","Brain - Cortex","Brain - Hippocampus","Colon - Sigmoid","Cells - Transformed fibroblasts","Brain - Anterior cingulate cortex (BA24)","Vagina","Esophagus - Muscularis","Adipose - Visceral (Omentum)","Brain - Hypothalamus","Pituitary","Artery - Tibial","Artery - Coronary","Thyroid","Liver","Artery - Aorta","Brain - Nucleus accumbens (basal ganglia)","Brain - Caudate (basal ganglia)","Brain - Putamen (basal ganglia)"],[4072,5736,4744,5007,5062,5914,5703,5301,5319,5163,5701,5745,5926,5904,5718,5620,6262,5937,5848,5651,5654,5768,5768,5858,5684,5742,5930,5818,5963,6150,5977,5850,6047,5904,5946,5946,6095,6223,6210,6368,6399,6158,6530,6606],[-0.3149293639422101,-0.3249170331853955,-0.3302594966413004,-0.3340266809559043,-0.3358493194197218,-0.3365057350523174,-0.3420804601264645,-0.3422896442145137,-0.343211476931061,-0.3437594696233894,-0.3445664317893053,-0.3499076937960567,-0.350324281374297,-0.3522488715609912,-0.3525316370929063,-0.3525498138108586,-0.3528575410096867,-0.3529845571734367,-0.3530366931046691,-0.3532899641347567,-0.3536514348617601,-0.3537676256522162,-0.3539592334790506,-0.3541174020366148,-0.3543558785508717,-0.3544350639605668,-0.3544757357982105,-0.3546674849223848,-0.3560868761578856,-0.3566979078196253,-0.3572304120208642,-0.3573252771707256,-0.35815387818665,-0.3584989262240016,-0.35862131621232,-0.3589598907990035,-0.3597578307753661,-0.363738017208288,-0.3639260432272751,-0.3655311800832424,-0.3659046505397448,-0.3661545582557043,-0.3727057810825189,-0.3735255617890338],["-5.3396<br/>[-5.6075, -5.1443]","-5.3400<br/>[-5.6073, -5.1453]","-5.3395<br/>[-5.6072, -5.1443]","-5.3395<br/>[-5.6072, -5.1443]","-5.3395<br/>[-5.6071, -5.1443]","-5.3397<br/>[-5.6071, -5.1448]","-5.3395<br/>[-5.6070, -5.1445]","-5.3394<br/>[-5.6071, -5.1442]","-5.3394<br/>[-5.6070, -5.1442]","-5.3393<br/>[-5.6071, -5.1441]","-5.3394<br/>[-5.6070, -5.1444]","-5.3393<br/>[-5.6069, -5.1442]","-5.3394<br/>[-5.6069, -5.1443]","-5.3393<br/>[-5.6069, -5.1442]","-5.3392<br/>[-5.6069, -5.1441]","-5.3392<br/>[-5.6069, -5.1440]","-5.3394<br/>[-5.6068, -5.1444]","-5.3392<br/>[-5.6068, -5.1441]","-5.3392<br/>[-5.6068, -5.1440]","-5.3392<br/>[-5.6069, -5.1440]","-5.3392<br/>[-5.6069, -5.1440]","-5.3392<br/>[-5.6069, -5.1441]","-5.3392<br/>[-5.6069, -5.1440]","-5.3392<br/>[-5.6069, -5.1441]","-5.3392<br/>[-5.6069, -5.1440]","-5.3392<br/>[-5.6069, -5.1440]","-5.3392<br/>[-5.6069, -5.1441]","-5.3392<br/>[-5.6069, -5.1440]","-5.3392<br/>[-5.6068, -5.1441]","-5.3390<br/>[-5.6067, -5.1439]","-5.3392<br/>[-5.6068, -5.1440]","-5.3392<br/>[-5.6068, -5.1440]","-5.3392<br/>[-5.6068, -5.1441]","-5.3391<br/>[-5.6068, -5.1440]","-5.3391<br/>[-5.6068, -5.1440]","-5.3391<br/>[-5.6068, -5.1440]","-5.3392<br/>[-5.6068, -5.1441]","-5.3391<br/>[-5.6067, -5.1440]","-5.3391<br/>[-5.6067, -5.1440]","-5.3391<br/>[-5.6067, -5.1440]","-5.3391<br/>[-5.6067, -5.1440]","-5.3390<br/>[-5.6067, -5.1438]","-5.3389<br/>[-5.6066, -5.1438]","-5.3389<br/>[-5.6065, -5.1438]"],["-4.8870<br/>[-6.5727, -4.2831]","-4.5952<br/>[-6.1495, -4.0312]","-4.8705<br/>[-6.4862, -4.2744]","-4.8355<br/>[-6.4687, -4.2417]","-4.8460<br/>[-6.4770, -4.2504]","-4.6633<br/>[-6.2233, -4.0862]","-4.7551<br/>[-6.3354, -4.1641]","-4.8722<br/>[-6.4908, -4.2777]","-4.8824<br/>[-6.4993, -4.2910]","-4.9590<br/>[-6.5702, -4.3689]","-4.7897<br/>[-6.3862, -4.1995]","-4.8610<br/>[-6.4443, -4.2771]","-4.8127<br/>[-6.4070, -4.2239]","-4.8478<br/>[-6.4490, -4.2578]","-4.9071<br/>[-6.4762, -4.3158]","-4.9454<br/>[-6.5081, -4.3548]","-4.7751<br/>[-6.3495, -4.1920]","-4.8459<br/>[-6.4678, -4.2457]","-4.8706<br/>[-6.5177, -4.2682]","-4.9463<br/>[-6.5366, -4.3562]","-4.9540<br/>[-6.5429, -4.3670]","-4.9136<br/>[-6.5008, -4.3272]","-4.9161<br/>[-6.5248, -4.3232]","-4.8918<br/>[-6.4787, -4.3051]","-4.9561<br/>[-6.5449, -4.3684]","-4.9353<br/>[-6.5224, -4.3476]","-4.8727<br/>[-6.4622, -4.2854]","-4.9108<br/>[-6.5036, -4.3222]","-4.8920<br/>[-6.4734, -4.3068]","-4.8361<br/>[-6.5432, -4.2216]","-4.9035<br/>[-6.4904, -4.3152]","-4.9506<br/>[-6.5327, -4.3644]","-4.9009<br/>[-6.4793, -4.3156]","-4.9532<br/>[-6.5033, -4.3662]","-4.9394<br/>[-6.5214, -4.3524]","-4.9469<br/>[-6.5255, -4.3609]","-4.9132<br/>[-6.4853, -4.3282]","-4.9427<br/>[-6.4772, -4.3581]","-4.9478<br/>[-6.5172, -4.3624]","-4.9319<br/>[-6.4838, -4.3522]","-4.9250<br/>[-6.4862, -4.3426]","-5.0099<br/>[-6.5735, -4.4250]","-5.0059<br/>[-6.5533, -4.4245]","-4.9954<br/>[-6.5118, -4.4137]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2]},{"name":"id","targets":0},{"name":"# of SNPs","targets":1},{"name":"log10.bf","targets":2},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":3},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-8c90653e74bfddaf3bed" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8c90653e74bfddaf3bed">{"x":{"filter":"none","vertical":false,"data":[["Pituitary","Kidney_Cortex","Brain_Cortex","Brain_Amygdala","Brain_Spinal_cord_(cervical_c-1)","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Frontal_Cortex_(BA9)","Whole_Blood","Spleen","Liver","Skin_Sun_Exposed_(Lower_leg)","Skin_Not_Sun_Exposed_(Suprapubic)","Nerve_Tibial","Colon_Transverse","Vagina","Testis","Esophagus_Mucosa","Lung","Cells_EBV-transformed_lymphocytes","Stomach","Muscle_Skeletal","Pancreas","Minor_Salivary_Gland","Ovary","Small_Intestine_Terminal_Ileum","Prostate","Breast_Mammary_Tissue","Brain_Cerebellum","Adipose_Visceral_(Omentum)","Brain_Hippocampus","Uterus","Adipose_Subcutaneous","Brain_Nucleus_accumbens_(basal_ganglia)","Colon_Sigmoid","Cells_Transformed_fibroblasts","Brain_Caudate_(basal_ganglia)","Brain_Putamen_(basal_ganglia)","Heart_Left_Ventricle","Heart_Atrial_Appendage","Brain_Hypothalamus","Thyroid","Adrenal_Gland","Brain_Substantia_nigra","Artery_Coronary","Brain_Cerebellar_Hemisphere","Artery_Tibial","Esophagus_Gastroesophageal_Junction","Esophagus_Muscularis","Artery_Aorta"],[9481,8969,11954,11191,12666,12957,13617,7210,7222,8164,6798,6872,12396,7496,6774,7684,6855,8350,7815,9260,8648,7674,8352,11511,8688,9103,9816,10879,9885,10921,12498,10142,11319,10936,10706,11653,11717,10999,11080,12180,11931,12167,12650,12332,13466,12507,14061,14219,14209],[2.195173532014957,0.8986594089471228,0.6566362370755277,0.6484907335802367,0.6442210610925924,0.6246171147457357,0.5703789818797198,-0.3127606257458823,-0.3292324273770611,-0.3440653759774164,-0.3516951188203818,-0.3530955267750791,-0.3636075730053737,-0.3767379954174706,-0.379783835113974,-0.3806916966526923,-0.3810376323532855,-0.3828454739052906,-0.3829932011395195,-0.3858271732464695,-0.3903180076039579,-0.3936842930874605,-0.4015662616333786,-0.4080058354664229,-0.4135346162909816,-0.414380384002403,-0.4167038363064509,-0.4211493305199431,-0.4276702738638264,-0.4277347444457876,-0.4324642273238732,-0.4332872503489386,-0.4350072765923551,-0.4369049042254841,-0.4398998502563631,-0.4405922095767472,-0.4428663299051397,-0.444246146780202,-0.4468720670965282,-0.448235912908757,-0.4537088152166049,-0.455078974436105,-0.4576928872670114,-0.4582413507170694,-0.4587136733196737,-0.4594444548734988,-0.4753290906666207,-0.4774897932380062,-0.4853413286386113],["-5.3393<br/>[-5.6060, -5.1451]","-5.3432<br/>[-5.6088, -5.1495]","-5.3384<br/>[-5.6051, -5.1442]","-5.3381<br/>[-5.6051, -5.1437]","-5.3371<br/>[-5.6043, -5.1424]","-5.3382<br/>[-5.6048, -5.1441]","-5.3379<br/>[-5.6045, -5.1438]","-5.3406<br/>[-5.6073, -5.1463]","-5.3403<br/>[-5.6071, -5.1458]","-5.3416<br/>[-5.6079, -5.1473]","-5.3393<br/>[-5.6067, -5.1445]","-5.3393<br/>[-5.6067, -5.1445]","-5.3420<br/>[-5.6066, -5.1496]","-5.3392<br/>[-5.6065, -5.1444]","-5.3388<br/>[-5.6064, -5.1436]","-5.3391<br/>[-5.6064, -5.1443]","-5.3388<br/>[-5.6064, -5.1436]","-5.3389<br/>[-5.6061, -5.1441]","-5.3387<br/>[-5.6062, -5.1437]","-5.3397<br/>[-5.6064, -5.1454]","-5.3391<br/>[-5.6062, -5.1445]","-5.3386<br/>[-5.6062, -5.1436]","-5.3385<br/>[-5.6060, -5.1436]","-5.3386<br/>[-5.6053, -5.1444]","-5.3382<br/>[-5.6057, -5.1431]","-5.3383<br/>[-5.6057, -5.1434]","-5.3384<br/>[-5.6056, -5.1438]","-5.3386<br/>[-5.6054, -5.1442]","-5.3380<br/>[-5.6054, -5.1431]","-5.3383<br/>[-5.6052, -5.1438]","-5.3387<br/>[-5.6051, -5.1447]","-5.3378<br/>[-5.6052, -5.1429]","-5.3381<br/>[-5.6051, -5.1435]","-5.3379<br/>[-5.6051, -5.1432]","-5.3377<br/>[-5.6051, -5.1429]","-5.3379<br/>[-5.6049, -5.1434]","-5.3378<br/>[-5.6049, -5.1433]","-5.3376<br/>[-5.6049, -5.1428]","-5.3375<br/>[-5.6049, -5.1427]","-5.3378<br/>[-5.6048, -5.1433]","-5.3372<br/>[-5.6046, -5.1424]","-5.3374<br/>[-5.6046, -5.1428]","-5.3375<br/>[-5.6045, -5.1430]","-5.3371<br/>[-5.6044, -5.1423]","-5.3377<br/>[-5.6044, -5.1435]","-5.3371<br/>[-5.6044, -5.1423]","-5.3370<br/>[-5.6040, -5.1425]","-5.3370<br/>[-5.6039, -5.1424]","-5.3365<br/>[-5.6037, -5.1418]"],["-2.7741<br/>[-2.7765, -2.7517]","-3.6844<br/>[-3.9381, -3.4008]","-3.7870<br/>[-4.0750, -3.5342]","-3.7962<br/>[-4.0443, -3.5711]","-3.9873<br/>[-4.0999, -3.8202]","-3.8171<br/>[-4.1070, -3.5576]","-3.8620<br/>[-4.1311, -3.6147]","-4.3779<br/>[-5.9317, -3.7949]","-4.4807<br/>[-6.0179, -3.9047]","-4.5848<br/>[-5.7325, -4.1275]","-4.6753<br/>[-6.3209, -4.0806]","-4.6782<br/>[-6.3217, -4.0832]","-4.5298<br/>[-5.8669, -4.0518]","-4.8508<br/>[-6.2861, -4.2919]","-5.0614<br/>[-6.6059, -4.4813]","-4.8676<br/>[-6.3262, -4.3168]","-5.0587<br/>[-6.5958, -4.4812]","-4.7792<br/>[-6.3507, -4.1955]","-4.8337<br/>[-6.4512, -4.2294]","-4.7428<br/>[-6.0504, -4.2104]","-4.8325<br/>[-6.2686, -4.2850]","-5.0441<br/>[-6.5444, -4.4714]","-5.0081<br/>[-6.4510, -4.4419]","-4.6803<br/>[-6.2315, -4.0840]","-5.1290<br/>[-6.5942, -4.5621]","-5.0386<br/>[-6.5023, -4.4745]","-4.9619<br/>[-6.3799, -4.4103]","-4.8718<br/>[-6.2581, -4.3183]","-5.0889<br/>[-6.5317, -4.5288]","-4.9313<br/>[-6.3292, -4.3736]","-4.8444<br/>[-6.2158, -4.3057]","-5.1283<br/>[-6.5716, -4.5650]","-4.9655<br/>[-6.3667, -4.4113]","-5.0478<br/>[-6.4205, -4.5027]","-5.1266<br/>[-6.5021, -4.5732]","-4.9851<br/>[-6.3860, -4.4288]","-5.0044<br/>[-6.3714, -4.4487]","-5.1395<br/>[-6.5454, -4.5863]","-5.1665<br/>[-6.5702, -4.6142]","-5.0144<br/>[-6.3621, -4.4633]","-5.1175<br/>[-6.5834, -4.5445]","-5.1002<br/>[-6.4826, -4.5511]","-5.0725<br/>[-6.4417, -4.5272]","-5.1199<br/>[-6.5519, -4.5417]","-4.9938<br/>[-6.3485, -4.4538]","-5.1109<br/>[-6.5645, -4.5363]","-5.1233<br/>[-6.4365, -4.5917]","-5.1320<br/>[-6.4410, -4.5980]","-5.2435<br/>[-6.5808, -4.7037]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2]},{"name":"id","targets":0},{"name":"# of SNPs","targets":1},{"name":"log10.bf","targets":2},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":3},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-f32b28fd418c4fff040d" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f32b28fd418c4fff040d">{"x":{"filter":"none","vertical":false,"data":[["9","14","5","4","17","7","1","19","2","13","12","15","18","10","6","20","11","8","16","3"],[9469,9348,11358,12270,8470,8199,7462,5891,8913,7566,7200,7701,7224,8330,8785,8051,10242,10113,10172,9998],[0.8650807306104074,0.5568402751195355,0.5237872402277454,0.454719139731209,-0.3585692856339899,-0.3628601793068456,-0.3649541703518021,-0.370676167244778,-0.3812320679915599,-0.3860762916025751,-0.3873023707814037,-0.392315873260535,-0.3965767426709652,-0.3986897851965558,-0.4052321703323733,-0.4092572883967365,-0.4223728469961903,-0.4230195090157963,-0.4300511135167322,-0.4361295230333666],["-5.3390<br/>[-5.6059, -5.1446]","-5.3382<br/>[-5.6056, -5.1434]","-5.4387<br/>[-5.8150, -5.2048]","-5.3378<br/>[-5.6086, -5.1434]","-5.3403<br/>[-5.6067, -5.1463]","-5.3396<br/>[-5.6065, -5.1451]","-5.3392<br/>[-5.6065, -5.1444]","-5.3391<br/>[-5.6068, -5.1439]","-5.3405<br/>[-5.6069, -5.1463]","-5.3391<br/>[-5.6064, -5.1443]","-5.3389<br/>[-5.6064, -5.1440]","-5.3390<br/>[-5.6063, -5.1442]","-5.3387<br/>[-5.6063, -5.1436]","-5.3388<br/>[-5.6061, -5.1440]","-5.3388<br/>[-5.6060, -5.1441]","-5.3385<br/>[-5.6060, -5.1435]","-5.3386<br/>[-5.6056, -5.1441]","-5.3382<br/>[-5.6054, -5.1435]","-5.3384<br/>[-5.6055, -5.1438]","-5.3381<br/>[-5.6054, -5.1433]"],["-3.7527<br/>[-3.9739, -3.4967]","-3.8348<br/>[-3.9680, -3.6526]","-3.9008<br/>[-4.0531, -3.6654]","-3.9129<br/>[-4.0856, -3.7203]","-4.5297<br/>[-6.0519, -3.9633]","-4.5590<br/>[-6.1800, -3.9697]","-4.5988<br/>[-6.3239, -3.9748]","-5.0100<br/>[-6.5491, -4.4224]","-4.6964<br/>[-5.9537, -4.1934]","-4.8305<br/>[-6.3434, -4.2599]","-4.9048<br/>[-6.4434, -4.3239]","-4.8899<br/>[-6.3608, -4.3304]","-5.0565<br/>[-6.5828, -4.4780]","-4.8417<br/>[-6.3969, -4.2547]","-4.8597<br/>[-6.3534, -4.2863]","-5.0525<br/>[-6.5474, -4.4786]","-4.8715<br/>[-6.2747, -4.3098]","-4.8759<br/>[-6.4136, -4.2787]","-4.9583<br/>[-6.3802, -4.4018]","-5.0675<br/>[-6.4894, -4.5100]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2]},{"name":"id","targets":0},{"name":"# of SNPs","targets":1},{"name":"log10.bf","targets":2},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":3},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of
<a
href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey
et al. (2017)</a>; see below.</p>
<p><img
src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">session_info</span>()</span></code></pre></div>
<pre><code>─ Session info ───────────────────────────────────────────────────────────────
 setting  value
 version  R version 4.4.1 (2024-06-14)
 os       macOS Sonoma 14.6.1
 system   aarch64, darwin20
 ui       X11
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       America/Los_Angeles
 date     2024-09-16
 pandoc   3.1.11 @ /Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/aarch64/ (via rmarkdown)

─ Packages ───────────────────────────────────────────────────────────────────
 package     * version    date (UTC) lib source
 bslib         0.8.0      2024-07-29 [1] CRAN (R 4.4.0)
 cachem        1.1.0      2024-05-16 [1] CRAN (R 4.4.0)
 callr         3.7.6      2024-03-25 [1] CRAN (R 4.4.0)
 cli           3.6.3      2024-06-21 [1] CRAN (R 4.4.0)
 crosstalk     1.2.1      2023-11-23 [1] CRAN (R 4.4.0)
 devtools      2.4.5      2022-10-11 [1] CRAN (R 4.4.0)
 digest        0.6.37     2024-08-19 [1] CRAN (R 4.4.1)
 dplyr       * 1.1.4      2023-11-17 [1] CRAN (R 4.4.0)
 DT          * 0.33       2024-04-04 [1] CRAN (R 4.4.0)
 ellipsis      0.3.2      2021-04-29 [1] CRAN (R 4.4.0)
 evaluate      0.24.0     2024-06-10 [1] CRAN (R 4.4.0)
 fansi         1.0.6      2023-12-08 [1] CRAN (R 4.4.0)
 fastmap       1.2.0      2024-05-15 [1] CRAN (R 4.4.0)
 fs            1.6.4      2024-04-25 [1] CRAN (R 4.4.0)
 generics      0.1.3      2022-07-05 [1] CRAN (R 4.4.0)
 getPass       0.2-4      2023-12-10 [1] CRAN (R 4.4.0)
 git2r         0.33.0     2023-11-26 [1] CRAN (R 4.4.0)
 glue          1.7.0      2024-01-09 [1] CRAN (R 4.4.0)
 htmltools     0.5.8.1    2024-04-04 [1] CRAN (R 4.4.0)
 htmlwidgets   1.6.4      2023-12-06 [1] CRAN (R 4.4.0)
 httpuv        1.6.15     2024-03-26 [1] CRAN (R 4.4.0)
 httr          1.4.7      2023-08-15 [1] CRAN (R 4.4.0)
 jquerylib     0.1.4      2021-04-26 [1] CRAN (R 4.4.0)
 jsonlite      1.8.8      2023-12-04 [1] CRAN (R 4.4.0)
 knitr         1.48       2024-07-07 [1] CRAN (R 4.4.0)
 later         1.3.2      2023-12-06 [1] CRAN (R 4.4.0)
 lifecycle     1.0.4      2023-11-07 [1] CRAN (R 4.4.0)
 magrittr      2.0.3      2022-03-30 [1] CRAN (R 4.4.0)
 memoise       2.0.1      2021-11-26 [1] CRAN (R 4.4.0)
 mime          0.12       2021-09-28 [1] CRAN (R 4.4.0)
 miniUI        0.1.1.1    2018-05-18 [1] CRAN (R 4.4.0)
 pillar        1.9.0      2023-03-22 [1] CRAN (R 4.4.0)
 pkgbuild      1.4.4      2024-03-17 [1] CRAN (R 4.4.0)
 pkgconfig     2.0.3      2019-09-22 [1] CRAN (R 4.4.0)
 pkgload       1.4.0      2024-06-28 [1] CRAN (R 4.4.0)
 plyr        * 1.8.9      2023-10-02 [1] CRAN (R 4.4.0)
 processx      3.8.4      2024-03-16 [1] CRAN (R 4.4.0)
 profvis       0.3.8      2023-05-02 [1] CRAN (R 4.4.0)
 promises      1.3.0      2024-04-05 [1] CRAN (R 4.4.0)
 ps            1.8.0      2024-09-12 [1] CRAN (R 4.4.1)
 purrr         1.0.2      2023-08-10 [1] CRAN (R 4.4.0)
 R.matlab    * 3.7.0      2022-08-25 [1] CRAN (R 4.4.0)
 R.methodsS3   1.8.2      2022-06-13 [1] CRAN (R 4.4.0)
 R.oo          1.26.0     2024-01-24 [1] CRAN (R 4.4.0)
 R.utils       2.12.3     2023-11-18 [1] CRAN (R 4.4.0)
 R6            2.5.1      2021-08-19 [1] CRAN (R 4.4.0)
 Rcpp          1.0.13     2024-07-17 [1] CRAN (R 4.4.0)
 remotes       2.5.0      2024-03-17 [1] CRAN (R 4.4.0)
 rlang         1.1.4      2024-06-04 [1] CRAN (R 4.4.0)
 rmarkdown     2.28       2024-08-17 [1] CRAN (R 4.4.0)
 rprojroot     2.0.4      2023-11-05 [1] CRAN (R 4.4.0)
 rstudioapi    0.16.0     2024-03-24 [1] CRAN (R 4.4.0)
 sass          0.4.9.9000 2024-07-11 [1] Github (rstudio/sass@9228fcf)
 sessioninfo   1.2.2      2021-12-06 [1] CRAN (R 4.4.0)
 shiny         1.9.1      2024-08-01 [1] CRAN (R 4.4.0)
 stringi       1.8.4      2024-05-06 [1] CRAN (R 4.4.0)
 stringr       1.5.1      2023-11-14 [1] CRAN (R 4.4.0)
 tibble        3.2.1      2023-03-20 [1] CRAN (R 4.4.0)
 tidyselect    1.2.1      2024-03-11 [1] CRAN (R 4.4.0)
 urlchecker    1.0.1      2021-11-30 [1] CRAN (R 4.4.0)
 usethis       3.0.0      2024-07-29 [1] CRAN (R 4.4.0)
 utf8          1.2.4      2023-10-22 [1] CRAN (R 4.4.0)
 vctrs         0.6.5      2023-12-01 [1] CRAN (R 4.4.0)
 whisker       0.4.1      2022-12-05 [1] CRAN (R 4.4.0)
 workflowr   * 1.7.1      2023-08-23 [1] CRAN (R 4.4.0)
 xfun          0.47       2024-08-17 [1] CRAN (R 4.4.0)
 xtable        1.8-4      2019-04-21 [1] CRAN (R 4.4.0)
 yaml          2.3.10     2024-07-26 [1] CRAN (R 4.4.0)

 [1] /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/library

──────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
