<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of haemoglobin</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of haemoglobin</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-10-05</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/1c85967d82bd54d5e76573a2360750fc083cedc5" target="_blank">1c85967</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/hb_2012.html" target="_blank">1c85967</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-05
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/67ec0cb779d9823fdea13861ce0499029ade165f/docs/hb_2012.html" target="_blank">67ec0cb</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/f451e7fc6fc66640969461e67be820157530e636/analysis/hb_2012.Rmd" target="_blank">f451e7f</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“hb_2012.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/23222517">“Seventy-Five Genetic Loci Influencing the Human Red Blood Cell” (<em>Nature</em>, 2012)</a>. The summary data file is available from the European Genome–Phenome Archive (EGA, <a href="http://www.ebi.ac.uk/ega" class="uri">http://www.ebi.ac.uk/ega</a>) under accession number EGAS00000000132.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["RNA Polymerase I Transcription<br/>(3586, reactome, BioSystems)","RMTs methylate histone arginines<br/>(3296, reactome, PC)","RNA Polymerase I Promoter Clearance<br/>(3560, reactome, PC)","G2/M Checkpoints<br/>(3590, reactome, BioSystems)","Negative epigenetic regulation of rRNA expression<br/>(3575, reactome, BioSystems)","Condensation of Prophase Chromosomes<br/>(3259, reactome, PC)","Chromosome Maintenance<br/>(3548, reactome, BioSystems)","Packaging Of Telomere Ends<br/>(2914, reactome, PC)","Meiotic recombination<br/>(3395, reactome, PC)","SIRT1 negatively regulates rRNA Expression<br/>(3162, reactome, PC)","Apoptosis<br/>(3733, reactome, BioSystems)","Cellular Senescence<br/>(3783, reactome, PC)","Type II interferon signaling (IFNG)<br/>(2952, wiki, BioSystems)","RNA Polymerase I Promoter Opening<br/>(3108, reactome, PC)","HDMs demethylate histones<br/>(2833, reactome, BioSystems)","TCF dependent signaling in response to WNT<br/>(3819, reactome, BioSystems)","Formation of Senescence-Associated Heterochromatin Foci (SAHF)<br/>(1631, reactome, PC)","DNA methylation<br/>(3125, reactome, PC)","Amyloids<br/>(3372, reactome, BioSystems)","Deposition of new CENPA-containing nucleosomes at the centromere<br/>(3112, reactome, PC)","G2/M DNA damage checkpoint<br/>(3301, reactome, BioSystems)","RHO GTPases activate PKNs<br/>(3457, reactome, BioSystems)","RNA Polymerase I, RNA Polymerase III, and Mitochondrial Transcription<br/>(3711, reactome, BioSystems)","HDACs deacetylate histones<br/>(3475, reactome, PC)","Alcoholism<br/>(3759, kegg, BioSystems)","M Phase<br/>(3872, reactome, BioSystems)","PRC2 methylates histones and DNA<br/>(3243, reactome, PC)","Transcriptional regulation by small RNAs<br/>(3529, reactome, PC)","DNA Damage/Telomere Stress Induced Senescence<br/>(3315, reactome, PC)","Cell Cycle Checkpoints<br/>(3773, reactome, BioSystems)","Activated PKN1 stimulates transcription of AR (androgen receptor) regulated genes KLK2 and KLK3<br/>(3150, reactome, BioSystems)","RNA Polymerase I Transcription<br/>(3572, reactome, PC)","NoRC negatively regulates rRNA expression<br/>(3546, reactome, PC)","Meiosis<br/>(3607, reactome, PC)","Signaling by WNT in cancer<br/>(3842, reactome, PC)","Epigenetic regulation of gene expression<br/>(3632, reactome, PC)","Signaling by Wnt<br/>(3891, reactome, BioSystems)","Transcription<br/>(3823, reactome, PC)","RNA Polymerase I Promoter Clearance<br/>(3576, reactome, BioSystems)","TCF dependent signaling in response to WNT<br/>(3813, reactome, PC)","Telomere Maintenance<br/>(3318, reactome, PC)","Epigenetic regulation of gene expression<br/>(3640, reactome, BioSystems)","Mitotic Prophase<br/>(3672, reactome, PC)","RHO GTPase Effectors<br/>(3869, reactome, BioSystems)","RNA Polymerase I Chain Elongation<br/>(3426, reactome, PC)","Cellular responses to stress<br/>(3901, reactome, BioSystems)","Activation of DNA fragmentation factor<br/>(1379, reactome, PC)","Systemic lupus erythematosus<br/>(3655, kegg, BioSystems)","Chromatin modifying enzymes<br/>(3805, reactome, PC)","Cellular responses to stress<br/>(3859, reactome, PC)","Regulatory RNA pathways<br/>(3664, reactome, BioSystems)","NoRC negatively regulates rRNA expression<br/>(3565, reactome, BioSystems)","RNA Polymerase I Chain Elongation<br/>(3445, reactome, BioSystems)","Senescence-Associated Secretory Phenotype (SASP)<br/>(3562, reactome, PC)","Hemostasis<br/>(3914, reactome, PC)","Apoptosis<br/>(3723, reactome, PC)","Programmed Cell Death<br/>(3737, reactome, BioSystems)","Gastric cancer network 1<br/>(2786, wiki, BioSystems)","Factors involved in megakaryocyte development and platelet production<br/>(3686, reactome, BioSystems)","Signaling events mediated by HDAC Class III<br/>(2674, pid, PC)","HDMs demethylate histones<br/>(2795, reactome, PC)","Factors involved in megakaryocyte development and platelet production<br/>(3679, reactome, PC)","Apoptotic execution phase<br/>(2929, reactome, PC)","RNF mutants show enhanced WNT signaling and proliferation<br/>(3820, reactome, PC)","Deposition of new CENPA-containing nucleosomes at the centromere<br/>(3245, reactome, BioSystems)","Signaling by Wnt<br/>(3887, reactome, PC)","Proteasome Degradation<br/>(3134, wiki, BioSystems)","Mitotic Prophase<br/>(3682, reactome, BioSystems)","M Phase<br/>(3871, reactome, PC)","formation of the beta-catenin:TCF transactivating complex<br/>(3435, reactome, PC)","Transcriptional misregulation in cancer<br/>(3760, kegg, BioSystems)","Amyloids<br/>(3351, reactome, PC)","RNA Polymerase I, RNA Polymerase III, and Mitochondrial Transcription<br/>(3705, reactome, PC)","HATs acetylate histones<br/>(3671, reactome, PC)","HATs acetylate histones<br/>(3693, reactome, BioSystems)","Viral carcinogenesis<br/>(3799, kegg, BioSystems)","Oxidative Stress Induced Senescence<br/>(3628, reactome, PC)","Transcriptional regulation by small RNAs<br/>(3538, reactome, BioSystems)","Chromatin modifying enzymes<br/>(3839, reactome, BioSystems)","Chromosome Maintenance<br/>(3486, reactome, PC)","Negative epigenetic regulation of rRNA expression<br/>(3559, reactome, PC)","Cell Cycle, Mitotic<br/>(3912, reactome, BioSystems)","Meiotic synapsis<br/>(3290, reactome, PC)","Amino acid and oligopeptide SLC transporters<br/>(2862, reactome, PC)","Transport of inorganic cations/anions and amino acids/oligopeptides<br/>(3471, reactome, BioSystems)","Transport of inorganic cations/anions and amino acids/oligopeptides<br/>(3469, reactome, PC)","Interferon Signaling<br/>(3794, reactome, BioSystems)","Cell Cycle, Mitotic<br/>(3911, reactome, PC)","Mitotic M-M/G1 phases<br/>(3890, reactome, PC)","Interferon gamma signaling<br/>(3462, reactome, BioSystems)","PKMTs methylate histone lysines<br/>(3145, reactome, BioSystems)","Signaling by Rho GTPases<br/>(3903, reactome, BioSystems)","Organic anion transporters<br/>(925, reactome, PC)","FOXM1 transcription factor network<br/>(2709, pid, BioSystems)","Regulatory RNA pathways<br/>(3600, reactome, PC)","FOXM1 transcription factor network<br/>(2770, pid, PC)","Endosomal Sorting Complex Required For Transport (ESCRT)<br/>(2236, reactome, PC)","Endosomal Sorting Complex Required For Transport (ESCRT)<br/>(2311, reactome, BioSystems)","Budding and maturation of HIV virion<br/>(2180, reactome, BioSystems)"],[107,73,103,108,105,69,101,48,83,63,154,183,49,59,44,222,16,61,80,60,73,88,143,91,170,288,68,99,75,177,62,105,101,112,244,119,318,225,105,214,75,121,132,274,85,366,13,126,203,258,129,103,87,103,490,149,157,42,134,38,43,133,49,223,68,312,61,133,284,86,170,79,140,132,136,197,117,100,241,93,103,463,72,46,90,90,192,457,316,89,62,375,8,39,111,42,27,28,25],[4474,3297,4251,6828,4591,2279,5960,1350,3095,1631,13568,11182,3532,1128,2639,14965,1220,1440,3431,2603,3950,4003,7623,3087,11882,20402,2001,4474,3314,11779,1874,4360,4294,5996,16684,6003,25654,14096,4365,14120,3855,6117,8153,20770,2965,26558,921,5152,12991,17718,7220,4408,3079,4861,50451,12857,13789,3115,11468,2816,2565,11383,4595,15119,3023,25081,3830,8249,20188,3621,16071,3328,7430,6341,6770,15250,5844,4570,16805,5542,4477,34671,3828,5122,10442,10355,13631,34264,22780,6455,3966,34945,743,3590,5519,3836,2132,2205,2344],[2641.30326751218,2578.96758250655,2314.75008040282,2200.87312309452,2131.77065865944,2090.91006260631,2090.3048557774,2079.94955570715,2047.28118814955,2040.41141413905,2034.96884976935,1960.89737057132,1941.85360250598,1938.37435005249,1864.0990441303,1851.95929491431,1838.58345586483,1814.99556944867,1792.27183172245,1790.49912711093,1785.49735723345,1693.20747664128,1667.78951969744,1651.33896179591,1651.18019946085,1647.62164128599,1627.0526026168,1591.99879481089,1585.52635984792,1583.68852812499,1580.33126973905,1576.66952190902,1539.11932667085,1536.75789893208,1528.53027975795,1521.01625335482,1508.22545000591,1502.22034561007,1484.92560616549,1481.51238186681,1468.80864338344,1463.59229650986,1431.49256986621,1428.1047391928,1424.04932349623,1404.21909793631,1403.85315264988,1392.93727127848,1387.82925768495,1382.09862293406,1344.62633572942,1292.91793139306,1267.38759797588,1255.48214443892,1249.32793482221,1199.20792095471,1198.57905366479,1184.62208232661,1134.92969030273,1045.62511548016,1044.5304949696,1043.85402884206,1024.39422180024,1007.92877143183,997.15733779032,994.53053566065,987.475130711596,986.843079740636,983.300270979735,932.604689225728,931.312701756272,920.555477694303,908.686351677359,897.219299179681,844.014741572905,782.375297878264,734.178191013928,718.660792453681,707.932055146432,707.246657508942,683.092857380173,666.793429945208,600.161595262486,149.558938399494,84.3828609069197,84.3793031686056,7.32262683667331,7.16335702336243,6.98940628464737,6.91319834160033,5.30461836871079,4.39988077560985,3.10499062338627,2.95310979582464,1.50268024451834,1.2846601563722,-0.153611414990038,-0.220350576830493,-0.224712251865507],["-5.0007<br/>[-5.0058, -4.9925]","-5.0012<br/>[-5.0062, -4.9929]","-5.0008<br/>[-5.0059, -4.9926]","-4.9998<br/>[-5.0049, -4.9916]","-5.0007<br/>[-5.0057, -4.9924]","-5.0016<br/>[-5.0066, -4.9933]","-5.0002<br/>[-5.0052, -4.9919]","-5.0020<br/>[-5.0070, -4.9937]","-5.0013<br/>[-5.0063, -4.9930]","-5.0018<br/>[-5.0069, -4.9936]","-4.9980<br/>[-5.0030, -4.9899]","-4.9981<br/>[-5.0032, -4.9899]","-5.0014<br/>[-5.0064, -4.9933]","-5.0020<br/>[-5.0071, -4.9938]","-5.0014<br/>[-5.0065, -4.9932]","-4.9968<br/>[-5.0019, -4.9886]","-5.0020<br/>[-5.0070, -4.9937]","-5.0019<br/>[-5.0070, -4.9937]","-5.0011<br/>[-5.0062, -4.9929]","-5.0015<br/>[-5.0065, -4.9932]","-5.0010<br/>[-5.0060, -4.9927]","-5.0009<br/>[-5.0060, -4.9927]","-4.9995<br/>[-5.0046, -4.9912]","-5.0013<br/>[-5.0063, -4.9930]","-4.9979<br/>[-5.0029, -4.9896]","-4.9945<br/>[-4.9995, -4.9862]","-5.0017<br/>[-5.0068, -4.9934]","-5.0007<br/>[-5.0058, -4.9925]","-5.0012<br/>[-5.0062, -4.9929]","-4.9979<br/>[-5.0029, -4.9896]","-5.0018<br/>[-5.0068, -4.9935]","-5.0008<br/>[-5.0058, -4.9925]","-5.0008<br/>[-5.0059, -4.9925]","-5.0002<br/>[-5.0052, -4.9919]","-4.9962<br/>[-5.0012, -4.9879]","-5.0001<br/>[-5.0052, -4.9919]","-4.9926<br/>[-4.9976, -4.9844]","-4.9970<br/>[-5.0020, -4.9887]","-5.0008<br/>[-5.0058, -4.9925]","-4.9972<br/>[-5.0022, -4.9890]","-5.0010<br/>[-5.0060, -4.9927]","-5.0001<br/>[-5.0051, -4.9918]","-4.9993<br/>[-5.0043, -4.9910]","-4.9943<br/>[-4.9994, -4.9861]","-5.0013<br/>[-5.0064, -4.9931]","-5.0625<br/>[-5.0667, -5.0555]","-5.0021<br/>[-5.0071, -4.9938]","-5.0005<br/>[-5.0055, -4.9922]","-4.9974<br/>[-5.0025, -4.9892]","-5.0223<br/>[-5.0262, -5.0159]","-4.9997<br/>[-5.0047, -4.9914]","-5.0008<br/>[-5.0058, -4.9925]","-5.0013<br/>[-5.0063, -4.9930]","-5.0006<br/>[-5.0056, -4.9923]","-5.1587<br/>[-5.1636, -5.1505]","-4.9983<br/>[-5.0033, -4.9902]","-4.9979<br/>[-5.0029, -4.9898]","-5.0012<br/>[-5.0063, -4.9930]","-5.0416<br/>[-5.0470, -5.0328]","-5.0014<br/>[-5.0064, -4.9931]","-5.0015<br/>[-5.0065, -4.9932]","-5.0416<br/>[-5.0470, -5.0329]","-5.0015<br/>[-5.0065, -4.9934]","-4.9968<br/>[-5.0018, -4.9886]","-5.0013<br/>[-5.0064, -4.9930]","-4.9928<br/>[-4.9979, -4.9846]","-5.0010<br/>[-5.0060, -4.9927]","-4.9993<br/>[-5.0043, -4.9910]","-4.9946<br/>[-4.9996, -4.9863]","-5.0011<br/>[-5.0061, -4.9928]","-5.0382<br/>[-5.0437, -5.0293]","-5.0012<br/>[-5.0062, -4.9929]","-4.9996<br/>[-5.0046, -4.9913]","-5.0000<br/>[-5.0051, -4.9918]","-4.9999<br/>[-5.0049, -4.9916]","-4.9965<br/>[-5.0016, -4.9883]","-5.0002<br/>[-5.0053, -4.9919]","-5.0007<br/>[-5.0057, -4.9924]","-4.9960<br/>[-5.0010, -4.9877]","-5.0003<br/>[-5.0054, -4.9921]","-5.0007<br/>[-5.0058, -4.9925]","-5.0027<br/>[-5.0064, -4.9966]","-5.0010<br/>[-5.0061, -4.9928]","-4.8966<br/>[-4.9006, -4.8901]","-4.8945<br/>[-4.8985, -4.8880]","-4.8946<br/>[-4.8985, -4.8881]","-4.8936<br/>[-4.8975, -4.8871]","-5.0028<br/>[-5.0065, -4.9968]","-5.0074<br/>[-5.0111, -5.0014]","-4.8964<br/>[-4.9003, -4.8899]","-5.0010<br/>[-5.0060, -4.9927]","-4.9891<br/>[-4.9941, -4.9810]","-4.8983<br/>[-4.9023, -4.8918]","-4.8972<br/>[-4.9012, -4.8907]","-5.0003<br/>[-5.0054, -4.9921]","-4.8971<br/>[-4.9011, -4.8906]","-4.8977<br/>[-4.9017, -4.8912]","-4.8977<br/>[-4.9017, -4.8912]","-4.8977<br/>[-4.9016, -4.8911]"],["-2.1232<br/>[-2.1232, -2.1211]","-1.9818<br/>[-1.9818, -1.9818]","-2.1067<br/>[-2.1067, -2.1033]","-2.2557<br/>[-2.2557, -2.2557]","-2.1551<br/>[-2.1551, -2.1551]","-1.7789<br/>[-1.7789, -1.7789]","-2.1386<br/>[-2.1386, -2.1386]","-1.5701<br/>[-1.5701, -1.5701]","-1.9949<br/>[-1.9949, -1.9949]","-1.6513<br/>[-1.6513, -1.6513]","-2.6211<br/>[-2.6211, -2.6211]","-2.4901<br/>[-2.4901, -2.4901]","-2.0983<br/>[-2.0983, -2.0983]","-1.5035<br/>[-1.5080, -1.4455]","-1.9952<br/>[-1.9961, -1.9832]","-2.5467<br/>[-2.5467, -2.5467]","-1.7405<br/>[-1.7406, -1.7381]","-1.7764<br/>[-1.7764, -1.7764]","-1.8672<br/>[-1.8672, -1.8672]","-1.8399<br/>[-1.8399, -1.8399]","-2.0645<br/>[-2.0645, -2.0645]","-1.9772<br/>[-1.9772, -1.9772]","-2.3955<br/>[-2.3955, -2.3955]","-1.9925<br/>[-1.9938, -1.9925]","-2.5580<br/>[-2.5580, -2.5580]","-2.9256<br/>[-2.9256, -2.9255]","-1.7876<br/>[-1.7876, -1.7876]","-2.1149<br/>[-2.1434, -2.1149]","-2.0173<br/>[-2.0173, -2.0173]","-2.6171<br/>[-2.6171, -2.6171]","-1.8182<br/>[-1.8182, -1.8182]","-2.2229<br/>[-2.2229, -2.2229]","-2.1937<br/>[-2.1937, -2.1937]","-2.3442<br/>[-2.3442, -2.3442]","-2.6220<br/>[-2.6220, -2.6217]","-2.3751<br/>[-2.3751, -2.3751]","-2.9240<br/>[-2.9240, -2.9240]","-2.8063<br/>[-2.8063, -2.8063]","-2.1962<br/>[-2.1962, -2.1962]","-2.6195<br/>[-2.6195, -2.6195]","-2.1829<br/>[-2.1829, -2.1829]","-2.3206<br/>[-2.3206, -2.3206]","-2.5275<br/>[-2.5286, -2.5275]","-3.0116<br/>[-3.0116, -3.0116]","-2.1678<br/>[-2.1678, -2.1670]","-2.9818<br/>[-2.9818, -2.9818]","-1.5428<br/>[-1.5428, -1.5428]","-2.2627<br/>[-2.2627, -2.2589]","-2.5773<br/>[-2.5773, -2.5773]","-2.8945<br/>[-2.8945, -2.8945]","-2.3196<br/>[-2.3196, -2.3184]","-2.1518<br/>[-2.1518, -2.1518]","-2.1635<br/>[-2.1635, -2.1635]","-2.2031<br/>[-2.2031, -2.2031]","-3.1620<br/>[-3.1621, -3.1606]","-2.6647<br/>[-2.6647, -2.6647]","-2.6740<br/>[-2.6740, -2.6740]","-2.1580<br/>[-2.1618, -2.1579]","-2.7011<br/>[-2.7011, -2.7011]","-2.1051<br/>[-2.1233, -2.1051]","-2.0648<br/>[-2.0648, -2.0648]","-2.6399<br/>[-2.6399, -2.6349]","-2.2688<br/>[-2.2688, -2.2688]","-2.8546<br/>[-2.8546, -2.8546]","-2.2015<br/>[-2.2015, -2.2014]","-3.1348<br/>[-3.1348, -3.1348]","-2.2210<br/>[-2.2210, -2.2210]","-2.6372<br/>[-2.6372, -2.6371]","-3.0488<br/>[-3.0488, -3.0488]","-2.2339<br/>[-2.2343, -2.2339]","-2.8519<br/>[-2.8528, -2.8519]","-2.2638<br/>[-2.2638, -2.2638]","-2.6647<br/>[-2.6647, -2.6647]","-2.4987<br/>[-2.4988, -2.4987]","-2.5285<br/>[-2.5508, -2.5285]","-2.9524<br/>[-2.9524, -2.8941]","-2.4863<br/>[-2.4863, -2.4863]","-2.4516<br/>[-2.4520, -2.4410]","-2.9234<br/>[-2.9234, -2.9234]","-2.4647<br/>[-2.4647, -2.4647]","-2.4531<br/>[-2.4535, -2.4457]","-3.2588<br/>[-3.2588, -3.2588]","-2.5406<br/>[-2.5406, -2.5406]","-2.6871<br/>[-2.6894, -2.6785]","-3.3924<br/>[-3.4131, -3.3648]","-3.3886<br/>[-3.4109, -3.3637]","-3.7773<br/>[-3.8198, -3.7483]","-3.9069<br/>[-3.9231, -3.8945]","-3.7398<br/>[-3.7483, -3.7324]","-3.4765<br/>[-3.5011, -3.4558]","-3.1013<br/>[-3.1133, -3.0907]","-4.0026<br/>[-4.0339, -3.9783]","-2.8650<br/>[-2.8709, -2.8591]","-3.1866<br/>[-3.2519, -3.1364]","-3.2641<br/>[-3.2646, -3.2638]","-3.3315<br/>[-3.5012, -3.2774]","-5.5387<br/>[-7.8915, -4.8501]","-5.5598<br/>[-7.8997, -4.8774]","-5.5854<br/>[-7.9024, -4.9014]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(ID, database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Brain - Caudate (basal ganglia)","Brain - Putamen (basal ganglia)","Liver","Spleen","Small Intestine - Terminal Ileum","Lung","Colon - Transverse","Stomach","Brain - Cerebellum","Nerve - Tibial","Adipose - Subcutaneous","Brain - Cerebellar Hemisphere","Testis","Brain - Cortex","Brain - Hippocampus","Adipose - Visceral (Omentum)","Esophagus - Gastroesophageal Junction","Whole Blood","Brain - Hypothalamus","Brain - Anterior cingulate cortex (BA24)","Colon - Sigmoid","Esophagus - Muscularis","Artery - Tibial","Artery - Coronary","Thyroid","Artery - Aorta","Esophagus - Mucosa","Heart - Left Ventricle","Heart - Atrial Appendage","Brain - Nucleus accumbens (basal ganglia)","Muscle - Skeletal","Cells - EBV-transformed lymphocytes","Pancreas","Adrenal Gland","Uterus","Pituitary","Breast - Mammary Tissue","Prostate","Vagina","Skin - Sun Exposed (Lower leg)","Skin - Not Sun Exposed (Suprapubic)","Ovary","Cells - Transformed fibroblasts","Brain - Frontal Cortex (BA9)"],[6907,6984,6707,4296,4993,5340,5548,5596,6068,6062,6000,6017,6273,6229,6110,6232,6150,6249,6250,6301,6256,6356,6426,6567,6549,6732,5954,5953,6211,6498,6013,5277,5461,6008,6034,6256,5950,5959,6160,6094,6172,6242,6480,6593],[347.222026200625,326.158805069528,299.64296102637,298.59606561902,298.462998345618,298.419105732794,298.368774091787,298.354511941428,298.303466547025,298.302518691521,298.299467800161,298.292202815061,298.283638594204,298.28107658095,298.280534715318,298.275127796468,298.26988004265,298.267802100519,298.260572939792,298.25828775628,298.249426085756,298.243477476873,298.231137816696,298.218396892997,298.199255268719,298.188901600321,286.636049792938,274.253601055035,248.978130765728,147.341423083344,121.558093259673,6.34100106178809,6.15708426778829,6.15007673721597,6.14100020233872,6.09079015590925,6.00385264684815,5.98800657521191,5.94503591276862,5.92431905795073,5.91502664811583,5.91329593477001,5.82453457498976,-0.456701041342029],["-4.8959<br/>[-4.8999, -4.8894]","-4.8959<br/>[-4.8998, -4.8893]","-4.8960<br/>[-4.8999, -4.8894]","-4.8969<br/>[-4.9009, -4.8904]","-4.8966<br/>[-4.9006, -4.8901]","-4.8965<br/>[-4.9005, -4.8900]","-4.8964<br/>[-4.9004, -4.8899]","-4.8964<br/>[-4.9004, -4.8899]","-4.8962<br/>[-4.9002, -4.8897]","-4.8962<br/>[-4.9002, -4.8897]","-4.8962<br/>[-4.9002, -4.8897]","-4.8962<br/>[-4.9002, -4.8897]","-4.8962<br/>[-4.9001, -4.8896]","-4.8961<br/>[-4.9001, -4.8896]","-4.8962<br/>[-4.9002, -4.8897]","-4.8961<br/>[-4.9001, -4.8896]","-4.8962<br/>[-4.9002, -4.8896]","-4.8961<br/>[-4.9001, -4.8896]","-4.8961<br/>[-4.9001, -4.8896]","-4.8961<br/>[-4.9001, -4.8896]","-4.8961<br/>[-4.9001, -4.8896]","-4.8961<br/>[-4.9001, -4.8896]","-4.8961<br/>[-4.9000, -4.8895]","-4.8960<br/>[-4.9000, -4.8895]","-4.8960<br/>[-4.9000, -4.8895]","-4.8959<br/>[-4.8999, -4.8894]","-4.8963<br/>[-4.9002, -4.8897]","-4.8963<br/>[-4.9002, -4.8897]","-4.8962<br/>[-4.9001, -4.8896]","-4.8960<br/>[-4.9000, -4.8895]","-4.8963<br/>[-4.9002, -4.8898]","-5.0355<br/>[-5.0400, -5.0279]","-5.0355<br/>[-5.0400, -5.0279]","-5.0352<br/>[-5.0397, -5.0276]","-5.0352<br/>[-5.0397, -5.0276]","-5.0351<br/>[-5.0396, -5.0275]","-5.0352<br/>[-5.0398, -5.0276]","-5.0352<br/>[-5.0397, -5.0276]","-5.0351<br/>[-5.0397, -5.0276]","-5.0352<br/>[-5.0397, -5.0276]","-5.0351<br/>[-5.0397, -5.0276]","-5.0351<br/>[-5.0396, -5.0275]","-5.0350<br/>[-5.0395, -5.0274]","-4.8960<br/>[-4.9000, -4.8895]"],["-2.9719<br/>[-2.9847, -2.9280]","-2.7396<br/>[-2.7396, -2.7396]","-2.9594<br/>[-2.9601, -2.9581]","-3.3305<br/>[-3.3320, -3.3284]","-3.3966<br/>[-3.3973, -3.3955]","-3.4258<br/>[-3.4265, -3.4247]","-3.4424<br/>[-3.4431, -3.4414]","-3.4463<br/>[-3.4468, -3.4455]","-3.4803<br/>[-3.4819, -3.4777]","-3.4811<br/>[-3.4815, -3.4803]","-3.4763<br/>[-3.4770, -3.4749]","-3.4771<br/>[-3.4782, -3.4751]","-3.4882<br/>[-3.4961, -3.4778]","-3.4918<br/>[-3.4933, -3.4895]","-3.4838<br/>[-3.4850, -3.4821]","-3.4927<br/>[-3.4936, -3.4914]","-3.4866<br/>[-3.4878, -3.4847]","-3.4936<br/>[-3.4948, -3.4919]","-3.4938<br/>[-3.4948, -3.4923]","-3.4972<br/>[-3.4983, -3.4956]","-3.4940<br/>[-3.4952, -3.4922]","-3.5009<br/>[-3.5021, -3.4991]","-3.5064<br/>[-3.5069, -3.5056]","-3.5158<br/>[-3.5163, -3.5150]","-3.5147<br/>[-3.5151, -3.5140]","-3.5266<br/>[-3.5271, -3.5258]","-2.1032<br/>[-2.1032, -2.1018]","-2.7318<br/>[-2.7320, -2.7306]","-2.7406<br/>[-2.7417, -2.7358]","-2.5898<br/>[-2.5898, -2.5898]","-2.8163<br/>[-2.8165, -2.8149]","-3.1184<br/>[-3.1201, -3.1165]","-3.1257<br/>[-3.1332, -3.1187]","-3.1743<br/>[-3.1762, -3.1718]","-3.1770<br/>[-3.1783, -3.1754]","-3.1920<br/>[-3.1938, -3.1894]","-3.1706<br/>[-3.1722, -3.1687]","-3.1716<br/>[-3.1729, -3.1699]","-3.1861<br/>[-3.1873, -3.1844]","-3.1814<br/>[-3.1826, -3.1798]","-3.1868<br/>[-3.1881, -3.1850]","-3.1918<br/>[-3.1930, -3.1900]","-3.2081<br/>[-3.2094, -3.2066]","-5.7267<br/>[-7.6414, -5.0615]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Kidney_Cortex","Prostate","Skin_Not_Sun_Exposed_(Suprapubic)","Skin_Sun_Exposed_(Lower_leg)","Minor_Salivary_Gland","Liver","Esophagus_Mucosa","Nerve_Tibial","Breast_Mammary_Tissue","Artery_Aorta","Vagina","Ovary","Cells_EBV-transformed_lymphocytes","Brain_Cerebellum","Colon_Sigmoid","Uterus","Esophagus_Gastroesophageal_Junction","Esophagus_Muscularis","Adipose_Visceral_(Omentum)","Cells_Transformed_fibroblasts","Whole_Blood","Spleen","Pancreas","Testis","Colon_Transverse","Lung","Muscle_Skeletal","Small_Intestine_Terminal_Ileum","Brain_Cortex","Stomach","Brain_Hippocampus","Pituitary","Brain_Amygdala","Brain_Anterior_cingulate_cortex_(BA24)","Adipose_Subcutaneous","Brain_Caudate_(basal_ganglia)","Heart_Left_Ventricle","Brain_Putamen_(basal_ganglia)","Heart_Atrial_Appendage","Brain_Frontal_Cortex_(BA9)","Brain_Nucleus_accumbens_(basal_ganglia)","Artery_Tibial","Brain_Hypothalamus","Brain_Spinal_cord_(cervical_c-1)","Brain_Substantia_nigra","Thyroid","Artery_Coronary","Adrenal_Gland","Brain_Cerebellar_Hemisphere"],[9442,9572,7246,7165,8894,8577,7228,13013,10335,14929,7135,12132,8212,11418,11409,13083,14669,14841,10349,11265,7524,7593,8037,8163,7931,8752,9030,9189,12579,9785,11513,10046,11758,13648,10616,12267,11522,12331,11609,14326,11892,13122,12761,13271,13279,12557,12986,12828,14092],[1126.4656010929,1020.23905062814,629.339569486801,485.489935180025,342.758808137068,335.616063822455,298.509559789921,288.416492371174,228.093371235644,223.36795945535,175.384679796863,164.546875639845,1.83610033190002,0.699481789790425,0.543996276929669,0.488597177939176,0.334007151651004,0.329825699378042,-0.0719753273581464,-0.394131144324237,-0.468211119935354,-0.47077837011681,-0.473471262612039,-0.474250715373885,-0.47503842585857,-0.48303476420172,-0.484424633770847,-0.488420271871522,-0.489201876097183,-0.49252569292256,-0.494281252116759,-0.495673477515384,-0.49644897514174,-0.497704407775232,-0.500483389609679,-0.50322805746867,-0.503501199274353,-0.50374801415185,-0.504262899769847,-0.504579347706414,-0.505426581247772,-0.507483534122026,-0.512373378356226,-0.515212465994663,-0.516304438340571,-0.517059544888259,-0.518214300742822,-0.519076905064157,-0.522910219072421],["-4.8949<br/>[-4.8989, -4.8884]","-4.9987<br/>[-5.0038, -4.9905]","-4.8957<br/>[-4.8997, -4.8892]","-4.8958<br/>[-4.8998, -4.8892]","-4.8959<br/>[-4.8998, -4.8895]","-4.8953<br/>[-4.8992, -4.8887]","-4.8958<br/>[-4.8997, -4.8892]","-4.8935<br/>[-4.8975, -4.8869]","-5.0335<br/>[-5.0380, -5.0259]","-4.8927<br/>[-4.8967, -4.8862]","-4.8958<br/>[-4.8998, -4.8893]","-4.8938<br/>[-4.8978, -4.8873]","-4.9273<br/>[-4.9316, -4.9203]","-4.8941<br/>[-4.8981, -4.8876]","-4.9606<br/>[-4.9652, -4.9529]","-4.9601<br/>[-4.9647, -4.9526]","-4.9593<br/>[-4.9639, -4.9517]","-4.9592<br/>[-4.9638, -4.9516]","-4.8948<br/>[-4.8988, -4.8884]","-4.8948<br/>[-4.8987, -4.8883]","-4.8956<br/>[-4.8996, -4.8891]","-4.8956<br/>[-4.8996, -4.8891]","-4.8954<br/>[-4.8994, -4.8889]","-4.8954<br/>[-4.8994, -4.8889]","-4.8955<br/>[-4.8995, -4.8889]","-4.8952<br/>[-4.8991, -4.8886]","-4.8950<br/>[-4.8990, -4.8885]","-4.8950<br/>[-4.8990, -4.8884]","-4.8937<br/>[-4.8976, -4.8871]","-4.8948<br/>[-4.8987, -4.8882]","-4.8941<br/>[-4.8981, -4.8876]","-4.8946<br/>[-4.8986, -4.8881]","-4.8940<br/>[-4.8980, -4.8875]","-4.8933<br/>[-4.8972, -4.8867]","-4.8944<br/>[-4.8984, -4.8879]","-4.8938<br/>[-4.8978, -4.8873]","-4.8941<br/>[-4.8981, -4.8875]","-4.8938<br/>[-4.8977, -4.8872]","-4.8940<br/>[-4.8980, -4.8875]","-4.8930<br/>[-4.8970, -4.8865]","-4.8939<br/>[-4.8979, -4.8874]","-4.8935<br/>[-4.8974, -4.8869]","-4.8936<br/>[-4.8976, -4.8871]","-4.8934<br/>[-4.8974, -4.8869]","-4.8934<br/>[-4.8974, -4.8869]","-4.8937<br/>[-4.8976, -4.8871]","-4.8935<br/>[-4.8975, -4.8870]","-4.8936<br/>[-4.8975, -4.8870]","-4.8931<br/>[-4.8970, -4.8865]"],["-2.8390<br/>[-2.8390, -2.8390]","-2.8931<br/>[-2.8938, -2.8931]","-2.5285<br/>[-2.5285, -2.5285]","-2.3720<br/>[-2.3720, -2.3720]","-3.1605<br/>[-3.1681, -3.1520]","-2.8421<br/>[-2.8421, -2.7656]","-2.3127<br/>[-2.3127, -2.3127]","-3.7383<br/>[-3.7402, -3.7317]","-2.8519<br/>[-2.8519, -2.8519]","-3.6348<br/>[-3.6393, -3.6317]","-1.7970<br/>[-1.7970, -1.7969]","-1.8507<br/>[-1.8530, -1.8192]","-3.5719<br/>[-3.8882, -3.3513]","-2.3730<br/>[-5.3631, -2.3437]","-3.7556<br/>[-3.7561, -3.7542]","-3.8000<br/>[-3.8145, -3.7878]","-3.8646<br/>[-3.8653, -3.8640]","-3.8697<br/>[-3.8704, -3.8690]","-2.0871<br/>[-5.8529, -1.8235]","-4.6751<br/>[-5.8049, -4.1109]","-5.7929<br/>[-7.6815, -5.1066]","-5.9770<br/>[-7.8763, -5.3168]","-5.7761<br/>[-7.6583, -5.1003]","-5.7692<br/>[-7.5730, -5.0847]","-6.0645<br/>[-7.9444, -5.4112]","-6.0200<br/>[-7.8471, -5.3523]","-5.8866<br/>[-7.7108, -5.2184]","-6.1821<br/>[-8.0127, -5.5278]","-5.2199<br/>[-6.9280, -4.5551]","-5.9978<br/>[-7.8074, -5.3273]","-5.4575<br/>[-7.1406, -4.8032]","-6.0973<br/>[-7.8975, -5.4433]","-5.4714<br/>[-7.1484, -4.8348]","-5.2630<br/>[-6.9493, -4.5918]","-6.0920<br/>[-7.8661, -5.4163]","-5.5684<br/>[-7.2274, -4.9050]","-5.7747<br/>[-7.4981, -5.0931]","-5.5715<br/>[-7.2294, -4.9057]","-5.7824<br/>[-7.5017, -5.1229]","-5.3217<br/>[-6.9813, -4.6506]","-5.7586<br/>[-7.4008, -5.1141]","-5.4834<br/>[-7.2080, -4.8562]","-5.7997<br/>[-7.4219, -5.1555]","-5.7674<br/>[-7.3952, -5.1327]","-5.8217<br/>[-7.4339, -5.1880]","-6.2776<br/>[-8.0027, -5.6230]","-6.0486<br/>[-7.7640, -5.3927]","-6.2843<br/>[-8.0025, -5.6230]","-5.8859<br/>[-7.5013, -5.2176]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["19","4","20","12","15","16","17","10","9","1","14","5","8","18","13","7","2","6","11","3"],[6156,12846,8496,7542,8077,10641,8865,8716,9943,7762,9744,11910,10559,7637,7914,8589,9313,9263,10776,10543],[2078.11650815223,615.717245587981,333.334188098834,298.094400407896,271.075609720133,170.971737581116,127.115739106449,80.2917555599606,74.8714918367116,65.6495155228662,1.3918492553442,-0.324076978362643,-0.388591052107903,-0.442227041385809,-0.446394304854519,-0.456601345297106,-0.467738809857509,-0.474231734015602,-0.496346030541693,-0.497953444003051],["-5.0001<br/>[-5.0051, -4.9918]","-4.9610<br/>[-4.9656, -4.9536]","-4.8953<br/>[-4.8992, -4.8887]","-4.8956<br/>[-4.8996, -4.8891]","-4.8954<br/>[-4.8994, -4.8889]","-4.8944<br/>[-4.8984, -4.8879]","-4.8951<br/>[-4.8991, -4.8886]","-4.8952<br/>[-4.8991, -4.8886]","-4.8947<br/>[-4.8987, -4.8882]","-4.9165<br/>[-4.9195, -4.9115]","-4.9267<br/>[-4.9310, -4.9197]","-4.8942<br/>[-4.8982, -4.8878]","-4.8950<br/>[-4.8989, -4.8886]","-4.8956<br/>[-4.8996, -4.8891]","-4.8955<br/>[-4.8995, -4.8890]","-4.8953<br/>[-4.8992, -4.8887]","-4.8950<br/>[-4.8989, -4.8884]","-4.8950<br/>[-4.8990, -4.8884]","-4.8944<br/>[-4.8983, -4.8878]","-4.8945<br/>[-4.8984, -4.8879]"],["-2.3195<br/>[-2.3195, -2.3195]","-3.1077<br/>[-3.1083, -3.1070]","-2.9697<br/>[-2.9697, -2.9697]","-3.5754<br/>[-3.5764, -3.5739]","-2.2602<br/>[-2.2602, -2.2602]","-3.1141<br/>[-3.1142, -3.1130]","-2.6835<br/>[-2.6835, -2.6827]","-3.3525<br/>[-3.3619, -3.3354]","-2.4935<br/>[-2.4939, -2.4881]","-3.0278<br/>[-3.0477, -2.9924]","-3.8495<br/>[-3.9882, -3.6536]","-4.4418<br/>[-6.1150, -3.9571]","-4.6597<br/>[-5.7794, -4.1130]","-5.0079<br/>[-6.7703, -4.3619]","-5.0273<br/>[-6.8144, -4.3173]","-5.1097<br/>[-6.8694, -4.4049]","-5.2688<br/>[-6.8399, -4.6522]","-5.4464<br/>[-6.9299, -4.8458]","-5.7339<br/>[-7.3870, -5.0839]","-5.9099<br/>[-7.6755, -5.2451]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1">devtools<span class="op">::</span><span class="kw">session_info</span>()</a></code></pre></div>
<pre><code>Session info -------------------------------------------------------------</code></pre>
<pre><code> setting  value                       
 version  R version 3.5.1 (2018-07-02)
 system   x86_64, darwin15.6.0        
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2018-10-05                  </code></pre>
<pre><code>Packages -----------------------------------------------------------------</code></pre>
<pre><code> package     * version date       source        
 assertthat    0.2.0   2017-04-11 CRAN (R 3.5.0)
 backports     1.1.2   2017-12-13 CRAN (R 3.5.0)
 base        * 3.5.1   2018-07-05 local         
 bindr         0.1.1   2018-03-13 CRAN (R 3.5.0)
 bindrcpp      0.2.2   2018-03-29 CRAN (R 3.5.0)
 compiler      3.5.1   2018-07-05 local         
 crayon        1.3.4   2017-09-16 CRAN (R 3.5.0)
 crosstalk     1.0.0   2016-12-21 CRAN (R 3.5.0)
 datasets    * 3.5.1   2018-07-05 local         
 devtools      1.13.6  2018-06-27 CRAN (R 3.5.0)
 digest        0.6.17  2018-09-12 CRAN (R 3.5.0)
 dplyr       * 0.7.6   2018-06-29 CRAN (R 3.5.1)
 DT          * 0.4     2018-01-30 CRAN (R 3.5.0)
 evaluate      0.11    2018-07-17 CRAN (R 3.5.0)
 git2r         0.23.0  2018-07-17 CRAN (R 3.5.0)
 glue          1.3.0   2018-07-17 CRAN (R 3.5.0)
 graphics    * 3.5.1   2018-07-05 local         
 grDevices   * 3.5.1   2018-07-05 local         
 htmltools     0.3.6   2017-04-28 CRAN (R 3.5.0)
 htmlwidgets   1.3     2018-09-30 CRAN (R 3.5.0)
 httpuv        1.4.5   2018-07-19 CRAN (R 3.5.0)
 jsonlite      1.5     2017-06-01 CRAN (R 3.5.0)
 knitr         1.20    2018-02-20 CRAN (R 3.5.0)
 later         0.7.5   2018-09-18 CRAN (R 3.5.0)
 magrittr      1.5     2014-11-22 CRAN (R 3.5.0)
 memoise       1.1.0   2017-04-21 CRAN (R 3.5.0)
 methods     * 3.5.1   2018-07-05 local         
 mime          0.5     2016-07-07 CRAN (R 3.5.0)
 pillar        1.3.0   2018-07-14 CRAN (R 3.5.0)
 pkgconfig     2.0.2   2018-08-16 CRAN (R 3.5.0)
 plyr        * 1.8.4   2016-06-08 CRAN (R 3.5.0)
 promises      1.0.1   2018-04-13 CRAN (R 3.5.0)
 purrr         0.2.5   2018-05-29 CRAN (R 3.5.0)
 R.matlab    * 3.6.2   2018-09-27 CRAN (R 3.5.0)
 R.methodsS3   1.7.1   2016-02-16 CRAN (R 3.5.0)
 R.oo          1.22.0  2018-04-22 CRAN (R 3.5.0)
 R.utils       2.7.0   2018-08-27 CRAN (R 3.5.0)
 R6            2.3.0   2018-10-04 CRAN (R 3.5.0)
 Rcpp          0.12.19 2018-10-01 CRAN (R 3.5.0)
 rlang         0.2.2   2018-08-16 CRAN (R 3.5.0)
 rmarkdown     1.10    2018-06-11 CRAN (R 3.5.0)
 rprojroot     1.3-2   2018-01-03 CRAN (R 3.5.0)
 shiny         1.1.0   2018-05-17 CRAN (R 3.5.0)
 stats       * 3.5.1   2018-07-05 local         
 stringi       1.2.4   2018-07-20 CRAN (R 3.5.0)
 stringr       1.3.1   2018-05-10 CRAN (R 3.5.0)
 tibble        1.4.2   2018-01-22 CRAN (R 3.5.0)
 tidyselect    0.2.4   2018-02-26 CRAN (R 3.5.0)
 tools         3.5.1   2018-07-05 local         
 utils       * 3.5.1   2018-07-05 local         
 whisker       0.3-2   2013-04-28 CRAN (R 3.5.0)
 withr         2.1.2   2018-03-15 CRAN (R 3.5.0)
 workflowr     1.1.1   2018-07-06 CRAN (R 3.5.0)
 xtable        1.8-3   2018-08-29 CRAN (R 3.5.0)
 yaml          2.2.0   2018-07-25 CRAN (R 3.5.0)</code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
