<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of high-density lipoprotein</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of high-density lipoprotein</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-10-05</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/1c85967d82bd54d5e76573a2360750fc083cedc5" target="_blank">1c85967</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/hdl_2010.html" target="_blank">1c85967</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-05
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/393e53719a8a269c2f7c021470c07e155d4ea594/docs/hdl_2010.html" target="_blank">393e537</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/353887d8b0c10f19544eb573704642edc5acfb12/analysis/hdl_2010.Rmd" target="_blank">353887d</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“hdl_2010.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/20686565">“Biological, Clinical and Population Relevance of 95 Loci for Blood Lipids” (<em>Nature</em>, 2010)</a>. The summary data file is available at <a href="http://csg.sph.umich.edu/willer/public/lipids2010" class="uri">http://csg.sph.umich.edu/willer/public/lipids2010</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["Statin Pathway<br/>(2412, wiki, BioSystems)","Lipoprotein metabolism<br/>(2378, reactome, PC)","Lipid and lipoprotein metabolism<br/>(2961, reactome, PC)","Lipid digestion, mobilization, and transport<br/>(3093, reactome, BioSystems)","Exosome, archaea<br/>(484, kegg, BioSystems)","mRNA decay by 3' to 5' exoribonuclease<br/>(1254, reactome, PC)","KSRP destabilizes mRNA<br/>(1706, reactome, PC)","Role of Ran in mitotic spindle regulation<br/>(1168, biocarta, BioCarta)","Exosome, eukaryotes<br/>(1192, kegg, BioSystems)","HDL-mediated lipid transport<br/>(1555, reactome, PC)","Chylomicron-mediated lipid transport<br/>(1698, reactome, PC)","Transport of gamma-carboxylated protein precursors from the endoplasmic reticulum to the Golgi apparatus<br/>(822, reactome, PC)","Gamma-carboxylation of protein precursors<br/>(915, reactome, PC)","Transcriptional activation of dbpb from mRNA<br/>(454, biocarta, BioCarta)","Removal of aminoterminal propeptides from gamma-carboxylated proteins<br/>(932, reactome, PC)","LDL endocytosis<br/>(673, reactome, PC)","Gamma-carboxylation, transport, and amino-terminal cleavage of proteins<br/>(1124, reactome, PC)","Triacylglyceride Synthesis<br/>(2097, wiki, BioSystems)","Vitamin B12 Metabolism<br/>(1860, wiki, BioSystems)","triacylglycerol degradation<br/>(1201, biocyc, BioSystems)","Cation-coupled Chloride cotransporters<br/>(792, reactome, PC)","Influenza Viral RNA Transcription and Replication<br/>(3638, reactome, PC)","Acylglycerol degradation<br/>(1290, kegg, BioSystems)","Influenza Viral RNA Transcription and Replication<br/>(3647, reactome, BioSystems)","Infectious disease<br/>(3900, reactome, BioSystems)","Influenza Life Cycle<br/>(3666, reactome, PC)","Influenza Life Cycle<br/>(3678, reactome, BioSystems)","Binding and Uptake of Ligands by Scavenger Receptors<br/>(2732, reactome, BioSystems)","Influenza Infection<br/>(3685, reactome, PC)","Binding and Uptake of Ligands by Scavenger Receptors<br/>(2742, reactome, PC)","Influenza Infection<br/>(3694, reactome, BioSystems)","Metabolism of RNA<br/>(3856, reactome, PC)","Triacylglycerol metabolism<br/>(76, panther, PC)","triacylglycerol degradation<br/>(1417, humancyc, PC)","PPAR signaling pathway<br/>(3239, kegg, BioSystems)","Fat digestion and absorption<br/>(2765, kegg, BioSystems)","Folate Metabolism<br/>(2138, wiki, BioSystems)","Role of PPAR-gamma Coactivators in Obesity and Thermogenesis<br/>(1071, biocarta, BioCarta)","Glycerolipid metabolism<br/>(3076, kegg, BioSystems)","HRD1/SEL1 ERAD complex<br/>(1446, kegg, BioSystems)","Low-density lipoprotein (LDL) pathway during atherogenesis<br/>(736, biocarta, BioCarta)","Host Interactions of HIV factors<br/>(3637, reactome, PC)","Host Interactions of HIV factors<br/>(3648, reactome, BioSystems)","ATF4 activates genes<br/>(2144, reactome, PC)","Visceral Fat Deposits and the Metabolic Syndrome<br/>(970, biocarta, BioCarta)","HIV Infection<br/>(3809, reactome, PC)","Retinoid metabolism and transport<br/>(2697, reactome, PC)","Retinoid metabolism and transport<br/>(2734, reactome, BioSystems)","HIV Infection<br/>(3814, reactome, BioSystems)","Selenium Pathway<br/>(3432, wiki, BioSystems)","Glycerolipid metabolism<br/>(1391, kegg, PC)","PERK regulates gene expression<br/>(2293, reactome, PC)","FXR and LXR Regulation of Cholesterol Metabolism<br/>(844, biocarta, BioCarta)","Antiviral mechanism by IFN-stimulated genes<br/>(3256, reactome, PC)","HIV Life Cycle<br/>(3680, reactome, PC)","ISG15 antiviral mechanism<br/>(3283, reactome, BioSystems)","Transport of Ribonucleoproteins into the Host Nucleus<br/>(2300, reactome, PC)","Nuclear import of Rev protein<br/>(2383, reactome, PC)","Fatty Acid Beta Oxidation<br/>(2542, wiki, BioSystems)","Nuclear import of Rev protein<br/>(2447, reactome, BioSystems)","HIV Life Cycle<br/>(3698, reactome, BioSystems)","NEP/NS2 Interacts with the Cellular Export Machinery<br/>(2290, reactome, PC)","Export of Viral Ribonucleoproteins from Nucleus<br/>(2371, reactome, PC)","Transport of Ribonucleoproteins into the Host Nucleus<br/>(2350, reactome, BioSystems)","Regulation of Glucokinase by Glucokinase Regulatory Protein<br/>(2297, reactome, PC)","Negative regulators of RIG-I/MDA5 signaling<br/>(2470, reactome, PC)","Vpr-mediated nuclear import of PICs<br/>(2392, reactome, PC)","NEP/NS2 Interacts with the Cellular Export Machinery<br/>(2349, reactome, BioSystems)","Export of Viral Ribonucleoproteins from Nucleus<br/>(2446, reactome, BioSystems)","Regulation of Glucokinase by Glucokinase Regulatory Protein<br/>(2344, reactome, BioSystems)","Transport of the SLBP independent Mature mRNA<br/>(2439, reactome, PC)","Interactions of Rev with host cellular proteins<br/>(2521, reactome, PC)","Interactions of Vpr with host cellular proteins<br/>(2427, reactome, PC)","Vpr-mediated nuclear import of PICs<br/>(2448, reactome, BioSystems)","Transport of the SLBP Dependant Mature mRNA<br/>(2481, reactome, PC)","Rev-mediated nuclear export of HIV RNA<br/>(2434, reactome, PC)","Orphan transporters<br/>(3498, reactome, PC)","Interactions of Rev with host cellular proteins<br/>(2574, reactome, BioSystems)","Transport of the SLBP independent Mature mRNA<br/>(2492, reactome, BioSystems)","Interactions of Vpr with host cellular proteins<br/>(2496, reactome, BioSystems)","Rev-mediated nuclear export of HIV RNA<br/>(2497, reactome, BioSystems)","Transport of the SLBP Dependant Mature mRNA<br/>(2535, reactome, BioSystems)","Transport of Mature mRNA Derived from an Intronless Transcript<br/>(2564, reactome, PC)","Nuclear Pore Complex (NPC) Disassembly<br/>(2472, reactome, PC)","Transport of Mature mRNAs Derived from Intronless Transcripts<br/>(2597, reactome, PC)","Transport of Mature mRNA Derived from an Intronless Transcript<br/>(2602, reactome, BioSystems)","Scavenging by Class B Receptors<br/>(567, reactome, PC)","Nuclear Pore Complex (NPC) Disassembly<br/>(2534, reactome, BioSystems)","Transport of Mature mRNAs Derived from Intronless Transcripts<br/>(2624, reactome, BioSystems)","Viral Messenger RNA Synthesis<br/>(2727, reactome, PC)","Late Phase of HIV Life Cycle<br/>(3635, reactome, PC)","Viral Messenger RNA Synthesis<br/>(2762, reactome, BioSystems)","Lysosome<br/>(3629, kegg, BioSystems)","retinol biosynthesis<br/>(1744, biocyc, BioSystems)","retinol biosynthesis<br/>(1780, humancyc, PC)","RIG-I/MDA5 mediated induction of IFN-alpha/beta pathways<br/>(3331, reactome, PC)","RIG-I/MDA5 mediated induction of IFN-alpha/beta pathways<br/>(3347, reactome, BioSystems)","Unfolded Protein Response (UPR)<br/>(3319, reactome, PC)","Unfolded Protein Response (UPR)<br/>(3430, reactome, BioSystems)","SREBF and miR33 in cholesterol and lipid homeostasis<br/>(1677, wiki, BioSystems)"],[30,30,50,58,4,11,17,10,10,15,17,7,8,4,8,6,10,23,19,10,7,121,11,123,359,130,132,40,134,41,136,257,2,13,67,41,24,9,56,13,6,121,123,25,8,209,39,40,214,85,13,28,7,69,133,71,28,30,33,31,137,28,30,29,28,32,30,29,31,29,31,33,31,31,32,31,96,34,32,32,32,33,34,32,35,35,5,33,36,40,120,41,117,17,18,77,78,75,85,16],[2314,2625,3969,4473,266,768,1121,633,647,1467,1140,245,312,268,331,512,428,1928,1509,915,563,7862,1050,7951,25071,8329,8418,4181,8586,4241,8675,16977,242,1276,6165,2638,2017,1009,5577,1229,569,8713,8805,2046,906,15078,4555,4555,15341,6873,1453,2241,708,5265,9784,5413,2151,2231,2399,2318,10042,2120,2120,2238,2120,2329,2258,2207,2207,2207,2286,2443,2326,2345,2338,2313,8648,2530,2373,2413,2400,2425,2492,2528,2544,2579,506,2615,2631,2922,8747,3009,9123,1332,1349,4944,4963,5991,6606,1425],[113.926243202026,113.331825297225,95.5213699618521,89.8248311366294,81.0801952850626,79.8128560504969,79.1737584116098,75.590329933673,75.3206999126494,68.4772956266237,59.7330298984967,45.9983521775178,45.2513496050825,41.7197015828896,41.6020091923876,41.385201552958,40.4472427126563,32.3009863437986,27.4385472000009,25.5388849967818,25.2336637991113,24.4972314059456,24.3996250583062,24.3814217309385,24.1367322122408,24.1342842553861,24.0249518795433,23.9189177531851,23.8743374756989,23.8102380067502,23.7483584121553,23.5074887424623,23.3443490243845,22.3106074444477,21.6217224505268,21.3035434100573,20.6023442426548,19.7231722125,19.6801676489894,19.3478466124119,19.0674310118394,18.5374921131794,18.4377364152177,18.2581873305786,18.0552382876295,18.0306114555775,17.8911799804717,17.8911799804717,17.8557780952709,17.6184974362369,17.4264167791213,17.3555333990359,17.2390893963588,17.2056719052467,17.1013950728951,17.0900564047093,17.0429817605736,16.9966087626437,16.8774137501788,16.8555260067335,16.8534314314484,16.8186075309059,16.8186075309059,16.8156358215206,16.8079207641934,16.7792574900288,16.7134983493147,16.6836143577145,16.6836143577145,16.6757228941115,16.5727608406042,16.5648749615677,16.5321791932153,16.4970931575988,16.4544312652584,16.4377566559186,16.4113466982663,16.3825572565677,16.3823755247856,16.3448802259438,16.2540494597348,16.2508532303764,16.1624044211698,16.1148775795585,16.0381533466559,15.9770665121165,15.9382488291198,15.9258469113185,15.8701972908967,15.6804153234579,15.6644406056259,15.5305664754163,14.6598000944299,14.0978147202276,13.9766299697834,12.6808232412818,12.6466619449859,12.5888877488284,11.7414433805024,11.2845790639399],["-3.8078<br/>[-3.8117, -3.7695]","-3.8092<br/>[-3.8132, -3.7709]","-3.8090<br/>[-3.8129, -3.7706]","-3.8091<br/>[-3.8131, -3.7708]","-3.7102<br/>[-3.7134, -3.6795]","-3.7101<br/>[-3.7133, -3.6793]","-3.7102<br/>[-3.7134, -3.6794]","-3.7103<br/>[-3.7134, -3.6795]","-3.7101<br/>[-3.7133, -3.6793]","-3.7658<br/>[-3.7693, -3.7314]","-3.7557<br/>[-3.7592, -3.7212]","-3.7123<br/>[-3.7155, -3.6814]","-3.7123<br/>[-3.7155, -3.6814]","-3.7126<br/>[-3.7158, -3.6817]","-3.7123<br/>[-3.7155, -3.6814]","-3.7403<br/>[-3.7437, -3.7077]","-3.7123<br/>[-3.7155, -3.6814]","-3.7383<br/>[-3.7417, -3.7050]","-3.7367<br/>[-3.7400, -3.7043]","-3.7357<br/>[-3.7391, -3.7026]","-3.7351<br/>[-3.7384, -3.7026]","-3.7250<br/>[-3.7282, -3.6934]","-3.7357<br/>[-3.7391, -3.7026]","-3.7250<br/>[-3.7282, -3.6933]","-3.7321<br/>[-3.7354, -3.7001]","-3.7250<br/>[-3.7282, -3.6934]","-3.7250<br/>[-3.7282, -3.6933]","-3.7284<br/>[-3.7317, -3.6967]","-3.7249<br/>[-3.7282, -3.6933]","-3.7284<br/>[-3.7317, -3.6967]","-3.7249<br/>[-3.7281, -3.6933]","-3.7284<br/>[-3.7316, -3.6966]","-3.7247<br/>[-3.7280, -3.6923]","-3.7357<br/>[-3.7391, -3.7025]","-3.7443<br/>[-3.7477, -3.7113]","-3.7366<br/>[-3.7399, -3.7041]","-3.7320<br/>[-3.7353, -3.6999]","-3.7214<br/>[-3.7247, -3.6899]","-3.7491<br/>[-3.7526, -3.7151]","-3.7329<br/>[-3.7362, -3.7006]","-3.7242<br/>[-3.7275, -3.6926]","-3.7189<br/>[-3.7222, -3.6877]","-3.7189<br/>[-3.7221, -3.6877]","-3.7325<br/>[-3.7358, -3.7003]","-3.7213<br/>[-3.7245, -3.6898]","-3.7219<br/>[-3.7251, -3.6904]","-3.7366<br/>[-3.7399, -3.7042]","-3.7366<br/>[-3.7399, -3.7042]","-3.7218<br/>[-3.7250, -3.6904]","-3.7414<br/>[-3.7447, -3.7088]","-3.7209<br/>[-3.7241, -3.6894]","-3.7325<br/>[-3.7358, -3.7002]","-3.7272<br/>[-3.7307, -3.6938]","-3.7188<br/>[-3.7220, -3.6875]","-3.7219<br/>[-3.7252, -3.6904]","-3.7188<br/>[-3.7220, -3.6875]","-3.7150<br/>[-3.7182, -3.6840]","-3.7150<br/>[-3.7182, -3.6840]","-3.7363<br/>[-3.7397, -3.7031]","-3.7150<br/>[-3.7182, -3.6840]","-3.7219<br/>[-3.7251, -3.6903]","-3.7149<br/>[-3.7181, -3.6838]","-3.7149<br/>[-3.7181, -3.6838]","-3.7150<br/>[-3.7181, -3.6839]","-3.7149<br/>[-3.7181, -3.6838]","-3.7374<br/>[-3.7407, -3.7048]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7149<br/>[-3.7181, -3.6839]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7345<br/>[-3.7378, -3.7022]","-3.7149<br/>[-3.7181, -3.6839]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7180, -3.6838]","-3.7148<br/>[-3.7179, -3.6837]","-3.7147<br/>[-3.7179, -3.6837]","-3.7215<br/>[-3.7248, -3.6900]","-3.7148<br/>[-3.7179, -3.6837]","-3.7147<br/>[-3.7179, -3.6837]","-3.7148<br/>[-3.7179, -3.6837]","-3.7177<br/>[-3.7209, -3.6865]","-3.7147<br/>[-3.7179, -3.6837]","-3.7130<br/>[-3.7162, -3.6821]","-3.7246<br/>[-3.7279, -3.6923]","-3.7246<br/>[-3.7279, -3.6923]","-3.7377<br/>[-3.7410, -3.7052]","-3.7377<br/>[-3.7410, -3.7052]","-3.7321<br/>[-3.7354, -3.7000]","-3.7319<br/>[-3.7353, -3.6998]","-3.7253<br/>[-3.7285, -3.6937]"],["-1.3258<br/>[-1.3444, -1.2359]","-1.3605<br/>[-1.3688, -1.3464]","-1.5122<br/>[-1.5355, -1.4855]","-1.5419<br/>[-1.6011, -1.5145]","-1.8793<br/>[-1.9364, -1.8198]","-2.3290<br/>[-2.3965, -2.2633]","-2.3895<br/>[-2.4294, -2.3274]","-2.3560<br/>[-2.4706, -2.2542]","-2.4488<br/>[-2.5018, -2.3813]","-1.3076<br/>[-1.3222, -1.2914]","-1.3250<br/>[-1.3789, -1.2998]","-1.6150<br/>[-1.6374, -1.5784]","-1.7034<br/>[-1.7232, -1.6680]","-1.7607<br/>[-1.8125, -1.7001]","-1.7640<br/>[-1.9863, -1.7298]","-1.0907<br/>[-1.0959, -1.0749]","-2.0062<br/>[-2.0355, -1.9567]","-1.7459<br/>[-1.7473, -1.7456]","-1.6802<br/>[-1.7563, -1.6242]","-1.4589<br/>[-1.5063, -1.4215]","-1.3911<br/>[-1.4021, -1.3885]","-2.4035<br/>[-2.4390, -2.3448]","-1.5154<br/>[-1.5531, -1.4770]","-2.4092<br/>[-2.4431, -2.3483]","-2.6668<br/>[-2.7023, -2.6155]","-2.4249<br/>[-2.4546, -2.3422]","-2.4302<br/>[-2.5052, -2.3689]","-2.2149<br/>[-2.2807, -2.1606]","-2.4373<br/>[-2.5209, -2.3790]","-2.2217<br/>[-2.2882, -2.1684]","-2.4438<br/>[-2.5245, -2.3822]","-2.5972<br/>[-2.6522, -2.5575]","-1.0795<br/>[-1.1375, -1.0209]","-1.6042<br/>[-1.6480, -1.5606]","-2.1598<br/>[-2.1710, -2.1380]","-1.9499<br/>[-1.9804, -1.9340]","-1.9153<br/>[-1.9524, -1.8983]","-2.0128<br/>[-2.0604, -1.9288]","-2.0975<br/>[-2.1152, -2.0710]","-1.7169<br/>[-1.7346, -1.7056]","-1.5925<br/>[-1.6188, -1.5269]","-2.5045<br/>[-2.6291, -2.4390]","-2.5106<br/>[-2.6329, -2.4422]","-1.8631<br/>[-1.9079, -1.8366]","-2.0384<br/>[-2.0972, -1.8672]","-2.6872<br/>[-2.7653, -2.5895]","-2.0290<br/>[-2.0299, -2.0290]","-2.0290<br/>[-2.0299, -2.0290]","-2.7005<br/>[-2.7716, -2.5994]","-2.2653<br/>[-2.3083, -2.2210]","-1.9684<br/>[-2.2490, -1.9267]","-1.9002<br/>[-1.9315, -1.8628]","-1.5231<br/>[-1.5918, -1.4445]","-2.4013<br/>[-2.5017, -2.3364]","-2.5702<br/>[-2.6869, -2.4987]","-2.4097<br/>[-2.5177, -2.3454]","-2.1107<br/>[-2.1742, -2.0269]","-2.1147<br/>[-2.1873, -2.0234]","-1.7448<br/>[-1.7577, -1.7342]","-2.1337<br/>[-2.1960, -2.0429]","-2.5883<br/>[-2.6966, -2.5112]","-2.1166<br/>[-2.1802, -2.0309]","-2.1166<br/>[-2.1802, -2.0309]","-2.1330<br/>[-2.1913, -2.0466]","-2.1177<br/>[-2.1802, -2.0323]","-1.9408<br/>[-1.9706, -1.9361]","-2.1394<br/>[-2.1969, -2.0535]","-2.1377<br/>[-2.1938, -2.0544]","-2.1377<br/>[-2.1938, -2.0544]","-2.1388<br/>[-2.1941, -2.0557]","-2.1515<br/>[-2.2046, -2.0834]","-2.1629<br/>[-2.2165, -2.0907]","-2.1560<br/>[-2.2162, -2.0888]","-2.1592<br/>[-2.2113, -2.0909]","-2.1631<br/>[-2.2134, -2.0948]","-2.1612<br/>[-2.2112, -2.0936]","-2.4700<br/>[-2.5472, -2.4121]","-2.1806<br/>[-2.2297, -2.1054]","-2.1698<br/>[-2.2168, -2.0991]","-2.1743<br/>[-2.2205, -2.1006]","-2.1794<br/>[-2.2233, -2.1051]","-2.1814<br/>[-2.2252, -2.1062]","-2.1915<br/>[-2.2343, -2.1169]","-2.1983<br/>[-2.2412, -2.1218]","-2.2025<br/>[-2.2442, -2.1236]","-2.2087<br/>[-2.2492, -2.1275]","-1.2116<br/>[-1.2583, -1.1959]","-2.2146<br/>[-2.2539, -2.1359]","-2.2178<br/>[-2.2568, -2.1377]","-2.2418<br/>[-2.2974, -2.1668]","-2.5903<br/>[-2.7027, -2.5054]","-2.2515<br/>[-2.2900, -2.1660]","-2.7576<br/>[-2.7773, -2.7379]","-1.7671<br/>[-1.8390, -1.7062]","-1.7721<br/>[-1.8261, -1.7102]","-2.2637<br/>[-2.3348, -2.1364]","-2.2660<br/>[-2.3362, -2.1376]","-2.3323<br/>[-2.4064, -2.2636]","-2.3843<br/>[-2.4602, -2.3217]","-1.8729<br/>[-1.9459, -1.8149]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(ID, database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Liver","Muscle - Skeletal","Adrenal Gland","Testis","Brain - Anterior cingulate cortex (BA24)","Brain - Cortex","Heart - Atrial Appendage","Brain - Hypothalamus","Brain - Frontal Cortex (BA9)","Brain - Putamen (basal ganglia)","Brain - Cerebellum","Small Intestine - Terminal Ileum","Artery - Tibial","Artery - Coronary","Brain - Nucleus accumbens (basal ganglia)","Adipose - Visceral (Omentum)","Esophagus - Gastroesophageal Junction","Skin - Sun Exposed (Lower leg)","Lung","Heart - Left Ventricle","Whole Blood","Breast - Mammary Tissue","Nerve - Tibial","Uterus","Colon - Transverse","Stomach","Skin - Not Sun Exposed (Suprapubic)","Brain - Cerebellar Hemisphere","Brain - Caudate (basal ganglia)","Pancreas","Spleen","Cells - EBV-transformed lymphocytes","Brain - Hippocampus","Esophagus - Mucosa","Colon - Sigmoid","Esophagus - Muscularis","Artery - Aorta","Prostate","Adipose - Subcutaneous","Vagina","Thyroid","Cells - Transformed fibroblasts","Ovary","Pituitary"],[6049,5351,5361,5554,5641,5558,5552,5641,5928,6274,5397,4477,5774,5895,5840,5565,5533,5430,4780,5343,5605,5307,5424,5411,4968,5012,5491,5376,6208,4876,3839,4742,5488,5352,5630,5725,6065,5325,5368,5508,5886,5844,5596,5617],[20.1779482479589,17.5156452012625,13.0640145885854,12.2270034263219,11.5226316579479,11.3583262930424,10.6946279377768,10.618990653961,10.2707556459392,10.2071534517045,9.73602040518488,9.73499877825709,9.69760201729083,9.65379329164745,8.71268643009798,8.65033864046293,8.32346337642094,8.0151290982851,7.90194815170935,7.42118705954695,7.35016213973658,7.30862805939315,7.24604275614003,6.71617919248609,6.63034087144231,6.57395568837911,6.14961736259321,6.03667110906785,5.84853882796849,5.70788148620553,5.32295828445346,5.23310251270908,5.14174719665212,4.81543856547326,3.96354205746793,3.92309347532381,3.21312673587092,1.82638136438621,1.59877462087125,1.42627506335266,1.32238552571937,0.85801524288701,0.172518971395434,0.0426198712799943],["-3.7190<br/>[-3.7222, -3.6877]","-3.7210<br/>[-3.7242, -3.6897]","-3.7230<br/>[-3.7262, -3.6914]","-3.7176<br/>[-3.7208, -3.6865]","-3.7204<br/>[-3.7236, -3.6890]","-3.7202<br/>[-3.7234, -3.6888]","-3.7222<br/>[-3.7254, -3.6908]","-3.7203<br/>[-3.7236, -3.6890]","-3.7201<br/>[-3.7233, -3.6887]","-3.7223<br/>[-3.7255, -3.6908]","-3.7184<br/>[-3.7216, -3.6872]","-3.7143<br/>[-3.7175, -3.6833]","-3.7155<br/>[-3.7187, -3.6846]","-3.7155<br/>[-3.7187, -3.6845]","-3.7225<br/>[-3.7258, -3.6910]","-3.7145<br/>[-3.7177, -3.6836]","-3.7162<br/>[-3.7194, -3.6851]","-3.7117<br/>[-3.7151, -3.6793]","-3.7154<br/>[-3.7186, -3.6845]","-3.7171<br/>[-3.7203, -3.6860]","-3.7179<br/>[-3.7212, -3.6867]","-3.7146<br/>[-3.7177, -3.6837]","-3.7145<br/>[-3.7177, -3.6836]","-3.7104<br/>[-3.7135, -3.6798]","-3.7100<br/>[-3.7131, -3.6793]","-3.7100<br/>[-3.7131, -3.6793]","-3.7158<br/>[-3.7193, -3.6831]","-3.7139<br/>[-3.7171, -3.6829]","-3.7225<br/>[-3.7257, -3.6909]","-3.7098<br/>[-3.7130, -3.6792]","-3.7099<br/>[-3.7130, -3.6792]","-3.7100<br/>[-3.7131, -3.6793]","-3.7180<br/>[-3.7213, -3.6868]","-3.7112<br/>[-3.7146, -3.6787]","-3.7162<br/>[-3.7194, -3.6851]","-3.7162<br/>[-3.7194, -3.6851]","-3.7144<br/>[-3.7176, -3.6835]","-3.7099<br/>[-3.7131, -3.6793]","-3.7145<br/>[-3.7176, -3.6835]","-3.7144<br/>[-3.7176, -3.6835]","-3.7143<br/>[-3.7175, -3.6834]","-3.7100<br/>[-3.7132, -3.6794]","-3.7100<br/>[-3.7131, -3.6794]","-3.7097<br/>[-3.7128, -3.6790]"],["-2.3651<br/>[-2.3872, -2.3253]","-2.4414<br/>[-2.4691, -2.3939]","-2.2923<br/>[-2.3029, -2.2629]","-2.5529<br/>[-2.6112, -2.4875]","-2.5104<br/>[-2.5526, -2.4514]","-2.5079<br/>[-2.5457, -2.4480]","-2.6563<br/>[-2.7321, -2.5696]","-2.6022<br/>[-2.7036, -2.5094]","-2.6352<br/>[-2.7409, -2.5393]","-2.6518<br/>[-2.7638, -2.5552]","-2.5561<br/>[-2.5991, -2.4831]","-2.4787<br/>[-2.5156, -2.4038]","-2.5880<br/>[-2.6909, -2.5037]","-2.5938<br/>[-2.6942, -2.5060]","-2.6195<br/>[-2.7231, -2.5274]","-2.3854<br/>[-2.5510, -2.3394]","-2.6938<br/>[-2.7669, -2.6121]","-2.6493<br/>[-2.7338, -2.5599]","-2.6534<br/>[-2.7823, -2.5480]","-2.7347<br/>[-2.8562, -2.6317]","-2.6393<br/>[-2.7384, -2.5541]","-2.7256<br/>[-2.8724, -2.6103]","-2.7345<br/>[-2.8820, -2.6118]","-2.7923<br/>[-2.9412, -2.6727]","-2.7906<br/>[-2.9447, -2.6721]","-2.8025<br/>[-2.9531, -2.6845]","-2.8197<br/>[-2.9469, -2.7160]","-2.9160<br/>[-3.0571, -2.8102]","-2.5875<br/>[-2.6857, -2.4985]","-2.9411<br/>[-3.1123, -2.8073]","-3.0098<br/>[-3.1724, -2.8599]","-3.0626<br/>[-3.2326, -2.9076]","-2.5315<br/>[-2.6353, -2.4454]","-3.0592<br/>[-3.1962, -2.9396]","-2.6274<br/>[-2.7119, -2.5830]","-2.6325<br/>[-2.7199, -2.5834]","-2.6687<br/>[-2.7718, -2.5793]","-2.7344<br/>[-2.8716, -2.6245]","-2.7642<br/>[-2.9029, -2.6489]","-2.7988<br/>[-2.9222, -2.6883]","-2.8218<br/>[-2.9568, -2.7116]","-2.8601<br/>[-3.0414, -2.7329]","-2.9816<br/>[-3.1729, -2.8660]","-3.0142<br/>[-3.1934, -2.8706]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Kidney_Cortex","Uterus","Adipose_Visceral_(Omentum)","Liver","Adipose_Subcutaneous","Adrenal_Gland","Breast_Mammary_Tissue","Whole_Blood","Stomach","Heart_Atrial_Appendage","Brain_Caudate_(basal_ganglia)","Spleen","Lung","Thyroid","Nerve_Tibial","Small_Intestine_Terminal_Ileum","Muscle_Skeletal","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Nucleus_accumbens_(basal_ganglia)","Esophagus_Muscularis","Brain_Putamen_(basal_ganglia)","Artery_Tibial","Heart_Left_Ventricle","Artery_Aorta","Esophagus_Gastroesophageal_Junction","Skin_Sun_Exposed_(Lower_leg)","Skin_Not_Sun_Exposed_(Suprapubic)","Artery_Coronary","Colon_Sigmoid","Pancreas","Testis","Cells_EBV-transformed_lymphocytes","Colon_Transverse","Ovary","Prostate","Brain_Cortex","Vagina","Brain_Substantia_nigra","Cells_Transformed_fibroblasts","Pituitary","Brain_Hypothalamus","Brain_Spinal_cord_(cervical_c-1)","Brain_Hippocampus","Minor_Salivary_Gland","Esophagus_Mucosa","Brain_Frontal_Cortex_(BA9)","Brain_Amygdala","Brain_Cerebellum","Brain_Cerebellar_Hemisphere"],[8546,12030,9405,7802,9709,11764,9406,6787,8856,10636,11221,6835,7959,11537,11965,8318,8255,12484,10900,13749,11293,12095,10541,13755,13579,6478,6561,11905,10522,7325,7350,7444,7141,11108,8708,11475,6416,12225,10340,9111,11716,12196,10490,8002,6519,13166,10757,10458,13068],[8.56159569154895,7.89804215047615,7.23323432897412,5.80917061821526,4.69685447572984,4.31604242244703,2.57713820315907,1.26300106748878,1.07159867536375,0.995181269338993,0.838788033209964,0.661348376393602,0.58432156391616,0.377699496841707,0.164939886425024,0.104107487073083,-0.233176519268453,-0.349384045903328,-0.493728462206134,-0.59933004979628,-0.615743476155149,-0.727771100720054,-0.753032816217442,-0.762473007312877,-0.800992485436152,-0.813015404797777,-0.815233835500331,-0.82693399432574,-0.841998038312565,-0.845356363831092,-0.916634123155574,-0.938087377293796,-0.969171729303845,-1.02013346202496,-1.04656598737408,-1.09931825679542,-1.11563095412984,-1.12186284035495,-1.13853388973771,-1.16696044155289,-1.17878204368592,-1.32211310674289,-1.34025745882639,-1.38788302504689,-1.49434923865068,-1.63850046579848,-2.89538814057726,-3.20827884508297,-3.37259245455917],["-3.7343<br/>[-3.7376, -3.7020]","-3.7268<br/>[-3.7301, -3.6943]","-3.7301<br/>[-3.7334, -3.6982]","-3.7296<br/>[-3.7329, -3.6978]","-3.7264<br/>[-3.7297, -3.6948]","-3.7187<br/>[-3.7219, -3.6873]","-3.7248<br/>[-3.7281, -3.6931]","-3.7135<br/>[-3.7167, -3.6827]","-3.7151<br/>[-3.7183, -3.6841]","-3.7136<br/>[-3.7168, -3.6828]","-3.7135<br/>[-3.7167, -3.6826]","-3.7114<br/>[-3.7145, -3.6806]","-3.7129<br/>[-3.7161, -3.6821]","-3.7184<br/>[-3.7216, -3.6870]","-3.7185<br/>[-3.7217, -3.6871]","-3.7134<br/>[-3.7166, -3.6825]","-3.7133<br/>[-3.7165, -3.6825]","-3.7130<br/>[-3.7162, -3.6821]","-3.7114<br/>[-3.7146, -3.6806]","-3.7112<br/>[-3.7146, -3.6787]","-3.7134<br/>[-3.7166, -3.6825]","-3.7076<br/>[-3.7107, -3.6769]","-3.7115<br/>[-3.7147, -3.6808]","-3.7077<br/>[-3.7108, -3.6770]","-3.7096<br/>[-3.7127, -3.6789]","-3.7093<br/>[-3.7125, -3.6786]","-3.7093<br/>[-3.7125, -3.6787]","-3.7085<br/>[-3.7117, -3.6779]","-3.7081<br/>[-3.7112, -3.6775]","-3.7093<br/>[-3.7124, -3.6786]","-3.7099<br/>[-3.7130, -3.6792]","-3.7087<br/>[-3.7119, -3.6781]","-3.7100<br/>[-3.7132, -3.6793]","-3.7097<br/>[-3.7128, -3.6789]","-3.7109<br/>[-3.7141, -3.6801]","-3.7131<br/>[-3.7163, -3.6822]","-3.7086<br/>[-3.7118, -3.6779]","-3.7094<br/>[-3.7126, -3.6786]","-3.7096<br/>[-3.7128, -3.6788]","-3.7085<br/>[-3.7116, -3.6779]","-3.7109<br/>[-3.7141, -3.6802]","-3.7071<br/>[-3.7103, -3.6765]","-3.7134<br/>[-3.7166, -3.6824]","-3.7105<br/>[-3.7137, -3.6797]","-3.7103<br/>[-3.7137, -3.6779]","-3.7126<br/>[-3.7158, -3.6816]","-3.7131<br/>[-3.7163, -3.6822]","-3.7100<br/>[-3.7131, -3.6792]","-3.7090<br/>[-3.7122, -3.6783]"],["-2.5532<br/>[-2.6316, -2.4866]","-2.7964<br/>[-2.8130, -2.7962]","-2.4955<br/>[-2.6603, -2.3812]","-2.5544<br/>[-2.6425, -2.4762]","-2.6572<br/>[-2.7468, -2.5924]","-2.8264<br/>[-2.8284, -2.8246]","-2.8192<br/>[-2.9193, -2.7285]","-3.0307<br/>[-3.1170, -2.8701]","-3.2214<br/>[-3.3474, -3.0946]","-3.2192<br/>[-3.3512, -3.0933]","-3.1092<br/>[-3.1681, -3.0043]","-3.3188<br/>[-3.4092, -3.2275]","-3.3764<br/>[-3.4810, -3.2391]","-3.1044<br/>[-3.2106, -2.9918]","-3.1206<br/>[-3.2357, -3.0257]","-3.0696<br/>[-3.2057, -2.9501]","-3.1431<br/>[-3.3015, -3.0078]","-3.1430<br/>[-3.2728, -3.0392]","-3.1484<br/>[-3.2936, -3.0317]","-3.3104<br/>[-3.4056, -3.2021]","-3.2835<br/>[-3.3890, -3.1484]","-3.3528<br/>[-3.4477, -3.2411]","-3.2891<br/>[-3.4510, -3.1692]","-3.3659<br/>[-3.4673, -3.2478]","-3.3839<br/>[-3.4794, -3.2645]","-3.4065<br/>[-3.5564, -3.2286]","-3.4071<br/>[-3.5575, -3.2329]","-3.3940<br/>[-3.5035, -3.2659]","-3.4076<br/>[-3.5106, -3.2782]","-3.4225<br/>[-3.5620, -3.2453]","-3.4831<br/>[-3.6523, -3.2824]","-3.5166<br/>[-3.6317, -3.3538]","-3.5416<br/>[-3.7338, -3.3147]","-3.5347<br/>[-3.6400, -3.3959]","-3.3053<br/>[-3.4446, -3.1590]","-3.1256<br/>[-3.2391, -3.0279]","-3.9162<br/>[-4.1064, -3.6768]","-3.6727<br/>[-3.7692, -3.5338]","-3.6946<br/>[-3.7760, -3.5685]","-3.7930<br/>[-3.9542, -3.5800]","-3.3807<br/>[-3.4920, -3.2512]","-4.0019<br/>[-4.1284, -3.8221]","-3.1407<br/>[-3.2692, -3.0239]","-3.5771<br/>[-3.6957, -3.4174]","-3.5939<br/>[-3.7161, -3.4393]","-3.2705<br/>[-3.3686, -3.1693]","-3.1817<br/>[-3.3054, -3.0739]","-3.4647<br/>[-3.5494, -3.3445]","-3.5711<br/>[-3.6385, -3.4688]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["20","18","3","10","13","17","9","19","5","1","14","12","11","4","7","6","8","16","15","2"],[7644,6884,9554,7849,7157,7959,9065,5553,10928,7016,8925,6819,9848,11776,7751,8324,9692,9733,7280,8429],[16.3864366966289,13.8129312275316,8.584444880067,4.96540025840368,4.15156973684737,3.11624469561057,2.99735492739141,0.551892954845,0.382508500862703,0.207606430771853,-0.193964619232023,-0.379730404216116,-0.453960007085027,-0.495797540216702,-0.52285707755788,-0.711704569927584,-0.714602034408835,-0.931500888784892,-1.30294708325036,-2.23240505596636],["-3.7193<br/>[-3.7225, -3.6881]","-3.7322<br/>[-3.7356, -3.7001]","-3.7288<br/>[-3.7320, -3.6969]","-3.7223<br/>[-3.7256, -3.6908]","-3.7232<br/>[-3.7265, -3.6917]","-3.7187<br/>[-3.7221, -3.6858]","-3.7211<br/>[-3.7243, -3.6896]","-3.7094<br/>[-3.7125, -3.6787]","-3.7121<br/>[-3.7152, -3.6813]","-3.7105<br/>[-3.7137, -3.6798]","-3.7131<br/>[-3.7163, -3.6822]","-3.7090<br/>[-3.7122, -3.6784]","-3.7094<br/>[-3.7125, -3.6787]","-3.7127<br/>[-3.7159, -3.6820]","-3.7126<br/>[-3.7158, -3.6818]","-3.7108<br/>[-3.7142, -3.6784]","-3.7119<br/>[-3.7151, -3.6811]","-3.7153<br/>[-3.7185, -3.6843]","-3.7105<br/>[-3.7138, -3.6780]","-3.7131<br/>[-3.7162, -3.6821]"],["-2.3777<br/>[-2.4013, -2.3408]","-2.3611<br/>[-2.4319, -2.2835]","-2.3676<br/>[-2.3720, -2.3418]","-2.7418<br/>[-2.8585, -2.6386]","-2.6143<br/>[-2.7321, -2.5986]","-2.7712<br/>[-2.8952, -2.6706]","-2.9118<br/>[-2.9661, -2.8311]","-2.9014<br/>[-3.1647, -2.7534]","-3.0818<br/>[-3.2426, -2.9852]","-3.0541<br/>[-3.2015, -2.9498]","-3.1954<br/>[-3.3344, -3.0599]","-3.2475<br/>[-3.3794, -3.1004]","-3.2821<br/>[-3.4498, -3.1123]","-3.3165<br/>[-3.4582, -3.1738]","-3.3200<br/>[-3.4670, -3.1550]","-3.4467<br/>[-3.5871, -3.2725]","-3.4437<br/>[-3.5589, -3.2986]","-2.9166<br/>[-3.1276, -2.8233]","-3.5378<br/>[-3.6809, -3.3643]","-3.1512<br/>[-3.3541, -2.9599]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1">devtools<span class="op">::</span><span class="kw">session_info</span>()</a></code></pre></div>
<pre><code>Session info -------------------------------------------------------------</code></pre>
<pre><code> setting  value                       
 version  R version 3.5.1 (2018-07-02)
 system   x86_64, darwin15.6.0        
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2018-10-05                  </code></pre>
<pre><code>Packages -----------------------------------------------------------------</code></pre>
<pre><code> package     * version date       source        
 assertthat    0.2.0   2017-04-11 CRAN (R 3.5.0)
 backports     1.1.2   2017-12-13 CRAN (R 3.5.0)
 base        * 3.5.1   2018-07-05 local         
 bindr         0.1.1   2018-03-13 CRAN (R 3.5.0)
 bindrcpp      0.2.2   2018-03-29 CRAN (R 3.5.0)
 compiler      3.5.1   2018-07-05 local         
 crayon        1.3.4   2017-09-16 CRAN (R 3.5.0)
 crosstalk     1.0.0   2016-12-21 CRAN (R 3.5.0)
 datasets    * 3.5.1   2018-07-05 local         
 devtools      1.13.6  2018-06-27 CRAN (R 3.5.0)
 digest        0.6.17  2018-09-12 CRAN (R 3.5.0)
 dplyr       * 0.7.6   2018-06-29 CRAN (R 3.5.1)
 DT          * 0.4     2018-01-30 CRAN (R 3.5.0)
 evaluate      0.11    2018-07-17 CRAN (R 3.5.0)
 git2r         0.23.0  2018-07-17 CRAN (R 3.5.0)
 glue          1.3.0   2018-07-17 CRAN (R 3.5.0)
 graphics    * 3.5.1   2018-07-05 local         
 grDevices   * 3.5.1   2018-07-05 local         
 htmltools     0.3.6   2017-04-28 CRAN (R 3.5.0)
 htmlwidgets   1.3     2018-09-30 CRAN (R 3.5.0)
 httpuv        1.4.5   2018-07-19 CRAN (R 3.5.0)
 jsonlite      1.5     2017-06-01 CRAN (R 3.5.0)
 knitr         1.20    2018-02-20 CRAN (R 3.5.0)
 later         0.7.5   2018-09-18 CRAN (R 3.5.0)
 magrittr      1.5     2014-11-22 CRAN (R 3.5.0)
 memoise       1.1.0   2017-04-21 CRAN (R 3.5.0)
 methods     * 3.5.1   2018-07-05 local         
 mime          0.5     2016-07-07 CRAN (R 3.5.0)
 pillar        1.3.0   2018-07-14 CRAN (R 3.5.0)
 pkgconfig     2.0.2   2018-08-16 CRAN (R 3.5.0)
 plyr        * 1.8.4   2016-06-08 CRAN (R 3.5.0)
 promises      1.0.1   2018-04-13 CRAN (R 3.5.0)
 purrr         0.2.5   2018-05-29 CRAN (R 3.5.0)
 R.matlab    * 3.6.2   2018-09-27 CRAN (R 3.5.0)
 R.methodsS3   1.7.1   2016-02-16 CRAN (R 3.5.0)
 R.oo          1.22.0  2018-04-22 CRAN (R 3.5.0)
 R.utils       2.7.0   2018-08-27 CRAN (R 3.5.0)
 R6            2.3.0   2018-10-04 CRAN (R 3.5.0)
 Rcpp          0.12.19 2018-10-01 CRAN (R 3.5.0)
 rlang         0.2.2   2018-08-16 CRAN (R 3.5.0)
 rmarkdown     1.10    2018-06-11 CRAN (R 3.5.0)
 rprojroot     1.3-2   2018-01-03 CRAN (R 3.5.0)
 shiny         1.1.0   2018-05-17 CRAN (R 3.5.0)
 stats       * 3.5.1   2018-07-05 local         
 stringi       1.2.4   2018-07-20 CRAN (R 3.5.0)
 stringr       1.3.1   2018-05-10 CRAN (R 3.5.0)
 tibble        1.4.2   2018-01-22 CRAN (R 3.5.0)
 tidyselect    0.2.4   2018-02-26 CRAN (R 3.5.0)
 tools         3.5.1   2018-07-05 local         
 utils       * 3.5.1   2018-07-05 local         
 whisker       0.3-2   2013-04-28 CRAN (R 3.5.0)
 withr         2.1.2   2018-03-15 CRAN (R 3.5.0)
 workflowr     1.1.1   2018-07-06 CRAN (R 3.5.0)
 xtable        1.8-3   2018-08-29 CRAN (R 3.5.0)
 yaml          2.2.0   2018-07-25 CRAN (R 3.5.0)</code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
