<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of adult human height</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; position: absolute; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine:empty
  { position: absolute; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of adult human height</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-06-27</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/b70131dd0275b0662811faab5f6765bcb7ef7a87" target="_blank">b70131d</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/b70131dd0275b0662811faab5f6765bcb7ef7a87/analysis/height_2014.Rmd" target="_blank">b70131d</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
wflow_publish(“height_2014.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/25282103">“Defining the Role of Common Variation in the Genomic and Biological Architecture of Adult Human Height” (<em>Nature Genetics</em>, 2014)</a>. The summary data file is available at <a href="https://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files" class="uri">https://portals.broadinstitute.org/collaboration/giant/index.php/GIANT_consortium_data_files</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["Endochondral Ossification<br/>(wiki, BioSystems)","Cellular responses to stress<br/>(reactome, BioSystems)","Cell Cycle, Mitotic<br/>(reactome, PC)","Cell Cycle, Mitotic<br/>(reactome, BioSystems)","Cellular responses to stress<br/>(reactome, PC)","Cellular Senescence<br/>(reactome, PC)","TGF-beta Receptor Signaling Pathway<br/>(wiki, BioSystems)","Integrated Breast Cancer Pathway<br/>(wiki, BioSystems)","Extracellular matrix organization<br/>(reactome, PC)","Extracellular matrix organization<br/>(reactome, BioSystems)","Infectious disease<br/>(reactome, BioSystems)","Signalling by NGF<br/>(reactome, BioSystems)","MAPK signaling pathway<br/>(kegg, BioSystems)","Pathways in cancer<br/>(kegg, BioSystems)","Endocytosis<br/>(kegg, BioSystems)","Chromatin modifying enzymes<br/>(reactome, BioSystems)","Metabolism of RNA<br/>(reactome, PC)","HTLV-I infection<br/>(kegg, BioSystems)","PI3K-Akt signaling pathway<br/>(kegg, BioSystems)","Hedgehog 'on' state<br/>(reactome, BioSystems)","Fc epsilon receptor (FCERI) signaling<br/>(reactome, BioSystems)","IRS-related events triggered by IGF1R<br/>(reactome, BioSystems)","Transcriptional misregulation in cancer<br/>(kegg, BioSystems)","Adipogenesis<br/>(wiki, BioSystems)","Integrated Pancreatic Cancer Pathway<br/>(wiki, BioSystems)","Signaling by Insulin receptor<br/>(reactome, BioSystems)","Signaling by Hedgehog<br/>(reactome, BioSystems)","NGF signalling via TRKA from the plasma membrane<br/>(reactome, BioSystems)","Signaling by ERBB4<br/>(reactome, BioSystems)","VEGFA-VEGFR2 Pathway<br/>(reactome, BioSystems)","Hemostasis<br/>(reactome, PC)","Signaling by VEGF<br/>(reactome, BioSystems)","Signaling by PDGF<br/>(reactome, BioSystems)","Signaling by SCF-KIT<br/>(reactome, BioSystems)","Insulin receptor signalling cascade<br/>(reactome, BioSystems)","Hemostasis<br/>(reactome, BioSystems)","Chromatin modifying enzymes<br/>(reactome, PC)","Downstream signal transduction<br/>(reactome, BioSystems)","Post-translational protein modification<br/>(reactome, BioSystems)","Elastic fibre formation<br/>(reactome, PC)","Hedgehog signaling pathway<br/>(kegg, BioSystems)","Proteoglycans in cancer<br/>(kegg, BioSystems)","Signaling by NGF<br/>(reactome, PC)","Organelle biogenesis and maintenance<br/>(reactome, BioSystems)","Metabolism of mRNA<br/>(reactome, PC)","Generic Transcription Pathway<br/>(reactome, PC)","Gastrin-CREB signalling pathway via PKC and MAPK<br/>(reactome, BioSystems)","IRS-related events triggered by IGF1R<br/>(reactome, PC)","Developmental Biology<br/>(reactome, PC)","Basal cell carcinoma<br/>(kegg, BioSystems)","IGF1R signaling cascade<br/>(reactome, PC)","IRS-mediated signalling<br/>(reactome, BioSystems)","Mitotic M-M/G1 phases<br/>(reactome, PC)","Signaling by Wnt<br/>(reactome, BioSystems)","Ras signaling pathway<br/>(kegg, BioSystems)","B Cell Receptor Signaling Pathway<br/>(wiki, BioSystems)","MAPK family signaling cascades<br/>(reactome, BioSystems)","Signaling by Wnt<br/>(reactome, PC)","RHO GTPase Effectors<br/>(reactome, BioSystems)","Signaling pathways regulating pluripotency of stem cells<br/>(kegg, BioSystems)","Signaling by Rho GTPases<br/>(reactome, BioSystems)","Signaling by ERBB2<br/>(reactome, BioSystems)","M Phase<br/>(reactome, PC)","Diseases of signal transduction<br/>(reactome, BioSystems)","GPCR ligand binding<br/>(reactome, PC)","Signaling by the B Cell Receptor (BCR)<br/>(reactome, BioSystems)","Signaling by WNT in cancer<br/>(reactome, PC)","M Phase<br/>(reactome, BioSystems)","B Cell Activation<br/>(reactome, PC)","GPCR ligand binding<br/>(reactome, BioSystems)","Signaling events mediated by the Hedgehog family<br/>(pid, BioSystems)","Signaling events mediated by the Hedgehog family<br/>(pid, PC)","Class I MHC mediated antigen processing & presentation<br/>(reactome, BioSystems)","Signaling by FGFR4<br/>(reactome, BioSystems)","Signaling by EGFR<br/>(reactome, BioSystems)","Downstream signaling of activated FGFR4<br/>(reactome, BioSystems)","Viral carcinogenesis<br/>(kegg, BioSystems)","Signaling by FGFR2<br/>(reactome, BioSystems)","Mitotic G1-G1/S phases<br/>(reactome, PC)","Androgen receptor signaling pathway<br/>(wiki, BioSystems)","Signalling to RAS<br/>(reactome, BioSystems)","Antigen processing: Ubiquitination & Proteasome degradation<br/>(reactome, BioSystems)","Signaling by Insulin receptor<br/>(reactome, PC)","Class I MHC mediated antigen processing & presentation<br/>(reactome, PC)","FoxO signaling pathway<br/>(kegg, BioSystems)","DAP12 signaling<br/>(reactome, BioSystems)","S Phase<br/>(reactome, PC)","S Phase<br/>(reactome, BioSystems)","Molecules associated with elastic fibres<br/>(reactome, PC)","Hepatitis B<br/>(kegg, BioSystems)","Signalling to ERKs<br/>(reactome, BioSystems)","Translation<br/>(reactome, BioSystems)","Prostate cancer<br/>(kegg, BioSystems)","Downstream signaling events of B Cell Receptor (BCR)<br/>(reactome, PC)","Hedgehog Signaling Pathway<br/>(wiki, BioSystems)","Downstream signaling events of B Cell Receptor (BCR)<br/>(reactome, BioSystems)","Translation<br/>(reactome, PC)","TGF-beta signaling pathway<br/>(kegg, BioSystems)","DAP12 interactions<br/>(reactome, BioSystems)","Organelle biogenesis and maintenance<br/>(reactome, PC)"],[65,366,457,463,258,183,149,147,254,259,359,410,246,385,252,241,257,250,335,83,330,271,170,130,190,292,132,344,303,302,490,309,335,298,271,496,203,313,333,41,50,195,273,311,211,442,406,87,476,55,90,268,316,318,220,153,264,312,274,138,375,311,284,256,421,191,244,288,188,426,22,23,256,304,325,301,197,307,131,84,231,213,112,242,128,315,118,121,30,142,238,137,86,163,22,164,136,80,330,143],[7573,24387,31432,31834,16444,10005,13241,13140,27448,28019,26162,42783,25117,40147,22581,15037,17586,18860,32932,6283,32021,25621,15096,12469,17485,27268,11887,34905,29159,29800,46399,30780,33892,27961,25586,46672,11365,31128,33101,4603,4377,19385,30785,25500,14412,24029,41928,7471,58142,4767,7756,25136,20814,23655,22366,16173,25128,23116,19097,11496,32907,30840,18629,24103,32984,18262,15362,18845,18077,33335,2490,2495,20181,30364,32798,30105,13863,30590,9246,7137,22070,17825,9498,19274,11400,31730,8468,8643,3418,11920,22881,9419,8149,15125,1800,15131,9416,6621,32565,11198],[68.8862016133663,64.3610613582384,60.2486186803681,59.7100882117568,54.5693688973819,49.4832435204049,48.8143837349395,48.7957551697753,48.4813176388026,48.4292324810719,47.192881516199,46.863880362955,46.2020160320493,45.8613924008229,45.3633885191687,45.3314209001459,43.9000313025381,42.1230163323821,41.2758073371355,40.2865720861261,39.8676031706658,39.7317215182171,39.293539326958,38.9067746763615,38.8330085774341,38.2571331725315,37.9308648214284,37.7048561004853,37.2290254647025,37.1321013568989,36.8071205148252,36.8005485343511,36.6512280778931,36.5421494109434,36.5418282905122,36.4444980069655,36.1243642163309,35.9139861083397,35.8346989503635,35.691951898713,35.6699503888098,35.5601492569533,35.4031770948915,35.4015455357779,35.0779728766463,35.0145369654243,34.7330358824933,34.6767843113591,34.510292642504,34.1098987487604,33.9766099235721,33.9267429111293,33.8440226420421,33.6091472904146,33.4806555041672,33.4340670155549,32.8187556034526,32.807225307214,32.7699488899132,32.363661299432,32.2686409299567,32.0226565400787,31.344773316669,31.2228382549267,31.2214666052147,31.0951313721818,31.0215374517961,31.0033669151105,30.9596070122248,30.7320757901699,30.6150775428271,30.4185498318495,30.3703097939564,30.1086231869801,30.0550614338482,30.015879257638,29.9131798631345,29.7269944288292,29.7145183065645,29.5801385574068,28.5356640553806,28.3756661437539,28.2776635415764,28.2032100157037,28.1838851097996,27.8226497645863,27.7156163354568,27.6633229414919,27.4576273365842,27.4532392299086,27.204816359787,26.7443321225499,26.7235892532488,26.7128310592725,26.7002508207139,26.6876590302971,26.5796551424261,26.3651920404217,26.2743797248527,25.3671081181942],["-1.9687<br/>[-2.0066, -1.9642]","-1.9688<br/>[-2.0067, -1.9643]","-1.9694<br/>[-2.0074, -1.9649]","-1.9694<br/>[-2.0073, -1.9649]","-1.9681<br/>[-2.0060, -1.9636]","-1.9678<br/>[-2.0056, -1.9633]","-1.9681<br/>[-2.0060, -1.9636]","-1.9678<br/>[-2.0057, -1.9633]","-1.9694<br/>[-2.0074, -1.9649]","-1.9695<br/>[-2.0074, -1.9649]","-1.9679<br/>[-2.0058, -1.9634]","-1.9695<br/>[-2.0074, -1.9649]","-1.9683<br/>[-2.0061, -1.9638]","-1.9705<br/>[-2.0084, -1.9659]","-1.9686<br/>[-2.0064, -1.9641]","-1.9684<br/>[-2.0064, -1.9639]","-1.9676<br/>[-2.0054, -1.9631]","-1.9683<br/>[-2.0062, -1.9638]","-1.9696<br/>[-2.0075, -1.9651]","-1.9669<br/>[-2.0047, -1.9624]","-1.9688<br/>[-2.0067, -1.9643]","-1.9685<br/>[-2.0063, -1.9639]","-1.9678<br/>[-2.0056, -1.9633]","-1.9676<br/>[-2.0054, -1.9631]","-1.9677<br/>[-2.0055, -1.9632]","-1.9682<br/>[-2.0060, -1.9637]","-1.9674<br/>[-2.0052, -1.9629]","-1.9688<br/>[-2.0066, -1.9642]","-1.9685<br/>[-2.0064, -1.9640]","-1.9681<br/>[-2.0059, -1.9636]","-1.9690<br/>[-2.0069, -1.9644]","-1.9681<br/>[-2.0059, -1.9636]","-1.9687<br/>[-2.0066, -1.9642]","-1.9685<br/>[-2.0063, -1.9639]","-1.9681<br/>[-2.0059, -1.9636]","-1.9689<br/>[-2.0068, -1.9644]","-1.9676<br/>[-2.0054, -1.9630]","-1.9684<br/>[-2.0063, -1.9639]","-1.9679<br/>[-2.0057, -1.9633]","-1.9672<br/>[-2.0051, -1.9627]","-1.9668<br/>[-2.0046, -1.9623]","-1.9683<br/>[-2.0062, -1.9638]","-1.9685<br/>[-2.0064, -1.9640]","-1.9674<br/>[-2.0052, -1.9629]","-1.9674<br/>[-2.0052, -1.9629]","-1.9678<br/>[-2.0056, -1.9633]","-1.9683<br/>[-2.0062, -1.9638]","-1.9674<br/>[-2.0052, -1.9629]","-1.9685<br/>[-2.0064, -1.9639]","-1.9668<br/>[-2.0045, -1.9623]","-1.9674<br/>[-2.0052, -1.9629]","-1.9679<br/>[-2.0057, -1.9633]","-1.9671<br/>[-2.0049, -1.9626]","-1.9675<br/>[-2.0053, -1.9630]","-1.9678<br/>[-2.0056, -1.9632]","-1.9675<br/>[-2.0053, -1.9630]","-1.9678<br/>[-2.0057, -1.9633]","-1.9675<br/>[-2.0053, -1.9629]","-1.9671<br/>[-2.0049, -1.9626]","-1.9675<br/>[-2.0053, -1.9630]","-1.9675<br/>[-2.0054, -1.9630]","-1.9684<br/>[-2.0063, -1.9639]","-1.9669<br/>[-2.0047, -1.9624]","-1.9683<br/>[-2.0061, -1.9637]","-1.9675<br/>[-2.0053, -1.9630]","-1.9679<br/>[-2.0058, -1.9634]","-1.9671<br/>[-2.0049, -1.9626]","-1.9669<br/>[-2.0047, -1.9624]","-1.9679<br/>[-2.0058, -1.9634]","-1.9675<br/>[-2.0053, -1.9630]","-1.9665<br/>[-2.0042, -1.9620]","-1.9665<br/>[-2.0042, -1.9620]","-1.9676<br/>[-2.0055, -1.9631]","-1.9683<br/>[-2.0062, -1.9638]","-1.9685<br/>[-2.0063, -1.9639]","-1.9683<br/>[-2.0061, -1.9638]","-1.9678<br/>[-2.0056, -1.9633]","-1.9683<br/>[-2.0061, -1.9638]","-1.9673<br/>[-2.0051, -1.9628]","-1.9671<br/>[-2.0049, -1.9626]","-1.9674<br/>[-2.0051, -1.9629]","-1.9674<br/>[-2.0053, -1.9629]","-1.9672<br/>[-2.0049, -1.9626]","-1.9675<br/>[-2.0054, -1.9630]","-1.9676<br/>[-2.0054, -1.9630]","-1.9682<br/>[-2.0061, -1.9637]","-1.9669<br/>[-2.0047, -1.9624]","-1.9669<br/>[-2.0047, -1.9624]","-1.9668<br/>[-2.0046, -1.9622]","-1.9677<br/>[-2.0055, -1.9631]","-1.9674<br/>[-2.0051, -1.9628]","-1.9667<br/>[-2.0044, -1.9622]","-1.9672<br/>[-2.0050, -1.9626]","-1.9677<br/>[-2.0055, -1.9631]","-1.9663<br/>[-2.0040, -1.9618]","-1.9677<br/>[-2.0055, -1.9631]","-1.9667<br/>[-2.0044, -1.9622]","-1.9668<br/>[-2.0045, -1.9622]","-1.9682<br/>[-2.0060, -1.9636]","-1.9666<br/>[-2.0043, -1.9621]"],["-1.2667<br/>[-1.2963, -1.2307]","-1.5056<br/>[-1.5228, -1.4936]","-1.5424<br/>[-1.5563, -1.5322]","-1.5444<br/>[-1.5576, -1.5334]","-1.4901<br/>[-1.5066, -1.4842]","-1.4464<br/>[-1.4537, -1.4421]","-1.4510<br/>[-1.4834, -1.4138]","-1.4324<br/>[-1.4542, -1.4015]","-1.5690<br/>[-1.5748, -1.5551]","-1.5705<br/>[-1.5759, -1.5562]","-1.5648<br/>[-1.6105, -1.5302]","-1.6252<br/>[-1.6397, -1.6080]","-1.5928<br/>[-1.6547, -1.5255]","-1.5629<br/>[-1.5696, -1.5312]","-1.4977<br/>[-1.5003, -1.4879]","-1.4626<br/>[-1.5069, -1.4440]","-1.4865<br/>[-1.5440, -1.4652]","-1.4931<br/>[-1.5131, -1.4689]","-1.5679<br/>[-1.6010, -1.5351]","-1.2999<br/>[-1.3584, -1.2769]","-1.6036<br/>[-1.6365, -1.5789]","-1.5138<br/>[-1.5140, -1.5076]","-1.4759<br/>[-1.4819, -1.4632]","-1.4854<br/>[-1.5162, -1.4505]","-1.5333<br/>[-1.5963, -1.4866]","-1.6164<br/>[-1.6527, -1.5783]","-1.4416<br/>[-1.4470, -1.4176]","-1.5719<br/>[-1.6276, -1.5373]","-1.6045<br/>[-1.6469, -1.5784]","-1.6243<br/>[-1.6676, -1.5991]","-1.7080<br/>[-1.7306, -1.6748]","-1.6296<br/>[-1.6698, -1.6010]","-1.6248<br/>[-1.6610, -1.5917]","-1.5794<br/>[-1.6193, -1.5269]","-1.6128<br/>[-1.6576, -1.5786]","-1.6985<br/>[-1.7371, -1.6752]","-1.4607<br/>[-1.5066, -1.4321]","-1.6167<br/>[-1.6508, -1.5886]","-1.6907<br/>[-1.7000, -1.6703]","-1.3266<br/>[-1.3698, -1.2865]","-1.2613<br/>[-1.3317, -1.2463]","-1.4996<br/>[-1.5037, -1.4865]","-1.6279<br/>[-1.6546, -1.5942]","-1.5796<br/>[-1.5985, -1.5577]","-1.4972<br/>[-1.5435, -1.4650]","-1.6250<br/>[-1.6536, -1.6049]","-1.6915<br/>[-1.7256, -1.6628]","-1.3858<br/>[-1.4237, -1.3474]","-1.7446<br/>[-1.7738, -1.7026]","-1.2820<br/>[-1.3071, -1.2487]","-1.3961<br/>[-1.4284, -1.3569]","-1.6220<br/>[-1.6612, -1.5869]","-1.5923<br/>[-1.6362, -1.5519]","-1.5965<br/>[-1.6351, -1.5670]","-1.6031<br/>[-1.6492, -1.5809]","-1.5581<br/>[-1.5951, -1.5241]","-1.6144<br/>[-1.6538, -1.5794]","-1.5971<br/>[-1.6352, -1.5671]","-1.6204<br/>[-1.6640, -1.5752]","-1.4429<br/>[-1.4772, -1.4014]","-1.7080<br/>[-1.7349, -1.6744]","-1.6188<br/>[-1.6634, -1.5475]","-1.6157<br/>[-1.6530, -1.5571]","-1.5851<br/>[-1.6110, -1.5673]","-1.6895<br/>[-1.7239, -1.6615]","-1.5575<br/>[-1.5785, -1.5112]","-1.5594<br/>[-1.5922, -1.5189]","-1.6047<br/>[-1.6537, -1.5470]","-1.5550<br/>[-1.5782, -1.5241]","-1.6931<br/>[-1.7204, -1.6616]","-1.2094<br/>[-1.2532, -1.1515]","-1.2096<br/>[-1.2631, -1.1335]","-1.5904<br/>[-1.6315, -1.5599]","-1.5681<br/>[-1.6590, -1.5423]","-1.5976<br/>[-1.6748, -1.5520]","-1.5668<br/>[-1.6521, -1.5421]","-1.5312<br/>[-1.5810, -1.4929]","-1.5720<br/>[-1.6593, -1.5431]","-1.4332<br/>[-1.4891, -1.3695]","-1.4151<br/>[-1.4752, -1.3724]","-1.5696<br/>[-1.6491, -1.5250]","-1.5817<br/>[-1.6222, -1.5507]","-1.4789<br/>[-1.5297, -1.4411]","-1.5949<br/>[-1.6335, -1.5618]","-1.4485<br/>[-1.4960, -1.4164]","-1.5901<br/>[-1.6708, -1.5534]","-1.4293<br/>[-1.4719, -1.3895]","-1.4328<br/>[-1.4734, -1.3907]","-1.3310<br/>[-1.3865, -1.2890]","-1.4717<br/>[-1.5241, -1.4281]","-1.6161<br/>[-1.6661, -1.5563]","-1.4000<br/>[-1.4651, -1.3657]","-1.3951<br/>[-1.4576, -1.3660]","-1.4863<br/>[-1.5571, -1.4435]","-1.2007<br/>[-1.2335, -1.1889]","-1.4853<br/>[-1.5571, -1.4435]","-1.3999<br/>[-1.4699, -1.3594]","-1.4042<br/>[-1.4519, -1.3647]","-1.6080<br/>[-1.6807, -1.5565]","-1.5353<br/>[-1.5829, -1.4883]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Artery - Aorta","Artery - Coronary","Pituitary","Artery - Tibial","Esophagus - Muscularis","Colon - Sigmoid","Esophagus - Gastroesophageal Junction","Adipose - Subcutaneous","Adipose - Visceral (Omentum)","Nerve - Tibial","Ovary","Prostate","Uterus","Skin - Not Sun Exposed (Suprapubic)","Breast - Mammary Tissue","Thyroid","Colon - Transverse","Adrenal Gland","Cells - Transformed fibroblasts","Stomach","Brain - Caudate (basal ganglia)","Brain - Cerebellum","Brain - Hippocampus","Cells - EBV-transformed lymphocytes","Brain - Hypothalamus","Brain - Putamen (basal ganglia)","Brain - Cerebellar Hemisphere","Pancreas","Vagina","Brain - Cortex","Lung","Brain - Nucleus accumbens (basal ganglia)","Liver","Muscle - Skeletal","Brain - Anterior cingulate cortex (BA24)","Testis","Brain - Frontal Cortex (BA9)","Skin - Sun Exposed (Lower leg)","Spleen","Esophagus - Mucosa","Small Intestine - Terminal Ileum","Heart - Left Ventricle","Heart - Atrial Appendage","Whole Blood"],[6517,6308,5553,6228,6018,5928,5817,5599,5737,5622,5728,5559,5632,5798,5454,6001,5192,5351,6073,5151,5945,5591,5493,4878,5519,6156,5762,5192,5717,5694,5079,5865,6487,5707,5786,5637,6156,5727,3821,5512,4748,5535,5547,5801],[20.9468184316883,14.4649597187659,13.5941510402003,13.1118848194883,13.0148183928349,11.7796225165304,11.3260358606895,11.1700670583592,9.09820661481993,8.99042640605908,8.68003884477149,8.52878493538681,8.4867755718057,8.34434578354722,8.19092848683128,7.65907559707973,6.81650585007826,6.69446979201861,6.5881687208051,6.0332216756551,6.00940472287845,5.58205377033169,5.35291549451701,5.33125506078197,5.14817051160825,5.13229750980688,4.65408342295365,4.30703688687621,4.30599693448932,4.25155303972393,4.2342913895442,3.9073199075845,3.82522569305536,3.76984069552096,3.55547150972297,3.31386280222122,3.02970396480579,2.55205261513189,2.05162452049783,1.83543710695662,1.50496445691981,1.46153234266814,1.38344249560351,0.422808231813606],["-1.9666<br/>[-2.0044, -1.9621]","-1.9662<br/>[-2.0040, -1.9617]","-1.9661<br/>[-2.0038, -1.9615]","-1.9662<br/>[-2.0039, -1.9616]","-1.9660<br/>[-2.0038, -1.9615]","-1.9659<br/>[-2.0036, -1.9614]","-1.9659<br/>[-2.0037, -1.9614]","-1.9659<br/>[-2.0036, -1.9614]","-1.9657<br/>[-2.0035, -1.9612]","-1.9657<br/>[-2.0034, -1.9612]","-1.9659<br/>[-2.0036, -1.9614]","-1.9657<br/>[-2.0034, -1.9612]","-1.9657<br/>[-2.0034, -1.9612]","-1.9656<br/>[-2.0033, -1.9611]","-1.9657<br/>[-2.0034, -1.9612]","-1.9658<br/>[-2.0035, -1.9613]","-1.9655<br/>[-2.0032, -1.9610]","-1.9658<br/>[-2.0035, -1.9613]","-1.9658<br/>[-2.0035, -1.9613]","-1.9656<br/>[-2.0033, -1.9611]","-1.9656<br/>[-2.0032, -1.9611]","-1.9657<br/>[-2.0034, -1.9612]","-1.9654<br/>[-2.0031, -1.9609]","-1.9656<br/>[-2.0033, -1.9611]","-1.9657<br/>[-2.0034, -1.9612]","-1.9655<br/>[-2.0032, -1.9610]","-1.9655<br/>[-2.0032, -1.9610]","-1.9656<br/>[-2.0033, -1.9611]","-1.9656<br/>[-2.0033, -1.9611]","-1.9656<br/>[-2.0032, -1.9610]","-1.9656<br/>[-2.0033, -1.9611]","-1.9655<br/>[-2.0032, -1.9610]","-1.9657<br/>[-2.0034, -1.9612]","-1.9657<br/>[-2.0034, -1.9612]","-1.9655<br/>[-2.0031, -1.9610]","-1.9655<br/>[-2.0032, -1.9610]","-1.9655<br/>[-2.0032, -1.9610]","-1.9655<br/>[-2.0032, -1.9610]","-1.9655<br/>[-2.0032, -1.9610]","-1.9655<br/>[-2.0031, -1.9610]","-1.9655<br/>[-2.0031, -1.9610]","-1.9655<br/>[-2.0031, -1.9610]","-1.9655<br/>[-2.0032, -1.9610]","-1.9654<br/>[-2.0031, -1.9609]"],["-1.4497<br/>[-1.5025, -1.4110]","-1.4783<br/>[-1.5568, -1.4359]","-1.4518<br/>[-1.5222, -1.3952]","-1.5067<br/>[-1.5588, -1.4412]","-1.4894<br/>[-1.5604, -1.4405]","-1.5083<br/>[-1.5671, -1.4459]","-1.5108<br/>[-1.5682, -1.4431]","-1.5505<br/>[-1.6197, -1.4837]","-1.5906<br/>[-1.6597, -1.5089]","-1.5915<br/>[-1.6598, -1.5095]","-1.5151<br/>[-1.5981, -1.4373]","-1.5296<br/>[-1.5975, -1.4554]","-1.5432<br/>[-1.6127, -1.4661]","-1.5430<br/>[-1.6046, -1.4642]","-1.6015<br/>[-1.6871, -1.5419]","-1.5583<br/>[-1.6320, -1.4749]","-1.5444<br/>[-1.6153, -1.4639]","-1.5220<br/>[-1.6213, -1.4545]","-1.6033<br/>[-1.6630, -1.5434]","-1.5555<br/>[-1.6480, -1.4850]","-1.6610<br/>[-1.7451, -1.5953]","-1.5692<br/>[-1.6633, -1.4955]","-1.6192<br/>[-1.7093, -1.5543]","-1.5660<br/>[-1.6425, -1.4733]","-1.5582<br/>[-1.6563, -1.4916]","-1.6908<br/>[-1.7688, -1.6179]","-1.5987<br/>[-1.6941, -1.5277]","-1.5909<br/>[-1.6502, -1.5210]","-1.6143<br/>[-1.6958, -1.5452]","-1.6177<br/>[-1.7212, -1.5314]","-1.6845<br/>[-1.7774, -1.6051]","-1.6189<br/>[-1.7065, -1.5369]","-1.6170<br/>[-1.7038, -1.5461]","-1.6226<br/>[-1.7334, -1.5343]","-1.6375<br/>[-1.7420, -1.5516]","-1.6340<br/>[-1.7153, -1.5559]","-1.6606<br/>[-1.7636, -1.5728]","-1.6372<br/>[-1.7319, -1.5675]","-1.5876<br/>[-1.6857, -1.4863]","-1.6545<br/>[-1.7412, -1.5750]","-1.6466<br/>[-1.7520, -1.5580]","-1.6887<br/>[-1.7852, -1.5998]","-1.7080<br/>[-1.8172, -1.6331]","-1.7572<br/>[-1.8690, -1.6744]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Cells_Transformed_fibroblasts","Pituitary","Liver","Testis","Artery_Aorta","Lung","Muscle_Skeletal","Artery_Tibial","Kidney_Cortex","Pancreas","Colon_Transverse","Stomach","Prostate","Adipose_Subcutaneous","Heart_Atrial_Appendage","Small_Intestine_Terminal_Ileum","Artery_Coronary","Cells_EBV-transformed_lymphocytes","Brain_Nucleus_accumbens_(basal_ganglia)","Brain_Caudate_(basal_ganglia)","Brain_Putamen_(basal_ganglia)","Adrenal_Gland","Adipose_Visceral_(Omentum)","Vagina","Brain_Hypothalamus","Spleen","Heart_Left_Ventricle","Uterus","Esophagus_Mucosa","Ovary","Brain_Substantia_nigra","Colon_Sigmoid","Nerve_Tibial","Breast_Mammary_Tissue","Brain_Amygdala","Esophagus_Gastroesophageal_Junction","Thyroid","Brain_Hippocampus","Whole_Blood","Minor_Salivary_Gland","Brain_Spinal_cord_(cervical_c-1)","Brain_Anterior_cingulate_cortex_(BA24)","Skin_Not_Sun_Exposed_(Suprapubic)","Skin_Sun_Exposed_(Lower_leg)","Brain_Cortex","Esophagus_Muscularis","Brain_Cerebellum","Brain_Frontal_Cortex_(BA9)","Brain_Cerebellar_Hemisphere"],[11244,9452,8345,7692,14472,8468,8136,12113,8453,7852,7892,9343,8780,10475,10235,8691,12892,7839,11472,11558,11808,12466,10407,6982,11231,7167,10321,12823,7153,11898,12907,11044,12428,10256,11392,14781,11769,11273,6401,7972,12966,12101,7039,6930,11912,14709,10543,13751,13477],[18.3254024800665,10.7799416439012,8.73311438536271,7.79971794095342,7.62132452338925,7.40724310774107,6.45591385879573,6.20231325525797,6.10520342656813,5.6322068974752,4.77647440625197,4.18999839322467,3.7586864862432,3.65114333283018,3.55290064848093,3.20960137966592,2.95246826188612,2.36614007409046,2.13817865007549,2.10910051986182,2.10553919250995,1.84597029889556,1.60638268073385,1.51436190210134,1.37448347892661,1.35899920609692,0.936502345116312,0.924115716341414,0.696000734349766,0.658098146651879,0.521329381468328,0.51686393949539,0.181686761601315,-0.141142235356972,-0.278460615627348,-0.325846106073278,-0.402031047725978,-0.577700159199306,-0.62390552405103,-0.680095627354523,-0.698009884505788,-0.815602515831223,-0.818523204534655,-0.836013016404087,-0.894576431090626,-0.905778696208704,-0.995007626710223,-1.26898030910312,-1.33647005967644],["-1.9668<br/>[-2.0046, -1.9623]","-1.9662<br/>[-2.0039, -1.9617]","-1.9660<br/>[-2.0037, -1.9615]","-1.9658<br/>[-2.0036, -1.9613]","-1.9663<br/>[-2.0040, -1.9618]","-1.9660<br/>[-2.0037, -1.9615]","-1.9660<br/>[-2.0037, -1.9615]","-1.9661<br/>[-2.0039, -1.9616]","-1.9657<br/>[-2.0034, -1.9612]","-1.9657<br/>[-2.0034, -1.9612]","-1.9656<br/>[-2.0033, -1.9611]","-1.9659<br/>[-2.0036, -1.9614]","-1.9658<br/>[-2.0035, -1.9613]","-1.9655<br/>[-2.0032, -1.9610]","-1.9657<br/>[-2.0034, -1.9612]","-1.9655<br/>[-2.0032, -1.9610]","-1.9660<br/>[-2.0037, -1.9615]","-1.9656<br/>[-2.0033, -1.9611]","-1.9653<br/>[-2.0030, -1.9608]","-1.9653<br/>[-2.0030, -1.9608]","-1.9653<br/>[-2.0030, -1.9608]","-1.9653<br/>[-2.0030, -1.9608]","-1.9655<br/>[-2.0032, -1.9610]","-1.9653<br/>[-2.0030, -1.9608]","-1.9653<br/>[-2.0029, -1.9608]","-1.9654<br/>[-2.0031, -1.9609]","-1.9654<br/>[-2.0031, -1.9609]","-1.9654<br/>[-2.0031, -1.9609]","-1.9654<br/>[-2.0030, -1.9609]","-1.9654<br/>[-2.0031, -1.9609]","-1.9650<br/>[-2.0026, -1.9605]","-1.9654<br/>[-2.0031, -1.9609]","-1.9652<br/>[-2.0029, -1.9607]","-1.9652<br/>[-2.0028, -1.9607]","-1.9653<br/>[-2.0030, -1.9608]","-1.9651<br/>[-2.0028, -1.9606]","-1.9652<br/>[-2.0028, -1.9607]","-1.9653<br/>[-2.0029, -1.9608]","-1.9653<br/>[-2.0030, -1.9608]","-1.9651<br/>[-2.0027, -1.9606]","-1.9653<br/>[-2.0029, -1.9608]","-1.9652<br/>[-2.0029, -1.9607]","-1.9651<br/>[-2.0028, -1.9606]","-1.9651<br/>[-2.0028, -1.9606]","-1.9651<br/>[-2.0028, -1.9606]","-1.9650<br/>[-2.0026, -1.9605]","-1.9651<br/>[-2.0027, -1.9606]","-1.9650<br/>[-2.0026, -1.9605]","-1.9648<br/>[-2.0024, -1.9603]"],["-1.5202<br/>[-1.5245, -1.4991]","-1.6519<br/>[-1.7125, -1.6037]","-1.6773<br/>[-1.7524, -1.6266]","-1.6747<br/>[-1.7396, -1.6154]","-1.7308<br/>[-1.7814, -1.6839]","-1.6234<br/>[-1.6920, -1.5489]","-1.6177<br/>[-1.7255, -1.5488]","-1.6672<br/>[-1.7159, -1.6089]","-1.6195<br/>[-1.6411, -1.5510]","-1.6311<br/>[-1.7414, -1.5748]","-1.6497<br/>[-1.7193, -1.5809]","-1.6160<br/>[-1.7609, -1.5526]","-1.7000<br/>[-1.7661, -1.5117]","-1.7992<br/>[-1.8652, -1.7252]","-1.7221<br/>[-1.7933, -1.6488]","-1.7263<br/>[-1.7742, -1.6676]","-1.7486<br/>[-1.8190, -1.6762]","-1.7020<br/>[-1.8104, -1.6294]","-1.8856<br/>[-1.9464, -1.8138]","-1.8858<br/>[-1.9471, -1.8218]","-1.8844<br/>[-1.9472, -1.8224]","-1.8045<br/>[-1.8634, -1.7459]","-1.7630<br/>[-1.8458, -1.6849]","-1.7946<br/>[-1.9001, -1.7298]","-1.8984<br/>[-1.9747, -1.8267]","-1.7317<br/>[-1.8240, -1.6499]","-1.7877<br/>[-1.8607, -1.7107]","-1.8169<br/>[-1.8889, -1.7401]","-1.7560<br/>[-1.8681, -1.6805]","-1.8198<br/>[-1.9508, -1.7506]","-1.9680<br/>[-2.0116, -1.9127]","-1.8272<br/>[-1.9239, -1.7407]","-1.8137<br/>[-1.8999, -1.7658]","-1.8231<br/>[-1.9085, -1.7480]","-1.8463<br/>[-1.9432, -1.7800]","-1.8572<br/>[-1.9361, -1.7849]","-1.8657<br/>[-1.9303, -1.8081]","-1.8642<br/>[-1.9641, -1.8005]","-1.8395<br/>[-1.9217, -1.7398]","-1.8609<br/>[-1.9398, -1.7744]","-1.8681<br/>[-1.9329, -1.7967]","-1.8850<br/>[-1.9988, -1.8207]","-1.8644<br/>[-1.9552, -1.7770]","-1.8663<br/>[-1.9556, -1.7781]","-1.8965<br/>[-1.9900, -1.8252]","-1.8913<br/>[-1.9789, -1.8245]","-1.9168<br/>[-1.9787, -1.8459]","-1.9288<br/>[-2.0155, -1.8670]","-1.9567<br/>[-2.0044, -1.9153]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["11","5","1","4","13","8","6","9","16","10","19","12","7","3","2","17","18","20","14","15"],[10787,11504,7326,12140,7302,10447,8797,9298,10604,8679,5413,7102,8239,10183,8963,8074,7012,8284,9267,7396],[25.1654085404261,24.8213217355843,19.084401363751,18.663779836813,18.542522604888,18.3653017366178,17.453916599611,14.879063895997,14.866394157512,14.5562618159097,11.9641167542146,11.5876455621886,11.549432076241,11.2822868826447,9.7520259749596,5.78100030318389,5.68494362289316,4.42385293849789,3.50531287495735,1.96820010357393],["-1.9664<br/>[-2.0041, -1.9619]","-1.9671<br/>[-2.0049, -1.9626]","-1.9664<br/>[-2.0042, -1.9619]","-1.9660<br/>[-2.0038, -1.9615]","-1.9663<br/>[-2.0040, -1.9618]","-1.9668<br/>[-2.0045, -1.9623]","-1.9661<br/>[-2.0038, -1.9616]","-1.9666<br/>[-2.0043, -1.9621]","-1.9665<br/>[-2.0043, -1.9620]","-1.9663<br/>[-2.0041, -1.9618]","-1.9662<br/>[-2.0040, -1.9617]","-1.9661<br/>[-2.0038, -1.9616]","-1.9658<br/>[-2.0035, -1.9613]","-1.9661<br/>[-2.0038, -1.9616]","-1.9662<br/>[-2.0039, -1.9617]","-1.9658<br/>[-2.0034, -1.9613]","-1.9658<br/>[-2.0035, -1.9612]","-1.9659<br/>[-2.0036, -1.9614]","-1.9656<br/>[-2.0033, -1.9611]","-1.9655<br/>[-2.0032, -1.9610]"],["-1.6307<br/>[-1.6887, -1.5868]","-1.5251<br/>[-1.5452, -1.4972]","-1.4583<br/>[-1.5315, -1.4224]","-1.6473<br/>[-1.7118, -1.6150]","-1.5520<br/>[-1.6006, -1.5159]","-1.5285<br/>[-1.5320, -1.5134]","-1.5866<br/>[-1.6435, -1.5347]","-1.5359<br/>[-1.5890, -1.5047]","-1.5622<br/>[-1.6353, -1.5157]","-1.5530<br/>[-1.6240, -1.4646]","-1.5172<br/>[-1.5908, -1.4512]","-1.5851<br/>[-1.6398, -1.5342]","-1.6158<br/>[-1.6803, -1.5866]","-1.6331<br/>[-1.6930, -1.5800]","-1.5715<br/>[-1.6457, -1.5191]","-1.6650<br/>[-1.7369, -1.5891]","-1.6123<br/>[-1.6638, -1.5636]","-1.6662<br/>[-1.7479, -1.5966]","-1.7592<br/>[-1.8447, -1.6898]","-1.7135<br/>[-1.8185, -1.6395]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">sessionInfo</span>()</a></code></pre></div>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.5

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] DT_0.4         plyr_1.8.4     dplyr_0.7.5    R.matlab_3.6.1

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.17      compiler_3.5.0    pillar_1.2.3     
 [4] later_0.7.3       git2r_0.21.0      workflowr_1.0.1  
 [7] bindr_0.1.1       R.methodsS3_1.7.1 R.utils_2.6.0    
[10] tools_3.5.0       digest_0.6.15     jsonlite_1.5     
[13] evaluate_0.10.1   tibble_1.4.2      pkgconfig_2.0.1  
[16] rlang_0.2.1       shiny_1.1.0       crosstalk_1.0.0  
[19] yaml_2.1.19       bindrcpp_0.2.2    stringr_1.3.1    
[22] knitr_1.20        htmlwidgets_1.2   rprojroot_1.3-2  
[25] tidyselect_0.2.4  glue_1.2.0        R6_2.2.2         
[28] rmarkdown_1.10    purrr_0.2.5       magrittr_1.5     
[31] whisker_0.3-2     backports_1.1.2   promises_1.0.1   
[34] htmltools_0.3.6   assertthat_0.2.0  mime_0.5         
[37] xtable_1.8-2      httpuv_1.4.4.1    stringi_1.2.3    
[40] R.oo_1.22.0      </code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.0.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
