<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of inflammatory bowel disease</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; position: absolute; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine:empty
  { position: absolute; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of inflammatory bowel disease</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-06-27</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/1d289622f08e5d8b484f7a48fd92cc9a597952d3" target="_blank">1d28962</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Unstaged changes:
    Modified:   analysis/index.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/1d289622f08e5d8b484f7a48fd92cc9a597952d3/analysis/ibd_2015.Rmd" target="_blank">1d28962</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
wflow_publish(“ibd_2015.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/26192919">“Association Analyses Identify 38 Susceptibility Loci for Inflammatory Bowel Disease and Highlight Shared Genetic Risk Across Populations” (<em>Nature Genetics</em>, 2015)</a>. The summary data file is available at <a href="https://www.ibdgenetics.org/downloads.html" class="uri">https://www.ibdgenetics.org/downloads.html</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["Inflammatory bowel disease (IBD)<br/>(kegg, BioSystems)","IL23-mediated signaling events<br/>(pid, PC)","Cytokine-cytokine receptor interaction<br/>(kegg, BioSystems)","IL27-mediated signaling events<br/>(pid, PC)","Jak-STAT signaling pathway<br/>(kegg, BioSystems)","IL12-mediated signaling events<br/>(pid, PC)","IL12-mediated signaling events<br/>(pid, BioSystems)","Toxoplasmosis<br/>(kegg, BioSystems)","NO2-dependent IL 12 Pathway in NK cells<br/>(biocarta, BioCarta)","Cytokine Signaling in Immune system<br/>(reactome, PC)","Th1/Th2 Differentiation<br/>(biocarta, BioCarta)","IL12 and Stat4 Dependent Signaling Pathway in Th1 Development<br/>(biocarta, BioCarta)","Selective expression of chemokine receptors during T-cell polarization<br/>(biocarta, BioCarta)","HTLV-I infection<br/>(kegg, BioSystems)","Leishmaniasis<br/>(kegg, BioSystems)","Allograft rejection<br/>(kegg, BioSystems)","Tuberculosis<br/>(kegg, BioSystems)","Interferon gamma signaling<br/>(reactome, PC)","Interferon gamma signaling<br/>(reactome, BioSystems)","Allograft Rejection<br/>(wiki, BioSystems)","Signaling by Interleukins<br/>(reactome, PC)","Cytokines and Inflammatory Response<br/>(biocarta, BioCarta)","Interferon Signaling<br/>(reactome, BioSystems)","PI3K-Akt signaling pathway<br/>(kegg, BioSystems)","Measles<br/>(kegg, BioSystems)","Interferon Signaling<br/>(reactome, PC)","Osteoclast differentiation<br/>(kegg, BioSystems)","IL6-mediated signaling events<br/>(pid, BioSystems)","IFN-gamma pathway<br/>(pid, PC)","Herpes simplex infection<br/>(kegg, BioSystems)","Signaling events mediated by PTP1B<br/>(pid, PC)","IL-3 Signaling Pathway<br/>(wiki, BioSystems)","B Cell Receptor Signaling Pathway<br/>(wiki, BioSystems)","Type II interferon signaling (IFNG)<br/>(wiki, BioSystems)","Interleukin signaling pathway<br/>(panther, PANTHER)","Interleukin signaling pathway<br/>(panther, PC)","IL6-mediated signaling events<br/>(pid, PC)","IFN-gamma pathway<br/>(pid, BioSystems)","TNF signaling pathway<br/>(kegg, BioSystems)","IL22 Soluble Receptor Signaling Pathway <br/>(biocarta, BioCarta)","Cytokine Network<br/>(biocarta, BioCarta)","GMCSF-mediated signaling events<br/>(pid, PC)","IL-6 Signaling Pathway<br/>(wiki, BioSystems)","IL12 signaling mediated by STAT4<br/>(pid, BioSystems)","Prolactin Signaling Pathway<br/>(wiki, BioSystems)","Influenza A<br/>(kegg, BioSystems)","Signaling events mediated by PTP1B<br/>(pid, BioSystems)","activated TAK1 mediates p38 MAPK activation<br/>(reactome, PC)","Leptin signaling pathway<br/>(wiki, BioSystems)","Pathways in cancer<br/>(kegg, BioSystems)","Cytokines and Inflammatory Response<br/>(wiki, BioSystems)","Signaling events mediated by TCPTP<br/>(pid, PC)","IL4-mediated signaling events<br/>(pid, PC)","Interleukin-6 signaling<br/>(reactome, PC)","IL12 signaling mediated by STAT4<br/>(pid, PC)","Dendritic cells in regulating TH1 and TH2 Development<br/>(biocarta, BioCarta)","Calcium signaling in the CD4+ TCR pathway<br/>(pid, PC)","Regulation of IFNG signaling<br/>(reactome, PC)","TRAF6 Mediated Induction of proinflammatory cytokines<br/>(reactome, PC)","Epstein-Barr virus infection<br/>(kegg, BioSystems)","MAP kinase activation in TLR cascade<br/>(reactome, PC)","IL4-mediated signaling events<br/>(pid, BioSystems)","JAK-STAT signaling<br/>(kegg, BioSystems)","Amoebiasis<br/>(kegg, BioSystems)","IL2-mediated signaling events<br/>(pid, BioSystems)","Intestinal immune network for IgA production<br/>(kegg, BioSystems)","Rheumatoid arthritis<br/>(kegg, BioSystems)","Cellular responses to stress<br/>(reactome, BioSystems)","Calcineurin-regulated NFAT-dependent transcription in lymphocytes<br/>(pid, PC)","IL2-mediated signaling events<br/>(pid, PC)","IL 18 Signaling Pathway<br/>(biocarta, BioCarta)","Interferon alpha/beta signaling<br/>(reactome, PC)","Natural killer cell mediated cytotoxicity<br/>(kegg, BioSystems)","Platelet activation, signaling and aggregation<br/>(reactome, BioSystems)","IL-5 Signaling Pathway<br/>(wiki, BioSystems)","MyD88 cascade initiated on plasma membrane<br/>(reactome, PC)","IL-7 Signaling Pathway<br/>(wiki, BioSystems)","TNF signaling<br/>(reactome, BioSystems)","TRAF6 mediated induction of NFkB and MAP kinases upon TLR7/8 or 9 activation<br/>(reactome, PC)","Adipocytokine signaling pathway<br/>(kegg, BioSystems)","IL3-mediated signaling events<br/>(pid, PC)","Glucocorticoid receptor regulatory network<br/>(pid, PC)","AP-1 transcription factor network<br/>(pid, PC)","Calcineurin-regulated NFAT-dependent transcription in lymphocytes<br/>(pid, BioSystems)","SHP2 signaling<br/>(pid, PC)","Type I diabetes mellitus<br/>(kegg, BioSystems)","African trypanosomiasis<br/>(kegg, BioSystems)","Chagas disease (American trypanosomiasis)<br/>(kegg, BioSystems)","Systemic lupus erythematosus<br/>(kegg, BioSystems)","Inflammation mediated by chemokine and cytokine signaling pathway<br/>(panther, PC)","HIF-1 signaling pathway<br/>(kegg, BioSystems)","Malaria<br/>(kegg, BioSystems)","T cell receptor signaling pathway<br/>(kegg, BioSystems)","Senescence and Autophagy<br/>(wiki, BioSystems)","Inflammation mediated by chemokine and cytokine signaling pathway<br/>(panther, PANTHER)","TGF Beta Signaling Pathway<br/>(wiki, BioSystems)","Selenium Pathway<br/>(wiki, BioSystems)","Downstream signaling in naive CD8+ T cells<br/>(pid, BioSystems)","Downstream signaling in naïve CD8+ T cells<br/>(pid, PC)","Spinal Cord Injury<br/>(wiki, BioSystems)"],[61,37,253,26,148,59,60,111,16,272,18,20,27,250,68,34,171,67,89,35,103,28,192,335,127,166,127,42,43,174,52,98,153,49,77,81,46,40,108,16,20,36,93,28,73,165,50,14,58,385,25,40,58,11,30,21,28,14,70,195,51,56,19,105,51,44,84,366,46,53,6,65,131,217,67,80,43,15,79,67,26,82,69,45,57,41,34,102,126,187,98,48,101,104,166,53,85,48,64,111],[4624,2990,16657,2005,10368,4463,4463,9404,1355,21356,1591,1880,1958,19760,4551,1521,12238,4550,6319,1477,10099,1729,13352,33422,9651,11279,11165,3593,4323,11795,4887,9327,16395,3413,6605,6845,4123,4025,10070,1274,1609,3307,9092,2268,6749,11692,4682,1427,5209,41009,2401,3624,5111,979,2499,1617,2614,1264,6590,14954,4985,4879,1398,11934,5295,2684,6153,26436,4306,5454,517,3366,9085,22897,6341,7553,4064,1008,7477,6437,2179,7287,6457,4222,6558,2559,4211,9722,5132,17464,10079,3929,9512,9324,16683,5544,7560,4807,5243,12304],[24.2422568941743,23.4902417928431,21.3120868752556,19.5348056718298,16.4446303660897,15.1985205860332,15.1985205860332,15.1598556111399,13.8392719330199,13.6740720997039,13.1776603057459,13.1766666035508,12.6441682699143,11.5874583804453,11.2354782626792,11.204820969025,10.9693219118259,10.380799552313,9.90571267107472,9.37527839499308,9.35321600917466,9.04358696369985,9.01339260961562,8.80407995566726,8.75089940014262,8.44016373998846,8.31208827477884,8.24262721722041,8.17854763352179,8.16439006743847,8.08828586503418,7.96765118723216,7.91027267971258,7.84662144164894,7.82309379519037,7.3407864443523,7.3011942354716,7.25326941801313,7.16930724391574,7.10804546896779,7.04943969966499,7.02734555435842,7.00044798893788,6.99898330178394,6.98320570312506,6.98079119383832,6.87837717639872,6.83457309289952,6.71987344468117,6.62954077337099,6.5557140591719,6.54458753001757,6.51080008664113,6.46383317281764,6.43147956253385,6.39706788126815,6.29832550423344,6.24482004463066,6.16866668851741,6.15040388000477,6.11195865036554,6.01417665187259,5.99954345347653,5.97927002356703,5.9595381609589,5.90841924991345,5.90818095872324,5.88528214211352,5.88356944826078,5.75904987599629,5.75202408703798,5.74693068168968,5.74491596390391,5.70224437996231,5.6696437396016,5.66699979481478,5.64857051474297,5.59308766134045,5.51550444880116,5.5090659641391,5.34863747143428,5.03128445199511,4.99287131734552,4.84691434447668,4.45950859612239,4.27314675687789,3.7769240026353,3.77263746761877,3.41284441623809,3.33832017368638,3.26633769665912,3.22739747561461,3.11744227173692,2.92606455225809,2.86329428973351,2.8360994808173,2.52652665989754,2.29106053631088,2.03804482158001,1.90031940798646],["-2.9292<br/>[-2.9404, -2.8644]","-2.9281<br/>[-2.9391, -2.8635]","-2.9314<br/>[-2.9425, -2.8663]","-2.9269<br/>[-2.9380, -2.8625]","-2.9286<br/>[-2.9397, -2.8640]","-2.9272<br/>[-2.9383, -2.8627]","-2.9272<br/>[-2.9383, -2.8627]","-2.9284<br/>[-2.9395, -2.8634]","-2.9256<br/>[-2.9366, -2.8613]","-2.9292<br/>[-2.9403, -2.8641]","-2.9252<br/>[-2.9362, -2.8609]","-2.9260<br/>[-2.9370, -2.8616]","-2.9251<br/>[-2.9361, -2.8608]","-2.9271<br/>[-2.9382, -2.8621]","-2.9261<br/>[-2.9371, -2.8613]","-2.9246<br/>[-2.9356, -2.8599]","-2.9284<br/>[-2.9394, -2.8633]","-2.9257<br/>[-2.9367, -2.8609]","-2.9259<br/>[-2.9369, -2.8611]","-2.9245<br/>[-2.9355, -2.8598]","-2.9266<br/>[-2.9376, -2.8617]","-2.9241<br/>[-2.9351, -2.8595]","-2.9263<br/>[-2.9374, -2.8615]","-2.9271<br/>[-2.9381, -2.8622]","-2.9271<br/>[-2.9382, -2.8622]","-2.9259<br/>[-2.9369, -2.8611]","-2.9264<br/>[-2.9375, -2.8615]","-2.9244<br/>[-2.9354, -2.8598]","-2.9253<br/>[-2.9363, -2.8606]","-2.9266<br/>[-2.9376, -2.8618]","-2.9259<br/>[-2.9369, -2.8611]","-2.9256<br/>[-2.9366, -2.8608]","-2.9278<br/>[-2.9389, -2.8626]","-2.9247<br/>[-2.9357, -2.8600]","-2.9256<br/>[-2.9366, -2.8612]","-2.9256<br/>[-2.9367, -2.8612]","-2.9244<br/>[-2.9354, -2.8598]","-2.9249<br/>[-2.9359, -2.8602]","-2.9251<br/>[-2.9362, -2.8603]","-2.9241<br/>[-2.9351, -2.8595]","-2.9239<br/>[-2.9350, -2.8593]","-2.9243<br/>[-2.9353, -2.8597]","-2.9260<br/>[-2.9371, -2.8612]","-2.9240<br/>[-2.9350, -2.8593]","-2.9250<br/>[-2.9360, -2.8603]","-2.9269<br/>[-2.9379, -2.8620]","-2.9254<br/>[-2.9365, -2.8607]","-2.9236<br/>[-2.9346, -2.8589]","-2.9250<br/>[-2.9360, -2.8603]","-2.9288<br/>[-2.9400, -2.8638]","-2.9240<br/>[-2.9351, -2.8594]","-2.9248<br/>[-2.9359, -2.8601]","-2.9250<br/>[-2.9360, -2.8603]","-2.9240<br/>[-2.9349, -2.8594]","-2.9240<br/>[-2.9350, -2.8593]","-2.9242<br/>[-2.9352, -2.8596]","-2.9233<br/>[-2.9343, -2.8587]","-2.9241<br/>[-2.9351, -2.8595]","-2.9250<br/>[-2.9360, -2.8603]","-2.9262<br/>[-2.9373, -2.8613]","-2.9248<br/>[-2.9358, -2.8602]","-2.9246<br/>[-2.9356, -2.8600]","-2.9243<br/>[-2.9352, -2.8596]","-2.9247<br/>[-2.9358, -2.8600]","-2.9254<br/>[-2.9365, -2.8606]","-2.9236<br/>[-2.9346, -2.8590]","-2.9243<br/>[-2.9353, -2.8596]","-2.9266<br/>[-2.9377, -2.8616]","-2.9239<br/>[-2.9349, -2.8592]","-2.9254<br/>[-2.9365, -2.8605]","-2.9233<br/>[-2.9344, -2.8588]","-2.9239<br/>[-2.9349, -2.8592]","-2.9246<br/>[-2.9357, -2.8599]","-2.9263<br/>[-2.9373, -2.8615]","-2.9252<br/>[-2.9362, -2.8605]","-2.9250<br/>[-2.9361, -2.8603]","-2.9241<br/>[-2.9351, -2.8594]","-2.9233<br/>[-2.9343, -2.8587]","-2.9250<br/>[-2.9360, -2.8603]","-2.9240<br/>[-2.9350, -2.8594]","-2.9233<br/>[-2.9343, -2.8588]","-2.9247<br/>[-2.9357, -2.8599]","-2.9244<br/>[-2.9355, -2.8597]","-2.9239<br/>[-2.9349, -2.8592]","-2.9248<br/>[-2.9358, -2.8601]","-2.9240<br/>[-2.9350, -2.8594]","-2.9246<br/>[-2.9357, -2.8599]","-2.9250<br/>[-2.9360, -2.8602]","-2.9242<br/>[-2.9352, -2.8595]","-2.9260<br/>[-2.9370, -2.8611]","-2.9246<br/>[-2.9356, -2.8599]","-2.9239<br/>[-2.9350, -2.8593]","-2.9244<br/>[-2.9354, -2.8597]","-2.9241<br/>[-2.9352, -2.8595]","-2.9259<br/>[-2.9370, -2.8611]","-2.9242<br/>[-2.9353, -2.8596]","-2.9244<br/>[-2.9354, -2.8596]","-2.9240<br/>[-2.9350, -2.8593]","-2.9240<br/>[-2.9350, -2.8593]","-2.9241<br/>[-2.9351, -2.8594]"],["-1.6401<br/>[-1.7300, -1.5344]","-1.6034<br/>[-1.7004, -1.5311]","-1.9825<br/>[-2.0910, -1.9060]","-1.4785<br/>[-1.5875, -1.3993]","-1.9684<br/>[-2.0690, -1.8796]","-1.7902<br/>[-1.9020, -1.6904]","-1.7902<br/>[-1.9020, -1.6904]","-1.9260<br/>[-2.0452, -1.8448]","-1.4697<br/>[-1.5859, -1.3796]","-2.1302<br/>[-2.2445, -2.0454]","-1.5115<br/>[-1.6412, -1.4277]","-1.5699<br/>[-1.7002, -1.4819]","-1.5597<br/>[-1.6933, -1.4652]","-2.2165<br/>[-2.3213, -2.1532]","-1.8319<br/>[-1.9786, -1.7350]","-1.5010<br/>[-1.6475, -1.4064]","-2.0790<br/>[-2.1765, -1.9814]","-1.8619<br/>[-1.9981, -1.7598]","-1.9554<br/>[-2.0859, -1.8553]","-1.5362<br/>[-1.6826, -1.4307]","-2.0506<br/>[-2.1725, -1.9508]","-1.6105<br/>[-1.7926, -1.5099]","-2.1432<br/>[-2.2370, -2.0466]","-2.3801<br/>[-2.4597, -2.3077]","-2.0712<br/>[-2.1880, -1.9652]","-2.1208<br/>[-2.2305, -2.0265]","-2.1016<br/>[-2.2371, -2.0036]","-1.8511<br/>[-2.0108, -1.7421]","-1.9310<br/>[-2.0799, -1.8242]","-2.1203<br/>[-2.2726, -2.0351]","-1.9421<br/>[-2.0926, -1.8355]","-2.1917<br/>[-2.3239, -2.1229]","-2.2028<br/>[-2.3203, -2.1004]","-1.8619<br/>[-2.0207, -1.7469]","-2.0167<br/>[-2.1538, -1.9113]","-2.0251<br/>[-2.1813, -1.9274]","-1.9169<br/>[-2.0745, -1.8021]","-1.9443<br/>[-2.0899, -1.8289]","-2.2164<br/>[-2.3584, -2.1584]","-1.5783<br/>[-1.7563, -1.4508]","-1.6441<br/>[-1.8473, -1.5372]","-2.0213<br/>[-2.1869, -1.8392]","-2.1094<br/>[-2.2388, -2.0019]","-1.8072<br/>[-1.9846, -1.6832]","-2.0707<br/>[-2.2071, -1.9656]","-2.1596<br/>[-2.2932, -2.0762]","-1.9702<br/>[-2.1272, -1.8580]","-1.5857<br/>[-1.6601, -1.5068]","-2.0062<br/>[-2.1524, -1.8782]","-2.3682<br/>[-2.5249, -2.2789]","-1.8428<br/>[-2.0144, -1.7279]","-1.8644<br/>[-2.0291, -1.7462]","-1.9875<br/>[-2.1352, -1.8629]","-1.5768<br/>[-1.7504, -1.4413]","-1.8564<br/>[-2.0202, -1.7243]","-1.6810<br/>[-1.9145, -1.5556]","-1.8737<br/>[-2.1762, -1.7371]","-1.6514<br/>[-1.8282, -1.5128]","-2.0338<br/>[-2.1783, -1.9184]","-2.2141<br/>[-2.3417, -2.1032]","-1.9664<br/>[-2.1273, -1.8389]","-1.9869<br/>[-2.1488, -1.8624]","-1.6872<br/>[-1.8668, -1.5494]","-2.3046<br/>[-2.4111, -2.2137]","-2.0374<br/>[-2.1885, -1.9220]","-1.8197<br/>[-1.9905, -1.6925]","-2.1986<br/>[-2.3821, -2.1030]","-2.3431<br/>[-2.4752, -2.2423]","-1.9724<br/>[-2.1502, -1.8504]","-2.0523<br/>[-2.2128, -1.9402]","-1.3944<br/>[-1.5504, -1.2202]","-1.9330<br/>[-2.1228, -1.8074]","-2.2712<br/>[-2.3888, -2.1922]","-2.2974<br/>[-2.4357, -2.2024]","-2.0641<br/>[-2.2163, -1.9418]","-2.0753<br/>[-2.2215, -1.9569]","-1.9898<br/>[-2.1522, -1.8699]","-1.6594<br/>[-1.9442, -1.5191]","-2.0815<br/>[-2.2473, -1.9731]","-2.2062<br/>[-2.3860, -2.0915]","-2.0362<br/>[-2.2426, -1.8323]","-2.1149<br/>[-2.2802, -1.9938]","-2.0711<br/>[-2.2508, -1.9537]","-2.0147<br/>[-2.1913, -1.8859]","-2.1192<br/>[-2.3006, -1.9965]","-1.9147<br/>[-2.1098, -1.7692]","-2.0929<br/>[-2.2606, -1.9596]","-2.2023<br/>[-2.3481, -2.0852]","-2.1345<br/>[-2.3341, -2.0044]","-2.3842<br/>[-2.5315, -2.2712]","-2.2586<br/>[-2.3954, -2.1784]","-2.0721<br/>[-2.2901, -1.9275]","-2.2760<br/>[-2.4499, -2.1537]","-2.2669<br/>[-2.4481, -2.1341]","-2.3966<br/>[-2.5368, -2.2760]","-2.1586<br/>[-2.3677, -2.0126]","-2.2581<br/>[-2.4539, -2.1162]","-2.1739<br/>[-2.3952, -2.0253]","-2.2116<br/>[-2.4357, -2.0639]","-2.3698<br/>[-2.5355, -2.2337]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Testis","Muscle - Skeletal","Brain - Cerebellum","Colon - Transverse","Brain - Cerebellar Hemisphere","Whole Blood","Heart - Atrial Appendage","Artery - Coronary","Pituitary","Adrenal Gland","Stomach","Ovary","Brain - Anterior cingulate cortex (BA24)","Thyroid","Small Intestine - Terminal Ileum","Adipose - Subcutaneous","Adipose - Visceral (Omentum)","Lung","Artery - Tibial","Uterus","Vagina","Esophagus - Muscularis","Brain - Cortex","Breast - Mammary Tissue","Heart - Left Ventricle","Esophagus - Gastroesophageal Junction","Colon - Sigmoid","Prostate","Spleen","Artery - Aorta","Brain - Caudate (basal ganglia)","Nerve - Tibial","Pancreas","Brain - Nucleus accumbens (basal ganglia)","Brain - Frontal Cortex (BA9)","Brain - Hypothalamus","Cells - EBV-transformed lymphocytes","Cells - Transformed fibroblasts","Brain - Hippocampus","Brain - Putamen (basal ganglia)","Skin - Not Sun Exposed (Suprapubic)","Skin - Sun Exposed (Lower leg)","Esophagus - Mucosa","Liver"],[5914,5787,5576,5424,5566,5789,5724,6244,5922,5667,5381,5945,5817,6282,4763,5652,5822,4957,6120,5780,5883,6067,5716,5657,5709,5940,6038,5676,3982,6453,6448,5708,5218,6131,6112,5834,5155,6195,5701,6539,5890,5898,5724,6420],[1.18052158782487,0.711362097559565,0.70915881920922,0.706576183791207,0.650864831036821,0.606956713015112,0.597214363600186,0.522168834179902,0.447700608521483,0.339615262854263,0.332014790682624,0.212318875127591,0.173568338252865,0.155595566257988,0.12145418359094,0.0889483601119853,0.0640817006961324,-0.0238470937800449,-0.0681372099715859,-0.069570230241043,-0.10155498553228,-0.102220269424597,-0.119759652919415,-0.123593953978144,-0.133289037685148,-0.153247886690955,-0.17947824595279,-0.234648212584865,-0.294452751301117,-0.313907269758572,-0.323897028265223,-0.361815963390444,-0.451003014977085,-0.451973275332124,-0.462123450784564,-0.582717440627794,-0.585057094963027,-0.593870154389815,-0.642470705040416,-0.661122739146788,-0.713612136005357,-0.789465820554376,-0.79566177994766,-1.1435371236374],["-2.9235<br/>[-2.9345, -2.8589]","-2.9226<br/>[-2.9336, -2.8582]","-2.9229<br/>[-2.9339, -2.8585]","-2.9233<br/>[-2.9343, -2.8586]","-2.9230<br/>[-2.9340, -2.8585]","-2.9234<br/>[-2.9344, -2.8588]","-2.9222<br/>[-2.9332, -2.8578]","-2.9226<br/>[-2.9336, -2.8581]","-2.9229<br/>[-2.9339, -2.8584]","-2.9226<br/>[-2.9336, -2.8581]","-2.9231<br/>[-2.9341, -2.8585]","-2.9226<br/>[-2.9336, -2.8581]","-2.9230<br/>[-2.9340, -2.8585]","-2.9229<br/>[-2.9339, -2.8584]","-2.9223<br/>[-2.9333, -2.8578]","-2.9226<br/>[-2.9336, -2.8581]","-2.9226<br/>[-2.9336, -2.8581]","-2.9227<br/>[-2.9337, -2.8582]","-2.9228<br/>[-2.9338, -2.8582]","-2.9229<br/>[-2.9339, -2.8583]","-2.9227<br/>[-2.9337, -2.8582]","-2.9227<br/>[-2.9337, -2.8582]","-2.9228<br/>[-2.9338, -2.8582]","-2.9224<br/>[-2.9334, -2.8579]","-2.9223<br/>[-2.9332, -2.8579]","-2.9227<br/>[-2.9337, -2.8581]","-2.9226<br/>[-2.9336, -2.8581]","-2.9227<br/>[-2.9337, -2.8582]","-2.9226<br/>[-2.9336, -2.8581]","-2.9228<br/>[-2.9338, -2.8583]","-2.9226<br/>[-2.9336, -2.8581]","-2.9224<br/>[-2.9334, -2.8579]","-2.9225<br/>[-2.9335, -2.8580]","-2.9225<br/>[-2.9335, -2.8580]","-2.9226<br/>[-2.9336, -2.8581]","-2.9226<br/>[-2.9336, -2.8581]","-2.9225<br/>[-2.9335, -2.8580]","-2.9224<br/>[-2.9334, -2.8580]","-2.9224<br/>[-2.9334, -2.8579]","-2.9222<br/>[-2.9332, -2.8578]","-2.9224<br/>[-2.9333, -2.8579]","-2.9223<br/>[-2.9333, -2.8578]","-2.9222<br/>[-2.9332, -2.8577]","-2.9217<br/>[-2.9327, -2.8573]"],["-2.3013<br/>[-2.5700, -2.1483]","-2.3326<br/>[-2.5967, -2.1510]","-2.3242<br/>[-2.6327, -2.1495]","-2.3394<br/>[-2.6548, -2.1777]","-2.3392<br/>[-2.6140, -2.1545]","-2.3366<br/>[-2.6255, -2.1504]","-2.3708<br/>[-2.6469, -2.2067]","-2.3816<br/>[-2.6612, -2.2092]","-2.3711<br/>[-2.6521, -2.1818]","-2.3819<br/>[-2.7158, -2.2088]","-2.3597<br/>[-2.7197, -2.1782]","-2.3991<br/>[-2.7155, -2.2090]","-2.4158<br/>[-2.7441, -2.2375]","-2.4181<br/>[-2.7349, -2.2353]","-2.4230<br/>[-2.6981, -2.2438]","-2.4217<br/>[-2.7505, -2.2390]","-2.4221<br/>[-2.7594, -2.2368]","-2.4313<br/>[-2.7500, -2.2417]","-2.4579<br/>[-2.7708, -2.2681]","-2.4415<br/>[-2.7572, -2.2370]","-2.4628<br/>[-2.7702, -2.2702]","-2.4651<br/>[-2.7533, -2.2684]","-2.4675<br/>[-2.7531, -2.2701]","-2.4557<br/>[-2.7554, -2.2435]","-2.4776<br/>[-2.7456, -2.2779]","-2.4751<br/>[-2.7560, -2.2726]","-2.4824<br/>[-2.7588, -2.2739]","-2.4779<br/>[-2.7911, -2.2685]","-2.4626<br/>[-2.7798, -2.2452]","-2.5139<br/>[-2.7819, -2.3009]","-2.5231<br/>[-2.7994, -2.3312]","-2.5120<br/>[-2.8017, -2.3010]","-2.5321<br/>[-2.8129, -2.3081]","-2.5583<br/>[-2.8088, -2.3665]","-2.5538<br/>[-2.8592, -2.3602]","-2.5810<br/>[-2.8346, -2.3676]","-2.5771<br/>[-2.8319, -2.3673]","-2.5813<br/>[-2.8415, -2.3634]","-2.5930<br/>[-2.8512, -2.3684]","-2.6106<br/>[-2.8503, -2.3966]","-2.6248<br/>[-2.8560, -2.4025]","-2.6570<br/>[-2.8677, -2.4389]","-2.6650<br/>[-2.8845, -2.4683]","-2.8201<br/>[-2.9711, -2.6171]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Testis","Lung","Spleen","Whole_Blood","Colon_Transverse","Small_Intestine_Terminal_Ileum","Kidney_Cortex","Minor_Salivary_Gland","Brain_Frontal_Cortex_(BA9)","Cells_EBV-transformed_lymphocytes","Colon_Sigmoid","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Hippocampus","Brain_Caudate_(basal_ganglia)","Adipose_Visceral_(Omentum)","Brain_Cortex","Muscle_Skeletal","Brain_Putamen_(basal_ganglia)","Brain_Amygdala","Stomach","Prostate","Ovary","Pancreas","Nerve_Tibial","Esophagus_Mucosa","Artery_Coronary","Esophagus_Gastroesophageal_Junction","Uterus","Brain_Nucleus_accumbens_(basal_ganglia)","Brain_Cerebellum","Vagina","Liver","Pituitary","Brain_Hypothalamus","Esophagus_Muscularis","Artery_Tibial","Cells_Transformed_fibroblasts","Heart_Left_Ventricle","Thyroid","Breast_Mammary_Tissue","Brain_Cerebellar_Hemisphere","Adrenal_Gland","Heart_Atrial_Appendage","Adipose_Subcutaneous","Brain_Substantia_nigra","Artery_Aorta","Brain_Spinal_cord_(cervical_c-1)","Skin_Not_Sun_Exposed_(Suprapubic)","Skin_Sun_Exposed_(Lower_leg)"],[8203,8226,7064,6871,7769,8777,9091,8401,13446,7719,10741,12874,10895,11730,9461,11837,8727,11780,11131,9329,9290,11491,7764,12414,6947,12245,14000,12172,11304,10727,6860,8480,9470,12054,14176,12633,10939,11062,12157,9787,13394,12535,11078,9892,12561,14270,12465,6870,6778],[3.22234663701264,2.30107346758643,1.84763547057344,1.07664920212839,0.914753032111477,0.084238485534543,-0.109660107702603,-0.226443272453151,-0.269571850757079,-0.280450346924138,-0.297371988938741,-0.396679094964794,-0.402873913669616,-0.471383746841018,-0.47437021820141,-0.55451637475088,-0.636199617839322,-0.698397032924846,-0.76762068533725,-0.767675444618065,-0.792598464976536,-0.797185731790443,-0.816330686000309,-0.863392527644905,-0.905627930970389,-0.914529799168932,-0.921776616640583,-0.940621795697514,-0.956789157684149,-0.958734608424411,-0.988093319946879,-0.994364627316457,-0.994473658352435,-1.00887907736365,-1.01206459789561,-1.01255929099626,-1.06084223222589,-1.08397006552489,-1.10347795260857,-1.11540966678585,-1.13969414768336,-1.14512945333877,-1.15060683267761,-1.24374614340724,-1.29104532710121,-1.31059151413052,-1.40142700870611,-2.15636438763395,-2.22890173239518],["-2.9238<br/>[-2.9349, -2.8596]","-2.9227<br/>[-2.9337, -2.8583]","-2.9235<br/>[-2.9345, -2.8589]","-2.9236<br/>[-2.9346, -2.8590]","-2.9233<br/>[-2.9344, -2.8587]","-2.9231<br/>[-2.9341, -2.8585]","-2.9232<br/>[-2.9342, -2.8586]","-2.9228<br/>[-2.9339, -2.8583]","-2.9229<br/>[-2.9339, -2.8583]","-2.9227<br/>[-2.9337, -2.8582]","-2.9234<br/>[-2.9344, -2.8588]","-2.9227<br/>[-2.9338, -2.8582]","-2.9225<br/>[-2.9335, -2.8580]","-2.9221<br/>[-2.9331, -2.8576]","-2.9228<br/>[-2.9338, -2.8583]","-2.9224<br/>[-2.9334, -2.8579]","-2.9221<br/>[-2.9331, -2.8576]","-2.9218<br/>[-2.9328, -2.8574]","-2.9222<br/>[-2.9332, -2.8577]","-2.9225<br/>[-2.9335, -2.8580]","-2.9220<br/>[-2.9330, -2.8576]","-2.9222<br/>[-2.9332, -2.8578]","-2.9233<br/>[-2.9343, -2.8587]","-2.9220<br/>[-2.9330, -2.8577]","-2.9223<br/>[-2.9333, -2.8578]","-2.9218<br/>[-2.9328, -2.8574]","-2.9228<br/>[-2.9338, -2.8582]","-2.9225<br/>[-2.9335, -2.8580]","-2.9221<br/>[-2.9330, -2.8575]","-2.9219<br/>[-2.9329, -2.8575]","-2.9222<br/>[-2.9332, -2.8577]","-2.9220<br/>[-2.9330, -2.8576]","-2.9223<br/>[-2.9333, -2.8578]","-2.9218<br/>[-2.9328, -2.8574]","-2.9225<br/>[-2.9335, -2.8580]","-2.9221<br/>[-2.9331, -2.8577]","-2.9223<br/>[-2.9333, -2.8578]","-2.9218<br/>[-2.9328, -2.8574]","-2.9223<br/>[-2.9333, -2.8578]","-2.9220<br/>[-2.9330, -2.8576]","-2.9217<br/>[-2.9327, -2.8573]","-2.9222<br/>[-2.9332, -2.8577]","-2.9217<br/>[-2.9327, -2.8573]","-2.9215<br/>[-2.9325, -2.8571]","-2.9217<br/>[-2.9327, -2.8573]","-2.9217<br/>[-2.9327, -2.8573]","-2.9213<br/>[-2.9323, -2.8570]","-2.9223<br/>[-2.9333, -2.8578]","-2.9222<br/>[-2.9332, -2.8578]"],["-2.2355<br/>[-2.3743, -2.1170]","-2.3974<br/>[-2.5096, -2.2515]","-2.2799<br/>[-2.4717, -2.1242]","-2.3314<br/>[-2.5890, -2.1752]","-2.3866<br/>[-2.6186, -2.2370]","-2.4872<br/>[-2.7543, -2.3231]","-2.5266<br/>[-2.7927, -2.3543]","-2.5249<br/>[-2.8002, -2.3503]","-2.5999<br/>[-2.8124, -2.4633]","-2.5171<br/>[-2.8087, -2.3236]","-2.5719<br/>[-2.8111, -2.4103]","-2.6204<br/>[-2.8179, -2.4747]","-2.6029<br/>[-2.8177, -2.4439]","-2.6268<br/>[-2.8240, -2.4767]","-2.5903<br/>[-2.8284, -2.4140]","-2.6407<br/>[-2.8323, -2.4790]","-2.6352<br/>[-2.8369, -2.4551]","-2.6739<br/>[-2.8448, -2.5136]","-2.6893<br/>[-2.8522, -2.5401]","-2.6697<br/>[-2.8602, -2.4846]","-2.6749<br/>[-2.8652, -2.4852]","-2.6886<br/>[-2.8624, -2.5128]","-2.4957<br/>[-2.7846, -2.3213]","-2.7095<br/>[-2.8691, -2.5420]","-2.7187<br/>[-2.8768, -2.5401]","-2.7273<br/>[-2.8726, -2.5728]","-2.7260<br/>[-2.9018, -2.5676]","-2.7255<br/>[-2.8860, -2.5430]","-2.7383<br/>[-2.8829, -2.5778]","-2.7372<br/>[-2.9079, -2.5780]","-2.7408<br/>[-2.9112, -2.5376]","-2.7411<br/>[-2.9309, -2.5557]","-2.7459<br/>[-2.8956, -2.5601]","-2.7570<br/>[-2.8899, -2.6066]","-2.7499<br/>[-2.8933, -2.5742]","-2.7526<br/>[-2.8931, -2.5805]","-2.7664<br/>[-2.9075, -2.5846]","-2.7735<br/>[-2.9131, -2.5867]","-2.7837<br/>[-2.9083, -2.6169]","-2.7906<br/>[-2.9497, -2.6236]","-2.7963<br/>[-2.9112, -2.6439]","-2.8006<br/>[-2.9135, -2.6503]","-2.8003<br/>[-2.9256, -2.6222]","-2.8466<br/>[-2.9626, -2.6705]","-2.8542<br/>[-2.9596, -2.6915]","-2.8563<br/>[-2.9775, -2.7180]","-2.9110<br/>[-3.0280, -2.7745]","-2.6719<br/>[-2.8572, -2.4677]","-2.6684<br/>[-2.8549, -2.4717]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["1","17","16","13","2","12","5","10","6","9","7","11","3","19","4","14","20","15","8","18"],[7414,8155,9991,7791,8910,7193,11380,8474,8929,9771,8534,10446,10132,5807,12311,9429,8103,7752,10380,7463],[15.7497376956849,3.68821597571714,2.444971723171,2.18838337786028,1.80072244739588,1.70609745394705,1.31329447957526,1.19552693616041,1.16759441765724,0.0348480333135494,-0.122894658265834,-0.163530953700427,-0.193358166474827,-0.226076900856387,-0.290958703903941,-0.513815692104727,-0.595636909361394,-0.629090028375703,-0.641183844071228,-0.650622623140238],["-2.9241<br/>[-2.9351, -2.8594]","-2.9251<br/>[-2.9361, -2.8602]","-2.9233<br/>[-2.9343, -2.8587]","-2.9239<br/>[-2.9349, -2.8593]","-2.9244<br/>[-2.9354, -2.8597]","-2.9234<br/>[-2.9344, -2.8588]","-2.9239<br/>[-2.9349, -2.8593]","-2.9235<br/>[-2.9346, -2.8589]","-2.9232<br/>[-2.9342, -2.8586]","-2.9228<br/>[-2.9338, -2.8583]","-2.9226<br/>[-2.9336, -2.8582]","-2.9219<br/>[-2.9329, -2.8576]","-2.9225<br/>[-2.9335, -2.8580]","-2.9228<br/>[-2.9338, -2.8582]","-2.9232<br/>[-2.9342, -2.8586]","-2.9224<br/>[-2.9334, -2.8579]","-2.9219<br/>[-2.9329, -2.8575]","-2.9223<br/>[-2.9333, -2.8578]","-2.9221<br/>[-2.9331, -2.8577]","-2.9219<br/>[-2.9329, -2.8575]"],["-2.1037<br/>[-2.1335, -2.0548]","-2.2160<br/>[-2.3904, -2.0814]","-2.2921<br/>[-2.4995, -2.1525]","-2.2867<br/>[-2.4936, -2.1415]","-2.2820<br/>[-2.4698, -2.1522]","-2.2745<br/>[-2.5055, -2.1042]","-2.4146<br/>[-2.6113, -2.2583]","-2.3594<br/>[-2.6287, -2.1821]","-2.3767<br/>[-2.6314, -2.2169]","-2.5502<br/>[-2.8048, -2.3782]","-2.5777<br/>[-2.8069, -2.4041]","-2.5915<br/>[-2.8229, -2.4162]","-2.6197<br/>[-2.8123, -2.4838]","-2.5658<br/>[-2.8129, -2.3624]","-2.5144<br/>[-2.7934, -2.3613]","-2.6780<br/>[-2.8545, -2.4983]","-2.6930<br/>[-2.8748, -2.4956]","-2.7064<br/>[-2.8785, -2.5113]","-2.7142<br/>[-2.8780, -2.5305]","-2.7068<br/>[-2.8941, -2.4994]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">sessionInfo</span>()</a></code></pre></div>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.5

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] DT_0.4         plyr_1.8.4     dplyr_0.7.5    R.matlab_3.6.1

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.17      compiler_3.5.0    pillar_1.2.3     
 [4] later_0.7.3       git2r_0.21.0      workflowr_1.0.1  
 [7] bindr_0.1.1       R.methodsS3_1.7.1 R.utils_2.6.0    
[10] tools_3.5.0       digest_0.6.15     jsonlite_1.5     
[13] evaluate_0.10.1   tibble_1.4.2      pkgconfig_2.0.1  
[16] rlang_0.2.1       shiny_1.1.0       crosstalk_1.0.0  
[19] yaml_2.1.19       bindrcpp_0.2.2    stringr_1.3.1    
[22] knitr_1.20        htmlwidgets_1.2   rprojroot_1.3-2  
[25] tidyselect_0.2.4  glue_1.2.0        R6_2.2.2         
[28] rmarkdown_1.10    purrr_0.2.5       magrittr_1.5     
[31] whisker_0.3-2     backports_1.1.2   promises_1.0.1   
[34] htmltools_0.3.6   assertthat_0.2.0  mime_0.5         
[37] xtable_1.8-2      httpuv_1.4.4.1    stringi_1.2.3    
[40] R.oo_1.22.0      </code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.0.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
