<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Full results of Zhu and Stephens (2017)</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Full results of Zhu and Stephens (2017)</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-10-06</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/956af9a7cec1ea4eaf65bab8d4badb5d3c9eb92c" target="_blank">956af9a</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/956af9a7cec1ea4eaf65bab8d4badb5d3c9eb92c/analysis/index.Rmd" target="_blank">956af9a</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-06
</td>
<td style="text-align:left;">
wflow_publish(“analysis/index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/da04234e1c4f81aaedc0343923f6e95311f8d8a5/docs/index.html" target="_blank">da04234</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-06
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/6b08034e177ee02cc6f64c718d3f846bd6cae2bb/analysis/index.Rmd" target="_blank">6b08034</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-06
</td>
<td style="text-align:left;">
wflow_publish(“analysis/index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/d1b708d41915f9f71f66f05a7944a55902538a8d/docs/index.html" target="_blank">d1b708d</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-06
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/d63db43f50ed992b278049bb6ba174bdf4bdcd54/analysis/index.Rmd" target="_blank">d63db43</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-06
</td>
<td style="text-align:left;">
wflow_publish(“analysis/index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/32eb1dddef3fdc5ea528cbf5dbc7410e7b53dd10/docs/index.html" target="_blank">32eb1dd</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-05
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/5c7ee1e666131480f28f7d1fa77607be416eae28/analysis/index.Rmd" target="_blank">5c7ee1e</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-05
</td>
<td style="text-align:left;">
wflow_publish(“analysis/index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/index.html" target="_blank">1c85967</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-05
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/810e15a892032529704da5592438cff4b8b5da9a/docs/index.html" target="_blank">810e15a</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-16
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/ebd42203e7bea43d2896a355db6a694cad4913c3/analysis/index.Rmd" target="_blank">ebd4220</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-16
</td>
<td style="text-align:left;">
wflow_publish(“analysis/index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/ecf07e13a2788bf0144d6999a355971cc7ada085/docs/index.html" target="_blank">ecf07e1</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-16
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/ec44ee063dc94abc188411676734ebc67b077323/analysis/index.Rmd" target="_blank">ec44ee0</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-09-16
</td>
<td style="text-align:left;">
wflow_publish(“analysis/index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/8f27e6da53ba7e25860b380fe8107baaf6cc10b9/docs/index.html" target="_blank">8f27e6d</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/c07c276af218dd70515756ede20af94359dd0c4f/analysis/index.Rmd" target="_blank">c07c276</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/c77f1e1e4fb23790eaf7f7c0d894503c631e9569/docs/index.html" target="_blank">c77f1e1</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/acbdfcb99d70923f5ae548e3198d8d0783984ea8/analysis/index.Rmd" target="_blank">acbdfcb</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/c1e0aacf45e2aefd15384836b4a519e7a2dd5f59/docs/index.html" target="_blank">c1e0aac</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/16a82f8d862e461c3a2e3d6312491cfac5572b73/analysis/index.Rmd" target="_blank">16a82f8</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/835a6be1710efd5de16e254f9110b7e23f46fc6a/docs/index.html" target="_blank">835a6be</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/447917ec2cdd654946d5a87236621515894f5bc7/analysis/index.Rmd" target="_blank">447917e</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/55087aeccdafb6f0eff926120889829b664ce1a7/docs/index.html" target="_blank">55087ae</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-01
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/07473ff4d2486e334e710d8b9e6af666ecd06ac6/analysis/index.Rmd" target="_blank">07473ff</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-01
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/2b357393870c1af4614638eb9b4423b8d05d05eb/docs/index.html" target="_blank">2b35739</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-30
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/e364292a90d4fd54c9c956ca2fb3770b18d42c1d/analysis/index.Rmd" target="_blank">e364292</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-30
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/de49400e509e7d1db688fe94b52307210fe208a5/docs/index.html" target="_blank">de49400</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-29
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/f80d46ea9d310f34ab2f6064e647c3a22ee8ec5b/analysis/index.Rmd" target="_blank">f80d46e</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-29
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/b30daba2887a6313b8f656cf57eee6c3086345fe/docs/index.html" target="_blank">b30daba</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/bc9746a9e817d6e677d5b256c8c759975d96229a/analysis/index.Rmd" target="_blank">bc9746a</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/c749ac73c66b895dc8f555a431e08160f17054db/docs/index.html" target="_blank">c749ac7</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/6006659014276b35f67c330a718a6349cb82f7b6/analysis/index.Rmd" target="_blank">6006659</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/0852676bda1f16f5333f8dc713f06ee17541bdaf/docs/index.html" target="_blank">0852676</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/46603f37950862f3be5fcfd49e01cbd6ad33f6d6/analysis/index.Rmd" target="_blank">46603f3</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
wflow_publish(“index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/ff6afd099209881334e83a9ee89642acaa830a53/docs/index.html" target="_blank">ff6afd0</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-26
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/bf31edf7399b4528d58a2cc7700bcb79724d604b/analysis/index.Rmd" target="_blank">bf31edf</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-26
</td>
<td style="text-align:left;">
wflow_publish(files = c(“analysis/index.Rmd”, “analysis/license.Rmd”))
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/33aa5b386ec7c6975848c7418c9da30e3cfdef51/docs/index.html" target="_blank">33aa5b3</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-26
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/02b392f1f926e36540994949f178d074e464988e/analysis/index.Rmd" target="_blank">02b392f</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-26
</td>
<td style="text-align:left;">
wflow_publish(“analysis/index.Rmd”)
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/cee1a927d6f181bac1305b4fdd1fe984707b9346/analysis/index.Rmd" target="_blank">cee1a92</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-26
</td>
<td style="text-align:left;">
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="overview" class="section level2">
<h2>Overview</h2>
<p>This is my online notebook to document and share the full results of genome-wide enrichment and prioritization analyses described in the manuscript:</p>
<blockquote>
<p>Xiang Zhu and Matthew Stephens (2017). A large-scale genome-wide enrichment analysis identifies new trait-associated genes, pathways and tissues across 31 human phenotypes. <em>bioRxiv.</em> <a href="https://doi.org/10.1101/160770">https://doi.org/10.1101/160770</a>.</p>
</blockquote>
<p>We developed a new statistical method, RSS-E, to generate the results for this study. The software that implements RSS-E is freely available at <a href="https://github.com/stephenslab/rss/tree/master/src_vb"><code>stephenslab/rss</code></a>. We also provide an end-to-end <a href="http://stephenslab.github.io/rss/Example-5">example</a> illustrating how to use RSS-E to perform the reported genome-wide enrichment and prioritization analyses of GWAS summary statistics. This software can be referenced in a journal’s “Code availability” section as <a href="xxx"></a>.</p>
<p>In addition, all 4,026 pre-processed gene sets used in this study (including 3,913 biological pathways and 113 tissue-based gene sets) are freely available at <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data"><code>xiangzhu/rss-gsea</code></a>. These gene sets can be referenced in a journal’s “Data availability” section as <a href="https://zenodo.org/badge/latestdoi/55633948"><img src="https://zenodo.org/badge/55633948.svg" alt="DOI" /></a>.</p>
<p>If you find the analysis results, the pre-processed gene sets, the statistical methods, and/or the open-source software useful for your work, <strong>please kindly cite our manuscript listed above, <a href="https://doi.org/10.1101/160770">Zhu and Stephens (2018)</a></strong>.</p>
<p>If you have any question about the notebook and/or the manuscript, please feel free to contact me: <a href="https://github.com/xiangzhu">Xiang Zhu</a>, <code>xiangzhu@uchicago.edu</code> or <code>xiangzhu@stanford.edu</code>.</p>
</div>
<div id="main-results" class="section level2">
<h2>Main results</h2>
<div id="anthropometric-phenotypes" class="section level4">
<h4>Anthropometric phenotypes</h4>
<ul>
<li><a href="height_2014.html">Adult human height (Wood et al. 2014)</a></li>
<li><a href="bmi_2015.html">Body mass index (Locke et al. 2015)</a></li>
<li><a href="waist_2015.html">Waist-to-hip ratio (Shungin et al. 2015)</a></li>
</ul>
</div>
<div id="hematopoietic-phenotypes" class="section level4">
<h4>Hematopoietic phenotypes</h4>
<ul>
<li><a href="hb_2012.html">Haemoglobin (Harst et al. 2012)</a></li>
<li><a href="mch_2012.html">Mean cell haemoglobin (Harst et al. 2012)</a></li>
<li><a href="mchc_2012.html">Mean cell haemoglobin concentration (Harst et al. 2012)</a></li>
<li><a href="mcv_2012.html">Mean cell volume (Harst et al. 2012)</a></li>
<li><a href="pcv_2012.html">Packed cell volume (Harst et al. 2012)</a></li>
<li><a href="rbc_2012.html">Red blood cell count (Harst et al. 2012)</a></li>
</ul>
</div>
<div id="immune-related-phenotypes" class="section level4">
<h4>Immune-related phenotypes</h4>
<ul>
<li><a href="ra_2014.html">Rheumatoid arthritis (Okada et al. 2014)</a></li>
<li><a href="cd_2015.html">Crohn’s disease (Liu et al. 2015)</a></li>
<li><a href="ibd_2015.html">Inflammatory bowel disease (Liu et al. 2015)</a></li>
<li><a href="uc_2015.html">Ulcerative colitis (Liu et al. 2015)</a></li>
</ul>
</div>
<div id="metabolic-phenotypes" class="section level4">
<h4>Metabolic phenotypes</h4>
<ul>
<li><a href="hdl_2010.html">High-density lipoprotein (Teslovich et al. 2010)</a></li>
<li><a href="ldl_2010.html">Low-density lipoprotein (Teslovich et al. 2010)</a></li>
<li><a href="tc_2010.html">Total cholesterol (Teslovich et al. 2010)</a></li>
<li><a href="tg_2010.html">Triglycerides (Teslovich et al. 2010)</a></li>
<li><a href="t2d_2012.html">Type 2 diabetes (Morris et al. 2012)</a></li>
<li><a href="fg_2012.html">Fasting glucose (Manning et al. 2012)</a></li>
<li><a href="fi_2012.html">Fasting insulin (Manning et al. 2012)</a></li>
<li><a href="hr_2013.html">Heart rate (Den Hoed et al. 2013)</a></li>
<li><a href="gout_2013.html">Gout (Köttgen et al. 2013)</a></li>
<li><a href="urate_2013.html">Serum urate (Köttgen et al. 2013)</a></li>
<li><a href="anm_2015.html">Age at natural menopause (Day et al. 2015)</a></li>
<li><a href="cad_2015.html">Coronary artery disease (Nikpay et al. 2015)</a></li>
<li><a href="mi_2015.html">Myocardial infarction (Nikpay et al. 2015)</a></li>
</ul>
</div>
<div id="neurological-phenotypes" class="section level4">
<h4>Neurological phenotypes</h4>
<ul>
<li><a href="load_2013.html">Alzheimer’s disease (Lambert et al. 2013)</a></li>
<li><a href="scz_2014.html">Schizophrenia (Ripke et al. 2014)</a></li>
<li><a href="als_2016.html">Amyotrophic lateral sclerosis (Rheenen et al. 2016)</a></li>
<li><a href="ds_2016.html">Depressive symptoms (Okbay et al. 2016)</a></li>
<li><a href="neu_2016.html">Neuroticism (Okbay et al. 2016)</a></li>
</ul>
</div>
<div id="gene-prioritization" class="section level4">
<h4>Gene prioritization</h4>
<ul>
<li><a href="ldl2010_path1698.html">Low-density lipoprotein (Teslovich et al. 2010) &amp; Chylomicron-mediated lipid transport</a></li>
<li><a href="height2014_path3208.html">Adult human height (Wood et al. 2014) &amp; Endochondral ossification</a></li>
<li><a href="ibd2015_path3850.html">Inflammatory bowel disease (Liu et al. 2015) &amp; Cytokine-cytokine receptor interaction</a></li>
<li><a href="ra2014_path2858.html">Rheumatoid arthritis (Okada et al. 2014) &amp; Lymphocyte NFAT-dependent transcription</a></li>
<li><a href="cad2015_path3059.html">Coronary artery disease (Nikpay et al. 2015) &amp; p75(NTR)-mediated signaling</a></li>
<li><a href="hdl2010_path3093.html">High-density lipoprotein (Teslovich et al. 2010) &amp; Lipid digestion, mobilization and transport</a></li>
</ul>
</div>
</div>
<div id="additional-resources" class="section level2">
<h2>Additional resources</h2>
<ul>
<li>How can I perform similar analyses on a new dataset of GWAS summary statistics using RSS-E?</li>
</ul>
<p>The software that generated results of this study is freely available at <a href="https://github.com/stephenslab/rss/tree/master/src_vb"><code>stephenslab/rss</code></a>. I also write a step-by-step <a href="http://stephenslab.github.io/rss/Example-5">RSS-E tutorial</a> that illustrates how to use this software to perform genome-wide enrichment and prioritization analyses on GWAS summary statistics.</p>
<ul>
<li>Compared with most existing enrichment methods, the most appealing feature of RSS-E is the automatic gene prioritization in light of inferred enrichments. Is this gene prioritization feature available in your software?</li>
</ul>
<p>Yes. This feature is implemented as function <a href="https://github.com/stephenslab/rss/blob/master/src_vb/compute_pip.m"><code>compute_pip.m</code></a> in RSS-E. The step-by-step <a href="http://stephenslab.github.io/rss/Example-5">RSS-E tutorial</a> illustrates how to use this function.</p>
<ul>
<li>There are two sanity checks for the more sophisticated RSS-E analysis in <a href="https://doi.org/10.1101/160770">Zhu and Stephens (2018)</a>: an eyeball test and a likelihood ratio calculation. Do you have software for these sanity checks?</li>
</ul>
<p>Yes. The eyeball test is simply plotting marginal distribution of GWAS z-scores, stratified by SNP-level annotations based on a given gene set. We used <a href="https://ggplot2.tidyverse.org/reference/geom_density.html"><code>ggplot2::geom_density</code></a> (default setting) in the present study.<br />
Regarding the likelihood ratio check, I write a stand-alone script <a href="https://github.com/stephenslab/rss/blob/master/misc/ash_lrt_31traits.R"><code>ash_lrt_31traits.R</code></a>. Please carefully read the instruction in this script. For more details of these two sanity checks, please see the caption of <a href="https://www.biorxiv.org/content/biorxiv/suppl/2018/07/16/160770.DC2/160770-3.pdf">Supplementary Figure 17</a> in <a href="https://doi.org/10.1101/160770">Zhu and Stephens (2018)</a>.</p>
<ul>
<li>Where can I download all 4,026 pre-processed gene sets?</li>
</ul>
<p>All 4,026 gene sets used in this study are freely available at <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data"><code>xiangzhu/rss-gsea</code></a>, where the folder <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data/biological_pathway"><code>biological_pathway</code></a> contains 3,913 biological pathways, and the folder <a href="https://github.com/xiangzhu/rss-gsea/tree/master/data/tissue_set"><code>tissue_set</code></a> contains 113 GTEx tissue-based gene sets. More details about these gene sets can be found <a href="gene_set.html">here</a>.</p>
<ul>
<li>Where can I find RSS-E “baseline” model fitting results of all 31 traits?</li>
</ul>
<p>You can find summary results of “baseline” model fitting at <a href="https://xiangzhu.github.io/rss-gsea-baseline/"><code>xiangzhu/rss-gsea-baseline</code></a>. For me, the baseline model fitting results are merely inferential “bases” for the enrichment model fitting results shown in the “Main results” section above. However, when I was presenting the enrichment results during my <a href="http://www.stat.uchicago.edu/~xiangzhu/THESIS_20170630.html">Ph.D. thesis defense</a>, <a href="http://jnpopgen.org/">Prof. John Novembre</a> and <a href="http://xinhelab.org/">Prof. Xin He</a> both pointed out these baseline results might be useful for other on-going research projects on the “fourth floor” (i.e. the <strong>fantastic</strong> computational space shared with the labs of Matthew Stephens, John Novembre and Xin He). Their comments motivated me to create a separate online notebook <a href="https://xiangzhu.github.io/rss-gsea-baseline/"><code>xiangzhu/rss-gsea-baseline</code></a> to share the baseline summary results.</p>
<ul>
<li>Where can I find “Round 1” RSS-E results of all 3,913 biological pathways?</li>
</ul>
<p>Currently you need to contact me directly to view our “Round 1” results of all 3913 pathways. When this work was under review, one referee pointed out that our previous online results, especially our “Round 1” analysis results, were “needlessly complicated” and did not have “any obvious benefit”. Hence, I removed the “Round 1” analysis results from this notebook to simplify the presentation. I sincerely hope that this change can address this referee’s comment.</p>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
