<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of low-density lipoprotein</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; position: absolute; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine:empty
  { position: absolute; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of low-density lipoprotein</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-07-02</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/4ee856560a24b05484fed9edf88b4a9a24610918" target="_blank">4ee8565</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Unstaged changes:
    Modified:   analysis/index.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/4ee856560a24b05484fed9edf88b4a9a24610918/analysis/ldl_2010.Rmd" target="_blank">4ee8565</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“ldl_2010.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/20686565">“Biological, Clinical and Population Relevance of 95 Loci for Blood Lipids” (<em>Nature</em>, 2010)</a>. The summary data file is available at <a href="http://csg.sph.umich.edu/willer/public/lipids2010" class="uri">http://csg.sph.umich.edu/willer/public/lipids2010</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["Chylomicron-mediated lipid transport<br/>(reactome, PC)","Statin Pathway<br/>(wiki, BioSystems)","Electric Transmission Across Gap Junctions<br/>(reactome, PC)","selenocysteine biosynthesis<br/>(humancyc, PC)","Gap junction assembly<br/>(reactome, PC)","Lipoprotein metabolism<br/>(reactome, PC)","Lipid and lipoprotein metabolism<br/>(reactome, PC)","Lipid digestion, mobilization, and transport<br/>(reactome, BioSystems)","Pyruvate metabolism<br/>(kegg, BioSystems)","LDL endocytosis<br/>(reactome, PC)","Carbon metabolism<br/>(kegg, BioSystems)","Nectin/Necl trans heterodimerization<br/>(reactome, PC)","tRNA splicing<br/>(humancyc, PC)","Vesicle-mediated transport<br/>(reactome, BioSystems)","Retinoid metabolism and transport<br/>(reactome, PC)","Retinoid metabolism and transport<br/>(reactome, BioSystems)","Basic mechanism of action of PPARa, PPARb(d) and PPARg and effects on gene expression<br/>(biocarta, BioCarta)","Scavenging by Class A Receptors<br/>(reactome, PC)","HDL-mediated lipid transport<br/>(reactome, PC)","Binding and Uptake of Ligands by Scavenger Receptors<br/>(reactome, BioSystems)","Binding and Uptake of Ligands by Scavenger Receptors<br/>(reactome, PC)","Diseases associated with visual transduction<br/>(reactome, PC)","Visual phototransduction<br/>(reactome, BioSystems)","Vitamin B12 Metabolism<br/>(wiki, BioSystems)","Scavenging by Class B Receptors<br/>(reactome, PC)","Atypical NF-kappaB pathway<br/>(pid, PC)","Golgi Associated Vesicle Biogenesis<br/>(reactome, PC)","Clathrin derived vesicle budding<br/>(reactome, PC)","Clathrin derived vesicle budding<br/>(reactome, BioSystems)","Organic cation/anion/zwitterion transport<br/>(reactome, BioSystems)","TNF-alpha/NF-kB Signaling Pathway<br/>(wiki, BioSystems)","Nectin adhesion pathway<br/>(pid, BioSystems)","Monoamine Transport<br/>(wiki, BioSystems)","FXR and LXR Regulation of Cholesterol Metabolism<br/>(biocarta, BioCarta)","Membrane Trafficking<br/>(reactome, PC)","EGFR1 Signaling Pathway<br/>(wiki, BioSystems)","Hemostasis<br/>(reactome, PC)","Hemostasis<br/>(reactome, BioSystems)","Membrane Trafficking<br/>(reactome, BioSystems)","Nectin adhesion pathway<br/>(pid, PC)","Validated nuclear estrogen receptor beta network<br/>(pid, PC)","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell<br/>(reactome, PC)","Selenium Pathway<br/>(wiki, BioSystems)","Low-density lipoprotein (LDL) pathway during atherogenesis<br/>(biocarta, BioCarta)","SREBP control of lipid synthesis<br/>(biocarta, BioCarta)","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell<br/>(reactome, BioSystems)","Regulation of CDC42 activity<br/>(pid, PC)","proline biosynthesis<br/>(biocyc, BioSystems)","Neurotransmitter Clearance In The Synaptic Cleft<br/>(reactome, PC)","proline biosynthesis<br/>(humancyc, PC)","ornithine de novo biosynthesis<br/>(biocyc, BioSystems)","Bile secretion<br/>(kegg, BioSystems)","Mitochondrial protein import<br/>(reactome, PC)","Proline biosynthesis, glutamate => proline<br/>(kegg, BioSystems)","ErbB signaling pathway<br/>(wiki, BioSystems)","Mitochondrial protein import<br/>(reactome, BioSystems)","TNF signaling pathway<br/>(kegg, BioSystems)","Fat digestion and absorption<br/>(kegg, BioSystems)","Organic cation transport<br/>(reactome, BioSystems)","Orphan transporters<br/>(reactome, PC)","Chronic myeloid leukemia<br/>(kegg, BioSystems)","Endocytosis<br/>(kegg, BioSystems)","Cytosolic tRNA aminoacylation<br/>(reactome, PC)","Stabilization and expansion of the E-cadherin adherens junction<br/>(pid, BioSystems)","Stabilization and expansion of the E-cadherin adherens junction<br/>(pid, PC)","SREBF and miR33 in cholesterol and lipid homeostasis<br/>(wiki, BioSystems)","SREBP signalling<br/>(wiki, BioSystems)","Adherens junctions interactions<br/>(reactome, PC)","Aminoacyl-tRNA...<br/>(kegg, PC)","Amyotrophic lateral sclerosis (ALS)<br/>(kegg, BioSystems)","Cell-cell junction organization<br/>(reactome, PC)","Aminoacyl-tRNA biosynthesis<br/>(kegg, BioSystems)","tRNA charging<br/>(humancyc, PC)","formation of the beta-catenin:TCF transactivating complex<br/>(reactome, PC)","Aminoacyl-tRNA biosynthesis, eukaryotes<br/>(kegg, BioSystems)","tRNA Aminoacylation<br/>(reactome, PC)","Factors involved in megakaryocyte development and platelet production<br/>(reactome, PC)","Factors involved in megakaryocyte development and platelet production<br/>(reactome, BioSystems)","Glucocorticoid receptor regulatory network<br/>(pid, PC)","Neurotrophin signaling pathway<br/>(kegg, BioSystems)","Lysosome<br/>(kegg, BioSystems)","BDNF signaling pathway<br/>(wiki, BioSystems)","Bacterial invasion of epithelial cells<br/>(kegg, BioSystems)","Fatty acid, triacylglycerol, and ketone body metabolism<br/>(reactome, PC)","T cell receptor signaling pathway<br/>(kegg, BioSystems)","EGF receptor signaling pathway<br/>(panther, PANTHER)","EGF receptor signaling pathway<br/>(panther, PC)","Gap junction trafficking and regulation<br/>(reactome, PC)","p75(NTR)-mediated signaling<br/>(pid, BioSystems)","p75(NTR)-mediated signaling<br/>(pid, PC)","ErbB signaling pathway<br/>(kegg, BioSystems)","Cell junction organization<br/>(reactome, PC)","Cell junction organization<br/>(reactome, BioSystems)","Fatty acid, triacylglycerol, and ketone body metabolism<br/>(reactome, BioSystems)","Herpes simplex infection<br/>(kegg, BioSystems)","Cell-Cell communication<br/>(reactome, PC)","Cell-Cell communication<br/>(reactome, BioSystems)","Adherens junction<br/>(kegg, BioSystems)","Insulin signaling pathway<br/>(kegg, BioSystems)","Cell adhesion molecules (CAMs)<br/>(kegg, BioSystems)"],[17,30,5,6,18,30,50,58,39,6,103,7,5,231,39,40,4,19,15,40,41,88,89,19,5,14,49,56,65,14,181,25,31,7,184,169,490,496,193,30,14,69,85,6,6,93,25,2,7,3,2,70,47,4,52,49,108,41,9,96,71,252,24,40,42,16,61,30,23,51,59,44,37,86,38,42,133,134,82,115,117,136,76,174,101,99,108,29,55,62,84,82,83,181,174,126,127,72,133,134],[1139,2311,489,486,1041,2618,3959,4463,3328,512,8279,665,389,19646,4541,4541,541,2338,1460,4171,4231,8917,8917,1505,506,1178,4536,4869,5666,1274,14394,2892,2987,705,14843,15633,45889,46250,15723,3415,1205,3435,6860,569,527,4488,3048,153,673,217,262,7366,3284,281,6444,3304,9398,2630,1088,8626,6425,22188,1846,4323,4538,1424,5236,5579,1808,4924,8006,3603,2970,3268,3062,3457,10328,10406,6707,10587,9100,14423,8696,17074,8904,10953,11687,2036,4590,5121,9785,10550,10550,17634,10783,17214,17214,9483,10995,14575],[65.5286624690027,63.9837738149057,59.8797570588217,59.73479824713,59.0653361166729,56.6878806592324,54.8308688379987,51.6830873839095,48.7388497147727,43.1581751272986,41.871735891333,34.5057544822095,34.0158074473022,34.0014596468015,33.9252369926093,33.9252369926093,33.8113592928551,30.5316948406917,30.2165240874165,30.1592842082679,29.7146171695189,27.134310164359,27.134310164359,26.9666542728607,25.8022576229393,24.6705223194266,23.0397853333724,22.7491253050679,21.6917692865133,19.9059845639386,18.1142382241471,17.9328682169879,17.5155358960695,16.7509830376869,16.5581616732514,16.4413439557194,16.3714941179733,16.2988043553166,16.0757663832413,16.0111113924227,15.5080364392251,15.4927708793939,15.3364756538234,15.1713752075767,14.5531240157203,14.5502944485425,14.5214913740828,14.4488367051166,14.377852565475,14.2010454066907,14.0376472579907,14.0327300437947,13.997924734111,13.970411418755,13.955440632834,13.9388253704654,13.8020954844745,13.8006498191217,13.7897046035896,13.6735683513346,13.4496389040791,13.3863942629125,12.9905103110063,12.9677839061592,12.857923409555,12.7867460606507,12.4257202085483,12.4197263932258,12.3709853727834,12.217200394721,11.9003803889121,11.5685870342031,11.5590014519879,11.5472442994237,11.4740001858512,11.4131009795859,11.2746188139444,11.2542371476551,11.1963723167759,11.088652036022,10.9911187232654,10.8164686367659,10.6968967540769,10.591947256885,10.2722965086959,10.2676885603383,10.19035646881,10.188256652565,10.1222510998736,9.9632166354649,9.71573777929333,9.30849739344155,9.30849739344155,8.88574816564796,8.70610600810732,8.44426320147171,8.44426320147171,8.18968988728376,7.53215266414573,5.74065586244143],["-3.7788<br/>[-3.7796, -3.7607]","-3.7963<br/>[-3.7972, -3.7776]","-3.7176<br/>[-3.7183, -3.7018]","-3.7237<br/>[-3.7244, -3.7077]","-3.7175<br/>[-3.7182, -3.7017]","-3.7891<br/>[-3.7899, -3.7706]","-3.7996<br/>[-3.8005, -3.7808]","-3.7998<br/>[-3.8007, -3.7809]","-3.7172<br/>[-3.7179, -3.7014]","-3.7453<br/>[-3.7461, -3.7285]","-3.7188<br/>[-3.7195, -3.7030]","-3.7566<br/>[-3.7574, -3.7394]","-3.7178<br/>[-3.7185, -3.7020]","-3.7904<br/>[-3.7912, -3.7721]","-3.7750<br/>[-3.7758, -3.7570]","-3.7750<br/>[-3.7758, -3.7570]","-3.7182<br/>[-3.7190, -3.7024]","-3.7606<br/>[-3.7614, -3.7432]","-3.7584<br/>[-3.7592, -3.7411]","-3.7658<br/>[-3.7666, -3.7483]","-3.7658<br/>[-3.7666, -3.7483]","-3.7762<br/>[-3.7770, -3.7583]","-3.7762<br/>[-3.7770, -3.7583]","-3.7534<br/>[-3.7542, -3.7363]","-3.7281<br/>[-3.7289, -3.7120]","-3.7310<br/>[-3.7318, -3.7148]","-3.7368<br/>[-3.7376, -3.7205]","-3.7368<br/>[-3.7375, -3.7204]","-3.7366<br/>[-3.7373, -3.7202]","-3.7279<br/>[-3.7287, -3.7118]","-3.7565<br/>[-3.7573, -3.7392]","-3.7585<br/>[-3.7593, -3.7412]","-3.7515<br/>[-3.7523, -3.7344]","-3.7310<br/>[-3.7318, -3.7148]","-3.7390<br/>[-3.7398, -3.7227]","-3.7648<br/>[-3.7656, -3.7474]","-3.7423<br/>[-3.7431, -3.7258]","-3.7422<br/>[-3.7430, -3.7257]","-3.7388<br/>[-3.7395, -3.7224]","-3.7583<br/>[-3.7591, -3.7410]","-3.7286<br/>[-3.7294, -3.7124]","-3.7565<br/>[-3.7573, -3.7393]","-3.7405<br/>[-3.7413, -3.7239]","-3.7294<br/>[-3.7302, -3.7132]","-3.7288<br/>[-3.7296, -3.7126]","-3.7562<br/>[-3.7571, -3.7391]","-3.7242<br/>[-3.7250, -3.7082]","-3.7176<br/>[-3.7184, -3.7018]","-3.7258<br/>[-3.7265, -3.7097]","-3.7176<br/>[-3.7184, -3.7018]","-3.7176<br/>[-3.7184, -3.7018]","-3.7490<br/>[-3.7498, -3.7321]","-3.7512<br/>[-3.7520, -3.7341]","-3.7176<br/>[-3.7183, -3.7018]","-3.7485<br/>[-3.7493, -3.7316]","-3.7512<br/>[-3.7520, -3.7341]","-3.7323<br/>[-3.7331, -3.7160]","-3.7418<br/>[-3.7426, -3.7252]","-3.7280<br/>[-3.7287, -3.7118]","-3.7414<br/>[-3.7422, -3.7248]","-3.7492<br/>[-3.7500, -3.7323]","-3.7711<br/>[-3.7719, -3.7535]","-3.7239<br/>[-3.7247, -3.7079]","-3.7487<br/>[-3.7495, -3.7317]","-3.7486<br/>[-3.7494, -3.7317]","-3.7333<br/>[-3.7341, -3.7169]","-3.7352<br/>[-3.7360, -3.7189]","-3.7555<br/>[-3.7563, -3.7383]","-3.7235<br/>[-3.7243, -3.7075]","-3.7487<br/>[-3.7495, -3.7318]","-3.7566<br/>[-3.7574, -3.7394]","-3.7236<br/>[-3.7243, -3.7076]","-3.7234<br/>[-3.7242, -3.7074]","-3.7307<br/>[-3.7314, -3.7144]","-3.7234<br/>[-3.7241, -3.7074]","-3.7234<br/>[-3.7242, -3.7075]","-3.7271<br/>[-3.7279, -3.7110]","-3.7271<br/>[-3.7278, -3.7110]","-3.7295<br/>[-3.7303, -3.7133]","-3.7257<br/>[-3.7265, -3.7096]","-3.7291<br/>[-3.7299, -3.7129]","-3.7251<br/>[-3.7258, -3.7090]","-3.7478<br/>[-3.7486, -3.7309]","-3.7395<br/>[-3.7403, -3.7230]","-3.7498<br/>[-3.7506, -3.7329]","-3.7506<br/>[-3.7514, -3.7336]","-3.7508<br/>[-3.7516, -3.7339]","-3.7193<br/>[-3.7201, -3.7035]","-3.7212<br/>[-3.7219, -3.7053]","-3.7211<br/>[-3.7219, -3.7052]","-3.7499<br/>[-3.7507, -3.7329]","-3.7590<br/>[-3.7598, -3.7417]","-3.7590<br/>[-3.7598, -3.7417]","-3.7393<br/>[-3.7401, -3.7229]","-3.7500<br/>[-3.7508, -3.7330]","-3.7599<br/>[-3.7607, -3.7426]","-3.7599<br/>[-3.7607, -3.7426]","-3.7478<br/>[-3.7486, -3.7309]","-3.7512<br/>[-3.7520, -3.7343]","-3.7546<br/>[-3.7554, -3.7375]"],["-1.2774<br/>[-1.2834, -1.2627]","-1.5392<br/>[-1.5725, -1.4891]","-2.1751<br/>[-2.3789, -2.1112]","-1.7521<br/>[-1.8271, -1.6890]","-2.6078<br/>[-2.6971, -2.5064]","-1.5831<br/>[-1.6058, -1.5154]","-1.7202<br/>[-1.7509, -1.6675]","-1.7801<br/>[-1.8697, -1.7114]","-2.8635<br/>[-2.9826, -2.7611]","-1.1480<br/>[-1.1974, -1.1065]","-2.7614<br/>[-2.7742, -2.7353]","-1.2916<br/>[-1.3364, -1.2628]","-1.9469<br/>[-2.0776, -1.8377]","-2.4246<br/>[-2.4904, -2.4071]","-2.0417<br/>[-2.0922, -2.0028]","-2.0417<br/>[-2.0922, -2.0028]","-2.0093<br/>[-2.2232, -1.8654]","-1.8387<br/>[-1.8849, -1.8228]","-1.6577<br/>[-1.6862, -1.6247]","-2.0253<br/>[-2.0752, -1.9878]","-2.0302<br/>[-2.0775, -1.9906]","-2.2859<br/>[-2.3353, -2.2584]","-2.2859<br/>[-2.3353, -2.2584]","-1.7118<br/>[-1.7320, -1.6801]","-1.5484<br/>[-1.6171, -1.3922]","-1.7628<br/>[-1.8119, -1.7139]","-2.2895<br/>[-2.3293, -2.2709]","-2.2888<br/>[-2.3691, -2.2358]","-2.3737<br/>[-2.4199, -2.3468]","-1.9347<br/>[-1.9360, -1.9219]","-2.5498<br/>[-2.5749, -2.5145]","-1.9092<br/>[-1.9806, -1.8416]","-2.0773<br/>[-2.0892, -2.0297]","-1.5895<br/>[-1.6309, -1.5648]","-2.6662<br/>[-2.6909, -2.6253]","-2.4905<br/>[-2.5256, -2.4715]","-3.0648<br/>[-3.0990, -3.0185]","-3.0679<br/>[-3.1015, -3.0207]","-2.6929<br/>[-2.7174, -2.6513]","-2.0394<br/>[-2.1404, -1.9773]","-2.0020<br/>[-2.1486, -1.9309]","-2.0182<br/>[-2.1592, -1.9627]","-2.4105<br/>[-2.4761, -2.3439]","-1.5524<br/>[-1.6058, -1.4724]","-1.7202<br/>[-1.9652, -1.4629]","-2.2429<br/>[-2.2543, -2.2264]","-2.5758<br/>[-2.7325, -2.4517]","-1.8148<br/>[-1.8809, -1.7369]","-1.8295<br/>[-1.8795, -1.7715]","-1.9436<br/>[-2.0311, -1.8467]","-2.0256<br/>[-2.1141, -1.9311]","-2.5025<br/>[-2.5613, -2.4557]","-2.1319<br/>[-2.1443, -2.0872]","-2.0606<br/>[-2.1435, -1.9671]","-2.3724<br/>[-2.4017, -2.3357]","-2.1341<br/>[-2.1680, -2.1066]","-2.5809<br/>[-2.6421, -2.5270]","-2.0826<br/>[-2.1357, -1.9849]","-1.8510<br/>[-1.8636, -1.8266]","-2.4590<br/>[-2.4953, -2.4131]","-2.3175<br/>[-2.3817, -2.2394]","-2.6405<br/>[-2.7033, -2.5950]","-2.3083<br/>[-2.4314, -2.2133]","-2.3206<br/>[-2.3476, -2.2064]","-2.3428<br/>[-2.3599, -2.3026]","-2.0147<br/>[-2.0991, -1.9418]","-2.3621<br/>[-2.4412, -2.2883]","-2.3351<br/>[-2.3494, -2.3277]","-2.3861<br/>[-2.5133, -2.2403]","-2.3926<br/>[-2.4080, -2.3689]","-2.4601<br/>[-2.4627, -2.4579]","-2.6208<br/>[-2.7378, -2.5228]","-2.5673<br/>[-2.6784, -2.4759]","-2.6103<br/>[-2.7376, -2.5066]","-2.5835<br/>[-2.6910, -2.4903]","-2.6035<br/>[-2.7315, -2.5038]","-2.9956<br/>[-3.1220, -2.9119]","-2.9995<br/>[-3.1230, -2.9143]","-2.8022<br/>[-2.8920, -2.7188]","-3.0625<br/>[-3.1094, -2.9884]","-2.8461<br/>[-2.9593, -2.7478]","-3.1098<br/>[-3.1445, -3.0494]","-2.4413<br/>[-2.4798, -2.4001]","-2.7519<br/>[-2.7525, -2.7506]","-2.4768<br/>[-2.5259, -2.4322]","-2.5229<br/>[-2.5725, -2.4768]","-2.5361<br/>[-2.5847, -2.4883]","-1.7398<br/>[-1.7398, -1.7397]","-2.7993<br/>[-2.9158, -2.7021]","-2.8378<br/>[-2.9531, -2.7335]","-2.5101<br/>[-2.5640, -2.4672]","-2.5258<br/>[-2.5365, -2.4890]","-2.5258<br/>[-2.5365, -2.4890]","-2.7452<br/>[-2.7659, -2.6773]","-2.5766<br/>[-2.6519, -2.5480]","-2.6648<br/>[-2.7278, -2.6523]","-2.6648<br/>[-2.7278, -2.6523]","-2.6625<br/>[-2.6961, -2.5677]","-2.5501<br/>[-2.5924, -2.5290]","-2.7183<br/>[-2.7750, -2.6386]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Liver","Adrenal Gland","Brain - Cerebellum","Brain - Cortex","Brain - Anterior cingulate cortex (BA24)","Brain - Frontal Cortex (BA9)","Brain - Caudate (basal ganglia)","Brain - Hippocampus","Brain - Hypothalamus","Brain - Nucleus accumbens (basal ganglia)","Brain - Putamen (basal ganglia)","Prostate","Pituitary","Uterus","Cells - EBV-transformed lymphocytes","Testis","Small Intestine - Terminal Ileum","Pancreas","Vagina","Skin - Sun Exposed (Lower leg)","Breast - Mammary Tissue","Skin - Not Sun Exposed (Suprapubic)","Cells - Transformed fibroblasts","Ovary","Brain - Cerebellar Hemisphere","Esophagus - Mucosa","Colon - Transverse","Esophagus - Muscularis","Artery - Coronary","Stomach","Esophagus - Gastroesophageal Junction","Colon - Sigmoid","Artery - Tibial","Artery - Aorta","Spleen","Heart - Atrial Appendage","Lung","Adipose - Subcutaneous","Heart - Left Ventricle","Adipose - Visceral (Omentum)","Nerve - Tibial","Thyroid","Muscle - Skeletal","Whole Blood"],[6031,5349,5375,5537,5624,5907,6189,5476,5626,5826,6256,5307,5604,5394,4730,5542,4468,4865,5494,5417,5295,5478,5827,5583,5355,5335,4954,5712,5881,5001,5522,5620,5761,6048,3831,5534,4765,5356,5326,5547,5408,5873,5338,5592],[21.4578019405333,15.1672217498984,13.7816714810347,12.4550511323185,12.3474155748387,12.0083834906341,11.8788244346006,11.7038143243531,11.6340140518975,11.3957528881788,11.1790822101452,1.62886415429628,1.56286294480845,1.50596887492392,1.49901040341518,1.49268942102397,1.2983885512164,1.2901543990953,0.964019482032602,0.884069370835655,0.70456756535334,0.650232256300225,0.569099852269415,0.465304531066982,0.302975391241404,0.106148661809763,0.0170223478776234,-0.00781711734942957,-0.0285619240058373,-0.0372114266807462,-0.0623353664627131,-0.0839274612820607,-0.123805695944385,-0.18007423906167,-0.246101534105525,-0.382041067322732,-0.399015228312724,-0.439093059386931,-0.439111723051002,-0.450610422956152,-0.462693952488546,-0.525683532279535,-0.707018936321186,-0.744797049090938],["-3.7600<br/>[-3.7608, -3.7429]","-3.7557<br/>[-3.7565, -3.7385]","-3.7524<br/>[-3.7532, -3.7354]","-3.7499<br/>[-3.7507, -3.7330]","-3.7499<br/>[-3.7507, -3.7330]","-3.7498<br/>[-3.7506, -3.7328]","-3.7483<br/>[-3.7491, -3.7314]","-3.7485<br/>[-3.7493, -3.7315]","-3.7485<br/>[-3.7493, -3.7315]","-3.7483<br/>[-3.7491, -3.7314]","-3.7483<br/>[-3.7491, -3.7314]","-3.7238<br/>[-3.7246, -3.7079]","-3.7242<br/>[-3.7249, -3.7082]","-3.7231<br/>[-3.7238, -3.7071]","-3.7229<br/>[-3.7237, -3.7070]","-3.7198<br/>[-3.7205, -3.7040]","-3.7209<br/>[-3.7216, -3.7050]","-3.7232<br/>[-3.7239, -3.7072]","-3.7221<br/>[-3.7228, -3.7062]","-3.7228<br/>[-3.7235, -3.7068]","-3.7216<br/>[-3.7223, -3.7057]","-3.7230<br/>[-3.7237, -3.7071]","-3.7216<br/>[-3.7223, -3.7057]","-3.7232<br/>[-3.7240, -3.7073]","-3.7203<br/>[-3.7210, -3.7044]","-3.7176<br/>[-3.7183, -3.7018]","-3.7172<br/>[-3.7179, -3.7014]","-3.7173<br/>[-3.7180, -3.7016]","-3.7173<br/>[-3.7180, -3.7016]","-3.7171<br/>[-3.7178, -3.7013]","-3.7171<br/>[-3.7178, -3.7013]","-3.7171<br/>[-3.7178, -3.7013]","-3.7169<br/>[-3.7177, -3.7012]","-3.7169<br/>[-3.7176, -3.7011]","-3.7174<br/>[-3.7181, -3.7016]","-3.7189<br/>[-3.7196, -3.7030]","-3.7172<br/>[-3.7179, -3.7014]","-3.7171<br/>[-3.7179, -3.7014]","-3.7183<br/>[-3.7190, -3.7025]","-3.7171<br/>[-3.7179, -3.7014]","-3.7171<br/>[-3.7178, -3.7013]","-3.7169<br/>[-3.7177, -3.7012]","-3.7168<br/>[-3.7175, -3.7010]","-3.7167<br/>[-3.7174, -3.7009]"],["-2.2306<br/>[-2.2702, -2.1915]","-2.2409<br/>[-2.3385, -2.1891]","-2.3396<br/>[-2.4116, -2.2850]","-2.3972<br/>[-2.4472, -2.3318]","-2.4085<br/>[-2.4564, -2.3442]","-2.4387<br/>[-2.4733, -2.3754]","-2.4609<br/>[-2.4880, -2.3907]","-2.4279<br/>[-2.4650, -2.3616]","-2.4400<br/>[-2.4739, -2.3715]","-2.4646<br/>[-2.4868, -2.4035]","-2.4900<br/>[-2.5116, -2.4336]","-2.7626<br/>[-2.8527, -2.6854]","-2.7846<br/>[-2.9100, -2.7431]","-2.7945<br/>[-2.8847, -2.7071]","-2.7443<br/>[-2.8832, -2.7052]","-2.8002<br/>[-2.9371, -2.6879]","-2.7836<br/>[-2.9206, -2.6691]","-2.7992<br/>[-2.9439, -2.6886]","-2.8232<br/>[-2.9937, -2.7503]","-2.8666<br/>[-2.9427, -2.7829]","-2.8547<br/>[-3.0296, -2.7370]","-2.8606<br/>[-2.9888, -2.7923]","-2.9032<br/>[-3.0690, -2.7637]","-2.8941<br/>[-3.0115, -2.8692]","-2.9669<br/>[-3.1526, -2.8357]","-2.9932<br/>[-3.2183, -2.8272]","-3.0265<br/>[-3.1965, -2.8826]","-3.0449<br/>[-3.2283, -2.8914]","-3.0544<br/>[-3.2373, -2.9009]","-3.0495<br/>[-3.2057, -2.9117]","-3.0620<br/>[-3.2303, -2.9117]","-3.0715<br/>[-3.2373, -2.9223]","-3.0843<br/>[-3.2560, -2.9357]","-3.1096<br/>[-3.2750, -2.9628]","-3.0752<br/>[-3.2663, -2.9085]","-3.1785<br/>[-3.3283, -3.0230]","-3.1623<br/>[-3.3440, -2.9866]","-3.1874<br/>[-3.3728, -3.0095]","-3.1945<br/>[-3.3609, -3.0277]","-3.1937<br/>[-3.3874, -3.0149]","-3.2015<br/>[-3.3811, -3.0247]","-3.2422<br/>[-3.4093, -3.0715]","-3.3607<br/>[-3.5161, -3.1756]","-3.3912<br/>[-3.5357, -3.2053]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Liver","Kidney_Cortex","Heart_Atrial_Appendage","Adrenal_Gland","Adipose_Subcutaneous","Small_Intestine_Terminal_Ileum","Adipose_Visceral_(Omentum)","Stomach","Minor_Salivary_Gland","Breast_Mammary_Tissue","Colon_Transverse","Brain_Cortex","Brain_Frontal_Cortex_(BA9)","Prostate","Brain_Nucleus_accumbens_(basal_ganglia)","Esophagus_Mucosa","Brain_Caudate_(basal_ganglia)","Brain_Putamen_(basal_ganglia)","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Cerebellum","Pancreas","Brain_Hippocampus","Ovary","Esophagus_Gastroesophageal_Junction","Brain_Amygdala","Skin_Sun_Exposed_(Lower_leg)","Skin_Not_Sun_Exposed_(Suprapubic)","Brain_Hypothalamus","Spleen","Artery_Aorta","Brain_Substantia_nigra","Whole_Blood","Esophagus_Muscularis","Artery_Tibial","Uterus","Heart_Left_Ventricle","Vagina","Cells_EBV-transformed_lymphocytes","Colon_Sigmoid","Artery_Coronary","Testis","Muscle_Skeletal","Thyroid","Lung","Pituitary","Brain_Cerebellar_Hemisphere","Cells_Transformed_fibroblasts","Nerve_Tibial","Brain_Spinal_cord_(cervical_c-1)"],[7790,8530,10618,11741,9689,8304,9373,8830,7990,9381,7128,11451,13135,8687,10873,6502,11195,11270,12460,10438,7317,10467,11076,13550,10733,6453,6537,11697,6811,13733,12194,6762,13725,12065,11999,10521,6398,7416,10500,11883,7339,8241,11516,7931,9084,13039,10323,11949,12168],[33.412024880446,8.51155122859739,8.36088822811218,7.72106662155477,1.75638369659217,0.768068518468028,0.574262061364489,0.088853445593211,-0.329292324088027,-0.386675266708668,-0.397100718139291,-0.446260165950369,-0.607549983046995,-0.670542067406141,-0.714923799570992,-0.71561344082355,-0.721833445946529,-0.74076463780338,-0.787156730479312,-0.859039523147883,-0.866759171479289,-0.911478772351024,-0.921289177246062,-0.930385904095957,-0.933022346974829,-0.960409886863702,-0.963001274832559,-0.967717267357328,-1.00184099927729,-1.00453323428705,-1.01008581453462,-1.03867978371957,-1.06595215459251,-1.07132853022051,-1.07483015271157,-1.07702459068192,-1.08334328716035,-1.09106426608409,-1.09243705390478,-1.11884282434952,-1.1248974166387,-1.12679918282616,-1.14105885434247,-1.15935741605709,-1.17504379191839,-1.17938386002736,-1.18096586539214,-1.19017455364736,-1.26138548300654],["-3.7749<br/>[-3.7757, -3.7573]","-3.7432<br/>[-3.7440, -3.7265]","-3.7218<br/>[-3.7226, -3.7059]","-3.7473<br/>[-3.7481, -3.7304]","-3.7202<br/>[-3.7210, -3.7044]","-3.7209<br/>[-3.7216, -3.7051]","-3.7247<br/>[-3.7254, -3.7088]","-3.7222<br/>[-3.7229, -3.7063]","-3.7204<br/>[-3.7211, -3.7046]","-3.7220<br/>[-3.7228, -3.7061]","-3.7189<br/>[-3.7196, -3.7031]","-3.7188<br/>[-3.7195, -3.7029]","-3.7183<br/>[-3.7190, -3.7024]","-3.7203<br/>[-3.7210, -3.7044]","-3.7191<br/>[-3.7199, -3.7033]","-3.7189<br/>[-3.7197, -3.7031]","-3.7190<br/>[-3.7198, -3.7032]","-3.7189<br/>[-3.7197, -3.7031]","-3.7185<br/>[-3.7192, -3.7026]","-3.7179<br/>[-3.7186, -3.7021]","-3.7179<br/>[-3.7186, -3.7021]","-3.7192<br/>[-3.7199, -3.7033]","-3.7154<br/>[-3.7161, -3.6997]","-3.7158<br/>[-3.7165, -3.7000]","-3.7190<br/>[-3.7198, -3.7032]","-3.7173<br/>[-3.7181, -3.7016]","-3.7173<br/>[-3.7181, -3.7015]","-3.7189<br/>[-3.7196, -3.7031]","-3.7166<br/>[-3.7173, -3.7008]","-3.7145<br/>[-3.7152, -3.6988]","-3.7186<br/>[-3.7193, -3.7028]","-3.7166<br/>[-3.7173, -3.7009]","-3.7156<br/>[-3.7163, -3.6999]","-3.7151<br/>[-3.7159, -3.6994]","-3.7156<br/>[-3.7163, -3.6999]","-3.7173<br/>[-3.7181, -3.7015]","-3.7183<br/>[-3.7190, -3.7025]","-3.7162<br/>[-3.7170, -3.7005]","-3.7152<br/>[-3.7159, -3.6994]","-3.7150<br/>[-3.7157, -3.6993]","-3.7158<br/>[-3.7166, -3.7001]","-3.7160<br/>[-3.7167, -3.7002]","-3.7155<br/>[-3.7163, -3.6998]","-3.7156<br/>[-3.7163, -3.6998]","-3.7155<br/>[-3.7162, -3.6997]","-3.7167<br/>[-3.7174, -3.7009]","-3.7154<br/>[-3.7162, -3.6997]","-3.7146<br/>[-3.7154, -3.6989]","-3.7149<br/>[-3.7156, -3.6992]"],["-2.1748<br/>[-2.2050, -2.1539]","-2.5274<br/>[-2.5454, -2.4894]","-3.2446<br/>[-3.3415, -3.1435]","-2.7222<br/>[-2.7393, -2.7079]","-2.8948<br/>[-3.0373, -2.7846]","-2.9636<br/>[-3.1228, -2.8479]","-2.9648<br/>[-3.2059, -2.8305]","-3.0904<br/>[-3.2657, -2.9673]","-3.1941<br/>[-3.3506, -3.0660]","-3.2285<br/>[-3.3656, -3.0973]","-3.1908<br/>[-3.3792, -3.0280]","-3.2695<br/>[-3.3944, -3.1563]","-3.3341<br/>[-3.4450, -3.2223]","-3.3442<br/>[-3.4668, -3.2087]","-3.3750<br/>[-3.4728, -3.2567]","-3.1914<br/>[-3.3913, -3.0179]","-3.3770<br/>[-3.4782, -3.2581]","-3.3872<br/>[-3.4848, -3.2715]","-3.4063<br/>[-3.5144, -3.2837]","-3.4492<br/>[-3.5652, -3.3019]","-3.4692<br/>[-3.6555, -3.2728]","-3.4907<br/>[-3.5857, -3.3602]","-3.4819<br/>[-3.7013, -3.3346]","-3.4901<br/>[-3.5673, -3.3880]","-3.5049<br/>[-3.5982, -3.3743]","-3.6179<br/>[-3.7657, -3.4370]","-3.6178<br/>[-3.7623, -3.4349]","-3.5220<br/>[-3.6158, -3.3895]","-3.6526<br/>[-3.8520, -3.4000]","-3.5324<br/>[-3.6255, -3.4029]","-3.5527<br/>[-3.6395, -3.4300]","-3.7320<br/>[-3.9171, -3.4963]","-3.5823<br/>[-3.6699, -3.4587]","-3.6064<br/>[-3.7102, -3.4560]","-3.6119<br/>[-3.7200, -3.4691]","-3.6488<br/>[-3.7466, -3.5042]","-3.4184<br/>[-3.5840, -3.2285]","-3.8024<br/>[-3.9878, -3.5662]","-3.6692<br/>[-3.7589, -3.5317]","-3.6671<br/>[-3.7581, -3.5308]","-3.9106<br/>[-4.0946, -3.6699]","-3.8265<br/>[-3.9945, -3.5879]","-3.6988<br/>[-3.8371, -3.5121]","-3.9606<br/>[-4.1278, -3.7383]","-3.8909<br/>[-4.0452, -3.6789]","-3.7220<br/>[-3.8036, -3.5918]","-3.8203<br/>[-3.9631, -3.6245]","-3.7697<br/>[-3.8758, -3.6171]","-3.9058<br/>[-4.0339, -3.7210]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["20","16","9","4","7","11","1","18","19","6","2","13","3","14","10","15","17","8","12","5"],[7632,9716,9058,11745,7741,9832,6985,6873,5539,8297,8395,7140,9525,8906,7839,7262,7941,9665,6804,10905],[21.9926221237335,11.7524804080877,9.29155703014007,3.81801166710491,3.07776152907557,1.94846759036667,1.5949200012806,0.520571602432786,0.264853535846885,0.0185285425273636,-0.0314638506956224,-0.168261585979361,-0.387312408270854,-0.493786624786828,-0.577788984114142,-0.679797201608212,-0.706400380593345,-0.837874088205992,-0.883514183836807,-0.883851489919066],["-3.7666<br/>[-3.7674, -3.7492]","-3.7280<br/>[-3.7288, -3.7119]","-3.7227<br/>[-3.7234, -3.7067]","-3.7267<br/>[-3.7274, -3.7106]","-3.7159<br/>[-3.7167, -3.7002]","-3.7191<br/>[-3.7199, -3.7033]","-3.7235<br/>[-3.7242, -3.7075]","-3.7203<br/>[-3.7210, -3.7045]","-3.7212<br/>[-3.7219, -3.7053]","-3.7212<br/>[-3.7219, -3.7053]","-3.7227<br/>[-3.7234, -3.7068]","-3.7194<br/>[-3.7202, -3.7036]","-3.7209<br/>[-3.7216, -3.7050]","-3.7207<br/>[-3.7215, -3.7048]","-3.7186<br/>[-3.7194, -3.7028]","-3.7168<br/>[-3.7175, -3.7010]","-3.7167<br/>[-3.7174, -3.7010]","-3.7178<br/>[-3.7186, -3.7021]","-3.7164<br/>[-3.7172, -3.7007]","-3.7155<br/>[-3.7162, -3.6998]"],["-2.2923<br/>[-2.2975, -2.2879]","-2.9533<br/>[-3.0684, -2.8740]","-3.0519<br/>[-3.1837, -2.9442]","-3.0861<br/>[-3.2008, -2.9901]","-3.2000<br/>[-3.3149, -3.0721]","-3.2256<br/>[-3.3742, -3.0913]","-2.8406<br/>[-2.9882, -2.7202]","-3.0014<br/>[-3.1625, -2.8786]","-3.0238<br/>[-3.1772, -2.8864]","-3.1185<br/>[-3.2928, -2.9731]","-3.1506<br/>[-3.3170, -3.0174]","-3.1703<br/>[-3.3395, -3.0165]","-3.2759<br/>[-3.4247, -3.1160]","-3.3303<br/>[-3.4544, -3.1833]","-3.3611<br/>[-3.5242, -3.1706]","-3.4394<br/>[-3.6148, -3.2395]","-3.4563<br/>[-3.6209, -3.2593]","-3.5616<br/>[-3.6969, -3.3845]","-3.7277<br/>[-3.9101, -3.4946]","-3.5899<br/>[-3.7020, -3.4318]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">sessionInfo</span>()</a></code></pre></div>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.5

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] DT_0.4         plyr_1.8.4     dplyr_0.7.5    R.matlab_3.6.1

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.17      compiler_3.5.0    pillar_1.2.3     
 [4] later_0.7.3       git2r_0.21.0      workflowr_1.0.1  
 [7] bindr_0.1.1       R.methodsS3_1.7.1 R.utils_2.6.0    
[10] tools_3.5.0       digest_0.6.15     jsonlite_1.5     
[13] evaluate_0.10.1   tibble_1.4.2      pkgconfig_2.0.1  
[16] rlang_0.2.1       shiny_1.1.0       crosstalk_1.0.0  
[19] yaml_2.1.19       bindrcpp_0.2.2    stringr_1.3.1    
[22] knitr_1.20        htmlwidgets_1.2   rprojroot_1.3-2  
[25] tidyselect_0.2.4  glue_1.2.0        R6_2.2.2         
[28] rmarkdown_1.10    purrr_0.2.5       magrittr_1.5     
[31] whisker_0.3-2     backports_1.1.2   promises_1.0.1   
[34] htmltools_0.3.6   assertthat_0.2.0  mime_0.5         
[37] xtable_1.8-2      httpuv_1.4.4.1    stringi_1.2.3    
[40] R.oo_1.22.0      </code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.0.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
