<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of mean cell haemoglobin</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; position: absolute; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine:empty
  { position: absolute; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of mean cell haemoglobin</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-07-02</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/45aeca367abba342c65d5da96149d38a699739bb" target="_blank">45aeca3</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Unstaged changes:
    Modified:   analysis/index.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/45aeca367abba342c65d5da96149d38a699739bb/analysis/mch_2012.Rmd" target="_blank">45aeca3</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“mch_2012.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/23222517">“Seventy-Five Genetic Loci Influencing the Human Red Blood Cell” (<em>Nature</em>, 2012)</a>. The summary data file is available from the European Genome–Phenome Archive (EGA, <a href="http://www.ebi.ac.uk/ega" class="uri">http://www.ebi.ac.uk/ega</a>) under accession number EGAS00000000132.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["Meiotic synapsis<br/>(reactome, PC)","Meiotic recombination<br/>(reactome, PC)","RNA Polymerase I Promoter Opening<br/>(reactome, PC)","PRC2 methylates histones and DNA<br/>(reactome, PC)","Condensation of Prophase Chromosomes<br/>(reactome, PC)","Packaging Of Telomere Ends<br/>(reactome, PC)","Telomere Maintenance<br/>(reactome, PC)","RMTs methylate histone arginines<br/>(reactome, PC)","DNA methylation<br/>(reactome, PC)","Activated PKN1 stimulates transcription of AR (androgen receptor) regulated genes KLK2 and KLK3<br/>(reactome, BioSystems)","HDACs deacetylate histones<br/>(reactome, PC)","DNA Damage/Telomere Stress Induced Senescence<br/>(reactome, PC)","RNA Polymerase I Chain Elongation<br/>(reactome, BioSystems)","NoRC negatively regulates rRNA expression<br/>(reactome, PC)","RNA Polymerase I Chain Elongation<br/>(reactome, PC)","Deposition of new CENPA-containing nucleosomes at the centromere<br/>(reactome, PC)","SIRT1 negatively regulates rRNA Expression<br/>(reactome, PC)","Activation of DNA fragmentation factor<br/>(reactome, PC)","G2/M DNA damage checkpoint<br/>(reactome, BioSystems)","HDMs demethylate histones<br/>(reactome, BioSystems)","Mitotic Prophase<br/>(reactome, BioSystems)","Type II interferon signaling (IFNG)<br/>(wiki, BioSystems)","Regulatory RNA pathways<br/>(reactome, BioSystems)","Transcriptional regulation by small RNAs<br/>(reactome, BioSystems)","Amyloids<br/>(reactome, BioSystems)","formation of the beta-catenin:TCF transactivating complex<br/>(reactome, PC)","Cell Cycle, Mitotic<br/>(reactome, BioSystems)","Deposition of new CENPA-containing nucleosomes at the centromere<br/>(reactome, BioSystems)","Regulatory RNA pathways<br/>(reactome, PC)","RNA Polymerase I Promoter Clearance<br/>(reactome, BioSystems)","PKMTs methylate histone lysines<br/>(reactome, BioSystems)","Negative epigenetic regulation of rRNA expression<br/>(reactome, PC)","Amyloids<br/>(reactome, PC)","Signaling by Wnt<br/>(reactome, BioSystems)","Gastric cancer network 1<br/>(wiki, BioSystems)","RNA Polymerase I Transcription<br/>(reactome, BioSystems)","Oxidative Stress Induced Senescence<br/>(reactome, PC)","Signaling events mediated by HDAC Class III<br/>(pid, PC)","Factors involved in megakaryocyte development and platelet production<br/>(reactome, PC)","HDMs demethylate histones<br/>(reactome, PC)","Hemostasis<br/>(reactome, PC)","Transcription<br/>(reactome, PC)","Viral carcinogenesis<br/>(kegg, BioSystems)","M Phase<br/>(reactome, BioSystems)","Alcoholism<br/>(kegg, BioSystems)","RNA Polymerase I, RNA Polymerase III, and Mitochondrial Transcription<br/>(reactome, BioSystems)","HATs acetylate histones<br/>(reactome, BioSystems)","RNA Polymerase I Promoter Clearance<br/>(reactome, PC)","Systemic lupus erythematosus<br/>(kegg, BioSystems)","RHO GTPases activate PKNs<br/>(reactome, BioSystems)","Formation of Senescence-Associated Heterochromatin Foci (SAHF)<br/>(reactome, PC)","Senescence-Associated Secretory Phenotype (SASP)<br/>(reactome, PC)","Chromatin modifying enzymes<br/>(reactome, BioSystems)","Epigenetic regulation of gene expression<br/>(reactome, BioSystems)","Cellular responses to stress<br/>(reactome, PC)","Proteasome Degradation<br/>(wiki, BioSystems)","Mitotic Prophase<br/>(reactome, PC)","RNA Polymerase I Transcription<br/>(reactome, PC)","Meiosis<br/>(reactome, PC)","Transcriptional regulation by small RNAs<br/>(reactome, PC)","G2/M Checkpoints<br/>(reactome, BioSystems)","Apoptosis<br/>(reactome, PC)","Cell Cycle, Mitotic<br/>(reactome, PC)","Chromatin modifying enzymes<br/>(reactome, PC)","NoRC negatively regulates rRNA expression<br/>(reactome, BioSystems)","Programmed Cell Death<br/>(reactome, BioSystems)","Negative epigenetic regulation of rRNA expression<br/>(reactome, BioSystems)","Chromosome Maintenance<br/>(reactome, PC)","Epigenetic regulation of gene expression<br/>(reactome, PC)","Mitotic M-M/G1 phases<br/>(reactome, PC)","Cellular responses to stress<br/>(reactome, BioSystems)","Factors involved in megakaryocyte development and platelet production<br/>(reactome, BioSystems)","Chromosome Maintenance<br/>(reactome, BioSystems)","TCF dependent signaling in response to WNT<br/>(reactome, BioSystems)","Signaling by Wnt<br/>(reactome, PC)","RNA Polymerase I, RNA Polymerase III, and Mitochondrial Transcription<br/>(reactome, PC)","RNF mutants show enhanced WNT signaling and proliferation<br/>(reactome, PC)","Signaling by Rho GTPases<br/>(reactome, BioSystems)","Regulation of Complement cascade<br/>(reactome, PC)","Complement cascade<br/>(reactome, BioSystems)","Staphylococcus aureus infection<br/>(kegg, BioSystems)","Complement cascade<br/>(reactome, PC)","Apoptotic execution phase<br/>(reactome, PC)","Complement and coagulation cascades<br/>(kegg, BioSystems)","Influenza Life Cycle<br/>(reactome, PC)","Influenza Life Cycle<br/>(reactome, BioSystems)","Influenza Infection<br/>(reactome, PC)","Influenza Infection<br/>(reactome, BioSystems)","Apoptosis<br/>(reactome, BioSystems)","Hemostasis<br/>(reactome, BioSystems)","L-cysteine degradation II<br/>(humancyc, PC)","M Phase<br/>(reactome, PC)","HATs acetylate histones<br/>(reactome, PC)","Protein processing in endoplasmic reticulum<br/>(kegg, BioSystems)","Lysosome<br/>(kegg, BioSystems)","Asparagine N-linked glycosylation<br/>(reactome, PC)","Amino acid and derivative metabolism<br/>(reactome, PC)","Metabolism of amino acids and derivatives<br/>(reactome, BioSystems)","Biosynthesis of antibiotics<br/>(kegg, BioSystems)"],[72,83,59,68,69,48,75,73,61,62,91,75,87,101,85,60,63,13,73,44,133,49,129,100,80,86,463,68,111,105,62,103,79,318,42,107,117,38,133,43,490,225,197,288,170,143,136,103,126,88,16,103,241,121,258,61,132,105,112,99,108,149,457,203,103,157,105,93,119,316,366,134,101,222,312,140,223,375,23,37,53,38,49,67,130,132,134,136,154,496,2,284,132,163,117,116,180,184,199],[3817,3088,1124,1996,2274,1348,3844,3289,1435,1868,3079,3310,3073,4284,2959,2598,1627,919,3944,2635,8237,3529,7201,4561,3424,3611,34618,3018,5505,4355,3961,4467,3321,25597,3102,4464,5837,2813,11375,2561,50376,14059,15224,20380,11858,7595,6755,4241,5145,3992,1219,4854,16772,6105,17699,3825,8141,4350,5980,4465,6814,12835,34211,12962,4398,13766,4581,5528,5991,22753,26525,11460,5946,14926,25039,7402,15094,34895,1609,2511,2612,2580,4590,5391,9296,9394,9576,9674,13546,50777,271,20166,6326,15429,10147,11416,16291,16598,17036],[2334.30300194528,1937.79851572912,1918.50650054222,1904.48506424587,1902.32062909948,1855.10309923047,1788.4808859637,1783.50387134096,1732.45845708695,1723.24945020348,1717.11677911961,1697.74838298193,1664.42214829136,1659.97350424833,1657.35471700142,1654.51489396512,1651.77687258086,1637.75281969219,1618.73303200294,1607.17221624921,1594.29481115855,1593.38038700399,1586.79052541222,1585.32380760984,1580.36179082495,1577.55214945968,1570.28752602337,1569.14055553551,1562.93921069052,1546.65788557252,1540.70975439729,1532.4714286705,1527.34594759943,1516.22352295388,1513.16862311567,1508.15004273388,1498.83714025605,1473.0655681626,1470.17046478717,1456.73630641364,1447.48758192891,1441.96810875949,1436.79815607435,1427.22001026708,1426.4321161198,1413.24965459144,1412.54403218245,1408.37765286335,1398.96028989285,1398.65407255145,1396.03032626283,1382.9648423454,1381.48245878866,1379.01386682583,1374.93535369905,1372.06405570367,1358.33685162015,1340.1275849737,1334.08437937275,1284.92030288206,1280.52341599601,1251.64165559794,1221.04720791253,1217.94579633962,1204.50192500915,1196.53150208285,1190.26097992196,1189.79851982251,1185.32888286246,1166.16315301711,1148.07153616033,1130.94862458489,1102.4733334232,1093.88119785965,1056.73915186369,1056.23840728863,1048.92171100868,1025.87053620677,955.93818521399,955.086705745184,955.056403402839,955.039919872621,937.008197712319,878.905876963737,869.812558849097,869.796737026965,869.758180212504,869.743195795626,763.142461073579,690.359281658255,455.235702788397,433.776460469803,37.0002272302686,1.18015829565694,1.06233948351205,0.656971465051469,0.319193027614452,0.318059926063028,0.0436635420114833],["-4.1518<br/>[-4.1518, -4.1518]","-4.1505<br/>[-4.1505, -4.1505]","-4.1459<br/>[-4.1459, -4.1459]","-4.1464<br/>[-4.1464, -4.1464]","-4.1514<br/>[-4.1514, -4.1514]","-4.1458<br/>[-4.1458, -4.1458]","-4.1456<br/>[-4.1456, -4.1456]","-4.1568<br/>[-4.1568, -4.1568]","-4.1458<br/>[-4.1458, -4.1458]","-4.1456<br/>[-4.1456, -4.1456]","-4.1452<br/>[-4.1452, -4.1452]","-4.1455<br/>[-4.1455, -4.1455]","-4.1453<br/>[-4.1453, -4.1453]","-4.1449<br/>[-4.1449, -4.1449]","-4.1453<br/>[-4.1453, -4.1453]","-4.1453<br/>[-4.1453, -4.1453]","-4.1457<br/>[-4.1457, -4.1457]","-4.1195<br/>[-4.1195, -4.1195]","-4.1466<br/>[-4.1466, -4.1466]","-4.1189<br/>[-4.1189, -4.1189]","-4.1558<br/>[-4.1558, -4.1558]","-4.1189<br/>[-4.1189, -4.1189]","-4.1508<br/>[-4.1508, -4.1508]","-4.1513<br/>[-4.1513, -4.1513]","-4.1451<br/>[-4.1451, -4.1451]","-4.1454<br/>[-4.1454, -4.1454]","-4.1812<br/>[-4.1812, -4.1812]","-4.1452<br/>[-4.1452, -4.1452]","-4.1509<br/>[-4.1509, -4.1509]","-4.1448<br/>[-4.1448, -4.1448]","-4.1187<br/>[-4.1187, -4.1187]","-4.1448<br/>[-4.1448, -4.1448]","-4.1451<br/>[-4.1451, -4.1451]","-4.1552<br/>[-4.1552, -4.1552]","-4.1190<br/>[-4.1190, -4.1190]","-4.1448<br/>[-4.1448, -4.1448]","-4.1444<br/>[-4.1444, -4.1444]","-4.1191<br/>[-4.1191, -4.1191]","-4.2214<br/>[-4.2214, -4.2214]","-4.1189<br/>[-4.1189, -4.1189]","-4.2433<br/>[-4.2433, -4.2433]","-4.1475<br/>[-4.1475, -4.1475]","-4.1653<br/>[-4.1653, -4.1653]","-4.1520<br/>[-4.1520, -4.1520]","-4.1558<br/>[-4.1558, -4.1558]","-4.1493<br/>[-4.1493, -4.1493]","-4.1448<br/>[-4.1448, -4.1448]","-4.1449<br/>[-4.1449, -4.1449]","-4.1447<br/>[-4.1447, -4.1447]","-4.1450<br/>[-4.1450, -4.1450]","-4.1194<br/>[-4.1194, -4.1194]","-4.1455<br/>[-4.1455, -4.1455]","-4.1530<br/>[-4.1530, -4.1530]","-4.1451<br/>[-4.1451, -4.1451]","-4.1609<br/>[-4.1609, -4.1609]","-4.1187<br/>[-4.1187, -4.1187]","-4.1559<br/>[-4.1559, -4.1559]","-4.1448<br/>[-4.1448, -4.1448]","-4.1565<br/>[-4.1565, -4.1565]","-4.1513<br/>[-4.1513, -4.1513]","-4.1463<br/>[-4.1463, -4.1463]","-4.1218<br/>[-4.1218, -4.1218]","-4.1814<br/>[-4.1814, -4.1814]","-4.1542<br/>[-4.1542, -4.1542]","-4.1448<br/>[-4.1448, -4.1448]","-4.1215<br/>[-4.1215, -4.1215]","-4.1448<br/>[-4.1448, -4.1448]","-4.1450<br/>[-4.1450, -4.1450]","-4.1451<br/>[-4.1451, -4.1451]","-4.1536<br/>[-4.1536, -4.1536]","-4.1646<br/>[-4.1646, -4.1646]","-4.2214<br/>[-4.2214, -4.2214]","-4.1448<br/>[-4.1448, -4.1448]","-4.1475<br/>[-4.1475, -4.1475]","-4.1554<br/>[-4.1554, -4.1554]","-4.1493<br/>[-4.1493, -4.1493]","-4.1475<br/>[-4.1475, -4.1475]","-4.1453<br/>[-4.1453, -4.1453]","-4.0311<br/>[-4.0311, -4.0311]","-4.0308<br/>[-4.0308, -4.0308]","-4.0308<br/>[-4.0308, -4.0308]","-4.0308<br/>[-4.0308, -4.0308]","-4.1227<br/>[-4.1227, -4.1227]","-4.0315<br/>[-4.0315, -4.0315]","-4.0427<br/>[-4.0427, -4.0427]","-4.0427<br/>[-4.0427, -4.0427]","-4.0426<br/>[-4.0426, -4.0426]","-4.0426<br/>[-4.0426, -4.0426]","-4.1216<br/>[-4.1216, -4.1216]","-4.2432<br/>[-4.2432, -4.2432]","-4.0443<br/>[-4.0443, -4.0443]","-4.1520<br/>[-4.1520, -4.1520]","-4.1450<br/>[-4.1450, -4.1450]","-4.0392<br/>[-4.0392, -4.0392]","-4.0330<br/>[-4.0330, -4.0330]","-4.0367<br/>[-4.0367, -4.0367]","-4.0444<br/>[-4.0444, -4.0444]","-4.0443<br/>[-4.0443, -4.0443]","-4.0384<br/>[-4.0384, -4.0384]"],["-1.9046<br/>[-1.9046, -1.9046]","-1.8260<br/>[-1.8260, -1.8256]","-1.2526<br/>[-1.2526, -1.2526]","-1.5449<br/>[-1.5449, -1.5449]","-1.6051<br/>[-1.6051, -1.6045]","-1.3708<br/>[-1.3708, -1.3708]","-1.9084<br/>[-1.9084, -1.9084]","-1.9373<br/>[-1.9373, -1.9373]","-1.4314<br/>[-1.4314, -1.4314]","-1.4603<br/>[-1.4603, -1.4603]","-1.7238<br/>[-1.7243, -1.7238]","-1.8959<br/>[-1.8959, -1.8959]","-1.7089<br/>[-1.7089, -1.7089]","-1.9852<br/>[-1.9852, -1.9844]","-1.8316<br/>[-1.8316, -1.8316]","-1.8595<br/>[-1.8595, -1.8595]","-1.5598<br/>[-1.5601, -1.5596]","-1.3855<br/>[-1.3855, -1.3855]","-2.0470<br/>[-2.0470, -2.0470]","-1.9540<br/>[-1.9540, -1.9540]","-2.3624<br/>[-2.3624, -2.3624]","-2.0805<br/>[-2.0805, -2.0796]","-2.2841<br/>[-2.2841, -2.2841]","-2.1070<br/>[-2.1070, -2.1070]","-2.0104<br/>[-2.0104, -2.0104]","-1.9294<br/>[-1.9294, -1.9289]","-2.9604<br/>[-2.9604, -2.9604]","-1.8397<br/>[-1.8397, -1.8397]","-2.1854<br/>[-2.1854, -2.1854]","-2.1746<br/>[-2.1746, -2.1613]","-2.0793<br/>[-2.0793, -2.0793]","-2.2006<br/>[-2.2006, -2.2006]","-1.7599<br/>[-1.7599, -1.7599]","-2.7585<br/>[-2.7585, -2.7585]","-2.0930<br/>[-2.0930, -2.0930]","-2.1859<br/>[-2.1859, -2.1859]","-2.2284<br/>[-2.2284, -2.2284]","-2.0254<br/>[-2.0255, -2.0253]","-2.3168<br/>[-2.3168, -2.3168]","-1.9078<br/>[-1.9079, -1.9078]","-3.0500<br/>[-3.0500, -3.0500]","-2.6382<br/>[-2.6382, -2.6382]","-2.6649<br/>[-2.6649, -2.6649]","-2.8692<br/>[-2.8692, -2.8692]","-2.5901<br/>[-2.5901, -2.5901]","-2.4356<br/>[-2.4356, -2.4356]","-2.1950<br/>[-2.1950, -2.1950]","-2.0570<br/>[-2.0570, -2.0570]","-2.1799<br/>[-2.1799, -2.1779]","-2.0440<br/>[-2.0440, -2.0440]","-1.4823<br/>[-1.4823, -1.4823]","-2.1849<br/>[-2.1849, -2.1849]","-2.7021<br/>[-2.7021, -2.7021]","-2.0770<br/>[-2.0770, -2.0770]","-2.7715<br/>[-2.7715, -2.7715]","-2.0879<br/>[-2.0879, -2.0879]","-2.4631<br/>[-2.4631, -2.4631]","-2.0697<br/>[-2.0697, -2.0697]","-2.2799<br/>[-2.2799, -2.2799]","-2.1397<br/>[-2.1397, -2.1397]","-2.3407<br/>[-2.3417, -2.3402]","-2.6580<br/>[-2.6580, -2.6580]","-2.9476<br/>[-2.9476, -2.9476]","-2.5834<br/>[-2.5866, -2.5833]","-2.1402<br/>[-2.1402, -2.1402]","-2.7120<br/>[-2.7123, -2.7107]","-2.2278<br/>[-2.2278, -2.2278]","-2.2678<br/>[-2.2695, -2.2652]","-2.2371<br/>[-2.2373, -2.2366]","-2.8463<br/>[-2.8463, -2.8463]","-2.8687<br/>[-2.8687, -2.8687]","-2.3135<br/>[-2.3135, -2.3135]","-2.3243<br/>[-2.3247, -2.3243]","-2.6735<br/>[-2.6735, -2.6734]","-2.8247<br/>[-2.8247, -2.8247]","-2.4157<br/>[-2.4160, -2.4157]","-2.7059<br/>[-2.7081, -2.7059]","-3.0877<br/>[-3.0967, -3.0545]","-2.4267<br/>[-2.4279, -2.4254]","-2.6192<br/>[-2.6210, -2.6174]","-2.6365<br/>[-2.6381, -2.6348]","-2.6310<br/>[-2.6327, -2.6292]","-2.2417<br/>[-2.2417, -2.2417]","-2.8553<br/>[-2.8553, -2.8553]","-3.0020<br/>[-3.0100, -2.9940]","-3.0067<br/>[-3.0146, -2.9985]","-3.0153<br/>[-3.0229, -3.0068]","-3.0198<br/>[-3.0273, -3.0112]","-2.7705<br/>[-2.7705, -2.7705]","-3.1527<br/>[-3.1527, -3.1527]","-1.3188<br/>[-1.3188, -1.3188]","-2.8329<br/>[-2.8329, -2.8329]","-2.4706<br/>[-2.4706, -2.4706]","-3.4738<br/>[-3.6701, -3.3269]","-3.4384<br/>[-3.4773, -3.4140]","-3.5945<br/>[-3.6830, -3.5334]","-3.5467<br/>[-3.5575, -3.5389]","-3.5545<br/>[-3.5653, -3.5466]","-3.4330<br/>[-3.4337, -3.4324]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Cells - Transformed fibroblasts","Thyroid","Heart - Atrial Appendage","Heart - Left Ventricle","Stomach","Ovary","Small Intestine - Terminal Ileum","Colon - Transverse","Nerve - Tibial","Uterus","Colon - Sigmoid","Adipose - Subcutaneous","Vagina","Spleen","Lung","Pancreas","Adipose - Visceral (Omentum)","Artery - Tibial","Breast - Mammary Tissue","Adrenal Gland","Muscle - Skeletal","Brain - Nucleus accumbens (basal ganglia)","Brain - Caudate (basal ganglia)","Skin - Sun Exposed (Lower leg)","Testis","Whole Blood","Brain - Cerebellum","Brain - Anterior cingulate cortex (BA24)","Brain - Cortex","Prostate","Esophagus - Muscularis","Artery - Aorta","Skin - Not Sun Exposed (Suprapubic)","Brain - Hippocampus","Pituitary","Esophagus - Mucosa","Brain - Hypothalamus","Cells - EBV-transformed lymphocytes","Brain - Cerebellar Hemisphere","Esophagus - Gastroesophageal Junction","Artery - Coronary","Brain - Putamen (basal ganglia)","Liver","Brain - Frontal Cortex (BA9)"],[6469,6542,6196,5937,5589,6229,4988,5542,6050,6023,6250,5988,6150,4289,5331,5450,6221,6418,5939,5998,5997,6489,6896,6087,6261,6237,6046,6285,6209,5948,6348,6723,6163,6100,6246,5944,6242,5270,5997,6144,6559,6973,6698,6576],[555.238904621375,433.056876703723,330.260202428436,277.730298311475,204.391686850645,196.12119666104,188.034594439918,187.924934352349,187.846869986805,187.834001835051,187.691455044164,187.12451589606,186.472788850558,167.917964772684,167.464246606493,166.741395542797,166.646793367143,166.568478540654,166.234009073673,166.198625085396,166.183084006876,166.151982819195,166.103715308588,166.096307987455,165.875506623547,165.784244732875,165.676290144781,165.658386818649,165.619150524192,165.530968335902,165.462571126327,160.297869933134,157.992857771636,157.644506657717,157.613472814748,153.121244480162,152.962104642068,93.4093089205635,93.3958946581014,93.3400864528292,93.1526637888612,39.8642344577721,28.5668393364205,-0.471200477657841],["-4.0341<br/>[-4.0341, -4.0341]","-4.0343<br/>[-4.0343, -4.0343]","-4.0342<br/>[-4.0342, -4.0342]","-4.0343<br/>[-4.0343, -4.0343]","-4.0304<br/>[-4.0304, -4.0304]","-4.0345<br/>[-4.0345, -4.0345]","-4.0304<br/>[-4.0304, -4.0304]","-4.0304<br/>[-4.0304, -4.0304]","-4.0300<br/>[-4.0300, -4.0300]","-4.0300<br/>[-4.0300, -4.0300]","-4.0302<br/>[-4.0302, -4.0302]","-4.0339<br/>[-4.0339, -4.0339]","-4.0302<br/>[-4.0302, -4.0302]","-4.0396<br/>[-4.0396, -4.0396]","-4.0391<br/>[-4.0391, -4.0391]","-4.0345<br/>[-4.0345, -4.0345]","-4.0343<br/>[-4.0343, -4.0343]","-4.0340<br/>[-4.0340, -4.0340]","-4.0339<br/>[-4.0339, -4.0339]","-4.0300<br/>[-4.0300, -4.0300]","-4.0301<br/>[-4.0301, -4.0301]","-4.0304<br/>[-4.0304, -4.0304]","-4.0302<br/>[-4.0302, -4.0302]","-4.0300<br/>[-4.0300, -4.0300]","-4.0302<br/>[-4.0302, -4.0302]","-4.0388<br/>[-4.0388, -4.0388]","-4.0347<br/>[-4.0347, -4.0347]","-4.0303<br/>[-4.0303, -4.0303]","-4.0303<br/>[-4.0303, -4.0303]","-4.0303<br/>[-4.0303, -4.0303]","-4.0302<br/>[-4.0302, -4.0302]","-4.0340<br/>[-4.0340, -4.0340]","-4.0342<br/>[-4.0342, -4.0342]","-4.0304<br/>[-4.0304, -4.0304]","-4.0302<br/>[-4.0302, -4.0302]","-4.0302<br/>[-4.0302, -4.0302]","-4.0303<br/>[-4.0303, -4.0303]","-4.0303<br/>[-4.0303, -4.0303]","-4.0304<br/>[-4.0304, -4.0304]","-4.0302<br/>[-4.0302, -4.0302]","-4.0298<br/>[-4.0298, -4.0298]","-4.0302<br/>[-4.0302, -4.0302]","-4.0347<br/>[-4.0347, -4.0347]","-4.0300<br/>[-4.0300, -4.0300]"],["-2.8536<br/>[-2.8542, -2.8533]","-3.0794<br/>[-3.1018, -3.0586]","-2.6156<br/>[-2.6156, -2.6154]","-3.2647<br/>[-3.2823, -3.2422]","-2.8001<br/>[-2.8206, -2.7781]","-2.9910<br/>[-2.9943, -2.9841]","-3.1981<br/>[-3.2153, -3.0622]","-3.2476<br/>[-3.2583, -3.2326]","-3.2909<br/>[-3.2986, -3.2798]","-3.2887<br/>[-3.2964, -3.2771]","-3.3021<br/>[-3.3092, -3.2949]","-3.1553<br/>[-3.1600, -3.1460]","-3.2309<br/>[-3.2389, -3.2152]","-2.9005<br/>[-2.9218, -2.8750]","-2.9998<br/>[-3.0120, -2.9801]","-3.1029<br/>[-3.1197, -3.0690]","-3.1745<br/>[-3.1863, -3.1602]","-3.1930<br/>[-3.2014, -3.1827]","-3.1529<br/>[-3.1627, -3.1365]","-3.2850<br/>[-3.2943, -3.2712]","-3.2827<br/>[-3.2926, -3.2681]","-3.3062<br/>[-3.3220, -3.2849]","-3.3329<br/>[-3.3482, -3.3122]","-3.2890<br/>[-3.2977, -3.2746]","-3.2864<br/>[-3.2957, -3.2698]","-3.0344<br/>[-3.0493, -2.9996]","-3.0855<br/>[-3.1161, -3.0237]","-3.2755<br/>[-3.2859, -3.2551]","-3.2674<br/>[-3.2779, -3.2454]","-3.2528<br/>[-3.2633, -3.2300]","-3.2838<br/>[-3.2935, -3.2626]","-2.9722<br/>[-2.9745, -2.9642]","-3.1503<br/>[-3.1771, -3.1195]","-3.2475<br/>[-3.2859, -3.2029]","-3.2623<br/>[-3.3007, -3.2217]","-3.0862<br/>[-3.1325, -3.0494]","-3.1339<br/>[-3.1638, -3.1000]","-3.1640<br/>[-3.2264, -3.1049]","-3.1850<br/>[-3.2656, -3.1077]","-3.2018<br/>[-3.2827, -3.1313]","-3.2672<br/>[-3.3215, -3.2133]","-3.3257<br/>[-3.4898, -3.1862]","-2.9716<br/>[-3.1081, -2.8770]","-3.9781<br/>[-4.4247, -3.5936]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Prostate","Whole_Blood","Pituitary","Spleen","Cells_EBV-transformed_lymphocytes","Colon_Sigmoid","Kidney_Cortex","Breast_Mammary_Tissue","Artery_Tibial","Artery_Aorta","Adipose_Subcutaneous","Adipose_Visceral_(Omentum)","Lung","Ovary","Brain_Cortex","Stomach","Pancreas","Brain_Nucleus_accumbens_(basal_ganglia)","Brain_Spinal_cord_(cervical_c-1)","Brain_Anterior_cingulate_cortex_(BA24)","Colon_Transverse","Skin_Sun_Exposed_(Lower_leg)","Small_Intestine_Terminal_Ileum","Skin_Not_Sun_Exposed_(Suprapubic)","Brain_Cerebellum","Thyroid","Uterus","Heart_Atrial_Appendage","Brain_Putamen_(basal_ganglia)","Brain_Caudate_(basal_ganglia)","Brain_Frontal_Cortex_(BA9)","Brain_Amygdala","Vagina","Testis","Esophagus_Mucosa","Muscle_Skeletal","Minor_Salivary_Gland","Brain_Hippocampus","Heart_Left_Ventricle","Cells_Transformed_fibroblasts","Brain_Cerebellar_Hemisphere","Brain_Substantia_nigra","Brain_Hypothalamus","Nerve_Tibial","Esophagus_Gastroesophageal_Junction","Artery_Coronary","Adrenal_Gland","Esophagus_Muscularis","Liver"],[9559,7501,10029,7573,8194,11390,9425,10329,13107,14909,10600,10338,8731,12111,12563,9758,8022,11876,13254,13626,7912,7155,9169,7237,11396,12535,13054,11594,12317,12247,14307,11744,7124,8159,7217,9014,8879,11498,11504,11252,14070,13263,12746,13001,14647,12968,12807,14820,8567],[927.988089328744,429.889964964424,428.497242375231,195.322405943156,195.069214330898,195.005173024821,134.175451479075,131.21771419028,130.993476956534,130.941194809987,71.2782797163727,71.2473403798365,63.3842564177242,15.3722874807407,0.00976675939336621,-0.00301799829835847,-0.00498412268892227,-0.00739098982882514,-0.0156083724494011,-0.0440359494031934,-0.128822173603702,-0.14216945692008,-0.147328545146393,-0.148249207543542,-0.250994056232207,-0.276149724749806,-0.279171039558497,-0.295083180944766,-0.301126530883968,-0.316946451683244,-0.424593738516589,-0.436893502630665,-0.489095732648753,-0.580159161548383,-0.644905505513625,-0.650823006500409,-0.666755594259197,-0.690343966998776,-0.707973125853927,-0.718004655497997,-0.746473957812506,-0.755536922158279,-0.755623269434881,-0.761383617592166,-0.763481306515559,-0.766950319711011,-0.768203982803194,-0.76963624832995,-65.8759706364891],["-4.0591<br/>[-4.0591, -4.0591]","-4.0424<br/>[-4.0424, -4.0424]","-4.0326<br/>[-4.0326, -4.0326]","-4.0349<br/>[-4.0349, -4.0349]","-4.0384<br/>[-4.0384, -4.0384]","-4.0318<br/>[-4.0318, -4.0318]","-4.0804<br/>[-4.0804, -4.0804]","-4.0282<br/>[-4.0282, -4.0282]","-4.0273<br/>[-4.0273, -4.0273]","-4.0273<br/>[-4.0273, -4.0273]","-4.0320<br/>[-4.0320, -4.0320]","-4.0319<br/>[-4.0319, -4.0319]","-4.0288<br/>[-4.0288, -4.0288]","-4.0351<br/>[-4.0351, -4.0351]","-4.0333<br/>[-4.0333, -4.0333]","-4.0325<br/>[-4.0325, -4.0325]","-4.0333<br/>[-4.0333, -4.0333]","-4.0294<br/>[-4.0294, -4.0294]","-4.0291<br/>[-4.0291, -4.0291]","-4.0329<br/>[-4.0329, -4.0329]","-4.0290<br/>[-4.0290, -4.0290]","-4.0335<br/>[-4.0335, -4.0335]","-4.0287<br/>[-4.0287, -4.0287]","-4.0334<br/>[-4.0334, -4.0334]","-4.0281<br/>[-4.0281, -4.0281]","-4.0319<br/>[-4.0319, -4.0319]","-4.0294<br/>[-4.0294, -4.0294]","-4.0317<br/>[-4.0317, -4.0317]","-4.0276<br/>[-4.0276, -4.0276]","-4.0276<br/>[-4.0276, -4.0276]","-4.0307<br/>[-4.0307, -4.0307]","-4.0298<br/>[-4.0298, -4.0298]","-4.0296<br/>[-4.0296, -4.0296]","-4.0292<br/>[-4.0292, -4.0292]","-4.0291<br/>[-4.0291, -4.0291]","-4.0287<br/>[-4.0287, -4.0287]","-4.0286<br/>[-4.0286, -4.0286]","-4.0279<br/>[-4.0279, -4.0279]","-4.0278<br/>[-4.0278, -4.0278]","-4.0279<br/>[-4.0279, -4.0279]","-4.0271<br/>[-4.0271, -4.0271]","-4.0271<br/>[-4.0271, -4.0271]","-4.0272<br/>[-4.0272, -4.0272]","-4.0271<br/>[-4.0271, -4.0271]","-4.0268<br/>[-4.0268, -4.0268]","-4.0271<br/>[-4.0271, -4.0271]","-4.0271<br/>[-4.0271, -4.0271]","-4.0267<br/>[-4.0267, -4.0267]","-4.0639<br/>[-4.0639, -4.0639]"],["-2.8759<br/>[-2.8759, -2.8758]","-3.0152<br/>[-3.0229, -3.0064]","-3.2945<br/>[-3.2992, -3.2883]","-3.4197<br/>[-3.4614, -3.3849]","-2.5588<br/>[-2.5597, -2.5530]","-3.6900<br/>[-3.7109, -3.6619]","-3.0757<br/>[-3.1186, -3.0325]","-3.4957<br/>[-3.5141, -3.4675]","-3.6046<br/>[-3.6196, -3.5807]","-3.6281<br/>[-3.6525, -3.5921]","-3.4420<br/>[-3.5261, -3.3616]","-3.4503<br/>[-3.5202, -3.3756]","-2.8850<br/>[-2.8914, -2.8676]","-3.0594<br/>[-3.0673, -3.0376]","-3.6925<br/>[-3.8851, -3.5448]","-3.6620<br/>[-3.8403, -3.5130]","-3.5697<br/>[-3.8295, -3.3974]","-3.7016<br/>[-3.8642, -3.5623]","-3.6456<br/>[-4.0259, -3.4551]","-3.7280<br/>[-3.9163, -3.5813]","-3.7003<br/>[-3.8532, -3.5029]","-3.7173<br/>[-3.8208, -3.5673]","-3.7163<br/>[-3.8983, -3.5009]","-3.7232<br/>[-3.8257, -3.5773]","-3.8273<br/>[-3.9599, -3.6676]","-3.8273<br/>[-4.0045, -3.6256]","-3.8063<br/>[-4.0841, -3.5754]","-3.8573<br/>[-3.9914, -3.6880]","-3.8679<br/>[-3.9974, -3.7080]","-3.8834<br/>[-4.0015, -3.7272]","-3.9757<br/>[-4.0954, -3.8301]","-3.9457<br/>[-4.2054, -3.7229]","-4.0169<br/>[-4.4392, -3.6580]","-4.2075<br/>[-4.6529, -3.8487]","-4.6686<br/>[-5.1226, -4.2528]","-4.4255<br/>[-4.8065, -4.0627]","-4.5260<br/>[-4.9218, -4.1470]","-4.4203<br/>[-4.7118, -4.1062]","-4.4815<br/>[-4.8338, -4.1322]","-4.5553<br/>[-4.9128, -4.2009]","-4.4909<br/>[-4.7629, -4.1946]","-4.5785<br/>[-4.9053, -4.2449]","-4.6296<br/>[-4.9559, -4.2847]","-4.6357<br/>[-4.9735, -4.2887]","-4.5162<br/>[-4.8321, -4.1800]","-4.6787<br/>[-5.0092, -4.3296]","-4.7055<br/>[-5.0466, -4.3477]","-4.5338<br/>[-4.8450, -4.2054]","-3.0248<br/>[-3.1110, -2.9490]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["19","20","14","7","13","17","12","1","3","18","8","11","2","5","6","10","15","9","16","4"],[6140,8488,9729,8568,7898,8837,7522,7736,10529,7626,10539,10758,9295,11895,9255,8702,8065,9930,10627,12826],[1362.96176829282,774.81140641051,516.454754656563,432.605906091826,360.266593691363,195.947957744102,166.498379526894,86.6696468223767,71.9405789689092,59.0734453691588,44.0170911081659,4.32687483203511,0.0170983760202163,-0.00130161143972428,-0.152573775384829,-0.360928796293996,-0.650612168329247,-0.713929737512444,-0.757134554716549,-0.834467761610733],["-4.1503<br/>[-4.1503, -4.1503]","-4.0427<br/>[-4.0427, -4.0427]","-4.0370<br/>[-4.0370, -4.0370]","-4.0330<br/>[-4.0330, -4.0330]","-4.0427<br/>[-4.0427, -4.0427]","-4.0313<br/>[-4.0313, -4.0313]","-4.0381<br/>[-4.0381, -4.0381]","-4.0410<br/>[-4.0410, -4.0410]","-4.0359<br/>[-4.0359, -4.0359]","-4.0368<br/>[-4.0368, -4.0368]","-4.0370<br/>[-4.0370, -4.0370]","-4.0327<br/>[-4.0327, -4.0327]","-4.0337<br/>[-4.0337, -4.0337]","-4.0343<br/>[-4.0343, -4.0343]","-4.0332<br/>[-4.0332, -4.0332]","-4.0335<br/>[-4.0335, -4.0335]","-4.0291<br/>[-4.0291, -4.0291]","-4.0289<br/>[-4.0289, -4.0289]","-4.0282<br/>[-4.0282, -4.0282]","-4.0275<br/>[-4.0275, -4.0275]"],["-2.2494<br/>[-2.2494, -2.2494]","-2.7729<br/>[-2.7731, -2.7724]","-3.2607<br/>[-3.2727, -3.2457]","-3.1705<br/>[-3.1887, -3.1610]","-3.1172<br/>[-3.1234, -3.1118]","-3.3120<br/>[-3.3794, -3.2338]","-3.1580<br/>[-3.1637, -3.1490]","-2.8046<br/>[-2.8059, -2.7901]","-3.3075<br/>[-3.3944, -3.2335]","-2.4585<br/>[-2.4630, -2.4516]","-3.2826<br/>[-3.3022, -3.2734]","-2.3477<br/>[-2.4135, -2.3398]","-3.4564<br/>[-3.8209, -3.1405]","-3.6104<br/>[-3.8202, -3.4483]","-3.6106<br/>[-3.8752, -3.4279]","-3.5397<br/>[-3.7909, -3.3642]","-4.0143<br/>[-4.3761, -3.7225]","-4.0807<br/>[-4.5301, -3.7347]","-4.1736<br/>[-4.4640, -3.8606]","-4.2832<br/>[-4.6341, -3.9234]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">sessionInfo</span>()</a></code></pre></div>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.5

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] DT_0.4         plyr_1.8.4     dplyr_0.7.5    R.matlab_3.6.1

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.17      compiler_3.5.0    pillar_1.2.3     
 [4] later_0.7.3       git2r_0.21.0      workflowr_1.0.1  
 [7] bindr_0.1.1       R.methodsS3_1.7.1 R.utils_2.6.0    
[10] tools_3.5.0       digest_0.6.15     jsonlite_1.5     
[13] evaluate_0.10.1   tibble_1.4.2      pkgconfig_2.0.1  
[16] rlang_0.2.1       shiny_1.1.0       crosstalk_1.0.0  
[19] yaml_2.1.19       bindrcpp_0.2.2    stringr_1.3.1    
[22] knitr_1.20        htmlwidgets_1.2   rprojroot_1.3-2  
[25] tidyselect_0.2.4  glue_1.2.0        R6_2.2.2         
[28] rmarkdown_1.10    purrr_0.2.5       magrittr_1.5     
[31] whisker_0.3-2     backports_1.1.2   promises_1.0.1   
[34] htmltools_0.3.6   assertthat_0.2.0  mime_0.5         
[37] xtable_1.8-2      httpuv_1.4.4.1    stringi_1.2.3    
[40] R.oo_1.22.0      </code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.0.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
