<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of neuroticism</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; position: absolute; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine:empty
  { position: absolute; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of neuroticism</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-06-27</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/031b9536b9719e2b0078a2da786db0b128eba48d" target="_blank">031b953</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Unstaged changes:
    Modified:   analysis/index.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/031b9536b9719e2b0078a2da786db0b128eba48d/analysis/neu_2016.Rmd" target="_blank">031b953</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-06-27
</td>
<td style="text-align:left;">
wflow_publish(“neu_2016.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/27089181">“Genetic Variants Associated with Subjective Well-Being, Depressive Symptoms, and Neuroticism Identified Through Genome-Wide Analyses” (<em>Nature Genetics</em>, 2016)</a>. The summary data file is available at <a href="https://www.thessgac.org/data" class="uri">https://www.thessgac.org/data</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["MCM complex<br/>(kegg, BioSystems)","Digestion of dietary carbohydrate<br/>(reactome, PC)","Unwinding of DNA<br/>(reactome, PC)","CDK Regulation of DNA Replication<br/>(biocarta, BioCarta)","2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA<br/>(humancyc, PC)","tetrapyrrole biosynthesis<br/>(biocyc, BioSystems)","Interactions of Tat with host cellular proteins<br/>(reactome, PC)","Unwinding of DNA<br/>(reactome, BioSystems)","Synthesis, secretion, and inactivation of Glucose-dependent Insulinotropic Polypeptide (GIP)<br/>(reactome, PC)","epoxysqualene biosynthesis<br/>(humancyc, PC)","Axonal growth inhibition (RHOA activation)<br/>(reactome, PC)","p75NTR regulates axonogenesis<br/>(reactome, PC)","p75(NTR)-mediated signaling<br/>(pid, PC)","Extrinsic Pathway<br/>(reactome, PC)","Transcription from mitochondrial promoters<br/>(reactome, PC)","Incretin synthesis, secretion, and inactivation<br/>(reactome, PC)","Beta oxidation of myristoyl-CoA to lauroyl-CoA<br/>(reactome, PC)","Autointegration results in viral DNA circles<br/>(reactome, PC)","Binding and entry of HIV virion<br/>(reactome, PC)","FOXA transcription factor networks<br/>(pid, PC)","phenylalanine degradation/tyrosine biosynthesis<br/>(biocyc, BioSystems)","Opposing roles of AIF in Apoptosis and Cell Survival<br/>(biocarta, BioCarta)","NAD phosphorylation and dephosphorylation<br/>(humancyc, PC)","Beta oxidation of palmitoyl-CoA to myristoyl-CoA<br/>(reactome, PC)","Tandem of pore domain in a weak inwardly rectifying K+ channels (TWIK)<br/>(reactome, PC)","Glycosphingolipid biosynthesis, globo-series, LacCer => Gb4Cer<br/>(kegg, BioSystems)","Degradation of aromatic compounds<br/>(kegg, BioSystems)","HIV-1 defeats host-mediated resistance by CEM15<br/>(biocarta, BioCarta)","Exosome, archaea<br/>(kegg, BioSystems)","Nef mediated downregulation of CD28 cell surface expression<br/>(reactome, PC)","Lactose synthesis<br/>(reactome, BioSystems)","cardiolipin biosynthesis<br/>(biocyc, BioSystems)","Calcitonin-like ligand receptors<br/>(reactome, PC)","E-cadherin signaling in keratinocytes<br/>(pid, PC)","Glypican 2 network<br/>(pid, PC)","Glypican pathway<br/>(pid, PC)","Mitochondrial Uncoupling Proteins<br/>(reactome, PC)","Succinate to proprionate conversion<br/>(panther, PC)","The AIM2 inflammasome<br/>(reactome, PC)","Relaxin receptors<br/>(reactome, PC)","Defective MUT causes methylmalonic aciduria mut type<br/>(reactome, BioSystems)","Abasic sugar-phosphate removal via the single-nucleotide replacement pathway<br/>(reactome, BioSystems)","Antisense Pathway<br/>(biocarta, BioCarta)","Cytochrome bc1 complex respiratory unit<br/>(kegg, BioSystems)","C6 deamination of adenosine<br/>(reactome, PC)","Hop Pathway in Cardiac Development<br/>(biocarta, BioCarta)","trans, trans-farnesyl diphosphate biosynthesis<br/>(humancyc, PC)","Cap binding complex<br/>(kegg, BioSystems)","Apoptosis<br/>(reactome, BioSystems)","Programmed Cell Death<br/>(reactome, BioSystems)","BTG family proteins and cell cycle regulation<br/>(biocarta, BioCarta)","Apoptosis<br/>(reactome, PC)","Regulation of Apoptosis<br/>(reactome, PC)","Sulfoglycolipids biosynthesis, ceramide/1-alkyl-2-acylglycerol => sulfatide/seminolipid<br/>(kegg, BioSystems)","Lipoic acid metabolism<br/>(kegg, PC)","PLK2 and PLK4 events<br/>(pid, PC)","Role of second messengers in netrin-1 signaling<br/>(reactome, PC)","cholesterol biosynthesis I<br/>(humancyc, PC)","Signaling mediated by p38-gamma and p38-delta<br/>(pid, PC)","APEX1-Independent Resolution of AP Sites via the Single Nucleotide Replacement Pathway<br/>(reactome, BioSystems)","Role of DCC in regulating apoptosis<br/>(reactome, PC)","Trafficking and processing of endosomal TLR<br/>(reactome, PC)","Caspase-mediated cleavage of cytoskeletal proteins<br/>(reactome, PC)","Netrin mediated repulsion signals<br/>(reactome, PC)","Cleavage of the damaged pyrimidine<br/>(reactome, BioSystems)","Deregulation of CDK5 in Alzheimers Disease<br/>(biocarta, BioCarta)","DCC mediated attractive signaling<br/>(reactome, PC)","DSCAM interactions<br/>(reactome, PC)","Base-Excision Repair, AP Site Formation<br/>(reactome, BioSystems)","Caspase activation via extrinsic apoptotic signalig pathway<br/>(reactome, BioSystems)","Cholesterol biosynthesis<br/>(panther, PC)","Proteasome, 19S regulatory particle (PA700)<br/>(kegg, BioSystems)","Netrin-1 signaling<br/>(reactome, PC)","Antigen processing and presentation<br/>(kegg, BioSystems)","Integrated Pancreatic Cancer Pathway<br/>(wiki, BioSystems)","Axon guidance mediated by Slit/Robo<br/>(panther, PC)","Netrin-mediated signaling events<br/>(pid, PC)","Cell-Cell communication<br/>(reactome, PC)","Cell-Cell communication<br/>(reactome, BioSystems)","Axon guidance mediated by netrin<br/>(panther, PC)","Steroid biosynthesis<br/>(kegg, PC)","Steroid biosynthesis<br/>(kegg, BioSystems)","Beta defensins<br/>(reactome, PC)","Beta defensins<br/>(reactome, BioSystems)","Axon guidance<br/>(kegg, BioSystems)","Defensins<br/>(reactome, PC)","Defensins<br/>(reactome, BioSystems)","Colorectal cancer<br/>(kegg, BioSystems)","Nuclear pore complex<br/>(kegg, BioSystems)","Cross-presentation of soluble exogenous antigens (endosomes)<br/>(reactome, PC)","Cross-presentation of soluble exogenous antigens (endosomes)<br/>(reactome, BioSystems)","Rev-mediated nuclear export of HIV RNA<br/>(reactome, PC)","Proteasome<br/>(kegg, BioSystems)","Regulation of Apoptosis<br/>(reactome, BioSystems)","Interactions of Rev with host cellular proteins<br/>(reactome, PC)","Rev-mediated nuclear export of HIV RNA<br/>(reactome, BioSystems)","Role of nicotinic acetylcholine receptors in the regulation of apoptosis<br/>(biocarta, BioCarta)","Interactions of Rev with host cellular proteins<br/>(reactome, BioSystems)","Arachidonate Epoxygenase / Epoxide Hydrolase<br/>(wiki, BioSystems)","spermine and spermidine degradation I<br/>(biocyc, BioSystems)"],[6,9,11,12,2,3,2,12,12,2,8,9,62,4,4,21,3,3,4,3,3,3,2,3,3,2,2,3,4,2,3,2,10,21,2,2,3,2,3,8,2,2,3,2,2,4,4,2,154,157,3,149,56,2,3,2,9,11,11,7,9,11,12,10,8,11,14,11,10,26,11,18,41,74,190,18,32,126,127,30,14,18,40,41,122,49,50,61,45,47,48,31,44,49,33,32,16,34,7,3],[480,378,1025,925,163,285,143,1104,1197,233,652,803,6029,290,331,1931,133,197,257,246,280,286,127,160,312,180,204,182,313,150,350,186,979,2396,96,175,176,138,200,663,150,221,147,122,261,435,201,201,14212,14466,230,13486,4984,172,248,122,1715,1281,928,563,1642,962,1122,2101,702,1113,3028,2376,815,3188,971,1366,7347,2908,19483,3690,5513,19033,19033,4629,1504,1964,930,930,18540,1207,1207,7181,3655,3143,3148,2699,3095,3509,2855,2794,1814,2950,643,192],[190.349225776758,168.864176219219,130.857552651858,116.175146825028,96.8615214900348,83.7583629004608,79.9990793618097,78.0109715609269,55.2543673893159,47.6533779896539,42.2292397284462,42.067167334342,40.7604299580373,35.5760235609061,32.3720480960201,31.7733760671916,31.1330744678135,29.9698516978958,27.3405133763317,23.1367038708271,22.5673786742413,21.3295494739836,20.3173781530275,19.4752059294431,18.8876780081182,18.6511333404338,18.2497591058326,16.9960849519533,15.6818259539155,14.9236258717373,14.0822638067041,14.01425859495,11.0580926101448,10.5945728160063,9.95054166406813,9.23689416202616,8.21271523653057,8.11111741728113,8.03255634670328,8.02853848577014,7.81286034579526,7.35772872452721,6.9693593100872,6.75083279625696,6.46416090556408,4.69503563430814,4.2571375140471,3.66092419385686,3.08538126483387,3.03674146082706,2.77819873656626,2.66042687475542,2.02774748218552,1.89916910134435,1.89269017146282,1.77770742573459,1.68982880676351,1.61210530643021,1.6067032246408,1.60658011822186,1.5744914015217,1.55388519935816,1.54717219728066,1.52257432682908,1.51416174188173,1.51272725376139,1.49496827897905,1.45126340016639,1.44902049623767,1.44232345420426,1.43377440417223,1.36939622639712,1.30742349423208,1.2569996472916,1.25666861401583,1.23858413593948,1.22141206301524,1.22042620737607,1.22042620737607,1.20602468007555,1.19316615282112,1.08083225861214,1.07977085280249,1.07977085280249,1.05762398463543,1.01021709245082,1.01021709245082,0.955711949996216,0.92562454405279,0.873643378327604,0.872147979552199,0.834906430605899,0.828764169438142,0.826255102546295,0.817640962796467,0.808581236321452,0.800118500776996,0.792209796852266,-0.249563577614721,-0.336184446167102],["-4.2692<br/>[-4.4624, -4.1703]","-4.2692<br/>[-4.4624, -4.1703]","-4.2691<br/>[-4.4623, -4.1703]","-4.2691<br/>[-4.4623, -4.1702]","-4.2693<br/>[-4.4625, -4.1705]","-4.2697<br/>[-4.4629, -4.1709]","-4.2693<br/>[-4.4625, -4.1704]","-4.2691<br/>[-4.4623, -4.1702]","-4.2771<br/>[-4.4742, -4.1769]","-4.2765<br/>[-4.4739, -4.1762]","-4.2764<br/>[-4.4736, -4.1761]","-4.2764<br/>[-4.4736, -4.1761]","-4.2752<br/>[-4.4724, -4.1750]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2770<br/>[-4.4740, -4.1767]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1705]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2692<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2695<br/>[-4.4626, -4.1707]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2694<br/>[-4.4636, -4.1705]","-4.2691<br/>[-4.4623, -4.1703]","-4.2691<br/>[-4.4622, -4.1703]","-4.2693<br/>[-4.4625, -4.1705]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2692<br/>[-4.4624, -4.1703]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2765<br/>[-4.4738, -4.1761]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1704]","-4.2854<br/>[-4.4791, -4.1852]","-4.2853<br/>[-4.4790, -4.1851]","-4.2693<br/>[-4.4625, -4.1704]","-4.2844<br/>[-4.4782, -4.1841]","-4.2818<br/>[-4.4763, -4.1815]","-4.2693<br/>[-4.4625, -4.1704]","-4.2693<br/>[-4.4625, -4.1705]","-4.2695<br/>[-4.4626, -4.1706]","-4.2747<br/>[-4.4734, -4.1751]","-4.2768<br/>[-4.4740, -4.1764]","-4.2691<br/>[-4.4622, -4.1702]","-4.2764<br/>[-4.4738, -4.1761]","-4.2745<br/>[-4.4732, -4.1748]","-4.2768<br/>[-4.4741, -4.1765]","-4.2690<br/>[-4.4622, -4.1702]","-4.2745<br/>[-4.4732, -4.1748]","-4.2764<br/>[-4.4737, -4.1760]","-4.2690<br/>[-4.4622, -4.1701]","-4.2746<br/>[-4.4734, -4.1749]","-4.2744<br/>[-4.4731, -4.1747]","-4.2764<br/>[-4.4737, -4.1760]","-4.2750<br/>[-4.4736, -4.1753]","-4.2764<br/>[-4.4737, -4.1760]","-4.2770<br/>[-4.4661, -4.1774]","-4.2761<br/>[-4.4741, -4.1766]","-4.2772<br/>[-4.4741, -4.1769]","-4.2719<br/>[-4.4696, -4.1729]","-4.2740<br/>[-4.4726, -4.1744]","-4.2743<br/>[-4.4728, -4.1747]","-4.2717<br/>[-4.4694, -4.1727]","-4.2717<br/>[-4.4694, -4.1727]","-4.2741<br/>[-4.4726, -4.1745]","-4.2761<br/>[-4.4734, -4.1758]","-4.2760<br/>[-4.4733, -4.1757]","-4.2695<br/>[-4.4626, -4.1706]","-4.2695<br/>[-4.4626, -4.1706]","-4.2745<br/>[-4.4719, -4.1754]","-4.2694<br/>[-4.4625, -4.1705]","-4.2694<br/>[-4.4625, -4.1705]","-4.2731<br/>[-4.4716, -4.1736]","-4.2696<br/>[-4.4636, -4.1710]","-4.2768<br/>[-4.4657, -4.1772]","-4.2768<br/>[-4.4657, -4.1772]","-4.2698<br/>[-4.4638, -4.1711]","-4.2767<br/>[-4.4656, -4.1771]","-4.2768<br/>[-4.4656, -4.1773]","-4.2698<br/>[-4.4638, -4.1711]","-4.2697<br/>[-4.4638, -4.1711]","-4.2751<br/>[-4.4650, -4.1754]","-4.2697<br/>[-4.4637, -4.1711]","-4.2699<br/>[-4.4629, -4.1711]","-4.2693<br/>[-4.4625, -4.1704]"],["-1.3766<br/>[-1.3766, -1.3761]","-1.6503<br/>[-1.6535, -1.6442]","-1.5698<br/>[-1.5699, -1.5696]","-1.6727<br/>[-1.6727, -1.6727]","-1.2920<br/>[-1.2936, -1.2864]","-2.1407<br/>[-2.1534, -2.1240]","-1.3538<br/>[-1.3683, -1.3366]","-1.5924<br/>[-1.5924, -1.5922]","-2.4000<br/>[-2.4682, -2.3483]","-1.8589<br/>[-1.8860, -1.8277]","-2.4608<br/>[-2.5059, -2.4137]","-2.5431<br/>[-2.5955, -2.4851]","-3.3638<br/>[-3.4490, -3.2488]","-2.1241<br/>[-2.1596, -2.0843]","-2.1786<br/>[-2.2158, -2.1291]","-2.5948<br/>[-2.6661, -2.5377]","-1.5039<br/>[-1.5190, -1.4888]","-1.9610<br/>[-1.9902, -1.9193]","-1.7776<br/>[-1.8006, -1.7467]","-1.9471<br/>[-2.0684, -1.8334]","-2.1151<br/>[-2.1439, -2.0759]","-2.1188<br/>[-2.1529, -2.0728]","-1.4765<br/>[-1.4978, -1.4563]","-1.7079<br/>[-1.7242, -1.6873]","-2.1656<br/>[-2.1909, -2.1302]","-1.9158<br/>[-1.9517, -1.8700]","-1.9797<br/>[-2.0066, -1.9458]","-1.6082<br/>[-1.6475, -1.5670]","-2.0272<br/>[-2.1438, -1.9354]","-1.3661<br/>[-1.3867, -1.3416]","-2.1959<br/>[-2.2404, -2.1442]","-1.7433<br/>[-1.7809, -1.7091]","-2.8820<br/>[-2.9804, -2.7525]","-2.7213<br/>[-2.7230, -2.7174]","-1.5980<br/>[-1.6660, -1.5323]","-1.9172<br/>[-1.9408, -1.8825]","-1.9031<br/>[-1.9420, -1.8599]","-1.8195<br/>[-1.8385, -1.7947]","-1.7432<br/>[-1.8235, -1.7051]","-2.4724<br/>[-2.5186, -2.4132]","-1.8524<br/>[-1.8748, -1.8240]","-1.8238<br/>[-1.8626, -1.7804]","-1.8194<br/>[-1.8619, -1.7687]","-1.7496<br/>[-1.7835, -1.7037]","-2.0700<br/>[-2.1127, -2.0183]","-2.0044<br/>[-2.0570, -1.9507]","-1.9543<br/>[-2.0000, -1.8986]","-1.9454<br/>[-2.0001, -1.8873]","-3.2101<br/>[-3.3869, -3.0924]","-3.2201<br/>[-3.3941, -3.1003]","-1.9740<br/>[-2.0571, -1.8911]","-3.2366<br/>[-3.3991, -3.1223]","-3.1289<br/>[-3.3109, -3.0073]","-1.9140<br/>[-1.9337, -1.8867]","-1.8861<br/>[-1.9156, -1.8529]","-1.5397<br/>[-1.6058, -1.4744]","-3.0192<br/>[-3.2077, -2.8262]","-2.8467<br/>[-3.0678, -2.7177]","-2.9079<br/>[-2.9652, -2.8094]","-2.7056<br/>[-2.7493, -2.6303]","-3.1027<br/>[-3.2081, -2.9600]","-2.8264<br/>[-2.9478, -2.6989]","-2.9714<br/>[-3.0462, -2.8630]","-3.1546<br/>[-3.3102, -2.9784]","-2.7976<br/>[-2.8445, -2.7135]","-3.0018<br/>[-3.0458, -2.9289]","-3.2106<br/>[-3.4280, -3.0241]","-3.2222<br/>[-3.3598, -3.0442]","-2.8624<br/>[-2.9092, -2.7805]","-3.2634<br/>[-3.4410, -3.0807]","-2.8850<br/>[-2.9832, -2.7569]","-2.7745<br/>[-2.9332, -2.6478]","-3.4551<br/>[-3.6726, -3.2745]","-3.1520<br/>[-3.3918, -2.9792]","-3.7198<br/>[-3.9004, -3.5576]","-3.4274<br/>[-3.5495, -3.2750]","-3.4523<br/>[-3.6610, -3.2447]","-3.7414<br/>[-3.8836, -3.5783]","-3.7414<br/>[-3.8836, -3.5783]","-3.4616<br/>[-3.6235, -3.2793]","-3.1178<br/>[-3.1740, -3.0304]","-3.2284<br/>[-3.2891, -3.1357]","-2.9075<br/>[-2.9650, -2.8219]","-2.9075<br/>[-2.9650, -2.8219]","-3.8042<br/>[-4.0445, -3.5993]","-3.0240<br/>[-3.0773, -2.9397]","-3.0240<br/>[-3.0773, -2.9397]","-3.6931<br/>[-3.8221, -3.5393]","-3.0908<br/>[-3.4462, -2.8624]","-3.0836<br/>[-3.3250, -2.9170]","-3.0845<br/>[-3.3262, -2.9180]","-3.1146<br/>[-3.3785, -2.9549]","-3.1077<br/>[-3.3431, -2.9610]","-3.1171<br/>[-3.3748, -2.9448]","-3.1287<br/>[-3.3919, -2.9577]","-3.1335<br/>[-3.3853, -2.9683]","-2.9863<br/>[-3.2316, -2.8250]","-3.1469<br/>[-3.4285, -2.9832]","-3.0627<br/>[-4.1493, -2.7545]","-3.6543<br/>[-5.3975, -3.0410]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Thyroid","Brain - Cerebellum","Brain - Cerebellar Hemisphere","Heart - Atrial Appendage","Brain - Hippocampus","Brain - Anterior cingulate cortex (BA24)","Brain - Cortex","Brain - Nucleus accumbens (basal ganglia)","Brain - Frontal Cortex (BA9)","Brain - Caudate (basal ganglia)","Brain - Hypothalamus","Brain - Putamen (basal ganglia)","Adrenal Gland","Cells - EBV-transformed lymphocytes","Pituitary","Liver","Muscle - Skeletal","Heart - Left Ventricle","Whole Blood","Spleen","Esophagus - Mucosa","Vagina","Small Intestine - Terminal Ileum","Lung","Colon - Transverse","Pancreas","Esophagus - Gastroesophageal Junction","Stomach","Ovary","Colon - Sigmoid","Uterus","Esophagus - Muscularis","Artery - Tibial","Cells - Transformed fibroblasts","Prostate","Testis","Breast - Mammary Tissue","Nerve - Tibial","Adipose - Visceral (Omentum)","Adipose - Subcutaneous","Skin - Not Sun Exposed (Suprapubic)","Skin - Sun Exposed (Lower leg)","Artery - Coronary","Artery - Aorta"],[6915,6553,6396,6701,6531,6703,6666,6862,6930,7294,6566,7369,6416,5602,6617,7023,6461,6442,6613,4664,6242,6483,5317,5614,5908,5782,6485,5958,6682,6604,6420,6695,6720,6797,6359,6805,6341,6436,6586,6374,6607,6522,6891,7004],[0.509008764346557,0.473190757900193,0.472945182481959,0.259296193450623,0.155902038689513,0.131310368397108,0.127078922752595,0.101977078261649,0.0913210315802674,0.0571382140618579,0.0552493911442937,0.0520563723667016,-0.156080285612782,-0.168192941126796,-0.24491592813199,-0.246886555998921,-0.384941590273925,-0.452359130049513,-0.461306420969293,-0.527583085208731,-0.552634019159298,-0.559849533831099,-0.564678576065957,-0.567291462437769,-0.570702460576614,-0.579075853244744,-0.587810228717329,-0.590054175456487,-0.591389286678712,-0.592162700156783,-0.594877118434499,-0.59615460271647,-0.599264521917624,-0.603020876808734,-0.603312173626027,-0.60500391027972,-0.60514177046508,-0.607142717762225,-0.607250294679058,-0.608231321001344,-0.612181875468851,-0.613984892758948,-0.617772118240034,-0.627828074677366],["-4.2814<br/>[-4.4808, -4.1800]","-4.2747<br/>[-4.4698, -4.1750]","-4.2747<br/>[-4.4698, -4.1750]","-4.2731<br/>[-4.4623, -4.1735]","-4.2771<br/>[-4.4711, -4.1776]","-4.2770<br/>[-4.4710, -4.1776]","-4.2770<br/>[-4.4710, -4.1775]","-4.2770<br/>[-4.4709, -4.1775]","-4.2769<br/>[-4.4709, -4.1774]","-4.2768<br/>[-4.4708, -4.1774]","-4.2766<br/>[-4.4707, -4.1770]","-4.2768<br/>[-4.4708, -4.1774]","-4.2747<br/>[-4.4699, -4.1749]","-4.2743<br/>[-4.4696, -4.1745]","-4.2740<br/>[-4.4692, -4.1742]","-4.2741<br/>[-4.4692, -4.1743]","-4.2705<br/>[-4.4622, -4.1720]","-4.2733<br/>[-4.4625, -4.1736]","-4.2734<br/>[-4.4625, -4.1737]","-4.2682<br/>[-4.4612, -4.1694]","-4.2682<br/>[-4.4611, -4.1694]","-4.2681<br/>[-4.4611, -4.1693]","-4.2680<br/>[-4.4609, -4.1692]","-4.2680<br/>[-4.4609, -4.1692]","-4.2680<br/>[-4.4609, -4.1693]","-4.2679<br/>[-4.4608, -4.1691]","-4.2679<br/>[-4.4608, -4.1692]","-4.2678<br/>[-4.4607, -4.1690]","-4.2680<br/>[-4.4608, -4.1693]","-4.2679<br/>[-4.4607, -4.1692]","-4.2678<br/>[-4.4607, -4.1691]","-4.2679<br/>[-4.4607, -4.1692]","-4.2678<br/>[-4.4607, -4.1691]","-4.2678<br/>[-4.4606, -4.1691]","-4.2677<br/>[-4.4606, -4.1689]","-4.2678<br/>[-4.4606, -4.1691]","-4.2677<br/>[-4.4606, -4.1689]","-4.2677<br/>[-4.4606, -4.1689]","-4.2677<br/>[-4.4606, -4.1690]","-4.2676<br/>[-4.4606, -4.1689]","-4.2676<br/>[-4.4605, -4.1689]","-4.2676<br/>[-4.4605, -4.1688]","-4.2676<br/>[-4.4605, -4.1689]","-4.2675<br/>[-4.4604, -4.1688]"],["-3.4536<br/>[-3.5778, -3.3159]","-3.3607<br/>[-3.5105, -3.2201]","-3.3604<br/>[-3.5021, -3.2262]","-3.6777<br/>[-3.8178, -3.5126]","-3.4149<br/>[-3.7012, -3.2198]","-3.4317<br/>[-3.7049, -3.2344]","-3.4344<br/>[-3.7039, -3.2398]","-3.4523<br/>[-3.7238, -3.2614]","-3.4594<br/>[-3.7240, -3.2656]","-3.4834<br/>[-3.7663, -3.2945]","-3.4788<br/>[-3.7214, -3.2944]","-3.4869<br/>[-3.7450, -3.2882]","-3.6146<br/>[-3.8271, -3.4276]","-3.6238<br/>[-3.8066, -3.4580]","-3.6988<br/>[-3.8764, -3.5365]","-3.6964<br/>[-3.9047, -3.5237]","-3.7717<br/>[-4.5316, -3.4903]","-3.6660<br/>[-3.8916, -3.4550]","-3.6750<br/>[-3.9151, -3.4710]","-4.2582<br/>[-5.1168, -3.8541]","-4.1639<br/>[-4.7467, -3.8495]","-4.1653<br/>[-4.7296, -3.8449]","-4.3490<br/>[-5.1759, -3.9571]","-4.2975<br/>[-5.1587, -3.9110]","-4.2630<br/>[-5.0730, -3.8830]","-4.3385<br/>[-5.1524, -3.9545]","-4.2626<br/>[-5.0606, -3.8900]","-4.3727<br/>[-5.1843, -3.9909]","-4.2624<br/>[-5.0000, -3.9124]","-4.2675<br/>[-5.0630, -3.8941]","-4.3124<br/>[-5.1044, -3.9385]","-4.2751<br/>[-5.1035, -3.9047]","-4.2877<br/>[-5.1144, -3.9198]","-4.2965<br/>[-5.0899, -3.9289]","-4.3757<br/>[-5.1755, -3.9986]","-4.3040<br/>[-5.1044, -3.9310]","-4.3922<br/>[-5.1937, -4.0160]","-4.3849<br/>[-5.1677, -4.0061]","-4.3542<br/>[-5.1401, -3.9747]","-4.4075<br/>[-5.2221, -4.0321]","-4.3832<br/>[-5.1829, -4.0061]","-4.4155<br/>[-5.2267, -4.0416]","-4.3657<br/>[-5.1575, -3.9965]","-4.4111<br/>[-5.1941, -4.0428]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Brain_Cortex","Muscle_Skeletal","Pituitary","Ovary","Spleen","Testis","Cells_EBV-transformed_lymphocytes","Artery_Coronary","Cells_Transformed_fibroblasts","Adipose_Subcutaneous","Skin_Sun_Exposed_(Lower_leg)","Skin_Not_Sun_Exposed_(Suprapubic)","Brain_Putamen_(basal_ganglia)","Brain_Cerebellum","Whole_Blood","Colon_Sigmoid","Nerve_Tibial","Stomach","Esophagus_Mucosa","Vagina","Kidney_Cortex","Pancreas","Colon_Transverse","Liver","Prostate","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Cerebellar_Hemisphere","Thyroid","Lung","Brain_Hippocampus","Brain_Frontal_Cortex_(BA9)","Minor_Salivary_Gland","Small_Intestine_Terminal_Ileum","Breast_Mammary_Tissue","Adipose_Visceral_(Omentum)","Brain_Amygdala","Adrenal_Gland","Brain_Caudate_(basal_ganglia)","Brain_Nucleus_accumbens_(basal_ganglia)","Artery_Tibial","Heart_Left_Ventricle","Brain_Hypothalamus","Uterus","Heart_Atrial_Appendage","Brain_Substantia_nigra","Esophagus_Muscularis","Artery_Aorta","Esophagus_Gastroesophageal_Junction","Brain_Spinal_cord_(cervical_c-1)"],[12762,9430,10359,12345,7901,8424,8421,13113,11219,10831,7322,7386,12570,11728,7914,11586,13281,10053,7437,7317,9629,8210,8353,8933,9851,13908,14304,12787,9031,11879,14418,9184,9523,10533,10629,12068,13120,12446,12202,13231,11688,13013,13203,11778,13455,14843,15021,14710,13381],[26.2682835462304,0.440210229776548,0.195533946068141,-0.0692352647568092,-0.178378532946302,-0.302717506679789,-0.316706636465756,-0.42004285071152,-0.445144256131623,-0.458797179637458,-0.486634668715401,-0.489021103826243,-0.489525115031453,-0.50029560071075,-0.515189226552765,-0.520366732027974,-0.52417892816938,-0.588506546659588,-0.607037275220108,-0.610826440394464,-0.633710763763505,-0.633827024005665,-0.658042997473194,-0.665942173526448,-0.671706040475893,-0.680852369778795,-0.687828253393475,-0.689275608454174,-0.693483830088707,-0.694891493789519,-0.698479621262007,-0.704329787948761,-0.714311138341079,-0.728635748077478,-0.732711682692343,-0.73442253656811,-0.736705800181701,-0.737937726169591,-0.739243839944154,-0.739780578114741,-0.744159921043038,-0.75607129045518,-0.763228583334143,-0.764066083729213,-0.775016886647212,-0.778716135009356,-0.789870964295002,-0.792127224388626,-0.793730137859334],["-4.2745<br/>[-4.4709, -4.1745]","-4.2756<br/>[-4.4639, -4.1763]","-4.2814<br/>[-4.4816, -4.1800]","-4.2733<br/>[-4.4702, -4.1732]","-4.2683<br/>[-4.4611, -4.1696]","-4.2731<br/>[-4.4666, -4.1739]","-4.2667<br/>[-4.4597, -4.1680]","-4.2740<br/>[-4.4706, -4.1740]","-4.2666<br/>[-4.4593, -4.1680]","-4.2663<br/>[-4.4599, -4.1676]","-4.2699<br/>[-4.4622, -4.1713]","-4.2698<br/>[-4.4622, -4.1713]","-4.2732<br/>[-4.4694, -4.1732]","-4.2707<br/>[-4.4623, -4.1725]","-4.2733<br/>[-4.4623, -4.1737]","-4.2716<br/>[-4.4649, -4.1725]","-4.2720<br/>[-4.4650, -4.1729]","-4.2692<br/>[-4.4614, -4.1707]","-4.2677<br/>[-4.4607, -4.1690]","-4.2677<br/>[-4.4606, -4.1689]","-4.2674<br/>[-4.4602, -4.1688]","-4.2672<br/>[-4.4601, -4.1685]","-4.2673<br/>[-4.4601, -4.1687]","-4.2673<br/>[-4.4599, -4.1686]","-4.2673<br/>[-4.4599, -4.1687]","-4.2675<br/>[-4.4597, -4.1692]","-4.2681<br/>[-4.4600, -4.1699]","-4.2679<br/>[-4.4605, -4.1695]","-4.2669<br/>[-4.4598, -4.1682]","-4.2674<br/>[-4.4597, -4.1690]","-4.2673<br/>[-4.4595, -4.1689]","-4.2667<br/>[-4.4596, -4.1680]","-4.2666<br/>[-4.4595, -4.1679]","-4.2664<br/>[-4.4592, -4.1678]","-4.2664<br/>[-4.4592, -4.1677]","-4.2667<br/>[-4.4592, -4.1682]","-4.2670<br/>[-4.4593, -4.1685]","-4.2668<br/>[-4.4593, -4.1683]","-4.2666<br/>[-4.4591, -4.1681]","-4.2670<br/>[-4.4594, -4.1685]","-4.2715<br/>[-4.4606, -4.1719]","-4.2665<br/>[-4.4589, -4.1680]","-4.2663<br/>[-4.4588, -4.1678]","-4.2713<br/>[-4.4604, -4.1716]","-4.2662<br/>[-4.4586, -4.1677]","-4.2663<br/>[-4.4585, -4.1678]","-4.2658<br/>[-4.4583, -4.1672]","-4.2660<br/>[-4.4583, -4.1675]","-4.2657<br/>[-4.4583, -4.1672]"],["-3.0701<br/>[-3.0786, -3.0523]","-3.5525<br/>[-3.7999, -3.3591]","-3.5270<br/>[-3.6952, -3.3881]","-3.6782<br/>[-3.7796, -3.5720]","-3.6566<br/>[-3.8340, -3.4906]","-3.7123<br/>[-4.0846, -3.4745]","-3.7862<br/>[-3.8933, -3.6704]","-3.8537<br/>[-4.0457, -3.6649]","-3.8889<br/>[-4.0607, -3.7377]","-3.9096<br/>[-4.0550, -3.7666]","-3.9371<br/>[-4.5203, -3.6696]","-3.9395<br/>[-4.5228, -3.6708]","-3.9280<br/>[-4.0923, -3.7752]","-3.8702<br/>[-4.4819, -3.5784]","-3.7225<br/>[-3.9634, -3.5025]","-3.9502<br/>[-4.2818, -3.7617]","-3.9343<br/>[-4.2827, -3.7289]","-4.0555<br/>[-4.6199, -3.7819]","-4.2594<br/>[-4.8152, -3.9554]","-4.2889<br/>[-4.8388, -3.9831]","-4.1571<br/>[-4.8158, -3.8037]","-4.2701<br/>[-4.9694, -3.9134]","-4.3705<br/>[-5.1340, -4.0067]","-4.3448<br/>[-5.0875, -4.0041]","-4.2796<br/>[-4.9883, -3.9355]","-4.1160<br/>[-4.7527, -3.7943]","-4.1257<br/>[-4.8007, -3.8173]","-4.1890<br/>[-4.7200, -3.9096]","-4.4947<br/>[-5.1988, -4.1552]","-4.2318<br/>[-4.9355, -3.9054]","-4.1424<br/>[-4.7993, -3.8295]","-4.5403<br/>[-5.3081, -4.1869]","-4.5543<br/>[-5.3532, -4.2031]","-4.4975<br/>[-5.2098, -4.1693]","-4.5107<br/>[-5.2201, -4.1831]","-4.3657<br/>[-5.0761, -4.0390]","-4.3115<br/>[-4.9388, -4.0107]","-4.3562<br/>[-5.0207, -4.0446]","-4.3762<br/>[-5.0936, -4.0521]","-4.3189<br/>[-4.9218, -4.0262]","-3.9680<br/>[-4.1653, -3.7842]","-4.3894<br/>[-5.0867, -4.0756]","-4.4067<br/>[-5.0964, -4.0949]","-3.9988<br/>[-4.1825, -3.8299]","-4.4422<br/>[-5.1437, -4.1308]","-4.3618<br/>[-5.0558, -4.0588]","-4.4066<br/>[-4.9638, -4.1366]","-4.4256<br/>[-5.0654, -4.1153]","-4.5520<br/>[-5.2282, -4.2318]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["8","1","19","12","7","6","9","11","17","14","2","13","15","16","18","10","20","5","4","3"],[10756,8494,6540,7719,9062,9648,10233,11104,9334,10063,10153,8478,8556,10894,7965,9213,8977,12195,12972,10847],[1.28631191195372,0.199788150897773,-0.178177255441324,-0.182862409120457,-0.46905219772038,-0.519426820509027,-0.541266370851156,-0.595446272819743,-0.601022453277084,-0.637991464660879,-0.643723892013679,-0.645799647947085,-0.646322943225992,-0.647411343483802,-0.66647461765959,-0.67274255919081,-0.684728221571923,-0.719517039546496,-0.725134789560204,-0.749978682843429],["-4.2753<br/>[-4.4718, -4.1753]","-4.2682<br/>[-4.4624, -4.1694]","-4.2679<br/>[-4.4608, -4.1692]","-4.2748<br/>[-4.4698, -4.1752]","-4.2704<br/>[-4.4620, -4.1721]","-4.2675<br/>[-4.4603, -4.1688]","-4.2698<br/>[-4.4613, -4.1715]","-4.2687<br/>[-4.4614, -4.1700]","-4.2728<br/>[-4.4618, -4.1732]","-4.2723<br/>[-4.4613, -4.1727]","-4.2670<br/>[-4.4597, -4.1684]","-4.2675<br/>[-4.4605, -4.1688]","-4.2674<br/>[-4.4603, -4.1687]","-4.2682<br/>[-4.4608, -4.1695]","-4.2672<br/>[-4.4600, -4.1684]","-4.2674<br/>[-4.4602, -4.1687]","-4.2671<br/>[-4.4600, -4.1684]","-4.2668<br/>[-4.4595, -4.1682]","-4.2672<br/>[-4.4597, -4.1687]","-4.2662<br/>[-4.4590, -4.1675]"],["-3.5248<br/>[-3.7032, -3.3773]","-3.6366<br/>[-3.8631, -3.4184]","-3.6313<br/>[-3.8575, -3.4471]","-3.6339<br/>[-3.9054, -3.4338]","-3.8639<br/>[-4.5281, -3.5900]","-3.9090<br/>[-4.5036, -3.5180]","-3.9544<br/>[-4.6543, -3.6782]","-4.0249<br/>[-4.6101, -3.7155]","-3.8099<br/>[-4.0399, -3.6030]","-3.8461<br/>[-4.0734, -3.6407]","-4.1858<br/>[-4.7230, -3.9099]","-4.3038<br/>[-4.9186, -3.9261]","-4.3121<br/>[-4.8611, -4.0125]","-4.1533<br/>[-4.7011, -3.8236]","-4.4947<br/>[-5.2858, -4.1397]","-4.3580<br/>[-4.9940, -4.0476]","-4.4535<br/>[-5.0763, -4.1368]","-4.3172<br/>[-4.8384, -4.0447]","-4.2825<br/>[-4.8574, -3.9826]","-4.5991<br/>[-5.3300, -4.2632]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">sessionInfo</span>()</a></code></pre></div>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.5

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] DT_0.4         plyr_1.8.4     dplyr_0.7.5    R.matlab_3.6.1

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.17      compiler_3.5.0    pillar_1.2.3     
 [4] later_0.7.3       git2r_0.21.0      workflowr_1.0.1  
 [7] bindr_0.1.1       R.methodsS3_1.7.1 R.utils_2.6.0    
[10] tools_3.5.0       digest_0.6.15     jsonlite_1.5     
[13] evaluate_0.10.1   tibble_1.4.2      pkgconfig_2.0.1  
[16] rlang_0.2.1       shiny_1.1.0       crosstalk_1.0.0  
[19] yaml_2.1.19       bindrcpp_0.2.2    stringr_1.3.1    
[22] knitr_1.20        htmlwidgets_1.2   rprojroot_1.3-2  
[25] tidyselect_0.2.4  glue_1.2.0        R6_2.2.2         
[28] rmarkdown_1.10    purrr_0.2.5       magrittr_1.5     
[31] whisker_0.3-2     backports_1.1.2   promises_1.0.1   
[34] htmltools_0.3.6   assertthat_0.2.0  mime_0.5         
[37] xtable_1.8-2      httpuv_1.4.4.1    stringi_1.2.3    
[40] R.oo_1.22.0      </code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.0.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
