<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Home</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Home</h1>

</div>


<p><strong>Last updated:</strong> 2018-07-02</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/07deea5001404401a8d53a61286f8b65f53173c2" target="_blank">07deea5</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/07deea5001404401a8d53a61286f8b65f53173c2/analysis/results.Rmd" target="_blank">07deea5</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“results.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="overview" class="section level2">
<h2>Overview</h2>
<p>This is my online notebook to document and share the full results of genome-wide enrichment analyses described in the manuscript:</p>
<blockquote>
<p>Xiang Zhu and Matthew Stephens (2017). A large-scale genome-wide enrichment analysis identifies new trait-associated genes, pathways and tissues across 31 human phenotypes. <em>bioRxiv.</em> <a href="https://doi.org/10.1101/160770" class="uri">https://doi.org/10.1101/160770</a>.</p>
</blockquote>
<p>The software that was used to generate these results is available at <a href="https://github.com/stephenslab/rss/tree/master/src_vb" class="uri">https://github.com/stephenslab/rss/tree/master/src_vb</a>. Here is an <a href="http://stephenslab.github.io/rss/Example-5">example</a> illustrating how to use this software to perform genome-wide enrichment analyses of GWAS summary statistics.</p>
<p>If you find results shown in this notebook and/or methods implemented in the software useful for your work, please cite the manuscript listed above, <a href="https://doi.org/10.1101/160770">Zhu and Stephens (2017)</a>.</p>
<p>If you have any question about this notebook or the software, please feel free to contact me: <a href="https://github.com/xiangzhu">Xiang Zhu</a>, <code>xiangzhu@uchicago.edu</code> or <code>xiangzhu@stanford.edu</code>.</p>
</div>
<div id="main-results" class="section level2">
<h2>Main results</h2>
<div id="anthropometric-phenotypes" class="section level4">
<h4>Anthropometric phenotypes</h4>
<ul>
<li><a href="height_2014.html">Adult human height (Wood et al. 2014)</a></li>
<li><a href="bmi_2015.html">Body mass index (Locke et al. 2015)</a></li>
<li><a href="waist_2015.html">Waist-to-hip ratio (Shungin et al. 2015)</a></li>
</ul>
</div>
<div id="hematopoietic-phenotypes" class="section level4">
<h4>Hematopoietic phenotypes</h4>
<ul>
<li><a href="hb_2012.html">Haemoglobin (Harst et al. 2012)</a></li>
<li><a href="mch_2012.html">Mean cell haemoglobin (Harst et al. 2012)</a></li>
<li><a href="mchc_2012.html">Mean cell haemoglobin concentration (Harst et al. 2012)</a></li>
<li><a href="mcv_2012.html">Mean cell volume (Harst et al. 2012)</a></li>
<li><a href="pcv_2012.html">Packed cell volume (Harst et al. 2012)</a></li>
<li><a href="rbc_2012.html">Red blood cell count (Harst et al. 2012)</a></li>
</ul>
</div>
<div id="immune-related-phenotypes" class="section level4">
<h4>Immune-related phenotypes</h4>
<ul>
<li><a href="ra_2014.html">Rheumatoid arthritis (Okada et al. 2014)</a></li>
<li><a href="cd_2015.html">Crohn’s disease (Liu et al. 2015)</a></li>
<li><a href="ibd_2015.html">Inflammatory bowel disease (Liu et al. 2015)</a></li>
<li><a href="uc_2015.html">Ulcerative colitis (Liu et al. 2015)</a></li>
</ul>
</div>
<div id="metabolic-phenotypes" class="section level4">
<h4>Metabolic phenotypes</h4>
<ul>
<li><a href="hdl_2010.html">High-density lipoprotein (Teslovich et al. 2010)</a></li>
<li><a href="ldl_2010.html">Low-density lipoprotein (Teslovich et al. 2010)</a></li>
<li><a href="tc_2010.html">Total cholesterol (Teslovich et al. 2010)</a></li>
<li><a href="tg_2010.html">Triglycerides (Teslovich et al. 2010)</a></li>
<li><a href="t2d_2012.html">Type 2 diabetes (Morris et al. 2012)</a></li>
<li><a href="fg_2012.html">Fasting glucose (Manning et al. 2012)</a></li>
<li><a href="fi_2012.html">Fasting insulin (Manning et al. 2012)</a></li>
<li><a href="hr_2013.html">Heart rate (Den Hoed et al. 2013)</a></li>
<li><a href="gout_2013.html">Gout (Köttgen et al. 2013)</a></li>
<li><a href="urate_2013.html">Serum urate (Köttgen et al. 2013)</a></li>
<li><a href="anm_2015.html">Age at natural menopause (Day et al. 2015)</a></li>
<li><a href="cad_2015.html">Coronary artery disease (Nikpay et al. 2015)</a></li>
<li><a href="mi_2015.html">Myocardial infarction (Nikpay et al. 2015)</a></li>
</ul>
</div>
<div id="neurological-phenotypes" class="section level4">
<h4>Neurological phenotypes</h4>
<ul>
<li><a href="load_2013.html">Alzheimer’s disease (Lambert et al. 2013)</a></li>
<li><a href="scz_2014.html">Schizophrenia (Ripke et al. 2014)</a></li>
<li><a href="als_2016.html">Amyotrophic lateral sclerosis (Rheenen et al. 2016)</a></li>
<li><a href="ds_2016.html">Depressive symptoms (Okbay et al. 2016)</a></li>
<li><a href="neu_2016.html">Neuroticism (Okbay et al. 2016)</a></li>
</ul>
</div>
</div>
<div id="additional-resources" class="section level2">
<h2>Additional resources</h2>
<ul>
<li>Where can I find “baseline” model fitting results of all 31 traits?</li>
</ul>
<p>You can find summary results of “baseline” model fitting at <a href="https://xiangzhu.github.io/rss-gsea-baseline" class="uri">https://xiangzhu.github.io/rss-gsea-baseline</a>. For me, the baseline model fitting results are merely inferential “bases” for the enrichment model fitting results shown in the “Main results” section above. However, when I was presenting the enrichment results during my Ph.D. <a href="http://www.stat.uchicago.edu/~xiangzhu/THESIS_20170630.html">thesis defense</a>, <a href="http://jnpopgen.org/">Prof. John Novembre</a> and <a href="http://xinhelab.org/">Prof. Xin He</a> both pointed out these baseline results might be useful for other on-going projects on the “fourth floor”. Their comments motivated me to create a separate <a href="https://xiangzhu.github.io/rss-gsea-baseline">online notebook</a> to share the baseline summary results.</p>
<ul>
<li>Where can I find “Round 1” results of all 3913 pathways?</li>
</ul>
<p>Currently you need to contact me directly to view our “Round 1” results of all 3913 pathways. When this work was under review, one referee pointed out that our online results, especially our “Round 1” analysis results, were “needlessly complicated” and did not have “any obvious benefit”. Hence, I removed the “Round 1” analysis results from this notebook to simplify the presentation. I hope that this change can address the referee’s comment.</p>
<ul>
<li>Where can I find the gene prioritization results?</li>
</ul>
<p>Currently you need to contact me directly to view our full prioritization results. Unlike the gene set enrichment results, the gene prioritization results cannot be easily tabulated, and thus I have not displayed them in this notebook yet.</p>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.0.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
