<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Full results of Zhu and Stephens (2018)</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>










<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
<li>
  <a href="https://stephenslab.github.io/rss/news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Full results of Zhu and Stephens
(2018)</h1>
<h4 class="author">Xiang Zhu</h4>
<h4 class="date">2017-04-12</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-09-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 2
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rss-gsea/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree0d0131b608937a899299b4dd8cf4ff994ecea3cbtargetblank0d0131ba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/rss-gsea/tree/0d0131b608937a899299b4dd8cf4ff994ecea3cb" target="_blank">0d0131b</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree0d0131b608937a899299b4dd8cf4ff994ecea3cbtargetblank0d0131ba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/rss-gsea/tree/0d0131b608937a899299b4dd8cf4ff994ecea3cb" target="_blank">0d0131b</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/results.Rmd</code>) and HTML
(<code>docs/results.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/dedfe81645d7f81d60203d25ab700459d91649ab/docs/results.html" target="_blank">dedfe81</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2024-09-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/72237818c1b8a1963ebe64d9a5776caf4d1ca79e/docs/results.html" target="_blank">7223781</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2019-02-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/6e12a0cfbfbbfea4a3991e6c49bde22ce9ab0b2f/docs/results.html" target="_blank">6e12a0c</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-12-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/85468f7ad4f78819d5660884f03c49fbf6374446/analysis/results.Rmd" target="_blank">85468f7</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-12-26
</td>
<td>
wflow_publish("analysis/results.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/aeaccc16301117a1b4008c2b6280dbe5661abc07/docs/results.html" target="_blank">aeaccc1</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/438bb1e54951f5afbe7bbac77e436813da7abbfe/analysis/results.Rmd" target="_blank">438bb1e</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-29
</td>
<td>
wflow_publish("analysis/results.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/7af8457c6ce2b74d2cc50cd814246b79fb471515/docs/results.html" target="_blank">7af8457</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/7d5ee7bdb98d521a0c6e51c179ad8341935174f3/analysis/results.Rmd" target="_blank">7d5ee7b</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-29
</td>
<td>
wflow_publish("analysis/results.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/426f458a3eed2dd1c954d34a34818820cff25e01/docs/results.html" target="_blank">426f458</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/d0967ce9a30dbb2e3f3621ca5664531394d0baa2/analysis/results.Rmd" target="_blank">d0967ce</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-19
</td>
<td>
wflow_publish("analysis/results.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/results.html" target="_blank">1c85967</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/8853a352ee7c4eb975e5e18f3be547b8b47f53d0/docs/results.html" target="_blank">8853a35</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/c400b73ce6fb917cb6737d50b377f14c70c60c1d/analysis/results.Rmd" target="_blank">c400b73</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-02
</td>
<td>
wflow_publish("results.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/9928c3f75594c8db329f2ce4982a861704173222/docs/results.html" target="_blank">9928c3f</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/07deea5001404401a8d53a61286f8b65f53173c2/analysis/results.Rmd" target="_blank">07deea5</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-02
</td>
<td>
wflow_publish("results.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="overview" class="section level2">
<h2><img src="https://static.thenounproject.com/png/568780-200.png" width="60"/>
Overview</h2>
<p>This is my online notebook to document and share the full results of
genome-wide enrichment and prioritization analyses described in the
article:</p>
<blockquote>
<p>Xiang Zhu and Matthew Stephens (2018). Large-scale genome-wide
enrichment analyses identify new trait-associated genes and pathways
across 31 human phenotypes. <em>Nature Communications</em> 9, 4361. <a
href="https://doi.org/10.1038/s41467-018-06805-x"
class="uri">https://doi.org/10.1038/s41467-018-06805-x</a>.</p>
</blockquote>
<p>We developed a new statistical method, RSS-E, to generate the results
for this study. The software that implements RSS-E is freely available
at <a
href="https://github.com/stephenslab/rss/tree/master/src_vb"><code>stephenslab/rss</code></a>.
We also provide an end-to-end <a
href="http://stephenslab.github.io/rss/Example-5">example</a>
illustrating how to use RSS-E to perform the reported genome-wide
enrichment and prioritization analyses of GWAS summary statistics. This
software can be referenced in a journal’s “Code availability” section as
<a href="https://doi.org/10.5281/zenodo.1473797"><img
src="https://zenodo.org/badge/DOI/10.5281/zenodo.1473797.svg"
alt="DOI" /></a>.</p>
<p>In addition, all 4,026 pre-processed gene sets used in this study
(including 3,913 biological pathways and 113 tissue-based gene sets) are
freely available at <a
href="https://github.com/xiangzhu/rss-gsea/tree/master/data"><code>xiangzhu/rss-gsea</code></a>.
These gene sets can be referenced in a journal’s “Data availability”
section as <a href="https://doi.org/10.5281/zenodo.1473807"><img
src="https://zenodo.org/badge/DOI/10.5281/zenodo.1473807.svg"
alt="DOI" /></a>.</p>
<p>If you find the analysis results, the pre-processed gene sets, the
statistical methods, and/or the open-source software useful for your
work, <strong>please kindly cite our article listed above, <a
href="https://www.nature.com/articles/s41467-018-06805-x">Zhu and
Stephens (2018)</a></strong>.</p>
<p>If you have any question about the notebook and/or the article,
please feel free to contact me: <a
href="https://github.com/xiangzhu">Xiang Zhu</a>,
<code>xiangzhu[at]uchicago[and/or]stanford.edu</code>.</p>
</div>
<div id="main-results" class="section level2">
<h2><img src="https://static.thenounproject.com/png/808252-200.png" width="60"/>
Main results</h2>
<div id="anthropometric-phenotypes" class="section level4">
<h4>Anthropometric phenotypes</h4>
<ul>
<li><a href="height_2014.html">Adult human height (Wood et
al. 2014)</a></li>
<li><a href="bmi_2015.html">Body mass index (Locke et al. 2015)</a></li>
<li><a href="waist_2015.html">Waist-to-hip ratio (Shungin et
al. 2015)</a></li>
</ul>
</div>
<div id="hematopoietic-phenotypes" class="section level4">
<h4>Hematopoietic phenotypes</h4>
<ul>
<li><a href="hb_2012.html">Haemoglobin (Harst et al. 2012)</a></li>
<li><a href="mch_2012.html">Mean cell haemoglobin (Harst et
al. 2012)</a></li>
<li><a href="mchc_2012.html">Mean cell haemoglobin concentration (Harst
et al. 2012)</a></li>
<li><a href="mcv_2012.html">Mean cell volume (Harst et
al. 2012)</a></li>
<li><a href="pcv_2012.html">Packed cell volume (Harst et
al. 2012)</a></li>
<li><a href="rbc_2012.html">Red blood cell count (Harst et
al. 2012)</a></li>
</ul>
</div>
<div id="immune-related-phenotypes" class="section level4">
<h4>Immune-related phenotypes</h4>
<ul>
<li><a href="ra_2014.html">Rheumatoid arthritis (Okada et
al. 2014)</a></li>
<li><a href="cd_2015.html">Crohn’s disease (Liu et al. 2015)</a></li>
<li><a href="ibd_2015.html">Inflammatory bowel disease (Liu et
al. 2015)</a></li>
<li><a href="uc_2015.html">Ulcerative colitis (Liu et al. 2015)</a></li>
</ul>
</div>
<div id="metabolic-phenotypes" class="section level4">
<h4>Metabolic phenotypes</h4>
<ul>
<li><a href="hdl_2010.html">High-density lipoprotein (Teslovich et
al. 2010)</a></li>
<li><a href="ldl_2010.html">Low-density lipoprotein (Teslovich et
al. 2010)</a></li>
<li><a href="tc_2010.html">Total cholesterol (Teslovich et
al. 2010)</a></li>
<li><a href="tg_2010.html">Triglycerides (Teslovich et
al. 2010)</a></li>
<li><a href="t2d_2012.html">Type 2 diabetes (Morris et
al. 2012)</a></li>
<li><a href="fg_2012.html">Fasting glucose (Manning et
al. 2012)</a></li>
<li><a href="fi_2012.html">Fasting insulin (Manning et
al. 2012)</a></li>
<li><a href="hr_2013.html">Heart rate (Den Hoed et al. 2013)</a></li>
<li><a href="gout_2013.html">Gout (Köttgen et al. 2013)</a></li>
<li><a href="urate_2013.html">Serum urate (Köttgen et al. 2013)</a></li>
<li><a href="anm_2015.html">Age at natural menopause (Day et
al. 2015)</a></li>
<li><a href="cad_2015.html">Coronary artery disease (Nikpay et
al. 2015)</a></li>
<li><a href="mi_2015.html">Myocardial infarction (Nikpay et
al. 2015)</a></li>
</ul>
</div>
<div id="neurological-phenotypes" class="section level4">
<h4>Neurological phenotypes</h4>
<ul>
<li><a href="load_2013.html">Alzheimer’s disease (Lambert et
al. 2013)</a></li>
<li><a href="scz_2014.html">Schizophrenia (Ripke et al. 2014)</a></li>
<li><a href="als_2016.html">Amyotrophic lateral sclerosis (Rheenen et
al. 2016)</a></li>
<li><a href="ds_2016.html">Depressive symptoms (Okbay et
al. 2016)</a></li>
<li><a href="neu_2016.html">Neuroticism (Okbay et al. 2016)</a></li>
</ul>
</div>
<div id="gene-prioritization" class="section level4">
<h4>Gene prioritization</h4>
<ul>
<li><a href="ldl2010_path1698.html">Low-density lipoprotein (Teslovich
et al. 2010) &amp; Chylomicron-mediated lipid transport</a></li>
<li><a href="height2014_path3208.html">Adult human height (Wood et
al. 2014) &amp; Endochondral ossification</a></li>
<li><a href="ibd2015_path3850.html">Inflammatory bowel disease (Liu et
al. 2015) &amp; Cytokine-cytokine receptor interaction</a></li>
<li><a href="ra2014_path2858.html">Rheumatoid arthritis (Okada et
al. 2014) &amp; Lymphocyte NFAT-dependent transcription</a></li>
<li><a href="cad2015_path3059.html">Coronary artery disease (Nikpay et
al. 2015) &amp; p75(NTR)-mediated signaling</a></li>
<li><a href="hdl2010_path3093.html">High-density lipoprotein (Teslovich
et al. 2010) &amp; Lipid digestion, mobilization and transport</a></li>
</ul>
</div>
</div>
<div id="additional-resources" class="section level2">
<h2><img src="https://static.thenounproject.com/png/824132-200.png" width="60"/>
Additional resources</h2>
<blockquote>
<p>How can I perform similar analyses on a new GWAS summary dataset
using RSS-E?</p>
</blockquote>
<p>The software that generated results of this study is freely available
at <a
href="https://github.com/stephenslab/rss/tree/master/src_vb"><code>stephenslab/rss</code></a>.
I also write a step-by-step <a
href="http://stephenslab.github.io/rss/Example-5">RSS-E tutorial</a>
that illustrates how to use this software to perform genome-wide
enrichment and prioritization analyses on GWAS summary statistics.</p>
<blockquote>
<p>Compared with most existing enrichment methods, the most appealing
feature of RSS-E is the automatic gene prioritization in light of
inferred enrichments. Is this gene prioritization feature available in
your software?</p>
</blockquote>
<p>Yes. This feature is implemented as function <a
href="https://github.com/stephenslab/rss/blob/master/src_vb/compute_pip.m"><code>compute_pip.m</code></a>
in RSS-E. The step-by-step <a
href="http://stephenslab.github.io/rss/Example-5">RSS-E tutorial</a>
illustrates how to use this function.</p>
<blockquote>
<p>There are two sanity checks for the more sophisticated RSS-E analysis
in <a href="https://www.nature.com/articles/s41467-018-06805-x">Zhu and
Stephens (2018)</a>: an eyeball test and a likelihood ratio calculation.
Do you have software for these sanity checks?</p>
</blockquote>
<p>Yes. The eyeball test is simply plotting marginal distribution of
GWAS z-scores, stratified by SNP-level annotations based on a given gene
set. Here we used <a
href="https://ggplot2.tidyverse.org/reference/geom_density.html"><code>ggplot2::geom_density</code></a>
(default setting). Regarding the likelihood ratio check, I write a
stand-alone script <a
href="https://github.com/stephenslab/rss/blob/master/misc/ash_lrt_31traits.R"><code>ash_lrt_31traits.R</code></a>.
Please carefully read the instruction in this script. For more details
of these two sanity checks, please see the caption of <a
href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-018-06805-x/MediaObjects/41467_2018_6805_MOESM1_ESM.pdf">Supplementary
Figure 17</a> in <a
href="https://www.nature.com/articles/s41467-018-06805-x">Zhu and
Stephens (2018)</a>.</p>
<blockquote>
<p>Where can I download all 4,026 pre-processed gene sets used in this
work?</p>
</blockquote>
<p>All 4,026 gene sets used in this study are freely available at <a
href="https://github.com/xiangzhu/rss-gsea/tree/master/data"><code>xiangzhu/rss-gsea</code></a>,
where the folder <a
href="https://github.com/xiangzhu/rss-gsea/tree/master/data/biological_pathway"><code>biological_pathway</code></a>
contains 3,913 biological pathways, and the folder <a
href="https://github.com/xiangzhu/rss-gsea/tree/master/data/tissue_set"><code>tissue_set</code></a>
contains 113 GTEx tissue-based gene sets. More details about these gene
sets can be found <a href="gene_set.html">here</a>.</p>
<blockquote>
<p>Where can I find RSS-E “baseline” model fitting results of all 31
traits?</p>
</blockquote>
<p>You can find summary results of “baseline” model fitting at <a
href="https://xiangzhu.github.io/rss-gsea-baseline/"><code>xiangzhu/rss-gsea-baseline</code></a>.
For me, the baseline model fitting results are merely inferential
“bases” for the enrichment model fitting results shown in the “Main
results” section above. However, when I was presenting the enrichment
results during my <a
href="http://www.stat.uchicago.edu/~xiangzhu/THESIS_20170630.html">Ph.D. thesis
defense</a>, <a href="http://jnpopgen.org/">Prof. John Novembre</a> and
<a href="http://xinhelab.org/">Prof. Xin He</a> both pointed out these
baseline results might be useful for other on-going research projects on
the “fourth floor” (i.e. the <strong>fantastic</strong> computational
space shared with the labs of Matthew Stephens, John Novembre and Xin
He). Their comments motivated me to create a separate online notebook <a
href="https://xiangzhu.github.io/rss-gsea-baseline/"><code>xiangzhu/rss-gsea-baseline</code></a>
to share the baseline summary results.</p>
<blockquote>
<p>Where can I find “Round 1” RSS-E results of all 3,913 biological
pathways?</p>
</blockquote>
<p>Currently you need to contact me directly to view our “Round 1”
results of all 3913 pathways. When this work was under review, one
referee pointed out that our previous online results, especially our
“Round 1” analysis results, were “needlessly complicated” and did not
have “any obvious benefit”. Hence, I removed the “Round 1” analysis
results from this notebook to simplify the presentation. I sincerely
hope that this change can address this referee’s comment.</p>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
