<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of total cholesterol</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Enrichment results of total cholesterol</h1>
<h4 class="author"><em>Xiang Zhu</em></h4>
<h4 class="date"><em>2017-04-12</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-10-05</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180626)</code> </summary></p>
<p>The command <code>set.seed(20180626)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> <a href="https://github.com/xiangzhu/rss-gsea/tree/1c85967d82bd54d5e76573a2360750fc083cedc5" target="_blank">1c85967</a> </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<details>
<summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/tc_2010.html" target="_blank">1c85967</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-10-05
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
<a href="https://cdn.rawgit.com/xiangzhu/rss-gsea/034b6e010cd7032fc2c03d8765be62fb19f7a411/docs/tc_2010.html" target="_blank">034b6e0</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
<a href="https://github.com/xiangzhu/rss-gsea/blob/d027d60d7a48b4448d18221e55687aa06e37a934/analysis/tc_2010.Rmd" target="_blank">d027d60</a>
</td>
<td style="text-align:left;">
Xiang Zhu
</td>
<td style="text-align:left;">
2018-07-02
</td>
<td style="text-align:left;">
wflow_publish(“tc_2010.Rmd”)
</td>
</tr>
</tbody>
</table>
</ul>
</details>
<hr />
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics published in the paper <a href="https://www.ncbi.nlm.nih.gov/pubmed/20686565">“Biological, Clinical and Population Relevance of 95 Loci for Blood Lipids” (<em>Nature</em>, 2010)</a>. The summary data file is available at <a href="http://csg.sph.umich.edu/willer/public/lipids2010" class="uri">http://csg.sph.umich.edu/willer/public/lipids2010</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “outside” the gene set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong> proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas the last two capture the magnitude of enrichment. For each gene set, we report these three quantities in the last three columns of tables below, on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div id="htmlwidget-983860bb2d7d6ad3a872" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-983860bb2d7d6ad3a872">{"x":{"filter":"none","data":[["C-MYC pathway<br/>(2041, pid, BioSystems)","C-MYC pathway<br/>(2148, pid, PC)","Glucose transport<br/>(2757, reactome, BioSystems)","Regulation of Glucokinase by Glucokinase Regulatory Protein<br/>(2297, reactome, PC)","Regulation of Glucokinase by Glucokinase Regulatory Protein<br/>(2344, reactome, BioSystems)","Electric Transmission Across Gap Junctions<br/>(536, reactome, PC)","tRNA splicing<br/>(596, humancyc, PC)","Basic mechanism of action of PPARa, PPARb(d) and PPARg and effects on gene expression<br/>(443, biocarta, BioCarta)","Visceral Fat Deposits and the Metabolic Syndrome<br/>(970, biocarta, BioCarta)","Nuclear Receptors in Lipid Metabolism and Toxicity<br/>(1508, biocarta, BioCarta)","Role of PPAR-gamma Coactivators in Obesity and Thermogenesis<br/>(1071, biocarta, BioCarta)","selenocysteine biosynthesis<br/>(722, humancyc, PC)","RXR and RAR heterodimerization with other nuclear receptor<br/>(2207, pid, PC)","Statin Pathway<br/>(2412, wiki, BioSystems)","Lipid and lipoprotein metabolism<br/>(2961, reactome, PC)","Lipid digestion, mobilization, and transport<br/>(3093, reactome, BioSystems)","Lipoprotein metabolism<br/>(2378, reactome, PC)","proline biosynthesis<br/>(141, biocyc, BioSystems)","proline biosynthesis<br/>(275, humancyc, PC)","ornithine de novo biosynthesis<br/>(169, biocyc, BioSystems)","Proline biosynthesis, glutamate => proline<br/>(498, kegg, BioSystems)","ornithine de novo biosynthesis<br/>(274, humancyc, PC)","Chylomicron-mediated lipid transport<br/>(1698, reactome, PC)","Abacavir transport and metabolism<br/>(1104, reactome, PC)","ACE Inhibitor Pathway<br/>(1539, wiki, BioSystems)","Renin-angiotensin system<br/>(1614, kegg, BioSystems)","Abacavir transmembrane transport<br/>(514, reactome, PC)","Neurotransmitter Clearance In The Synaptic Cleft<br/>(810, reactome, PC)","Vesicle-mediated transport<br/>(3831, reactome, BioSystems)","Binding and Uptake of Ligands by Scavenger Receptors<br/>(2732, reactome, BioSystems)","Binding and Uptake of Ligands by Scavenger Receptors<br/>(2742, reactome, PC)","Retinoid metabolism and transport<br/>(2697, reactome, PC)","Retinoid metabolism and transport<br/>(2734, reactome, BioSystems)","Organic cation/anion/zwitterion transport<br/>(1403, reactome, PC)","Alzheimer's disease<br/>(3739, kegg, BioSystems)","Diseases associated with visual transduction<br/>(3452, reactome, PC)","Visual phototransduction<br/>(3461, reactome, BioSystems)","HDL-mediated lipid transport<br/>(1555, reactome, PC)","Scavenging by Class A Receptors<br/>(1835, reactome, PC)","Vitamin B12 Metabolism<br/>(1860, wiki, BioSystems)","Nectin/Necl trans heterodimerization<br/>(808, reactome, PC)","Fatty acid, triacylglycerol, and ketone body metabolism<br/>(3766, reactome, PC)","Fatty acid, triacylglycerol, and ketone body metabolism<br/>(3779, reactome, BioSystems)","PPARA activates gene expression<br/>(3584, reactome, PC)","PPARA activates gene expression<br/>(3595, reactome, BioSystems)","Regulation of lipid metabolism by Peroxisome proliferator-activated receptor alpha (PPARalpha)<br/>(3597, reactome, PC)","Regulation of lipid metabolism by Peroxisome proliferator-activated receptor alpha (PPARalpha)<br/>(3610, reactome, BioSystems)","Nectin adhesion pathway<br/>(2186, pid, BioSystems)","LDL endocytosis<br/>(673, reactome, PC)","EGFR1 Signaling Pathway<br/>(3757, wiki, BioSystems)","Alzheimers Disease<br/>(3337, wiki, BioSystems)","Nectin adhesion pathway<br/>(2381, pid, PC)","Respiratory electron transport<br/>(3291, reactome, PC)","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell<br/>(3261, reactome, PC)","Defective CHST14 causes EDS, musculocontractural type<br/>(751, reactome, BioSystems)","Organic cation transport<br/>(926, reactome, PC)","A tetrasaccharide linker sequence is required for GAG synthesis<br/>(2088, reactome, BioSystems)","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell<br/>(3487, reactome, BioSystems)","Syndecan-3-mediated signaling events<br/>(1894, pid, PC)","Organic cation/anion/zwitterion transport<br/>(1522, reactome, BioSystems)","Chondroitin sulfate biosynthesis<br/>(1755, reactome, PC)","Amine compound SLC transporters<br/>(2327, reactome, PC)","Norepinephrine Neurotransmitter Release Cycle<br/>(1707, reactome, PC)","Endocytosis<br/>(3848, kegg, BioSystems)","Na+/Cl- dependent neurotransmitter transporters<br/>(1768, reactome, PC)","Neurotransmitter Release Cycle<br/>(2721, reactome, PC)","Selenium Pathway<br/>(3432, wiki, BioSystems)","Neurotransmitter Release Cycle<br/>(2897, reactome, BioSystems)","Organic cation transport<br/>(1076, reactome, BioSystems)","Cohesin Loading onto Chromatin<br/>(975, reactome, BioSystems)","Adherens junctions interactions<br/>(2366, reactome, PC)","Fat digestion and absorption<br/>(2765, kegg, BioSystems)","Orphan transporters<br/>(3498, reactome, PC)","Transmission across Chemical Synapses<br/>(3802, reactome, BioSystems)","Cell-cell junction organization<br/>(3103, reactome, PC)","Cell adhesion molecules (CAMs)<br/>(3688, kegg, BioSystems)","Electron Transport Chain<br/>(3449, wiki, BioSystems)","Cell junction organization<br/>(3385, reactome, PC)","Cell junction organization<br/>(3397, reactome, BioSystems)","Cell-Cell communication<br/>(3652, reactome, PC)","Cell-Cell communication<br/>(3657, reactome, BioSystems)","Herpes simplex infection<br/>(3768, kegg, BioSystems)","TNF-alpha/NF-kB Signaling Pathway<br/>(3781, wiki, BioSystems)","Neuronal System<br/>(3864, reactome, PC)","Chronic myeloid leukemia<br/>(3289, kegg, BioSystems)","Scavenging by Class B Receptors<br/>(567, reactome, PC)","Mitochondrial protein import<br/>(2887, reactome, PC)","Monoamine Transport<br/>(2452, wiki, BioSystems)","LKB1 signaling events<br/>(2774, pid, PC)","Chromatin modifying enzymes<br/>(3805, reactome, PC)","Chromatin modifying enzymes<br/>(3839, reactome, BioSystems)","SLC-mediated transmembrane transport<br/>(3852, reactome, BioSystems)","Metabolism of carbohydrates<br/>(3849, reactome, BioSystems)","SLC-mediated transmembrane transport<br/>(3847, reactome, PC)","HATs acetylate histones<br/>(3671, reactome, PC)","HATs acetylate histones<br/>(3693, reactome, BioSystems)","Carbohydrate metabolism<br/>(3841, reactome, PC)","Hexose transport<br/>(2893, reactome, BioSystems)","Glucose transport<br/>(2691, reactome, PC)","Hexose uptake<br/>(2847, reactome, PC)"],[22,25,41,28,29,5,5,4,8,14,9,6,26,30,50,58,30,2,3,2,4,3,17,10,14,15,5,7,231,40,41,39,40,13,157,88,89,15,19,19,7,174,181,107,109,110,112,25,6,169,77,30,72,69,6,8,23,93,20,14,18,29,17,252,18,40,85,47,9,8,30,41,96,200,59,134,87,82,83,126,127,174,181,269,71,5,47,31,42,203,241,254,253,251,132,136,244,47,39,45],[1717,1937,3156,2120,2207,489,390,543,906,1372,1009,486,2905,2314,3969,4473,2625,153,217,262,281,305,1140,1026,1403,1472,613,673,19684,4183,4243,4555,4555,1027,14683,8940,8940,1467,2347,1509,666,17117,17676,11219,11318,11527,11626,2900,512,15665,9557,3423,4868,3451,452,841,2937,4510,1668,1277,2343,3166,2034,22230,1854,4661,6872,5499,1091,721,5592,2638,8649,24018,8024,14598,5809,10577,10577,17250,17250,10810,14429,32090,6438,506,3298,2992,3591,11667,15124,23944,23181,23473,5705,6107,22614,3645,3001,3490],[1046.71297038265,1030.28057706807,833.167736335236,833.115109164907,833.071238617996,189.412834516806,177.78667497423,177.606383982322,176.519771704914,176.513044553226,176.423292674786,136.651722355881,135.600403658823,127.100950038157,122.69621790464,120.46859127399,117.920825753058,107.296825018183,107.009357779274,106.830601775116,106.783206225496,106.694501749244,105.44046429875,92.7095375729518,92.4177766411033,92.306490486481,91.7966797796013,82.9032332225167,65.8193195674651,65.7480962726918,65.4064751492399,64.6114076553592,64.6114076553592,63.0634896671963,62.2600756659802,60.0740049626721,60.0740049626721,59.8517866692081,57.9973025606874,56.8794544735574,48.2210247522355,45.2811871416823,44.8074170145675,43.8792314614443,43.7693079329668,43.7680415185022,43.6785397232853,42.7157227956977,42.5213265472326,42.4481937268736,42.3689586787171,41.3946922430321,40.6150444093429,40.4683606234445,39.483711575291,38.4208985755171,38.0521436936433,38.0239488153225,37.5638421585216,37.4438464769592,37.0535321876074,36.6888290432167,36.626335263684,36.1917445663657,36.0239955426929,35.5570001462603,35.007134320934,34.8192630174718,34.2823940652204,34.2108113389209,33.4222511081726,33.2861119387045,32.8412258368669,32.3963908078288,32.2865546596225,31.5848750803951,31.2203209780455,30.9615949836417,30.9615949836417,30.8411480412691,30.8411480412691,29.456792792058,29.3813293082727,29.0392145240454,28.2441535165126,27.8424633252401,27.4933984572174,27.0280169769773,26.5944645829211,11.5055797344586,10.5202917192331,4.5808792825295,3.63777074909445,3.41635699391635,3.23941239555911,3.12340305431543,2.58855473113178,2.1802497234865,0.980640425662793,0.86097385271057],["-4.0271<br/>[-4.0397, -4.0140]","-4.0270<br/>[-4.0396, -4.0139]","-4.0354<br/>[-4.0481, -4.0223]","-4.0314<br/>[-4.0442, -4.0182]","-4.0314<br/>[-4.0441, -4.0182]","-4.0275<br/>[-4.0402, -4.0144]","-4.0308<br/>[-4.0434, -4.0178]","-4.0287<br/>[-4.0413, -4.0158]","-4.0285<br/>[-4.0410, -4.0155]","-4.0287<br/>[-4.0412, -4.0158]","-4.0284<br/>[-4.0410, -4.0155]","-4.0366<br/>[-4.0495, -4.0233]","-4.0416<br/>[-4.0545, -4.0283]","-4.2045<br/>[-4.2202, -4.1886]","-4.2140<br/>[-4.2294, -4.1983]","-4.2138<br/>[-4.2292, -4.1981]","-4.1939<br/>[-4.2088, -4.1788]","-4.0277<br/>[-4.0403, -4.0146]","-4.0276<br/>[-4.0403, -4.0145]","-4.0276<br/>[-4.0403, -4.0145]","-4.0276<br/>[-4.0402, -4.0145]","-4.0276<br/>[-4.0402, -4.0145]","-4.1559<br/>[-4.1697, -4.1419]","-4.0410<br/>[-4.0540, -4.0275]","-4.0272<br/>[-4.0398, -4.0141]","-4.0271<br/>[-4.0398, -4.0141]","-4.0411<br/>[-4.0542, -4.0276]","-4.0365<br/>[-4.0494, -4.0232]","-4.1290<br/>[-4.1404, -4.1176]","-4.1093<br/>[-4.1212, -4.0974]","-4.1093<br/>[-4.1212, -4.0973]","-4.1368<br/>[-4.1500, -4.1233]","-4.1368<br/>[-4.1500, -4.1233]","-4.0410<br/>[-4.0540, -4.0275]","-4.0649<br/>[-4.0786, -4.0507]","-4.1353<br/>[-4.1485, -4.1219]","-4.1353<br/>[-4.1485, -4.1219]","-4.1054<br/>[-4.1201, -4.0902]","-4.1019<br/>[-4.1144, -4.0891]","-4.0943<br/>[-4.1087, -4.0793]","-4.0765<br/>[-4.0903, -4.0623]","-4.0899<br/>[-4.1041, -4.0752]","-4.0897<br/>[-4.1039, -4.0750]","-4.0869<br/>[-4.1010, -4.0723]","-4.0868<br/>[-4.1010, -4.0722]","-4.0868<br/>[-4.1009, -4.0722]","-4.0867<br/>[-4.1008, -4.0721]","-4.0784<br/>[-4.0918, -4.0644]","-4.0878<br/>[-4.1001, -4.0752]","-4.1092<br/>[-4.1239, -4.0939]","-4.0669<br/>[-4.0806, -4.0527]","-4.0782<br/>[-4.0916, -4.0641]","-4.0399<br/>[-4.0528, -4.0265]","-4.0762<br/>[-4.0898, -4.0622]","-4.0320<br/>[-4.0448, -4.0188]","-4.0410<br/>[-4.0541, -4.0276]","-4.0330<br/>[-4.0455, -4.0200]","-4.0758<br/>[-4.0894, -4.0617]","-4.0315<br/>[-4.0443, -4.0183]","-4.0408<br/>[-4.0539, -4.0274]","-4.0313<br/>[-4.0440, -4.0180]","-4.0375<br/>[-4.0501, -4.0244]","-4.0363<br/>[-4.0492, -4.0231]","-4.0911<br/>[-4.1055, -4.0762]","-4.0360<br/>[-4.0489, -4.0227]","-4.0363<br/>[-4.0490, -4.0232]","-4.1004<br/>[-4.1129, -4.0877]","-4.0360<br/>[-4.0487, -4.0228]","-4.0409<br/>[-4.0540, -4.0275]","-4.0274<br/>[-4.0401, -4.0144]","-4.0745<br/>[-4.0882, -4.0603]","-4.0969<br/>[-4.1091, -4.0844]","-4.0698<br/>[-4.0816, -4.0579]","-4.0332<br/>[-4.0455, -4.0204]","-4.0738<br/>[-4.0874, -4.0596]","-4.0732<br/>[-4.0866, -4.0593]","-4.0300<br/>[-4.0427, -4.0168]","-4.0776<br/>[-4.0914, -4.0633]","-4.0776<br/>[-4.0914, -4.0633]","-4.0799<br/>[-4.0938, -4.0655]","-4.0799<br/>[-4.0938, -4.0655]","-4.0657<br/>[-4.0796, -4.0514]","-4.0878<br/>[-4.1017, -4.0733]","-4.0298<br/>[-4.0421, -4.0170]","-4.0668<br/>[-4.0804, -4.0527]","-4.0621<br/>[-4.0735, -4.0504]","-4.0686<br/>[-4.0825, -4.0543]","-4.0715<br/>[-4.0852, -4.0574]","-4.0583<br/>[-4.0717, -4.0445]","-4.0501<br/>[-4.0633, -4.0364]","-4.0488<br/>[-4.0620, -4.0352]","-4.0478<br/>[-4.0607, -4.0344]","-4.0385<br/>[-4.0511, -4.0254]","-4.0438<br/>[-4.0567, -4.0304]","-4.0293<br/>[-4.0418, -4.0163]","-4.0291<br/>[-4.0416, -4.0161]","-4.0347<br/>[-4.0473, -4.0215]","-4.0352<br/>[-4.0479, -4.0221]","-4.0315<br/>[-4.0442, -4.0184]","-4.0313<br/>[-4.0440, -4.0182]"],["-2.6270<br/>[-2.6270, -2.6270]","-2.4957<br/>[-2.4957, -2.4955]","-2.7861<br/>[-2.8034, -2.7706]","-2.7205<br/>[-2.7233, -2.7174]","-2.7379<br/>[-2.7408, -2.7348]","-2.3832<br/>[-2.3881, -2.3763]","-2.0218<br/>[-2.0925, -1.9856]","-2.1719<br/>[-2.2285, -2.1237]","-2.4580<br/>[-2.4806, -2.4350]","-2.5343<br/>[-2.6101, -2.4603]","-2.4904<br/>[-2.5238, -2.4571]","-2.0407<br/>[-2.0671, -2.0118]","-2.2301<br/>[-2.2315, -2.2253]","-1.5561<br/>[-1.5603, -1.5521]","-1.7678<br/>[-1.7782, -1.7556]","-1.8140<br/>[-1.8258, -1.7991]","-1.6233<br/>[-1.6507, -1.5736]","-1.8704<br/>[-1.8834, -1.8531]","-2.0225<br/>[-2.0351, -2.0053]","-2.1051<br/>[-2.1169, -2.0886]","-2.1361<br/>[-2.1474, -2.1205]","-2.1714<br/>[-2.1830, -2.1554]","-1.3426<br/>[-1.3893, -1.2226]","-2.2541<br/>[-2.2579, -2.2502]","-2.6841<br/>[-2.7554, -2.6290]","-2.6984<br/>[-2.7480, -2.6572]","-2.0519<br/>[-2.0609, -2.0411]","-1.9987<br/>[-2.0392, -1.9648]","-2.6104<br/>[-2.6203, -2.6060]","-2.1280<br/>[-2.1291, -2.1273]","-2.1344<br/>[-2.1350, -2.1340]","-2.1174<br/>[-2.1467, -2.0528]","-2.1174<br/>[-2.1467, -2.0528]","-2.1897<br/>[-2.2141, -2.1600]","-2.6935<br/>[-2.6961, -2.6870]","-2.3987<br/>[-2.4012, -2.3967]","-2.3987<br/>[-2.4012, -2.3967]","-1.6307<br/>[-1.6607, -1.5981]","-1.9402<br/>[-1.9666, -1.8544]","-1.6769<br/>[-1.6799, -1.6606]","-1.6483<br/>[-1.6631, -1.6290]","-2.6747<br/>[-2.6946, -2.6482]","-2.6877<br/>[-2.7062, -2.6612]","-2.5610<br/>[-2.5819, -2.5456]","-2.5654<br/>[-2.5839, -2.5488]","-2.5666<br/>[-2.5865, -2.5490]","-2.5698<br/>[-2.5895, -2.5518]","-2.2400<br/>[-2.2773, -2.2064]","-1.3143<br/>[-1.3484, -1.3021]","-2.6336<br/>[-2.6562, -2.6157]","-2.7353<br/>[-2.8207, -2.7120]","-2.3054<br/>[-2.3427, -2.2695]","-2.5487<br/>[-2.5543, -2.5354]","-2.3097<br/>[-2.3442, -2.2712]","-2.0520<br/>[-2.1518, -1.9866]","-2.1619<br/>[-2.2061, -2.1184]","-2.8295<br/>[-2.8645, -2.7936]","-2.4055<br/>[-2.4424, -2.3718]","-2.6822<br/>[-2.7330, -2.6296]","-2.3485<br/>[-2.3817, -2.3094]","-2.8461<br/>[-2.8866, -2.8041]","-2.6624<br/>[-2.6882, -2.6283]","-2.4807<br/>[-2.5195, -2.4437]","-2.7576<br/>[-2.7704, -2.7342]","-2.5109<br/>[-2.5456, -2.4700]","-2.7631<br/>[-2.7811, -2.7338]","-2.3922<br/>[-2.4021, -2.3782]","-2.8306<br/>[-2.8460, -2.8046]","-2.2011<br/>[-2.2464, -2.1560]","-2.3282<br/>[-2.3698, -2.2869]","-2.6576<br/>[-2.7037, -2.4995]","-2.0336<br/>[-2.0875, -1.9869]","-2.5520<br/>[-2.5985, -2.5293]","-3.2838<br/>[-3.3770, -3.1757]","-2.8391<br/>[-2.8460, -2.8115]","-3.0728<br/>[-3.0752, -3.0713]","-3.0820<br/>[-3.1113, -3.0397]","-2.8822<br/>[-2.9344, -2.6777]","-2.8822<br/>[-2.9344, -2.6777]","-3.0815<br/>[-3.0852, -3.0814]","-3.0815<br/>[-3.0852, -3.0814]","-2.8561<br/>[-2.8822, -2.8484]","-2.7075<br/>[-2.7161, -2.6903]","-3.2678<br/>[-3.2736, -3.2600]","-2.6057<br/>[-2.6257, -2.5871]","-1.4703<br/>[-1.4829, -1.4620]","-2.4673<br/>[-2.4710, -2.4331]","-2.3771<br/>[-2.3827, -2.3770]","-2.4768<br/>[-2.5017, -2.4660]","-2.9728<br/>[-3.0173, -2.9271]","-3.0781<br/>[-3.1325, -3.0298]","-3.4100<br/>[-3.5296, -3.2515]","-3.4792<br/>[-3.5560, -3.4099]","-3.4839<br/>[-3.6191, -3.3738]","-3.0646<br/>[-3.1553, -2.9910]","-3.0945<br/>[-3.1896, -3.0173]","-3.5451<br/>[-3.6493, -3.4596]","-3.0222<br/>[-3.1683, -2.9127]","-3.1479<br/>[-3.4265, -2.9865]","-3.2345<br/>[-3.4964, -3.0643]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(ID, database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 3, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div id="htmlwidget-0aaad034f4771159c3b9" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-0aaad034f4771159c3b9">{"x":{"filter":"none","data":[["Adrenal Gland","Liver","Brain - Cerebellum","Brain - Cortex","Brain - Anterior cingulate cortex (BA24)","Brain - Frontal Cortex (BA9)","Brain - Hippocampus","Brain - Caudate (basal ganglia)","Brain - Nucleus accumbens (basal ganglia)","Brain - Hypothalamus","Brain - Putamen (basal ganglia)","Uterus","Prostate","Ovary","Cells - EBV-transformed lymphocytes","Pancreas","Pituitary","Skin - Not Sun Exposed (Suprapubic)","Skin - Sun Exposed (Lower leg)","Testis","Vagina","Breast - Mammary Tissue","Cells - Transformed fibroblasts","Small Intestine - Terminal Ileum","Esophagus - Muscularis","Spleen","Lung","Esophagus - Mucosa","Heart - Left Ventricle","Muscle - Skeletal","Artery - Coronary","Artery - Tibial","Artery - Aorta","Colon - Transverse","Stomach","Esophagus - Gastroesophageal Junction","Colon - Sigmoid","Brain - Cerebellar Hemisphere","Whole Blood","Adipose - Subcutaneous","Adipose - Visceral (Omentum)","Nerve - Tibial","Thyroid","Heart - Atrial Appendage"],[5361,6048,5395,5558,5640,5927,5487,6207,5839,5640,6273,5412,5325,5597,4742,4876,5617,5491,5430,5556,5508,5307,5845,4477,5725,3839,4780,5352,5344,5353,5895,5774,6066,4968,5012,5533,5630,5376,5604,5368,5565,5424,5886,5553],[52.0704554065734,49.3930228142375,48.0683537636795,45.5679449337702,45.3939236963975,44.9309613689794,44.168485122197,43.7259782596276,43.2078287470558,42.993083687603,42.5795668348474,15.4969548512149,15.2204093908507,11.8158171177728,11.4442912583134,11.323443013394,11.2320265923393,11.0557493903728,11.0038032467042,10.4323795904761,10.1476221544046,10.0567218825599,9.91884469654199,8.95632521403064,6.60875326483083,4.4674850728692,4.41952569143933,4.37213270295694,4.15600582602265,4.03453787007166,3.09980117937957,3.00844910018291,2.87977806443035,2.80757809943047,2.7869292483961,2.68070440795212,2.65259784404762,2.05387158601534,0.853367757813608,0.482281816026395,0.481431445708902,0.405456899455688,0.289076641127626,0.136154964136661],["-4.0732<br/>[-4.0872, -4.0588]","-4.0902<br/>[-4.1037, -4.0764]","-4.0687<br/>[-4.0821, -4.0547]","-4.0671<br/>[-4.0808, -4.0529]","-4.0671<br/>[-4.0808, -4.0529]","-4.0669<br/>[-4.0806, -4.0527]","-4.0629<br/>[-4.0766, -4.0488]","-4.0626<br/>[-4.0763, -4.0484]","-4.0627<br/>[-4.0765, -4.0485]","-4.0628<br/>[-4.0765, -4.0486]","-4.0625<br/>[-4.0763, -4.0484]","-4.0437<br/>[-4.0567, -4.0303]","-4.0435<br/>[-4.0565, -4.0301]","-4.0391<br/>[-4.0519, -4.0258]","-4.0391<br/>[-4.0519, -4.0257]","-4.0390<br/>[-4.0519, -4.0257]","-4.0389<br/>[-4.0517, -4.0256]","-4.0387<br/>[-4.0516, -4.0254]","-4.0388<br/>[-4.0516, -4.0255]","-4.0434<br/>[-4.0563, -4.0300]","-4.0352<br/>[-4.0480, -4.0219]","-4.0351<br/>[-4.0480, -4.0219]","-4.0349<br/>[-4.0477, -4.0217]","-4.0438<br/>[-4.0568, -4.0304]","-4.0300<br/>[-4.0428, -4.0168]","-4.0264<br/>[-4.0390, -4.0134]","-4.0262<br/>[-4.0388, -4.0132]","-4.0260<br/>[-4.0386, -4.0130]","-4.0258<br/>[-4.0384, -4.0128]","-4.0256<br/>[-4.0382, -4.0125]","-4.0303<br/>[-4.0430, -4.0171]","-4.0302<br/>[-4.0429, -4.0171]","-4.0301<br/>[-4.0428, -4.0170]","-4.0303<br/>[-4.0431, -4.0171]","-4.0303<br/>[-4.0430, -4.0171]","-4.0301<br/>[-4.0428, -4.0169]","-4.0300<br/>[-4.0428, -4.0169]","-4.0308<br/>[-4.0432, -4.0180]","-4.0265<br/>[-4.0389, -4.0136]","-4.0260<br/>[-4.0386, -4.0131]","-4.0260<br/>[-4.0385, -4.0130]","-4.0259<br/>[-4.0385, -4.0130]","-4.0257<br/>[-4.0383, -4.0127]","-4.0255<br/>[-4.0381, -4.0124]"],["-2.3941<br/>[-2.4299, -2.3595]","-2.3339<br/>[-2.3490, -2.3264]","-2.4511<br/>[-2.4853, -2.4179]","-2.5017<br/>[-2.5209, -2.4737]","-2.5068<br/>[-2.5261, -2.4827]","-2.5320<br/>[-2.5502, -2.5086]","-2.5059<br/>[-2.5280, -2.4777]","-2.5500<br/>[-2.5689, -2.5209]","-2.5479<br/>[-2.5619, -2.5299]","-2.5193<br/>[-2.5314, -2.4954]","-2.5730<br/>[-2.5859, -2.5481]","-2.7538<br/>[-2.8213, -2.6925]","-2.7600<br/>[-2.8312, -2.6952]","-2.8766<br/>[-2.9906, -2.7902]","-2.8919<br/>[-3.0037, -2.7959]","-2.9197<br/>[-3.0229, -2.8320]","-2.9401<br/>[-3.0217, -2.8645]","-2.9775<br/>[-3.0793, -2.8873]","-2.9574<br/>[-3.0297, -2.8808]","-2.5487<br/>[-2.5823, -2.5234]","-2.9874<br/>[-3.1285, -2.8709]","-2.9965<br/>[-3.0973, -2.9177]","-3.0307<br/>[-3.1792, -2.9167]","-2.7300<br/>[-2.7947, -2.6543]","-3.1340<br/>[-3.2387, -3.0269]","-3.3346<br/>[-3.5407, -3.1536]","-3.3554<br/>[-3.6227, -3.1674]","-3.4442<br/>[-3.6687, -3.2541]","-3.5568<br/>[-3.7007, -3.3965]","-3.6474<br/>[-3.7223, -3.5366]","-3.0422<br/>[-3.1920, -2.9266]","-3.0540<br/>[-3.1942, -2.9413]","-3.0898<br/>[-3.2210, -2.9802]","-3.0654<br/>[-3.1776, -2.9620]","-3.0721<br/>[-3.1812, -2.9698]","-3.1074<br/>[-3.2241, -2.9985]","-3.1163<br/>[-3.2348, -3.0092]","-3.1297<br/>[-3.2630, -3.0066]","-3.2403<br/>[-3.5601, -3.0732]","-3.3255<br/>[-3.7029, -3.1028]","-3.3258<br/>[-3.7174, -3.0948]","-3.3707<br/>[-3.7143, -3.1542]","-3.4558<br/>[-3.7538, -3.2527]","-3.5379<br/>[-3.7558, -3.3576]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div id="htmlwidget-9dd0ac780160f191cf9f" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-9dd0ac780160f191cf9f">{"x":{"filter":"none","data":[["Testis","Liver","Kidney_Cortex","Adrenal_Gland","Breast_Mammary_Tissue","Adipose_Subcutaneous","Small_Intestine_Terminal_Ileum","Adipose_Visceral_(Omentum)","Uterus","Stomach","Brain_Cortex","Brain_Caudate_(basal_ganglia)","Brain_Putamen_(basal_ganglia)","Brain_Nucleus_accumbens_(basal_ganglia)","Brain_Frontal_Cortex_(BA9)","Prostate","Minor_Salivary_Gland","Brain_Hippocampus","Brain_Amygdala","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Hypothalamus","Brain_Substantia_nigra","Artery_Tibial","Artery_Aorta","Esophagus_Gastroesophageal_Junction","Heart_Atrial_Appendage","Heart_Left_Ventricle","Colon_Sigmoid","Artery_Coronary","Esophagus_Muscularis","Colon_Transverse","Esophagus_Mucosa","Thyroid","Vagina","Nerve_Tibial","Brain_Cerebellum","Cells_Transformed_fibroblasts","Brain_Cerebellar_Hemisphere","Whole_Blood","Spleen","Ovary","Cells_EBV-transformed_lymphocytes","Pancreas","Skin_Sun_Exposed_(Lower_leg)","Skin_Not_Sun_Exposed_(Suprapubic)","Muscle_Skeletal","Lung","Pituitary","Brain_Spinal_cord_(cervical_c-1)"],[7350,7802,8549,11764,9405,9708,8318,9405,12030,8858,11476,11221,11293,10900,13167,8710,8003,10489,10756,12484,11718,12225,12095,13755,13582,10637,10542,10522,11905,13752,7141,6519,11536,6415,11965,10458,10342,13068,6787,6835,11108,7446,7325,6475,6558,8257,7960,9113,12198],[831.624249328303,78.3018068012931,47.33184794992,38.5302348195227,8.76710498352084,7.10365218679871,7.00076792924858,4.46206092714211,3.47606082078827,3.45912546581733,2.47374851612024,2.43508707805053,2.41127313691331,2.41041679833768,2.28939426529292,1.91219123422169,1.81919700188814,1.69378070105506,1.65833163349494,1.5875085727687,1.55844908526278,1.5172162375537,0.686210709514993,0.633230195471363,0.613431569452819,0.592498805066991,0.556161305763991,0.432985339464681,0.364528212952307,0.313794039513618,0.175882809410624,0.0849480160320504,-0.0102210704555253,-0.0150314809925611,-0.166370834736919,-0.237266598615805,-0.312335353563115,-0.409497108081871,-0.529549788358677,-0.531151135749514,-0.578908327331269,-0.58102441372352,-0.653375816287891,-0.656502647582483,-0.659668300290412,-0.759506349467074,-0.760032638351012,-0.781047302410597,-0.840224235878113],["-4.0292<br/>[-4.0419, -4.0160]","-4.1244<br/>[-4.1383, -4.1102]","-4.0613<br/>[-4.0748, -4.0474]","-4.0603<br/>[-4.0740, -4.0462]","-4.0383<br/>[-4.0512, -4.0250]","-4.0312<br/>[-4.0436, -4.0184]","-4.0424<br/>[-4.0553, -4.0290]","-4.0380<br/>[-4.0499, -4.0258]","-4.0387<br/>[-4.0515, -4.0255]","-4.0371<br/>[-4.0498, -4.0239]","-4.0319<br/>[-4.0447, -4.0186]","-4.0320<br/>[-4.0448, -4.0187]","-4.0319<br/>[-4.0447, -4.0187]","-4.0320<br/>[-4.0449, -4.0188]","-4.0312<br/>[-4.0440, -4.0179]","-4.0330<br/>[-4.0459, -4.0198]","-4.0327<br/>[-4.0453, -4.0195]","-4.0323<br/>[-4.0451, -4.0190]","-4.0322<br/>[-4.0450, -4.0189]","-4.0315<br/>[-4.0443, -4.0182]","-4.0317<br/>[-4.0445, -4.0185]","-4.0315<br/>[-4.0443, -4.0182]","-4.0273<br/>[-4.0400, -4.0141]","-4.0265<br/>[-4.0393, -4.0134]","-4.0268<br/>[-4.0395, -4.0137]","-4.0324<br/>[-4.0453, -4.0191]","-4.0234<br/>[-4.0360, -4.0103]","-4.0278<br/>[-4.0406, -4.0147]","-4.0273<br/>[-4.0401, -4.0141]","-4.0267<br/>[-4.0395, -4.0136]","-4.0289<br/>[-4.0415, -4.0159]","-4.0256<br/>[-4.0381, -4.0126]","-4.0277<br/>[-4.0404, -4.0145]","-4.0254<br/>[-4.0380, -4.0124]","-4.0228<br/>[-4.0354, -4.0097]","-4.0259<br/>[-4.0381, -4.0133]","-4.0278<br/>[-4.0405, -4.0146]","-4.0248<br/>[-4.0370, -4.0122]","-4.0257<br/>[-4.0382, -4.0128]","-4.0257<br/>[-4.0382, -4.0128]","-4.0231<br/>[-4.0358, -4.0101]","-4.0254<br/>[-4.0379, -4.0124]","-4.0251<br/>[-4.0376, -4.0121]","-4.0252<br/>[-4.0378, -4.0122]","-4.0252<br/>[-4.0378, -4.0122]","-4.0243<br/>[-4.0369, -4.0113]","-4.0244<br/>[-4.0370, -4.0113]","-4.0239<br/>[-4.0366, -4.0109]","-4.0227<br/>[-4.0353, -4.0096]"],["-3.2629<br/>[-3.2635, -3.2618]","-2.1730<br/>[-2.1733, -2.1728]","-2.6778<br/>[-2.6973, -2.6451]","-2.9090<br/>[-2.9453, -2.7858]","-3.2648<br/>[-3.2761, -3.2443]","-3.0056<br/>[-3.0826, -2.9365]","-2.9796<br/>[-3.0592, -2.9091]","-3.1407<br/>[-3.1897, -3.0676]","-3.3275<br/>[-3.3776, -3.2751]","-3.2783<br/>[-3.3976, -3.1828]","-3.5058<br/>[-3.5972, -3.4259]","-3.5131<br/>[-3.5917, -3.4378]","-3.5213<br/>[-3.5962, -3.4492]","-3.5195<br/>[-3.5823, -3.4559]","-3.5728<br/>[-3.6600, -3.4935]","-3.5334<br/>[-3.6265, -3.4369]","-3.5497<br/>[-3.6167, -3.4787]","-3.6431<br/>[-3.7114, -3.5657]","-3.6602<br/>[-3.7229, -3.5851]","-3.7037<br/>[-3.7826, -3.6174]","-3.7118<br/>[-3.7627, -3.6474]","-3.7345<br/>[-3.7808, -3.6735]","-3.6144<br/>[-3.7393, -3.5260]","-3.6452<br/>[-3.7842, -3.5388]","-3.6531<br/>[-3.7861, -3.5546]","-3.6193<br/>[-3.7056, -3.5285]","-3.8801<br/>[-4.0090, -3.7205]","-3.6857<br/>[-3.7178, -3.6404]","-3.7244<br/>[-3.7689, -3.6644]","-3.7626<br/>[-3.8226, -3.6886]","-3.5520<br/>[-3.8423, -3.3650]","-3.5856<br/>[-3.8533, -3.4003]","-3.7485<br/>[-3.9833, -3.5633]","-3.6599<br/>[-3.8690, -3.5090]","-3.8584<br/>[-4.0424, -3.7167]","-3.8031<br/>[-4.2359, -3.5959]","-3.9686<br/>[-4.0284, -3.8919]","-3.9720<br/>[-4.3730, -3.7663]","-4.0609<br/>[-4.6237, -3.7500]","-4.0665<br/>[-4.6114, -3.7618]","-4.1698<br/>[-4.5117, -3.9498]","-4.1704<br/>[-4.6838, -3.8654]","-4.3601<br/>[-4.9472, -3.9406]","-4.4653<br/>[-5.0402, -4.0784]","-4.4741<br/>[-5.0338, -4.0873]","-5.0646<br/>[-5.7795, -4.6252]","-5.2371<br/>[-5.9826, -4.7814]","-5.1312<br/>[-5.7859, -4.6972]","-5.1708<br/>[-5.7800, -4.7541]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div id="htmlwidget-798b325e8770e69abc2e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-798b325e8770e69abc2e">{"x":{"filter":"none","data":[["20","19","18","16","4","9","2","1","3","13","7","10","11","6","5","14","17","8","12","15"],[7643,5554,6884,9734,11777,9065,8428,7017,9555,7158,7753,7847,9849,8324,10930,8926,7959,9695,6819,7282],[55.970799696169,27.3087120739373,7.88511859738518,6.26711909026508,5.06908520013794,3.05299825116557,2.85681693600718,2.30723977369433,2.08889751871904,1.38297087010023,1.2946823994595,0.979125154029098,0.87361947479212,0.743196150846694,0.703216350441884,-0.0725796340809404,-0.446874773299393,-0.571003711405703,-0.606908675667966,-0.620297551273622],["-4.0948<br/>[-4.1085, -4.0809]","-4.0337<br/>[-4.0463, -4.0205]","-4.0380<br/>[-4.0509, -4.0247]","-4.0482<br/>[-4.0610, -4.0349]","-4.0472<br/>[-4.0603, -4.0337]","-4.0377<br/>[-4.0507, -4.0242]","-4.0342<br/>[-4.0466, -4.0213]","-4.0308<br/>[-4.0433, -4.0179]","-4.0351<br/>[-4.0477, -4.0222]","-4.0340<br/>[-4.0469, -4.0207]","-4.0245<br/>[-4.0372, -4.0115]","-4.0334<br/>[-4.0462, -4.0201]","-4.0284<br/>[-4.0411, -4.0153]","-4.0289<br/>[-4.0416, -4.0157]","-4.0281<br/>[-4.0408, -4.0149]","-4.0284<br/>[-4.0411, -4.0153]","-4.0253<br/>[-4.0378, -4.0124]","-4.0246<br/>[-4.0371, -4.0118]","-4.0250<br/>[-4.0377, -4.0120]","-4.0251<br/>[-4.0377, -4.0121]"],["-2.3841<br/>[-2.3978, -2.3757]","-2.9095<br/>[-2.9242, -2.8848]","-3.0074<br/>[-3.0289, -2.9716]","-3.0385<br/>[-3.1179, -2.9721]","-3.2008<br/>[-3.2643, -3.1419]","-3.2865<br/>[-3.3405, -3.2287]","-3.3060<br/>[-3.4646, -3.1791]","-3.1110<br/>[-3.2692, -2.9924]","-3.3306<br/>[-3.4399, -3.2453]","-3.3116<br/>[-3.4884, -3.1811]","-3.3975<br/>[-3.4455, -3.3599]","-3.4552<br/>[-3.5753, -3.3509]","-3.5068<br/>[-3.6736, -3.3860]","-3.5121<br/>[-3.6481, -3.3986]","-3.5885<br/>[-3.7127, -3.4789]","-3.7910<br/>[-3.9294, -3.6306]","-3.9721<br/>[-4.5890, -3.6324]","-4.2156<br/>[-4.6942, -3.9181]","-4.3832<br/>[-5.0583, -3.9503]","-4.4467<br/>[-4.9502, -4.0868]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2]}],"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data) {\nDTWidget.formatRound(this, row, data, 2, 3, 3, ',', '.');\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey et al. (2017)</a>; see below.</p>
<p><img src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
</div>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1">devtools<span class="op">::</span><span class="kw">session_info</span>()</a></code></pre></div>
<pre><code>Session info -------------------------------------------------------------</code></pre>
<pre><code> setting  value                       
 version  R version 3.5.1 (2018-07-02)
 system   x86_64, darwin15.6.0        
 ui       X11                         
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2018-10-05                  </code></pre>
<pre><code>Packages -----------------------------------------------------------------</code></pre>
<pre><code> package     * version date       source        
 assertthat    0.2.0   2017-04-11 CRAN (R 3.5.0)
 backports     1.1.2   2017-12-13 CRAN (R 3.5.0)
 base        * 3.5.1   2018-07-05 local         
 bindr         0.1.1   2018-03-13 CRAN (R 3.5.0)
 bindrcpp      0.2.2   2018-03-29 CRAN (R 3.5.0)
 compiler      3.5.1   2018-07-05 local         
 crayon        1.3.4   2017-09-16 CRAN (R 3.5.0)
 crosstalk     1.0.0   2016-12-21 CRAN (R 3.5.0)
 datasets    * 3.5.1   2018-07-05 local         
 devtools      1.13.6  2018-06-27 CRAN (R 3.5.0)
 digest        0.6.17  2018-09-12 CRAN (R 3.5.0)
 dplyr       * 0.7.6   2018-06-29 CRAN (R 3.5.1)
 DT          * 0.4     2018-01-30 CRAN (R 3.5.0)
 evaluate      0.11    2018-07-17 CRAN (R 3.5.0)
 git2r         0.23.0  2018-07-17 CRAN (R 3.5.0)
 glue          1.3.0   2018-07-17 CRAN (R 3.5.0)
 graphics    * 3.5.1   2018-07-05 local         
 grDevices   * 3.5.1   2018-07-05 local         
 htmltools     0.3.6   2017-04-28 CRAN (R 3.5.0)
 htmlwidgets   1.3     2018-09-30 CRAN (R 3.5.0)
 httpuv        1.4.5   2018-07-19 CRAN (R 3.5.0)
 jsonlite      1.5     2017-06-01 CRAN (R 3.5.0)
 knitr         1.20    2018-02-20 CRAN (R 3.5.0)
 later         0.7.5   2018-09-18 CRAN (R 3.5.0)
 magrittr      1.5     2014-11-22 CRAN (R 3.5.0)
 memoise       1.1.0   2017-04-21 CRAN (R 3.5.0)
 methods     * 3.5.1   2018-07-05 local         
 mime          0.5     2016-07-07 CRAN (R 3.5.0)
 pillar        1.3.0   2018-07-14 CRAN (R 3.5.0)
 pkgconfig     2.0.2   2018-08-16 CRAN (R 3.5.0)
 plyr        * 1.8.4   2016-06-08 CRAN (R 3.5.0)
 promises      1.0.1   2018-04-13 CRAN (R 3.5.0)
 purrr         0.2.5   2018-05-29 CRAN (R 3.5.0)
 R.matlab    * 3.6.2   2018-09-27 CRAN (R 3.5.0)
 R.methodsS3   1.7.1   2016-02-16 CRAN (R 3.5.0)
 R.oo          1.22.0  2018-04-22 CRAN (R 3.5.0)
 R.utils       2.7.0   2018-08-27 CRAN (R 3.5.0)
 R6            2.3.0   2018-10-04 CRAN (R 3.5.0)
 Rcpp          0.12.19 2018-10-01 CRAN (R 3.5.0)
 rlang         0.2.2   2018-08-16 CRAN (R 3.5.0)
 rmarkdown     1.10    2018-06-11 CRAN (R 3.5.0)
 rprojroot     1.3-2   2018-01-03 CRAN (R 3.5.0)
 shiny         1.1.0   2018-05-17 CRAN (R 3.5.0)
 stats       * 3.5.1   2018-07-05 local         
 stringi       1.2.4   2018-07-20 CRAN (R 3.5.0)
 stringr       1.3.1   2018-05-10 CRAN (R 3.5.0)
 tibble        1.4.2   2018-01-22 CRAN (R 3.5.0)
 tidyselect    0.2.4   2018-02-26 CRAN (R 3.5.0)
 tools         3.5.1   2018-07-05 local         
 utils       * 3.5.1   2018-07-05 local         
 whisker       0.3-2   2013-04-28 CRAN (R 3.5.0)
 withr         2.1.2   2018-03-15 CRAN (R 3.5.0)
 workflowr     1.1.1   2018-07-06 CRAN (R 3.5.0)
 xtable        1.8-3   2018-08-29 CRAN (R 3.5.0)
 yaml          2.2.0   2018-07-25 CRAN (R 3.5.0)</code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
