<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Xiang Zhu" />

<meta name="date" content="2017-04-12" />

<title>Enrichment results of serum urate</title>

<script src="site_libs/header-attrs-2.28/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>



<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">rss-gsea</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
<li>
  <a href="https://stephenslab.github.io/rss/news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xiangzhu/rss-gsea">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Enrichment results of serum urate</h1>
<h4 class="author">Xiang Zhu</h4>
<h4 class="date">2017-04-12</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-09-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>rss-gsea/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20180626code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20180626)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20180626code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20180626)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree0d0131b608937a899299b4dd8cf4ff994ecea3cbtargetblank0d0131ba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/xiangzhu/rss-gsea/tree/0d0131b608937a899299b4dd8cf4ff994ecea3cb" target="_blank">0d0131b</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomxiangzhurssgseatree0d0131b608937a899299b4dd8cf4ff994ecea3cbtargetblank0d0131ba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/xiangzhu/rss-gsea/tree/0d0131b608937a899299b4dd8cf4ff994ecea3cb" target="_blank">0d0131b</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/urate_2013.Rmd</code>) and HTML
(<code>docs/urate_2013.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/dedfe81645d7f81d60203d25ab700459d91649ab/docs/urate_2013.html" target="_blank">dedfe81</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2024-09-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/72237818c1b8a1963ebe64d9a5776caf4d1ca79e/docs/urate_2013.html" target="_blank">7223781</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2019-02-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/73f3c80eafd4669d84f817685bc7f2ded7510544/docs/urate_2013.html" target="_blank">73f3c80</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/1c85967d82bd54d5e76573a2360750fc083cedc5/docs/urate_2013.html" target="_blank">1c85967</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-10-05
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/xiangzhu/rss-gsea/20037a0520b9583007465aedce6837150bfa7fef/docs/urate_2013.html" target="_blank">20037a0</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/xiangzhu/rss-gsea/blob/c61fb1a357f6abcfd3b2e67f142ad7756bf191cf/analysis/urate_2013.Rmd" target="_blank">c61fb1a</a>
</td>
<td>
Xiang Zhu
</td>
<td>
2018-07-01
</td>
<td>
wflow_publish("urate_2013.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="input-data" class="section level2">
<h2>Input data</h2>
<p>Results below were generated from the GWAS summary statistics
published in the paper <a
href="https://www.ncbi.nlm.nih.gov/pubmed/23263486">“Genome-Wide
Association Analyses Identify 18 New Loci Associated with Serum Urate
Concentrations” (<em>Nature Genetics</em>, 2013)</a>. The summary data
file is available at <a
href="http://metabolomics.helmholtz-muenchen.de/gugc"
class="uri">http://metabolomics.helmholtz-muenchen.de/gugc</a>.</p>
</div>
<div id="analysis-results" class="section level2">
<h2>Analysis results</h2>
<p>Enrichment analyses are summarized by the following three
quantities.</p>
<ul>
<li><strong>BF:</strong> Bayes factor comparing the enrichment model
against the baseline model;</li>
<li><strong>Outside <span class="math inline">\(\pi\)</span>:</strong>
proportion of trait-associated SNPs that are “outside” the gene
set;</li>
<li><strong>Inside <span class="math inline">\(\pi\)</span>:</strong>
proportion of trait-associated SNPs that are “inside” the gene set.</li>
</ul>
<p>The first quantity reflects the significance of enrichment, whereas
the last two capture the magnitude of enrichment. For each gene set, we
report these three quantities in the last three columns of tables below,
on log 10 scale.</p>
<div id="biological-pathways" class="section level3">
<h3>Biological pathways</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-31fbc4f90f9b8b0a7aa7" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-31fbc4f90f9b8b0a7aa7">{"x":{"filter":"none","vertical":false,"data":[["Class II GLUTs<br/>(365, reactome, PC)","Facilitative Na+-independent glucose transporters<br/>(1434, reactome, BioSystems)","Facilitative Na+-independent glucose transporters<br/>(1225, reactome, PC)","Transport of glucose and other sugars, bile salts and organic acids, metal ions and amine compounds<br/>(3496, reactome, PC)","SLC-mediated transmembrane transport<br/>(3847, reactome, PC)","SLC-mediated transmembrane transport<br/>(3852, reactome, BioSystems)","Transport of glucose and other sugars, bile salts and organic acids, metal ions and amine compounds<br/>(3518, reactome, BioSystems)","Hypertrophy Model<br/>(1921, wiki, BioSystems)","Platelet activation, signaling and aggregation<br/>(3815, reactome, BioSystems)","Platelet activation, signaling and aggregation<br/>(3811, reactome, PC)","Platelet degranulation<br/>(3307, reactome, PC)","Response to elevated platelet cytosolic Ca2+<br/>(3355, reactome, PC)","Hemostasis<br/>(3914, reactome, PC)","Hemostasis<br/>(3916, reactome, BioSystems)","proline biosynthesis<br/>(141, biocyc, BioSystems)","PI3K-Akt signaling pathway<br/>(3897, kegg, BioSystems)","pentose phosphate pathway (oxidative branch)<br/>(102, humancyc, PC)","Pentose phosphate pathway, oxidative phase, glucose 6P => ribulose 5P<br/>(329, kegg, BioSystems)","Abacavir transmembrane transport<br/>(514, reactome, PC)","Abacavir transport and metabolism<br/>(1104, reactome, PC)","VxPx cargo-targeting to cilium<br/>(1917, reactome, BioSystems)","Signaling by PDGF<br/>(3896, reactome, BioSystems)","Irinotecan Pathway<br/>(1451, wiki, BioSystems)","HIF-2-alpha transcription factor network<br/>(2466, pid, PC)","Fluoropyrimidine Activity<br/>(2543, wiki, BioSystems)","IRAK4 deficiency (TLR5)<br/>(462, reactome, BioSystems)","ABC transporters<br/>(2792, kegg, BioSystems)","Cargo trafficking to the periciliary membrane<br/>(2947, reactome, BioSystems)","Iron uptake and transport<br/>(2693, reactome, PC)","ornithine de novo biosynthesis<br/>(169, biocyc, BioSystems)","Proline biosynthesis, glutamate => proline<br/>(498, kegg, BioSystems)","ornithine de novo biosynthesis<br/>(274, humancyc, PC)","HIF-1-alpha transcription factor network<br/>(3190, pid, BioSystems)","Post-transcriptional silencing by small RNAs<br/>(812, reactome, PC)","HIF-1-alpha transcription factor network<br/>(3201, pid, PC)","Ketone body catabolism<br/>(212, reactome, PC)","Type II diabetes mellitus<br/>(1922, wiki, BioSystems)","Bile secretion<br/>(3277, kegg, BioSystems)","ketolysis<br/>(420, humancyc, PC)","Integrins in angiogenesis<br/>(3113, pid, PC)","Integrins in angiogenesis<br/>(2878, pid, BioSystems)","HATs acetylate histones<br/>(3671, reactome, PC)","HATs acetylate histones<br/>(3693, reactome, BioSystems)","Signaling by PDGF<br/>(3776, reactome, PC)","Chromatin modifying enzymes<br/>(3839, reactome, BioSystems)","Chromatin modifying enzymes<br/>(3805, reactome, PC)","Alpha9 beta1 integrin signaling events<br/>(1988, pid, PC)","Endochondral Ossification<br/>(3208, wiki, BioSystems)","Osteopontin Signaling<br/>(1294, wiki, BioSystems)","Beta3 integrin cell surface interactions<br/>(2717, pid, PC)","FGF signaling pathway<br/>(3037, pid, PC)","FGF signaling pathway<br/>(2903, pid, BioSystems)","Osteopontin-mediated events<br/>(2431, pid, PC)","Osteopontin-mediated events<br/>(2508, pid, BioSystems)","Osteoclast Signaling<br/>(1687, wiki, BioSystems)","Ketone body metabolism<br/>(549, reactome, PC)","glycine/serine biosynthesis<br/>(98, humancyc, PC)","Direct p53 effectors<br/>(3683, pid, BioSystems)","Regulation of toll-like receptor signaling pathway<br/>(3521, wiki, BioSystems)","Toll-like receptor signaling pathway<br/>(3557, kegg, BioSystems)","Toll-like receptor signaling pathway<br/>(3525, wiki, BioSystems)","Direct p53 effectors<br/>(3703, pid, PC)","Alpha4 beta1 integrin signaling events<br/>(2513, pid, PC)","Beta1 integrin cell surface interactions<br/>(3177, pid, PC)","BDNF signaling pathway<br/>(3695, wiki, BioSystems)","Degradation of the extracellular matrix<br/>(3442, reactome, PC)","TGF Beta Signaling Pathway<br/>(3031, wiki, BioSystems)","Extracellular matrix organization<br/>(3861, reactome, BioSystems)","Interleukin-11 Signaling Pathway<br/>(2737, wiki, BioSystems)","Focal Adhesion<br/>(3774, wiki, BioSystems)","ECM-receptor interaction<br/>(3433, kegg, BioSystems)","Integrin cell surface interactions<br/>(3212, reactome, PC)","DNA polymerase alpha / primase complex<br/>(324, kegg, BioSystems)","Osteoblast Signaling<br/>(1453, wiki, BioSystems)","Focal adhesion<br/>(3803, kegg, BioSystems)","C-MYC pathway<br/>(2041, pid, BioSystems)","Extracellular matrix organization<br/>(3851, reactome, PC)","C-MYC pathway<br/>(2148, pid, PC)","ECM proteoglycans<br/>(3036, reactome, PC)","Carbohydrate metabolism<br/>(3841, reactome, PC)","Metabolism of carbohydrates<br/>(3849, reactome, BioSystems)","Trk receptor signaling mediated by the MAPK pathway<br/>(2319, pid, BioSystems)","Validated targets of C-MYC transcriptional activation<br/>(3343, pid, PC)","Validated targets of C-MYC transcriptional activation<br/>(3364, pid, BioSystems)","dTMP de novo biosynthesis (mitochondrial)<br/>(259, humancyc, PC)","HuR stabilizes mRNA<br/>(918, reactome, PC)","Regulation of Glucokinase by Glucokinase Regulatory Protein<br/>(2297, reactome, PC)","Regulation of Glucokinase by Glucokinase Regulatory Protein<br/>(2344, reactome, BioSystems)","Glucose transport<br/>(2691, reactome, PC)","Glucose transport<br/>(2757, reactome, BioSystems)","Hexose uptake<br/>(2847, reactome, PC)","Hexose transport<br/>(2893, reactome, BioSystems)","pentose phosphate pathway<br/>(639, biocyc, BioSystems)","Pentose Phosphate Pathway<br/>(889, wiki, BioSystems)","4-hydroxyproline degradation<br/>(344, humancyc, PC)","Salvage pyrimidine deoxyribonucleotides<br/>(61, panther, PC)","Rho-Selective Guanine Exchange Factor AKAP13 Mediates Stress Fiber Formation<br/>(1348, biocarta, BioCarta)","Ascorbate degradation<br/>(120, panther, PANTHER)","proline biosynthesis<br/>(275, humancyc, PC)","Small interfering RNA (siRNA) biogenesis<br/>(1049, reactome, PC)"],[4,13,11,95,251,254,98,20,217,211,74,79,490,496,2,335,2,3,5,10,20,335,13,32,33,4,42,49,39,2,4,3,64,7,65,3,20,70,4,60,46,132,136,179,241,203,22,65,11,40,54,47,31,32,16,5,2,133,98,102,98,139,33,64,136,87,53,259,40,177,85,66,3,13,200,22,254,25,54,244,253,28,78,79,3,8,28,29,39,41,45,47,5,7,4,2,12,2,3,9],[500,1110,953,8823,24228,24715,9230,1832,22410,22066,7024,7898,47497,47869,164,33164,146,248,623,1050,2384,34087,1193,2761,3870,227,4448,4890,3260,271,307,318,6123,484,6719,219,1737,7558,289,6317,4670,5987,6394,20136,15750,12183,2675,7547,848,4340,5067,4457,3285,3362,1526,349,85,11597,6824,6902,6919,11932,3359,7558,14875,9054,5412,27794,3449,19437,10730,6892,159,1334,23092,1800,27313,2020,6295,23294,23886,2591,5877,5877,184,928,2229,2315,3141,3298,3642,3799,331,429,328,235,1424,149,234,555],[1817.898458006694,1651.625725075521,1642.599363303567,1558.06453422305,1329.776517838277,1308.428509544722,1156.242159887835,592.8281311648666,456.594733092567,372.9343206920205,348.8162553875852,307.9560720793123,301.832831593335,256.6758600016709,152.29315657259,140.6957404006748,117.9485035646194,117.5122094508421,108.158050154849,104.9790087364719,101.457528154516,93.35196153857319,89.94443313812995,87.81384740243102,85.16589963525799,84.14588727257851,79.79346818649381,78.42844809029174,68.36787706983789,67.17936623931492,67.17040004944251,67.04521198626371,61.64921341372962,61.57607419506952,58.16665578544207,57.5222212075324,54.91977570791429,46.2347287628912,26.32339027284415,24.30522130721007,23.6918169429454,22.75849484532015,22.75142564244718,22.2300404250068,22.05906648377593,21.57229454343588,21.47216581107808,20.48350979875967,19.77150198570948,19.10151875599376,15.67061178862446,15.60382810048418,15.5292877152512,15.31132560861194,15.2716327055267,14.29653032818661,14.12501603102889,12.52067972603451,12.49471500425496,12.42881031587035,12.35393909983493,12.321667589956,11.54578776806497,11.4116797213819,10.8963340340215,10.84929647593571,10.75705992746226,10.2973624080663,9.764786959097266,9.640809497637978,9.419243681922641,9.300797105237342,8.532941074844695,7.277714190290262,7.035311114084281,4.31757331943377,4.249858852409218,4.239659761875697,4.034025510678649,3.754293258511237,3.418015558206936,3.027798477081233,2.72123227081455,2.72123227081455,2.240593683399813,1.599905969323946,0.9961700039893444,0.9937854359371894,0.9024247233660218,0.8646654089381967,0.8410793447809709,0.8387245465450982,0.1603333605987075,0.08431363802844551,-0.1044289067595094,-0.113254113119857,-0.1135555535825514,-0.1219348369116778,-0.1397136060863308,-0.2091901216133172],["-4.2689<br/>[-4.2689, -4.2689]","-4.2687<br/>[-4.2687, -4.2687]","-4.2688<br/>[-4.2688, -4.2688]","-4.2812<br/>[-4.2812, -4.2812]","-4.2915<br/>[-4.2915, -4.2915]","-4.2913<br/>[-4.2913, -4.2913]","-4.2810<br/>[-4.2810, -4.2810]","-4.2316<br/>[-4.2316, -4.2316]","-4.2231<br/>[-4.2231, -4.2231]","-4.2232<br/>[-4.2232, -4.2232]","-4.2294<br/>[-4.2294, -4.2294]","-4.2291<br/>[-4.2291, -4.2291]","-4.2197<br/>[-4.2197, -4.2197]","-4.2195<br/>[-4.2195, -4.2195]","-4.0321<br/>[-4.0321, -4.0321]","-4.0422<br/>[-4.0422, -4.0422]","-4.0322<br/>[-4.0322, -4.0322]","-4.0321<br/>[-4.0321, -4.0321]","-4.0456<br/>[-4.0456, -4.0456]","-4.0455<br/>[-4.0455, -4.0455]","-4.0450<br/>[-4.0450, -4.0450]","-4.0365<br/>[-4.0365, -4.0365]","-4.0454<br/>[-4.0454, -4.0454]","-4.0541<br/>[-4.0541, -4.0541]","-4.0443<br/>[-4.0443, -4.0443]","-4.0321<br/>[-4.0321, -4.0321]","-4.0441<br/>[-4.0441, -4.0441]","-4.0442<br/>[-4.0442, -4.0442]","-4.0445<br/>[-4.0445, -4.0445]","-4.0321<br/>[-4.0321, -4.0321]","-4.0321<br/>[-4.0321, -4.0321]","-4.0321<br/>[-4.0321, -4.0321]","-4.0482<br/>[-4.0482, -4.0482]","-4.0320<br/>[-4.0320, -4.0320]","-4.0479<br/>[-4.0479, -4.0479]","-4.0321<br/>[-4.0321, -4.0321]","-4.0321<br/>[-4.0321, -4.0321]","-4.0428<br/>[-4.0428, -4.0428]","-4.0321<br/>[-4.0321, -4.0321]","-4.0437<br/>[-4.0437, -4.0437]","-4.0444<br/>[-4.0444, -4.0444]","-4.0344<br/>[-4.0344, -4.0344]","-4.0343<br/>[-4.0343, -4.0343]","-4.0378<br/>[-4.0378, -4.0378]","-4.0304<br/>[-4.0304, -4.0304]","-4.0319<br/>[-4.0319, -4.0319]","-4.0403<br/>[-4.0403, -4.0403]","-4.0428<br/>[-4.0428, -4.0428]","-4.0411<br/>[-4.0411, -4.0411]","-4.0401<br/>[-4.0401, -4.0401]","-4.0352<br/>[-4.0352, -4.0352]","-4.0354<br/>[-4.0354, -4.0354]","-4.0401<br/>[-4.0401, -4.0401]","-4.0400<br/>[-4.0400, -4.0400]","-4.0363<br/>[-4.0363, -4.0363]","-4.0321<br/>[-4.0321, -4.0321]","-4.0367<br/>[-4.0367, -4.0367]","-4.0322<br/>[-4.0322, -4.0322]","-4.0386<br/>[-4.0386, -4.0386]","-4.0386<br/>[-4.0386, -4.0386]","-4.0385<br/>[-4.0385, -4.0385]","-4.0320<br/>[-4.0320, -4.0320]","-4.0355<br/>[-4.0355, -4.0355]","-4.0383<br/>[-4.0383, -4.0383]","-4.0353<br/>[-4.0353, -4.0353]","-4.0332<br/>[-4.0332, -4.0332]","-4.0347<br/>[-4.0347, -4.0347]","-4.0253<br/>[-4.0253, -4.0253]","-4.0353<br/>[-4.0353, -4.0353]","-4.0379<br/>[-4.0379, -4.0379]","-4.0370<br/>[-4.0370, -4.0370]","-4.0340<br/>[-4.0340, -4.0340]","-4.0322<br/>[-4.0322, -4.0322]","-4.0363<br/>[-4.0363, -4.0363]","-4.0410<br/>[-4.0410, -4.0410]","-4.0359<br/>[-4.0359, -4.0359]","-4.0255<br/>[-4.0255, -4.0255]","-4.0359<br/>[-4.0359, -4.0359]","-4.0296<br/>[-4.0296, -4.0296]","-4.0273<br/>[-4.0273, -4.0273]","-4.0271<br/>[-4.0271, -4.0271]","-4.0311<br/>[-4.0311, -4.0311]","-4.0343<br/>[-4.0343, -4.0343]","-4.0343<br/>[-4.0343, -4.0343]","-4.0366<br/>[-4.0366, -4.0366]","-4.0318<br/>[-4.0318, -4.0318]","-4.0358<br/>[-4.0358, -4.0358]","-4.0358<br/>[-4.0358, -4.0358]","-4.0354<br/>[-4.0354, -4.0354]","-4.0354<br/>[-4.0354, -4.0354]","-4.0352<br/>[-4.0352, -4.0352]","-4.0352<br/>[-4.0352, -4.0352]","-4.0321<br/>[-4.0321, -4.0321]","-4.0320<br/>[-4.0320, -4.0320]","-4.0321<br/>[-4.0321, -4.0321]","-4.0321<br/>[-4.0321, -4.0321]","-4.0316<br/>[-4.0316, -4.0316]","-4.0322<br/>[-4.0322, -4.0322]","-4.0321<br/>[-4.0321, -4.0321]","-4.0320<br/>[-4.0320, -4.0320]"],["-1.0641<br/>[-1.0641, -1.0641]","-1.4100<br/>[-1.4100, -1.4100]","-1.3226<br/>[-1.3226, -1.3226]","-2.2960<br/>[-2.2960, -2.2960]","-2.6958<br/>[-2.6983, -2.6958]","-2.7894<br/>[-2.7894, -2.7894]","-2.3737<br/>[-2.3834, -2.3737]","-1.6305<br/>[-1.6305, -1.6305]","-2.8002<br/>[-2.8002, -2.8002]","-2.8208<br/>[-2.8208, -2.8208]","-2.3522<br/>[-2.3522, -2.3522]","-2.4491<br/>[-2.4497, -2.4491]","-3.1179<br/>[-3.1179, -3.1130]","-3.1338<br/>[-3.1338, -3.1338]","-1.9066<br/>[-1.9135, -1.8964]","-2.8017<br/>[-2.8017, -2.7996]","-1.8607<br/>[-1.8632, -1.8568]","-2.0910<br/>[-2.0933, -2.0873]","-1.4878<br/>[-1.4891, -1.4844]","-1.7334<br/>[-1.7367, -1.7173]","-1.8435<br/>[-1.8435, -1.8435]","-2.6882<br/>[-2.6882, -2.6778]","-1.7278<br/>[-1.7287, -1.7219]","-1.9300<br/>[-1.9320, -1.9300]","-2.1124<br/>[-2.1349, -2.0968]","-2.0520<br/>[-2.0549, -2.0478]","-2.1564<br/>[-2.1640, -2.1258]","-2.2111<br/>[-2.2115, -2.2111]","-2.1949<br/>[-2.1950, -2.1853]","-2.1175<br/>[-2.1316, -2.0968]","-2.1408<br/>[-2.1798, -2.0982]","-2.1559<br/>[-2.1970, -2.1107]","-2.3010<br/>[-2.3013, -2.3010]","-1.9473<br/>[-1.9707, -1.9190]","-2.4014<br/>[-2.4018, -2.4014]","-2.0351<br/>[-2.0392, -2.0291]","-1.8561<br/>[-1.8562, -1.8550]","-2.6348<br/>[-2.6349, -2.6348]","-1.9754<br/>[-1.9822, -1.9683]","-2.8299<br/>[-2.8438, -2.8238]","-2.6824<br/>[-2.7284, -2.6713]","-3.1521<br/>[-3.1679, -3.0707]","-3.1806<br/>[-3.1964, -3.0991]","-3.2627<br/>[-3.2821, -3.2546]","-3.5667<br/>[-3.5765, -3.5616]","-3.4753<br/>[-3.4786, -3.4735]","-2.4963<br/>[-2.5103, -2.4866]","-3.0299<br/>[-3.0381, -3.0259]","-2.1170<br/>[-2.1387, -2.0684]","-2.6794<br/>[-2.7213, -2.6450]","-2.7782<br/>[-2.8129, -2.7629]","-2.7305<br/>[-2.7624, -2.7166]","-2.7203<br/>[-2.7293, -2.7162]","-2.7371<br/>[-2.7463, -2.7327]","-2.3988<br/>[-2.4471, -2.3341]","-1.9326<br/>[-1.9355, -1.9302]","-1.4425<br/>[-1.4516, -1.4353]","-3.3731<br/>[-3.4196, -3.3626]","-3.0588<br/>[-3.1007, -3.0537]","-3.0646<br/>[-3.0871, -3.0546]","-3.0741<br/>[-3.0951, -3.0641]","-3.3958<br/>[-3.4337, -3.3742]","-2.8541<br/>[-2.8777, -2.8443]","-3.0883<br/>[-3.0964, -3.0847]","-3.3577<br/>[-3.3685, -3.3532]","-3.1595<br/>[-3.1713, -3.1543]","-3.0640<br/>[-3.0828, -3.0542]","-3.3799<br/>[-3.3968, -3.3555]","-2.7618<br/>[-2.8030, -2.7395]","-3.3549<br/>[-3.3981, -3.3151]","-3.0419<br/>[-3.0851, -2.9579]","-3.0032<br/>[-3.0434, -2.9709]","-1.8819<br/>[-1.9001, -1.8588]","-2.5318<br/>[-2.6271, -2.4198]","-3.4698<br/>[-3.4974, -3.4597]","-2.7759<br/>[-2.7771, -2.7752]","-3.4817<br/>[-3.5178, -3.4545]","-2.8260<br/>[-2.8272, -2.8252]","-3.4593<br/>[-3.4938, -3.4138]","-3.7132<br/>[-3.7785, -3.6803]","-3.7527<br/>[-3.8058, -3.7243]","-3.0649<br/>[-3.1162, -3.0040]","-3.2853<br/>[-3.2892, -3.2827]","-3.2853<br/>[-3.2892, -3.2827]","-2.2376<br/>[-2.2647, -2.1831]","-2.9304<br/>[-2.9674, -2.8639]","-3.3024<br/>[-3.3384, -3.2733]","-3.3189<br/>[-3.3549, -3.2896]","-3.4580<br/>[-3.4888, -3.4319]","-3.4819<br/>[-3.5103, -3.4574]","-3.5267<br/>[-3.5538, -3.5030]","-3.5446<br/>[-3.5721, -3.5204]","-2.5295<br/>[-4.9754, -2.1630]","-2.5541<br/>[-5.1864, -2.2720]","-4.9285<br/>[-7.0496, -4.2485]","-4.4549<br/>[-6.7174, -3.7406]","-4.7343<br/>[-6.2333, -4.0996]","-3.5804<br/>[-5.9630, -2.8642]","-4.6284<br/>[-6.8896, -3.8549]","-4.3710<br/>[-6.2789, -3.6984]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Pathway name<br/>(ID, database, repository)<\/th>\n      <th># of genes<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2,3]},{"name":"Pathway name<br/>(ID, database, repository)","targets":0},{"name":"# of genes","targets":1},{"name":"# of SNPs","targets":2},{"name":"Log10 BF","targets":3},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":4},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
<div id="tissue-highly-expressed-genes" class="section level3">
<h3>Tissue highly expressed genes</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-c98d9295234ac2628b54" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c98d9295234ac2628b54">{"x":{"filter":"none","vertical":false,"data":[["Liver","Muscle - Skeletal","Brain - Cortex","Brain - Frontal Cortex (BA9)","Testis","Stomach","Brain - Hippocampus","Brain - Anterior cingulate cortex (BA24)","Artery - Tibial","Cells - EBV-transformed lymphocytes","Brain - Cerebellum","Adrenal Gland","Esophagus - Mucosa","Spleen","Heart - Left Ventricle","Small Intestine - Terminal Ileum","Lung","Colon - Transverse","Pancreas","Brain - Cerebellar Hemisphere","Skin - Not Sun Exposed (Suprapubic)","Breast - Mammary Tissue","Adipose - Subcutaneous","Whole Blood","Nerve - Tibial","Uterus","Skin - Sun Exposed (Lower leg)","Vagina","Esophagus - Gastroesophageal Junction","Adipose - Visceral (Omentum)","Colon - Sigmoid","Prostate","Esophagus - Muscularis","Brain - Hypothalamus","Pituitary","Thyroid","Heart - Atrial Appendage","Ovary","Brain - Nucleus accumbens (basal ganglia)","Cells - Transformed fibroblasts","Artery - Coronary","Artery - Aorta","Brain - Caudate (basal ganglia)","Brain - Putamen (basal ganglia)"],[6290,5654,5833,6179,5835,5260,5735,5899,6023,4961,5681,5620,5579,4024,5631,4680,4997,5216,5096,5630,5771,5555,5611,5828,5674,5667,5701,5765,5789,5820,5892,5573,5974,5871,5881,6146,5838,5863,6089,6091,6148,6318,6456,6530],[3.013899638394633,2.811664695126906,2.285962980217909,2.219612726135562,0.899947461810266,0.812159645438472,0.7579187116521625,0.7386170911513743,-0.1504315500078358,-0.2103970763373561,-0.2223317023053974,-0.3492245891543146,-0.3838197101818505,-0.3882631886074271,-0.396817458590821,-0.4067906114602114,-0.4126050579028796,-0.4132618381288974,-0.4159836941501249,-0.4228668448125582,-0.424049595269862,-0.4280433505791045,-0.4286770417679631,-0.4296874712364138,-0.4297184402536737,-0.4299263103255905,-0.4302515763068382,-0.4323963797121529,-0.4325811572836511,-0.4334363487722913,-0.4348878738754014,-0.4359520405459281,-0.4366445879265972,-0.4391093386504032,-0.4402461891116339,-0.4404533149019232,-0.4405215260005054,-0.4422123687261366,-0.4439629179464471,-0.4468832450284005,-0.4478056335441895,-0.4515010526893347,-0.4519040155719752,-0.4523731068150166],["-4.0296<br/>[-4.0296, -4.0296]","-4.0344<br/>[-4.0344, -4.0344]","-4.0302<br/>[-4.0302, -4.0302]","-4.0301<br/>[-4.0301, -4.0301]","-4.0343<br/>[-4.0343, -4.0343]","-4.0345<br/>[-4.0345, -4.0345]","-4.0298<br/>[-4.0298, -4.0298]","-4.0298<br/>[-4.0298, -4.0298]","-4.0300<br/>[-4.0300, -4.0300]","-4.0303<br/>[-4.0303, -4.0303]","-4.0300<br/>[-4.0300, -4.0300]","-4.0299<br/>[-4.0299, -4.0299]","-4.0299<br/>[-4.0299, -4.0299]","-4.0306<br/>[-4.0306, -4.0306]","-4.0299<br/>[-4.0299, -4.0299]","-4.0303<br/>[-4.0303, -4.0303]","-4.0301<br/>[-4.0301, -4.0301]","-4.0301<br/>[-4.0301, -4.0301]","-4.0301<br/>[-4.0301, -4.0301]","-4.0299<br/>[-4.0299, -4.0299]","-4.0298<br/>[-4.0298, -4.0298]","-4.0299<br/>[-4.0299, -4.0299]","-4.0299<br/>[-4.0299, -4.0299]","-4.0298<br/>[-4.0298, -4.0298]","-4.0299<br/>[-4.0299, -4.0299]","-4.0299<br/>[-4.0299, -4.0299]","-4.0299<br/>[-4.0299, -4.0299]","-4.0298<br/>[-4.0298, -4.0298]","-4.0298<br/>[-4.0298, -4.0298]","-4.0298<br/>[-4.0298, -4.0298]","-4.0298<br/>[-4.0298, -4.0298]","-4.0299<br/>[-4.0299, -4.0299]","-4.0297<br/>[-4.0297, -4.0297]","-4.0298<br/>[-4.0298, -4.0298]","-4.0298<br/>[-4.0298, -4.0298]","-4.0297<br/>[-4.0297, -4.0297]","-4.0298<br/>[-4.0298, -4.0298]","-4.0298<br/>[-4.0298, -4.0298]","-4.0297<br/>[-4.0297, -4.0297]","-4.0297<br/>[-4.0297, -4.0297]","-4.0297<br/>[-4.0297, -4.0297]","-4.0296<br/>[-4.0296, -4.0296]","-4.0295<br/>[-4.0295, -4.0295]","-4.0295<br/>[-4.0295, -4.0295]"],["-3.4634<br/>[-3.4959, -3.4130]","-3.3499<br/>[-3.4302, -3.2549]","-3.2882<br/>[-3.4034, -3.2030]","-3.3154<br/>[-3.4324, -3.2271]","-3.6109<br/>[-3.7587, -3.4255]","-3.6712<br/>[-3.7198, -3.5934]","-3.7206<br/>[-3.7577, -3.6513]","-3.7397<br/>[-3.7701, -3.6823]","-4.0276<br/>[-4.9218, -3.7569]","-4.0446<br/>[-4.9141, -3.6985]","-4.0660<br/>[-4.9592, -3.6935]","-4.3493<br/>[-5.6051, -3.7563]","-4.5914<br/>[-5.7432, -4.0041]","-5.0588<br/>[-6.2316, -4.4750]","-4.8124<br/>[-5.5328, -4.3623]","-5.1603<br/>[-6.2914, -4.5790]","-5.1253<br/>[-6.2611, -4.5402]","-5.0421<br/>[-6.1569, -4.4596]","-5.1614<br/>[-6.3077, -4.5770]","-5.0686<br/>[-6.2113, -4.4795]","-5.0591<br/>[-6.1366, -4.4886]","-5.2622<br/>[-6.3509, -4.6959]","-5.2449<br/>[-6.3378, -4.6751]","-5.1567<br/>[-6.2486, -4.5823]","-5.2384<br/>[-6.3314, -4.6649]","-5.2509<br/>[-6.3407, -4.6779]","-5.2392<br/>[-6.3323, -4.6641]","-5.2715<br/>[-6.3545, -4.6927]","-5.2618<br/>[-6.3470, -4.6816]","-5.2709<br/>[-6.3530, -4.7074]","-5.2745<br/>[-6.3547, -4.7067]","-5.6029<br/>[-6.7904, -5.0012]","-5.2823<br/>[-6.3589, -4.7100]","-5.4386<br/>[-6.5868, -4.8342]","-5.4911<br/>[-6.6326, -4.8842]","-5.3050<br/>[-6.3738, -4.7416]","-5.5470<br/>[-6.7209, -4.9319]","-5.6341<br/>[-6.8059, -5.0355]","-5.4750<br/>[-6.6111, -4.8778]","-5.6519<br/>[-6.8108, -5.0395]","-5.6379<br/>[-6.7947, -5.0449]","-5.6766<br/>[-6.8238, -5.0727]","-5.5396<br/>[-6.6558, -4.9414]","-5.4905<br/>[-6.6090, -4.8904]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2]},{"name":"id","targets":0},{"name":"# of SNPs","targets":1},{"name":"log10.bf","targets":2},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":3},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
<div id="tissue-selectively-expressed-genes" class="section level3">
<h3>Tissue selectively expressed genes</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-8c90653e74bfddaf3bed" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8c90653e74bfddaf3bed">{"x":{"filter":"none","vertical":false,"data":[["Testis","Kidney_Cortex","Brain_Spinal_cord_(cervical_c-1)","Brain_Amygdala","Brain_Cortex","Brain_Anterior_cingulate_cortex_(BA24)","Brain_Frontal_Cortex_(BA9)","Muscle_Skeletal","Liver","Adrenal_Gland","Artery_Tibial","Artery_Coronary","Prostate","Pituitary","Thyroid","Cells_Transformed_fibroblasts","Heart_Left_Ventricle","Skin_Sun_Exposed_(Lower_leg)","Skin_Not_Sun_Exposed_(Suprapubic)","Stomach","Brain_Hippocampus","Brain_Nucleus_accumbens_(basal_ganglia)","Brain_Caudate_(basal_ganglia)","Brain_Putamen_(basal_ganglia)","Ovary","Nerve_Tibial","Uterus","Minor_Salivary_Gland","Lung","Artery_Aorta","Esophagus_Mucosa","Vagina","Colon_Transverse","Cells_EBV-transformed_lymphocytes","Small_Intestine_Terminal_Ileum","Spleen","Pancreas","Whole_Blood","Adipose_Subcutaneous","Adipose_Visceral_(Omentum)","Breast_Mammary_Tissue","Esophagus_Gastroesophageal_Junction","Esophagus_Muscularis","Colon_Sigmoid","Brain_Cerebellum","Heart_Atrial_Appendage","Brain_Hypothalamus","Brain_Substantia_nigra","Brain_Cerebellar_Hemisphere"],[7597,8840,12477,11042,11799,12784,13443,8535,8071,12023,12360,12156,9001,9362,11809,10553,10864,6704,6781,9126,10782,11175,11511,11581,11367,12250,12272,8245,8177,14043,6750,6670,7419,7681,8583,7050,7556,7046,9976,9695,9694,13866,14030,10766,10753,10955,12027,12484,13332],[692.1613874666301,210.7637420644726,197.7317985300375,191.620356887782,191.4896943941163,191.3149193426446,191.1962287523497,7.283268043914299,6.472507542099154,5.448500912549732,5.184524849620286,4.264854617585761,4.157569197704165,3.462658712752833,2.344788491731966,1.113928907831628,0.89052474263827,0.7681338433192587,0.7613079227946135,0.6488788149138666,0.538718951320533,0.5278963981042188,0.5095933927405866,0.5065639723860899,0.4628895698774144,0.4502370503280084,0.4402339532849185,0.4114023888328335,0.4041729974386987,0.2716450826643282,-0.3798692455491709,-0.4004801252001363,-0.4206905421794998,-0.4362203768619027,-0.4445262925519428,-0.45470196080122,-0.4554530994896513,-0.46104871137221,-0.4668391492444204,-0.4748525666897814,-0.4920842498133309,-0.498630313298321,-0.5019853671724728,-0.5022853122465103,-0.5075400853363456,-0.5163432695045949,-0.5304998559848584,-0.5375194112627265,-0.5402016320556041],["-4.0338<br/>[-4.0338, -4.0338]","-4.0555<br/>[-4.0555, -4.0555]","-4.0317<br/>[-4.0317, -4.0317]","-4.0276<br/>[-4.0276, -4.0276]","-4.0273<br/>[-4.0273, -4.0273]","-4.0269<br/>[-4.0269, -4.0269]","-4.0266<br/>[-4.0266, -4.0266]","-4.0332<br/>[-4.0332, -4.0332]","-4.0335<br/>[-4.0335, -4.0335]","-4.0354<br/>[-4.0354, -4.0354]","-4.0300<br/>[-4.0300, -4.0300]","-4.0275<br/>[-4.0275, -4.0275]","-4.0356<br/>[-4.0356, -4.0356]","-4.0283<br/>[-4.0283, -4.0283]","-4.0318<br/>[-4.0318, -4.0318]","-4.0281<br/>[-4.0281, -4.0281]","-4.0277<br/>[-4.0277, -4.0277]","-4.0339<br/>[-4.0339, -4.0339]","-4.0339<br/>[-4.0339, -4.0339]","-4.0330<br/>[-4.0330, -4.0330]","-4.0277<br/>[-4.0277, -4.0277]","-4.0276<br/>[-4.0276, -4.0276]","-4.0274<br/>[-4.0274, -4.0274]","-4.0274<br/>[-4.0274, -4.0274]","-4.0320<br/>[-4.0320, -4.0320]","-4.0271<br/>[-4.0271, -4.0271]","-4.0272<br/>[-4.0272, -4.0272]","-4.0309<br/>[-4.0309, -4.0309]","-4.0315<br/>[-4.0315, -4.0315]","-4.0264<br/>[-4.0264, -4.0264]","-4.0340<br/>[-4.0340, -4.0340]","-4.0340<br/>[-4.0340, -4.0340]","-4.0292<br/>[-4.0292, -4.0292]","-4.0290<br/>[-4.0290, -4.0290]","-4.0287<br/>[-4.0287, -4.0287]","-4.0293<br/>[-4.0293, -4.0293]","-4.0291<br/>[-4.0291, -4.0291]","-4.0293<br/>[-4.0293, -4.0293]","-4.0281<br/>[-4.0281, -4.0281]","-4.0282<br/>[-4.0282, -4.0282]","-4.0282<br/>[-4.0282, -4.0282]","-4.0265<br/>[-4.0265, -4.0265]","-4.0265<br/>[-4.0265, -4.0265]","-4.0278<br/>[-4.0278, -4.0278]","-4.0278<br/>[-4.0278, -4.0278]","-4.0277<br/>[-4.0277, -4.0277]","-4.0272<br/>[-4.0272, -4.0272]","-4.0270<br/>[-4.0270, -4.0270]","-4.0267<br/>[-4.0267, -4.0267]"],["-3.2607<br/>[-3.2705, -3.2516]","-2.8022<br/>[-2.8391, -2.7640]","-3.2651<br/>[-3.2918, -3.2521]","-3.3194<br/>[-3.3392, -3.3016]","-3.3475<br/>[-3.3652, -3.3268]","-3.3833<br/>[-3.4004, -3.3634]","-3.4062<br/>[-3.4240, -3.3886]","-3.2497<br/>[-3.3026, -3.1913]","-3.1970<br/>[-3.2730, -3.1196]","-3.3021<br/>[-3.4090, -3.2137]","-3.4279<br/>[-3.5060, -3.3567]","-3.4536<br/>[-3.5157, -3.3991]","-3.3655<br/>[-3.4668, -3.2531]","-3.6365<br/>[-3.6688, -3.5871]","-3.5388<br/>[-3.5710, -3.4455]","-3.6532<br/>[-3.9733, -3.5065]","-3.7594<br/>[-4.0081, -3.6097]","-3.7253<br/>[-3.8233, -3.5922]","-3.7317<br/>[-3.8280, -3.5978]","-3.8736<br/>[-3.9501, -3.7874]","-3.9458<br/>[-4.0292, -3.8177]","-3.9559<br/>[-4.0449, -3.8242]","-3.9734<br/>[-4.0573, -3.8398]","-3.9766<br/>[-4.0604, -3.8502]","-3.9731<br/>[-4.3031, -3.7946]","-4.0293<br/>[-4.0857, -3.9293]","-4.0028<br/>[-4.3432, -3.8293]","-3.9138<br/>[-4.1138, -3.8055]","-3.8918<br/>[-4.0686, -3.7719]","-4.1133<br/>[-4.2690, -4.0153]","-4.4769<br/>[-5.6047, -3.8900]","-4.6117<br/>[-5.7174, -4.0283]","-4.7764<br/>[-5.5326, -4.2136]","-4.8703<br/>[-5.7192, -4.3417]","-4.8780<br/>[-5.5953, -4.3377]","-5.2397<br/>[-6.3653, -4.6317]","-5.0930<br/>[-6.1815, -4.5133]","-5.4820<br/>[-6.5701, -4.9026]","-4.8700<br/>[-5.9134, -4.2897]","-4.9886<br/>[-6.0210, -4.4049]","-5.2641<br/>[-6.2887, -4.6879]","-4.9204<br/>[-5.8076, -4.3993]","-4.9370<br/>[-5.8181, -4.4108]","-5.2694<br/>[-6.1143, -4.7571]","-5.3455<br/>[-6.3038, -4.7844]","-5.5170<br/>[-6.4927, -4.9434]","-5.5976<br/>[-6.5193, -5.0410]","-5.6680<br/>[-6.6075, -5.1063]","-5.4777<br/>[-6.3768, -4.9229]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Tissue name<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2]},{"name":"id","targets":0},{"name":"# of SNPs","targets":1},{"name":"log10.bf","targets":2},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":3},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
<div id="cluster-distinctively-expressed-genes" class="section level3">
<h3>Cluster distinctively expressed genes</h3>
<div class="datatables html-widget html-fill-item" id="htmlwidget-f32b28fd418c4fff040d" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f32b28fd418c4fff040d">{"x":{"filter":"none","vertical":false,"data":[["4","9","5","19","10","14","2","6","20","13","1","7","8","17","11","18","12","16","15","3"],[12121,9353,11218,5789,8187,9229,8790,8666,7948,7466,7332,8091,9983,8294,10144,7142,7070,10028,7579,9834],[421.3746725278255,198.7054962985966,53.95484302661902,9.580579582237682,8.026133029390634,4.871172768425112,4.148239083341885,4.004530287548624,2.81389047125993,2.714810375323562,2.371801090245189,2.279206936385801,2.192380987107274,0.7318121207172591,0.5167357994509076,-0.01881494642231826,-0.2911640546760107,-0.3364191167712018,-0.4192174863448203,-0.4809662629840151],["-4.2066<br/>[-4.2066, -4.2066]","-4.0330<br/>[-4.0330, -4.0330]","-4.0417<br/>[-4.0417, -4.0417]","-4.0343<br/>[-4.0343, -4.0343]","-4.0400<br/>[-4.0400, -4.0400]","-4.0286<br/>[-4.0286, -4.0286]","-4.0376<br/>[-4.0376, -4.0376]","-4.0334<br/>[-4.0334, -4.0334]","-4.0289<br/>[-4.0289, -4.0289]","-4.0336<br/>[-4.0336, -4.0336]","-4.0382<br/>[-4.0382, -4.0382]","-4.0333<br/>[-4.0333, -4.0333]","-4.0326<br/>[-4.0326, -4.0326]","-4.0288<br/>[-4.0288, -4.0288]","-4.0280<br/>[-4.0280, -4.0280]","-4.0299<br/>[-4.0299, -4.0299]","-4.0294<br/>[-4.0294, -4.0294]","-4.0283<br/>[-4.0283, -4.0283]","-4.0291<br/>[-4.0291, -4.0291]","-4.0281<br/>[-4.0281, -4.0281]"],["-2.5129<br/>[-2.5129, -2.5129]","-3.1093<br/>[-3.1380, -3.0975]","-2.4804<br/>[-2.4804, -2.4804]","-2.8906<br/>[-2.9569, -2.8331]","-3.0217<br/>[-3.1060, -2.9474]","-3.2519<br/>[-3.4095, -3.1372]","-3.4344<br/>[-3.4623, -3.3919]","-3.1491<br/>[-3.3080, -3.0300]","-3.5531<br/>[-3.5958, -3.4874]","-3.3852<br/>[-3.5486, -3.2433]","-3.5137<br/>[-3.5600, -3.4460]","-3.5613<br/>[-3.6025, -3.4976]","-3.6230<br/>[-3.6895, -3.5463]","-3.7783<br/>[-3.9101, -3.6328]","-3.9487<br/>[-4.0043, -3.8502]","-3.8424<br/>[-4.6501, -3.4927]","-4.2112<br/>[-5.0555, -3.8082]","-4.3027<br/>[-5.3012, -3.8443]","-4.6123<br/>[-5.7086, -4.0442]","-4.9908<br/>[-5.8279, -4.5126]"]],"container":"<table class=\"white-space: nowrap\">\n  <thead>\n    <tr>\n      <th>Cluster ID<\/th>\n      <th># of SNPs<\/th>\n      <th>Log10 BF<\/th>\n      <th>Log10 outside &pi;<br/>(mean, 95% CI)<\/th>\n      <th>Log10 inside &pi;<br/>(mean, 95% CI)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[1,2]},{"name":"id","targets":0},{"name":"# of SNPs","targets":1},{"name":"log10.bf","targets":2},{"name":"Log10 outside &pi;<br/>(mean, 95% CI)","targets":3},{"name":"Log10 inside &pi;<br/>(mean, 95% CI)","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
<p>The relationship between tissues and clusters is shown in Figure 1 of
<a
href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006599">Dey
et al. (2017)</a>; see below.</p>
<p><img
src="http://journals.plos.org/plosgenetics/article/figure/image?size=large&amp;id=10.1371/journal.pgen.1006599.g001" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">session_info</span>()</span></code></pre></div>
<pre><code>─ Session info ───────────────────────────────────────────────────────────────
 setting  value
 version  R version 4.4.1 (2024-06-14)
 os       macOS Sonoma 14.6.1
 system   aarch64, darwin20
 ui       X11
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       America/Los_Angeles
 date     2024-09-16
 pandoc   3.1.11 @ /Applications/RStudio.app/Contents/Resources/app/quarto/bin/tools/aarch64/ (via rmarkdown)

─ Packages ───────────────────────────────────────────────────────────────────
 package     * version    date (UTC) lib source
 bslib         0.8.0      2024-07-29 [1] CRAN (R 4.4.0)
 cachem        1.1.0      2024-05-16 [1] CRAN (R 4.4.0)
 callr         3.7.6      2024-03-25 [1] CRAN (R 4.4.0)
 cli           3.6.3      2024-06-21 [1] CRAN (R 4.4.0)
 crosstalk     1.2.1      2023-11-23 [1] CRAN (R 4.4.0)
 devtools      2.4.5      2022-10-11 [1] CRAN (R 4.4.0)
 digest        0.6.37     2024-08-19 [1] CRAN (R 4.4.1)
 dplyr       * 1.1.4      2023-11-17 [1] CRAN (R 4.4.0)
 DT          * 0.33       2024-04-04 [1] CRAN (R 4.4.0)
 ellipsis      0.3.2      2021-04-29 [1] CRAN (R 4.4.0)
 evaluate      0.24.0     2024-06-10 [1] CRAN (R 4.4.0)
 fansi         1.0.6      2023-12-08 [1] CRAN (R 4.4.0)
 fastmap       1.2.0      2024-05-15 [1] CRAN (R 4.4.0)
 fs            1.6.4      2024-04-25 [1] CRAN (R 4.4.0)
 generics      0.1.3      2022-07-05 [1] CRAN (R 4.4.0)
 getPass       0.2-4      2023-12-10 [1] CRAN (R 4.4.0)
 git2r         0.33.0     2023-11-26 [1] CRAN (R 4.4.0)
 glue          1.7.0      2024-01-09 [1] CRAN (R 4.4.0)
 htmltools     0.5.8.1    2024-04-04 [1] CRAN (R 4.4.0)
 htmlwidgets   1.6.4      2023-12-06 [1] CRAN (R 4.4.0)
 httpuv        1.6.15     2024-03-26 [1] CRAN (R 4.4.0)
 httr          1.4.7      2023-08-15 [1] CRAN (R 4.4.0)
 jquerylib     0.1.4      2021-04-26 [1] CRAN (R 4.4.0)
 jsonlite      1.8.8      2023-12-04 [1] CRAN (R 4.4.0)
 knitr         1.48       2024-07-07 [1] CRAN (R 4.4.0)
 later         1.3.2      2023-12-06 [1] CRAN (R 4.4.0)
 lifecycle     1.0.4      2023-11-07 [1] CRAN (R 4.4.0)
 magrittr      2.0.3      2022-03-30 [1] CRAN (R 4.4.0)
 memoise       2.0.1      2021-11-26 [1] CRAN (R 4.4.0)
 mime          0.12       2021-09-28 [1] CRAN (R 4.4.0)
 miniUI        0.1.1.1    2018-05-18 [1] CRAN (R 4.4.0)
 pillar        1.9.0      2023-03-22 [1] CRAN (R 4.4.0)
 pkgbuild      1.4.4      2024-03-17 [1] CRAN (R 4.4.0)
 pkgconfig     2.0.3      2019-09-22 [1] CRAN (R 4.4.0)
 pkgload       1.4.0      2024-06-28 [1] CRAN (R 4.4.0)
 plyr        * 1.8.9      2023-10-02 [1] CRAN (R 4.4.0)
 processx      3.8.4      2024-03-16 [1] CRAN (R 4.4.0)
 profvis       0.3.8      2023-05-02 [1] CRAN (R 4.4.0)
 promises      1.3.0      2024-04-05 [1] CRAN (R 4.4.0)
 ps            1.8.0      2024-09-12 [1] CRAN (R 4.4.1)
 purrr         1.0.2      2023-08-10 [1] CRAN (R 4.4.0)
 R.matlab    * 3.7.0      2022-08-25 [1] CRAN (R 4.4.0)
 R.methodsS3   1.8.2      2022-06-13 [1] CRAN (R 4.4.0)
 R.oo          1.26.0     2024-01-24 [1] CRAN (R 4.4.0)
 R.utils       2.12.3     2023-11-18 [1] CRAN (R 4.4.0)
 R6            2.5.1      2021-08-19 [1] CRAN (R 4.4.0)
 Rcpp          1.0.13     2024-07-17 [1] CRAN (R 4.4.0)
 remotes       2.5.0      2024-03-17 [1] CRAN (R 4.4.0)
 rlang         1.1.4      2024-06-04 [1] CRAN (R 4.4.0)
 rmarkdown     2.28       2024-08-17 [1] CRAN (R 4.4.0)
 rprojroot     2.0.4      2023-11-05 [1] CRAN (R 4.4.0)
 rstudioapi    0.16.0     2024-03-24 [1] CRAN (R 4.4.0)
 sass          0.4.9.9000 2024-07-11 [1] Github (rstudio/sass@9228fcf)
 sessioninfo   1.2.2      2021-12-06 [1] CRAN (R 4.4.0)
 shiny         1.9.1      2024-08-01 [1] CRAN (R 4.4.0)
 stringi       1.8.4      2024-05-06 [1] CRAN (R 4.4.0)
 stringr       1.5.1      2023-11-14 [1] CRAN (R 4.4.0)
 tibble        3.2.1      2023-03-20 [1] CRAN (R 4.4.0)
 tidyselect    1.2.1      2024-03-11 [1] CRAN (R 4.4.0)
 urlchecker    1.0.1      2021-11-30 [1] CRAN (R 4.4.0)
 usethis       3.0.0      2024-07-29 [1] CRAN (R 4.4.0)
 utf8          1.2.4      2023-10-22 [1] CRAN (R 4.4.0)
 vctrs         0.6.5      2023-12-01 [1] CRAN (R 4.4.0)
 whisker       0.4.1      2022-12-05 [1] CRAN (R 4.4.0)
 workflowr   * 1.7.1      2023-08-23 [1] CRAN (R 4.4.0)
 xfun          0.47       2024-08-17 [1] CRAN (R 4.4.0)
 xtable        1.8-4      2019-04-21 [1] CRAN (R 4.4.0)
 yaml          2.3.10     2024-07-26 [1] CRAN (R 4.4.0)

 [1] /Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/library

──────────────────────────────────────────────────────────────────────────────</code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
