---
title: "Full results of Zhu and Stephens (2017)"
author: "Xiang Zhu"
date: 2017-04-12
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: true
---

[`stephenslab/rss`]: https://github.com/stephenslab/rss/tree/master/src_vb 
[zenodo-geneset]: https://zenodo.org/badge/latestdoi/55633948
[`xiangzhu/rss-gsea`]: https://github.com/xiangzhu/rss-gsea/tree/master/data
[`xiangzhu/rss-gsea-baseline`]: https://xiangzhu.github.io/rss-gsea-baseline/ 

## Overview

This is my online notebook to document and share the full results
of genome-wide enrichment analyses described in the manuscript:

> Xiang Zhu and Matthew Stephens (2017).
> A large-scale genome-wide enrichment analysis identifies new
> trait-associated genes, pathways and tissues across 31 human phenotypes.
> *bioRxiv.* [https://doi.org/10.1101/160770](https://doi.org/10.1101/160770).

The software that generated these results is freely available at [`stephenslab/rss`][].
We also provide an end-to-end [example](http://stephenslab.github.io/rss/Example-5)
illustrating how to use this software to perform
genome-wide enrichment analyses of GWAS summary statistics.
This software can be referenced in a journal's "Code availability" section
as [][zenodo-software].

In addition, all 4,026 pre-processed gene sets used in this study
(including 3,913 biological pathways and 113 tissue-based gene sets)
are freely available at [`xiangzhu/rss-gsea`][].
These gene sets can be referenced in a journal's "Data availability" section
as [![DOI](https://zenodo.org/badge/55633948.svg)][zenodo-geneset].

If you find the analysis results, the pre-processed gene sets,
the statistical methods, and/or the software useful for your work,
**please kindly cite our manuscript listed above,
[Zhu and Stephens (2017)](https://doi.org/10.1101/160770)**.

If you have any question about the notebook and/or the manuscript,
please feel free to contact me: [Xiang Zhu](https://github.com/xiangzhu),
`xiangzhu@uchicago.edu` or `xiangzhu@stanford.edu`.

## Main results

#### Anthropometric phenotypes

- [Adult human height (Wood et al. 2014)](height_2014.html)
- [Body mass index (Locke et al. 2015)](bmi_2015.html)
- [Waist-to-hip ratio (Shungin et al. 2015)](waist_2015.html)

#### Hematopoietic phenotypes

- [Haemoglobin (Harst et al. 2012)](hb_2012.html)
- [Mean cell haemoglobin (Harst et al. 2012)](mch_2012.html)
- [Mean cell haemoglobin concentration (Harst et al. 2012)](mchc_2012.html)
- [Mean cell volume (Harst et al. 2012)](mcv_2012.html)
- [Packed cell volume (Harst et al. 2012)](pcv_2012.html)
- [Red blood cell count (Harst et al. 2012)](rbc_2012.html)

#### Immune-related phenotypes

- [Rheumatoid arthritis (Okada et al. 2014)](ra_2014.html)
- [Crohn's disease (Liu et al. 2015)](cd_2015.html)
- [Inflammatory bowel disease (Liu et al. 2015)](ibd_2015.html)
- [Ulcerative colitis (Liu et al. 2015)](uc_2015.html)

#### Metabolic phenotypes

- [High-density lipoprotein (Teslovich et al. 2010)](hdl_2010.html)
- [Low-density lipoprotein (Teslovich et al. 2010)](ldl_2010.html)
- [Total cholesterol (Teslovich et al. 2010)](tc_2010.html)
- [Triglycerides (Teslovich et al. 2010)](tg_2010.html)
- [Type 2 diabetes (Morris et al. 2012)](t2d_2012.html)
- [Fasting glucose (Manning et al. 2012)](fg_2012.html)
- [Fasting insulin (Manning et al. 2012)](fi_2012.html)
- [Heart rate (Den Hoed et al. 2013)](hr_2013.html)
- [Gout (Köttgen et al. 2013)](gout_2013.html)
- [Serum urate (Köttgen et al. 2013)](urate_2013.html)
- [Age at natural menopause (Day et al. 2015)](anm_2015.html)
- [Coronary artery disease (Nikpay et al. 2015)](cad_2015.html)
- [Myocardial infarction (Nikpay et al. 2015)](mi_2015.html)

#### Neurological phenotypes

- [Alzheimer's disease (Lambert et al. 2013)](load_2013.html)
- [Schizophrenia (Ripke et al. 2014)](scz_2014.html)
- [Amyotrophic lateral sclerosis (Rheenen et al. 2016)](als_2016.html)
- [Depressive symptoms (Okbay et al. 2016)](ds_2016.html)
- [Neuroticism (Okbay et al. 2016)](neu_2016.html)

## Additional resources

- How can I perform similar analyses on a new dataset of GWAS summary statistics?

The software that generated results of this study is available at [`stephenslab/rss`][].
I also write a step-by-step [tutorial](http://stephenslab.github.io/rss/Example-5)
illustrating how to use this software to perform genome-wide enrichment
and prioritization analyses on GWAS summary statistics.

- Where can I download all 4,026 pre-processed gene sets?

All 4,026 gene sets used in this study are freely available at [`xiangzhu/rss-gsea`],
where the folder [`biological_pathway`](https://github.com/xiangzhu/rss-gsea/tree/master/data/biological_pathway)
contains 3,913 biological pathways,
and the folder [`tissue_set`](https://github.com/xiangzhu/rss-gsea/tree/master/data/tissue_set)
contains 113 GTEx tissue-based gene sets.
More details about these gene sets can be found [here](gene_set.html).

- Where can I find "baseline" model fitting results of all 31 traits?

You can find summary results of "baseline" model fitting at [`xiangzhu/rss-gsea-baseline`][].
For me, the baseline model fitting results are merely inferential "bases" for the
enrichment model fitting results shown in the "Main results" section above.
However, when I was presenting the enrichment results during my
[Ph.D. thesis defense](http://www.stat.uchicago.edu/~xiangzhu/THESIS_20170630.html),
[Prof. John Novembre](http://jnpopgen.org/) and
[Prof. Xin He](http://xinhelab.org/) both pointed out these baseline results
might be useful for other on-going projects on the "fourth floor"
(i.e. the great computational space shared with the labs of Matthew Stephens, John Novembre and Xin He).
Their comments motivated me to create a separate online notebook
[`xiangzhu/rss-gsea-baseline`][] to share the baseline summary results.

- Where can I find "Round 1" results of all 3,913 biological pathways?

Currently you need to contact me directly to view our "Round 1" results of all 3913 pathways.
When this work was under review, one referee pointed out that our online results, especially our
"Round 1" analysis results, were "needlessly complicated" and did not have "any obvious benefit".
Hence, I removed the "Round 1" analysis results from this notebook to simplify the presentation.
I hope that this change can address the referee's comment.

- Where can I find the gene prioritization results?

Currently you need to contact me directly to view our full prioritization results. 
Unlike the gene set enrichment results, the gene prioritization results
cannot be easily tabulated, and thus I have not displayed them in this notebook yet.
